<!DOCTYPE html>

<html>

<head>
    <meta charset="UTF-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, minimum-scale=1.0">
<title>Chapter II</title>


<script type="text/x-mathjax-config">
	MathJax.Hub.Config({
    "HTML-CSS" : {
        availableFonts : ["STIX"],
        preferredFont : "Gyre Pagella",
        webFont : "STIX-Web",
        imageFont : null
    },
	  TeX: {
		equationNumbers: {
		  autoNumber: "AMS"
		}
	  },
	  tex2jax: {
		inlineMath: [['$','$']], 
		displayMath: [['$$','$$']],
		processEscapes: true,
	  }
	});
  </script>
<style>
    .eqnos {
        display: inline-block;
        position: relative;
        width: 100%;
    }

    .eqnos br {
        display: none;
    }

    .eqnos-number {
        position: absolute;
        right: 0em;
        top: 50%;
        line-height: 0;
    }
</style>

<script>
    window.MathJax = {
        tex: {
            tags: 'all'
        }
    };
</script>
<script>
    MathJax.Hub.Config({
        tex2jax: {
            inlineMath: [['$', '$'], ['\\(', '\\)']]
        }
    });
</script>
<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>

<link rel="icon" href="https://mrazomej.github.io/phd//favicon.ico"
    type="image/x-icon">
<link href="https://fonts.googleapis.com/css?family=Nanum+Myeongjo&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Alegreya|Lora&display=swap" rel="stylesheet">

<link rel="stylesheet"
    href="https://mrazomej.github.io/phd//doks-theme/assets/css/style.css">
</head>

<body class="grey" data-spy="scroll" data-target=".js-scrollspy">
    


	<header class="site-header">
		<div class="container">
			<div class="row">
				<div class="col-xs-12">
					
						<a href="https://mrazomej.github.io/phd//" class="site-header__logo"><i class="icon icon--chevron-left"></i> Home</a>
					
					
				</div><!-- /.col -->
			</div><!-- /.row -->
		</div><!-- /.container -->
	</header><!-- /.site-header -->


    <div class="hero-subheader" style="background-image: url(https://mrazomej.github.io/phd//doks-theme/assets/images/paihia.jpg); background-position: center; object-fit: cover; width:100%;">

        <div class="container" style="background-color: rgba(0,0,0, 0.75); border:0.75px solid white;">
            <div class="row">
                <div class="col-md-7">
                    <div class="align-container" data-mh>
                        <div class="align-inner">
                            
                            <h2 class="hero-subheader__title">Chapter II</h2>
                            
                            
                            <h1 class="hero-subheader__title">A Predictive Theory of Allosteric Induction
</h1>
                            
                            
                            <!-- 
									
										<a href="https://mrazomej.github.io/phd/chapter_01" class="btn btn--dark btn--rounded btn--w-icon btn--w-icon-left">
											<i class="icon icon--arrow-left"></i>	
										</a>
									
									
										<a href="https://mrazomej.github.io/phd/chapter_03" class="btn btn--dark btn--rounded btn--w-icon">
											<i class="icon icon--arrow-right"></i>
										</a>
									
								 -->
                        </div><!-- /.align-inner -->
                    </div><!-- /.align-container -->
                </div><!-- /.col -->
                
                <div class="col-md-5 col-md-offset-0 hidden-xs hidden-sm">
                    <div class="align-container" data-mh>
                        <div class="align-inner">
                            <div class="hero-subheader__author">
                                <p class="hero-subheader__author-title">
                                    Summary
                                    <i class="icon icon--chevron-down"></i>
                                </p><!-- /.hero-subheader__author-title -->
                                <p>Despite lacking a nervous system, single bacterial cells are capable of making decisions given signals from their surroundings. How can individual molecules sense and transmit these signals? The answer comes from one of the crowning scientific achievements of the past century: allostery. Simply stated, allostery is the property of certain macromolecules to exist in multiple conformations with different properties. For example, transcription factors —proteins that control gene expression— can be active (able to bind the DNA) or inactive (unable to bind DNA) depending on the concentration of a signaling molecule.
In this chapter, I write down a theoretical model that predicts the expression level of a gene regulated by an allosteric transcription factor. We then test the model experimentally, showing that the model was able to predict how changes to the regulation of the gene translate to changes in the cellular response.
</p>
                            </div><!-- /.hero-subheader__author -->
                        </div><!-- /.align-inner -->
                    </div><!-- /.align-container -->
                </div><!-- /.col -->
                
            </div><!-- /.row -->
        </div><!-- /.container -->
    </div><!-- /.hero-subheader -->
    
    <nav class="page-nav">
        <div class="container">
            <div class="row">
                <div class="col-xs-12">
                    
                    <a href="https://mrazomej.github.io/phd/chapter_01"
                        class="page-nav__item page-nav__item--prev">
                        <i class="icon icon--arrow-left"></i>
                        Chapter 1
                    </a><!-- /.page-nav__item -->
                    
                    
                    <a href="https://mrazomej.github.io/phd/chapter_03"
                        class="page-nav__item page-nav__item--next">
                        Chapter 3
                        <i class="icon icon--arrow-right"></i>
                    </a><!-- /.page-nav__item -->
                    
                </div><!-- /.col -->
            </div><!-- /.row -->
        </div><!-- /.container -->
    </nav><!-- /.page-nav -->
    

    <div class="section">
        <div class="container">
            <div class="row">
                <div class="col-md-7">
                    <div class="content">
                        <strong>Published as …</strong>
<hr/>
<p>A version of this chapter originally appeared as Razo-Mejia, M.†, Barnes, S.L.†, Belliveau, N.M.†, Chure, G.†, Einav, T.†, Lewis, M., and Phillips, R. (2018). Tuning transcriptional regulation through signaling: A predictive theory of allosteric induction. Cell Systems 6, 456-469.e10. DOI:https://doi.org/10.1016/j.cels.2018.02.004. † M.R.M, S.L.B, N.M.B, G.C., and T.E. contributed equally to this work from the theoretical underpinnings to the experimental design and execution. M.R.M, S.L.B, N.M.B, G.C, T.E., and R.P. wrote the paper. M.L. provided guidance and advice.</p>
<h2 id="abstract">Abstract</h2>
<p>Allosteric regulation is found across all domains of life, yet we still lack simple, predictive theories that directly link the experimentally tunable parameters of a system to its input-output response. To that end, we present a general theory of allosteric transcriptional regulation using the Monod-Wyman-Changeux model. We rigorously test this model using the ubiquitous simple repression motif in bacteria by first predicting the behavior of strains that span a large range of repressor copy numbers and DNA binding strengths and then constructing and measuring their response. Our model not only accurately captures the induction profiles of these strains but also enables us to derive analytic expressions for key properties such as the dynamic range and <span class="math inline">\([EC_{50}]\)</span>. Finally, we derive an expression for the free energy of allosteric repressors which enables us to collapse our experimental data onto a single master curve that captures the diverse phenomenology of the induction profiles.</p>
<h2 id="introduction">Introduction</h2>
<p>Understanding how organisms sense and respond to changes in their environment has long been a central theme of biological inquiry. At the cellular level, this interaction is mediated by a diverse collection of molecular signaling pathways. A pervasive mechanism of signaling in these pathways is allosteric regulation, in which the binding of a ligand induces a conformational change in some target molecule, triggering a signaling cascade <span class="citation" data-cites="Lindsley2006"> [<a href="#ref-Lindsley2006" role="doc-biblioref">35</a>]</span>. One of the most important examples of such signaling is offered by transcriptional regulation, where a transcription factor’s propensity to bind to DNA will be altered upon binding to an allosteric effector.</p>
<p>Despite allostery’s ubiquity, we lack a formal, rigorous, and generalizable framework for studying its effects across the broad variety of contexts in which it appears. A key example of this is transcriptional regulation, in which allosteric transcription factors can be induced or corepressed by binding to a ligand. An allosteric transcription factor can adopt multiple conformational states, each of which has its own affinity for the ligand and for its DNA target site. <em>In vitro</em> studies have rigorously quantified the equilibria of different conformational states for allosteric transcription factors and measured the affinities of these states to the ligand <span class="citation" data-cites="Harman2001 Lanfranco2017"> [<a href="#ref-Lanfranco2017" role="doc-biblioref">1</a>,<a href="#ref-Harman2001" role="doc-biblioref">2</a>]</span>. In spite of these experimental observations, the lack of a coherent quantitative model for allosteric transcriptional regulation has made it impossible to predict the behavior of even a simple genetic circuit across a range of regulatory parameters.</p>
<p>The ability to predict circuit behavior robustly–that is, across both broad ranges of parameters and regulatory architectures–is important for multiple reasons. First, in the context of a specific gene, accurate prediction demonstrates that all components relevant to the gene’s behavior have been identified and characterized to sufficient quantitative precision. Second, in the context of genetic circuits in general, robust prediction validates the model that generated the prediction. Possessing a validated model also has implications for future work. For example, when we have sufficient confidence in the model, a single data set can be used to accurately extrapolate a system’s behavior in other conditions. Moreover, there is an essential distinction between a predictive model, which is used to predict a system’s behavior given a set of input variables, and a retroactive model, which is used to describe the behavior of data that has already been obtained. We note that even some of the most careful and rigorous analysis of transcriptional regulation often entails only a retroactive reflection on a single experiment. This raises the fear that each regulatory architecture may require a unique analysis that cannot carry over to other systems, a worry that is exacerbated by the prevalent use of phenomenological functions (e.g. Hill functions or ratios of polynomials) that can analyze a single data set but cannot be used to extrapolate a system’s behavior in other conditions <span class="citation" data-cites="Setty2003 Poelwijk2011 Vilar2013 Rogers2015 Rohlhill2017"> [<a href="#ref-Poelwijk2011" role="doc-biblioref">9</a>,<a href="#ref-Rogers2015" role="doc-biblioref">12</a>,<a href="#ref-Rohlhill2017" role="doc-biblioref">13</a>,<a href="#ref-Setty2003" role="doc-biblioref">26</a>,<a href="#ref-Vilar2013" role="doc-biblioref">66</a>]</span>.</p>
<p>This work explores what happens when theory takes center stage, namely, we first write down the equations governing a system and describe its expected behavior across a wide array of experimental conditions, and only then do we set out to experimentally confirm these results. Building upon previous work <span class="citation" data-cites="Garcia2011c Brewster2014 Weinert2014"> [<a href="#ref-Weinert2014" role="doc-biblioref">47</a>,<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>,<a href="#ref-Brewster2014" role="doc-biblioref">70</a>]</span> and the work of Monod, Wyman, and Changeux <span class="citation" data-cites="MONOD1965"> [<a href="#ref-MONOD1965" role="doc-biblioref">57</a>]</span>, we present a statistical mechanical rendering of allostery in the context of induction and corepression (shown schematically in Fig. 1 and henceforth referred to as the MWC model) and use it as the basis of parameter-free predictions which we then test experimentally. More specifically, we study the simple repression motif – a widespread bacterial genetic regulatory architecture in which binding of a transcription factor occludes binding of an RNA polymerase, thereby inhibiting transcription initiation. The MWC model stipulates that an allosteric protein fluctuates between two distinct conformations – an active and inactive state – in thermodynamic equilibrium <span class="citation" data-cites="MONOD1965"> [<a href="#ref-MONOD1965" role="doc-biblioref">57</a>]</span>. During induction, for example, effector binding increases the probability that a repressor will be in the inactive state, weakening its ability to bind to the promoter and resulting in increased expression. To test the predictions of our model across a wide range of operator binding strengths and repressor copy numbers, we design an <em>E. coli</em> genetic construct in which the binding probability of a repressor regulates gene expression of a fluorescent reporter.</p>
<p>In total, the work presented here demonstrates that one extremely compact set of parameters can be applied self-consistently and predictively to different regulatory situations including simple repression on the chromosome, cases in which decoy binding sites for repressor are put on plasmids, cases in which multiple genes compete for the same regulatory machinery, cases involving multiple binding sites for repressor leading to DNA looping, and induction by signaling <span class="citation" data-cites="Garcia2011c Garcia2011b Brewster2012 Boedicker2013a Boedicker2013b Brewster2014"> [<a href="#ref-Boedicker2013b" role="doc-biblioref">15</a>,<a href="#ref-Garcia2011b" role="doc-biblioref">27</a>,<a href="#ref-Boedicker2013a" role="doc-biblioref">61</a>,<a href="#ref-Brewster2012" role="doc-biblioref">68</a>–<a href="#ref-Brewster2014" role="doc-biblioref">70</a>]</span>. Thus, rather than viewing the behavior of each circuit as giving rise to its own unique input-output response, the MWC model provides a means to characterize these seemingly diverse behaviors using a single unified framework governed by a small set of parameters.</p>
<figure>
<img src="ch2_fig01.png" id="fig:ch2_fig01" data-short-caption="Transcriptional regulatory architectures involving an allosteric repressor." alt="Figure 1: Transcription regulation architectures involving an allosteric repressor. We consider a promoter regulated solely by an allosteric repressor. When bound, the repressor prevents RNAP from binding and initiating transcription. Induction is characterized by the addition of an effector which binds to the repressor and stabilizes the inactive state (defined as the state which has a low affinity for DNA), thereby increasing gene expression. In corepression, the effector stabilizes the repressor’s active state and thus further reduces gene expression. We list several characterized examples of induction and corepression that support different physiological roles in E. coli  [21,58]. A schematic regulatory response of the two architectures shown in Panel plotting the fold-change in gene expression as a function of effector concentration, where fold-change is defined as the ratio of gene expression in the presence versus the absence of repressor. We consider the following key phenotypic properties that describe each response curve: the minimum response (leakiness), the maximum response (saturation), the difference between the maximum and minimum response (dynamic range), the concentration of ligand which generates a fold-change halfway between the minimal and maximal response ([EC_{50}]), and the log-log slope at the midpoint of the response (effective Hill coefficient). (C) Over time we have refined our understanding of simple repression architectures. A first round of experiments used colorimetric assays and quantitative Western blots to investigate how single-site repression is modified by the repressor copy number and repressor-DNA binding energy  [69]. A second round of experiments used video microscopy to probe how the copy number of the promoter and presence of competing repressor binding sites affect gene expression, and we use this data set to determine the free energy difference between the repressor’s inactive and active conformations  [47]. Here we used flow cytometry to determine the inducer-repressor dissociation constants and demonstrate that with these parameters we can predict a priori the behavior of the system for any repressor copy number, DNA binding energy, gene copy number, and inducer concentration." /><figcaption aria-hidden="true">Figure 1: <strong>Transcription regulation architectures involving an allosteric repressor.</strong> We consider a promoter regulated solely by an allosteric repressor. When bound, the repressor prevents RNAP from binding and initiating transcription. Induction is characterized by the addition of an effector which binds to the repressor and stabilizes the inactive state (defined as the state which has a low affinity for DNA), thereby increasing gene expression. In corepression, the effector stabilizes the repressor’s active state and thus further reduces gene expression. We list several characterized examples of induction and corepression that support different physiological roles in <em>E. coli</em> <span class="citation" data-cites="Huang2011 Li2014"> [<a href="#ref-Huang2011" role="doc-biblioref">21</a>,<a href="#ref-Li2014" role="doc-biblioref">58</a>]</span>. A schematic regulatory response of the two architectures shown in Panel plotting the fold-change in gene expression as a function of effector concentration, where fold-change is defined as the ratio of gene expression in the presence versus the absence of repressor. We consider the following key phenotypic properties that describe each response curve: the minimum response (leakiness), the maximum response (saturation), the difference between the maximum and minimum response (dynamic range), the concentration of ligand which generates a fold-change halfway between the minimal and maximal response (<span class="math inline">\([EC_{50}]\)</span>), and the log-log slope at the midpoint of the response (effective Hill coefficient). (C) Over time we have refined our understanding of simple repression architectures. A first round of experiments used colorimetric assays and quantitative Western blots to investigate how single-site repression is modified by the repressor copy number and repressor-DNA binding energy <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>]</span>. A second round of experiments used video microscopy to probe how the copy number of the promoter and presence of competing repressor binding sites affect gene expression, and we use this data set to determine the free energy difference between the repressor’s inactive and active conformations <span class="citation" data-cites="Weinert2014"> [<a href="#ref-Weinert2014" role="doc-biblioref">47</a>]</span>. Here we used flow cytometry to determine the inducer-repressor dissociation constants and demonstrate that with these parameters we can predict <em>a priori</em> the behavior of the system for any repressor copy number, DNA binding energy, gene copy number, and inducer concentration.</figcaption>
</figure>
<h2 id="results">Results</h2>
<h3 id="characterizing-transcription-factor-induction-using-the-monod-wyman-changeux-mwc-model">Characterizing Transcription Factor Induction using the Monod-Wyman-Changeux (MWC) Model</h3>
<p>We begin by considering a simple repression genetic architecture in which the binding of an allosteric repressor occludes the binding of RNA polymerase (RNAP) to the DNA <span class="citation" data-cites="Ackers1982 Buchler2003"> [<a href="#ref-Ackers1982" role="doc-biblioref">53</a>,<a href="#ref-Buchler2003" role="doc-biblioref">59</a>]</span>. When an effector (hereafter referred to as an "inducer" for the case of induction) binds to the repressor, it shifts the repressor’s allosteric equilibrium towards the inactive state as specified by the MWC model <span class="citation" data-cites="MONOD1965"> [<a href="#ref-MONOD1965" role="doc-biblioref">57</a>]</span>. This causes the repressor to bind more weakly to the operator, which increases gene expression. Simple repression motifs in the absence of inducer have been previously characterized by an equilibrium model where the probability of each state of repressor and RNAP promoter occupancy is dictated by the Boltzmann distribution <span class="citation" data-cites="Ackers1982 Buchler2003 Vilar2003 Bintu2005a Garcia2011c Brewster2014"> [<a href="#ref-Vilar2003" role="doc-biblioref">33</a>,<a href="#ref-Ackers1982" role="doc-biblioref">53</a>,<a href="#ref-Buchler2003" role="doc-biblioref">59</a>,<a href="#ref-Bintu2005a" role="doc-biblioref">63</a>,<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>,<a href="#ref-Brewster2014" role="doc-biblioref">70</a>]</span> (we note that non-equilibrium models of simple repression have been shown to have the same functional form that we derive below <span class="citation" data-cites="Phillips2015a"> [<a href="#ref-Phillips2015a" role="doc-biblioref">67</a>]</span>). We extend these models to consider allostery by accounting for the equilibrium state of the repressor through the MWC model.</p>
<p>Thermodynamic models of gene expression begin by enumerating all possible states of the promoter and their corresponding statistical weights. As shown in Fig. 2(A) , the promoter can either be empty, occupied by RNAP, or occupied by either an active or inactive repressor. The probability of binding to the promoter will be affected by the protein copy number, which we denote as <span class="math inline">\(P\)</span> for RNAP, <span class="math inline">\(R_{A}\)</span> for active repressor, and <span class="math inline">\(R_{I}\)</span> for inactive repressor. We note that repressors fluctuate between the active and inactive conformation in thermodynamic equilibrium, such that <span class="math inline">\(R_{A}\)</span> and <span class="math inline">\(R_{I}\)</span> will remain constant for a given inducer concentration <span class="citation" data-cites="MONOD1965"> [<a href="#ref-MONOD1965" role="doc-biblioref">57</a>]</span>. We assign the repressor a different DNA binding affinity in the active and inactive state. In addition to the specific binding sites at the promoter, we assume that there are <span class="math inline">\(N_{NS}\)</span> non-specific binding sites elsewhere (i.e. on parts of the genome outside the simple repression architecture) where the RNAP or the repressor can bind. All specific binding energies are measured relative to the average non-specific binding energy. Thus, <span class="math inline">\(\Delta\varepsilon_{P}\)</span> represents the energy difference between the specific and non-specific binding for RNAP to the DNA. Likewise, <span class="math inline">\(\Delta\varepsilon_{RA}\)</span> and <span class="math inline">\(\Delta\varepsilon_{RI}\)</span> represent the difference in specific and non-specific binding energies for repressor in the active or inactive state, respectively.</p>
<figure>
<img src="ch2_fig02.png" id="fig:ch2_fig02" data-short-caption="States and weights for the simple repression motif." alt="Figure 2: States and weights for the simple repression motif. RNAP (light blue) and a repressor compete for binding to a promoter of interest. There are R_A repressors in the active state (red) and R_I repressors in the inactive state (purple). The difference in energy between a repressor bound to the promoter of interest versus another non-specific site elsewhere on the DNA equals \Delta\varepsilon_{RA} in the active state and \Delta\varepsilon_{RI} in the inactive state; the P RNAP have a corresponding energy difference \Delta\varepsilon_{P} relative to non-specific binding on the DNA. N_{NS} represents the number of non-specific binding sites for both RNAP and repressor. A repressor has an active conformation (red, left column) and an inactive conformation (purple, right column), with the energy difference between these two states given by \Delta \varepsilon_{AI}. The inducer (blue circle) at concentration c is capable of binding to the repressor with dissociation constants K_A in the active state and K_I in the inactive state. The eight states for a dimer with n=2 inducer binding sites are shown along with the sums of the active and inactive states." /><figcaption aria-hidden="true">Figure 2: <strong>States and weights for the simple repression motif.</strong> RNAP (light blue) and a repressor compete for binding to a promoter of interest. There are <span class="math inline">\(R_A\)</span> repressors in the active state (red) and <span class="math inline">\(R_I\)</span> repressors in the inactive state (purple). The difference in energy between a repressor bound to the promoter of interest versus another non-specific site elsewhere on the DNA equals <span class="math inline">\(\Delta\varepsilon_{RA}\)</span> in the active state and <span class="math inline">\(\Delta\varepsilon_{RI}\)</span> in the inactive state; the <span class="math inline">\(P\)</span> RNAP have a corresponding energy difference <span class="math inline">\(\Delta\varepsilon_{P}\)</span> relative to non-specific binding on the DNA. <span class="math inline">\(N_{NS}\)</span> represents the number of non-specific binding sites for both RNAP and repressor. A repressor has an active conformation (red, left column) and an inactive conformation (purple, right column), with the energy difference between these two states given by <span class="math inline">\(\Delta \varepsilon_{AI}\)</span>. The inducer (blue circle) at concentration <span class="math inline">\(c\)</span> is capable of binding to the repressor with dissociation constants <span class="math inline">\(K_A\)</span> in the active state and <span class="math inline">\(K_I\)</span> in the inactive state. The eight states for a dimer with <span class="math inline">\(n=2\)</span> inducer binding sites are shown along with the sums of the active and inactive states.</figcaption>
</figure>
<p>Thermodynamic models of transcription <span class="citation" data-cites="Ackers1982 Buchler2003 Vilar2003 Bintu2005 Bintu2005a Kuhlman2007 Daber2011a Garcia2011c Brewster2014 Weinert2014"> [<a href="#ref-Vilar2003" role="doc-biblioref">33</a>,<a href="#ref-Weinert2014" role="doc-biblioref">47</a>,<a href="#ref-Ackers1982" role="doc-biblioref">53</a>,<a href="#ref-Daber2011a" role="doc-biblioref">55</a>,<a href="#ref-Buchler2003" role="doc-biblioref">59</a>,<a href="#ref-Bintu2005" role="doc-biblioref">62</a>–<a href="#ref-Kuhlman2007" role="doc-biblioref">64</a>,<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>,<a href="#ref-Brewster2014" role="doc-biblioref">70</a>]</span> posit that gene expression is proportional to the probability that the RNAP is bound to the promoter <span class="math inline">\(p_{\text{bound}}\)</span>, which is given by <span class="math display">\[
p_\text{bound} = 
\frac{\frac{P}{N_{NS}}e^{-\beta \Delta\varepsilon_{P}}}{1+\frac{R_A}{N_{NS}}
e^{-\beta \Delta\varepsilon_{RA}} + 
\frac{R_I}{N_{NS}}e^{-\beta \Delta\varepsilon_{RI}} + 
\frac{P}{N_{NS}}e^{-\beta\Delta\varepsilon_{P}}},
\label{eq:p_bound_definition}
\]</span> with <span class="math inline">\(\beta = \frac{1}{k_BT}\)</span> where <span class="math inline">\(k_B\)</span> is the Boltzmann constant and <span class="math inline">\(T\)</span> is the temperature of the system. As <span class="math inline">\(k_BT\)</span> is the natural unit of energy at the molecular length scale, we treat the products <span class="math inline">\(\beta \Delta\varepsilon_{j}\)</span> as single parameters within our model. Measuring <span class="math inline">\(p_{\text{bound}}\)</span> directly is fraught with experimental difficulties, as determining the exact proportionality between expression and <span class="math inline">\(p_{\text{bound}}\)</span> is not straightforward. Instead, we measure the fold-change in gene expression due to the presence of the repressor. We define fold-change as the ratio of gene expression in the presence of repressor relative to expression in the absence of repressor (i.e. constitutive expression), namely, <span class="math display">\[
\text{fold-change}
\equiv \frac{p_\text{bound}(R &gt; 0)}{p_\text{bound}(R = 0)}.
\label{eq:fold_change_definition}
\]</span> We can simplify this expression using two well-justified approximations: (1) <span class="math inline">\(\frac{P}{N_{NS}}e^{-\beta\Delta\varepsilon_{P}}\ll 1\)</span> implying that the RNAP binds weakly to the promoter (<span class="math inline">\(N_{NS} = 4.6 \times 10^6\)</span>, <span class="math inline">\(P \approx 10^3\)</span> <span class="citation" data-cites="Klumpp2008"> [<a href="#ref-Klumpp2008" role="doc-biblioref">65</a>]</span>, <span class="math inline">\(\Delta\varepsilon_{P} \approx -2 \,\, \text{to} \, -5~k_B T\)</span> <span class="citation" data-cites="Brewster2012"> [<a href="#ref-Brewster2012" role="doc-biblioref">68</a>]</span>, so that <span class="math inline">\(\frac{P}{N_{NS}}e^{-\beta\Delta\varepsilon_{P}} \approx 0.01\)</span>) and (2) <span class="math inline">\(\frac{R_I}{N_{NS}}e^{-\beta \Delta\varepsilon_{RI}} \ll 1 + \frac{R_A}{N_{NS}} e^{-\beta\Delta\varepsilon_{RA}}\)</span> which reflects our assumption that the inactive repressor binds weakly to the promoter of interest. Using these approximations, the fold-change reduces to the form <span class="math display">\[
\text{fold-change} \approx
\left(1+\frac{R_A}{N_{NS}}e^{-\beta \Delta\varepsilon_{RA}}\right)^{-1} \equiv
\left( 1+p_A(c) \frac{R}{N_{NS}}e^{-\beta \Delta\varepsilon_{RA}}
\right)^{-1},
\label{eq:fold_change_approx}
\]</span> where in the last step we have introduced the fraction <span class="math inline">\(p_A(c)\)</span> of repressors in the active state given a concentration <span class="math inline">\(c\)</span> of inducer, such that <span class="math inline">\(R_A(c)=p_A(c) R\)</span>. Since inducer binding shifts the repressors from the active to the inactive state, <span class="math inline">\(p_A(c)\)</span> grows smaller as <span class="math inline">\(c\)</span> increases <span class="citation" data-cites="Marzen2013"> [<a href="#ref-Marzen2013" role="doc-biblioref">44</a>]</span>.</p>
<p>We use the MWC model to compute the probability <span class="math inline">\(p_A(c)\)</span> that a repressor with <span class="math inline">\(n\)</span> inducer binding sites will be active. The value of <span class="math inline">\(p_A(c)\)</span> is given by the sum of the weights of the active repressor states divided by the sum of the weights of all possible repressor states (see Fig. 2(B)), namely, <span class="math display">\[
p_A(c) = \frac{\left(1+\frac{c}{K_A}\right)^n}{\left(1+\frac{c}{K_A}\right)^n +
e^{-\beta \Delta \varepsilon_{AI} }\left(1+\frac{c}{K_I}\right)^n},
\label{eq:p_active}
\]</span> where <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span> represent the dissociation constant between the inducer and repressor in the active and inactive states, respectively, and <span class="math inline">\(\Delta \varepsilon_{AI} = \varepsilon_{I} - \varepsilon_{A}\)</span> is the free energy difference between a repressor in the inactive and active state (the quantity <span class="math inline">\(e^{-\Delta \beta \varepsilon_{AI}}\)</span> is sometimes denoted by <span class="math inline">\(L\)</span> <span class="citation" data-cites="MONOD1965 Marzen2013"> [<a href="#ref-Marzen2013" role="doc-biblioref">44</a>,<a href="#ref-MONOD1965" role="doc-biblioref">57</a>]</span> or <span class="math inline">\(K_{\text{RR}*}\)</span> <span class="citation" data-cites="Daber2011a"> [<a href="#ref-Daber2011a" role="doc-biblioref">55</a>]</span>). In this equation, <span class="math inline">\(\frac{c}{K_A}\)</span> and <span class="math inline">\(\frac{c}{K_I}\)</span> represent the change in free energy when an inducer binds to a repressor in the active or inactive state, respectively, while <span class="math inline">\(e^{-\beta \Delta \varepsilon_{AI} }\)</span> represents the change in free energy when the repressor changes from the active to inactive state in the absence of inducer. Thus, a repressor which favors the active state in the absence of inducer (<span class="math inline">\(\Delta \varepsilon_{AI} &gt; 0\)</span>) will be driven towards the inactive state upon inducer binding when <span class="math inline">\(K_I &lt; K_A\)</span>. The specific case of a repressor dimer with <span class="math inline">\(n=2\)</span> inducer binding sites is shown in Fig. 2(B).</p>
<p>Substituting <span class="math inline">\(p_A(c)\)</span> from Eq. <span class="math inline">\(\ref{eq:p_active}\)</span> into Eq. <span class="math inline">\(\ref{eq:fold_change_approx}\)</span> yields the general formula for induction of a simple repression regulatory architecture <span class="citation" data-cites="Phillips2015a"> [<a href="#ref-Phillips2015a" role="doc-biblioref">67</a>]</span>, namely, <span class="math display">\[
\text{fold-change} = 
\left( 1+\frac{\left(1+\frac{c}{K_A}\right)^n}{\left(1+\frac{c}{K_A}\right)^n +
e^{-\beta \Delta \varepsilon_{AI} }\left(1+\frac{c}{K_I}\right)^n}
\frac{R}{N_{NS}}e^{-\beta \Delta\varepsilon_{RA}} \right)^{-1}.
\label{eq:fold_change_full}
\]</span> While we have used the specific case of simple repression with induction to craft this model, the same mathematics describe the case of corepression in which binding of an allosteric effector stabilizes the active state of the repressor and decreases gene expression (see Fig. 1(B)). Interestingly, we shift from induction (governed by <span class="math inline">\(K_I &lt; K_A\)</span>) to corepression (<span class="math inline">\(K_I &gt; K_A\)</span>) as the ligand transitions from preferentially binding to the inactive repressor state to stabilizing the active state. Furthermore, this general approach can be used to describe a variety of other motifs such as activation, multiple repressor binding sites, and combinations of activator and repressor binding sites <span class="citation" data-cites="Bintu2005 Brewster2014 Weinert2014"> [<a href="#ref-Weinert2014" role="doc-biblioref">47</a>,<a href="#ref-Bintu2005" role="doc-biblioref">62</a>,<a href="#ref-Brewster2014" role="doc-biblioref">70</a>]</span>.</p>
<p>The formula presented in Eq. <span class="math inline">\(\ref{eq:fold_change_full}\)</span> enables us to make precise quantitative statements about induction profiles. Motivated by the broad range of predictions implied by Eq. <span class="math inline">\(\ref{eq:fold_change_full}\)</span>, we designed a series of experiments using the <em>lac</em> system in <em>E. coli</em> to tune the control parameters for a simple repression genetic circuit. As discussed in Fig. 1(C), previous studies from our lab have provided well-characterized values for many of the parameters in our experimental system, leaving only the values of the the MWC parameters (<span class="math inline">\(K_A\)</span>, <span class="math inline">\(K_I\)</span>, and <span class="math inline">\(\Delta \varepsilon_{AI}\)</span>) to be determined. We note that while previous studies have obtained values for <span class="math inline">\(K_A\)</span>, <span class="math inline">\(K_I\)</span>, and <span class="math inline">\(L=e^{-\beta \Delta \varepsilon_{AI}}\)</span> <span class="citation" data-cites="OGorman1980 Daber2011a"> [<a href="#ref-OGorman1980" role="doc-biblioref">46</a>,<a href="#ref-Daber2011a" role="doc-biblioref">55</a>]</span>, they were either based upon biochemical experiments or <em>in vivo</em> conditions involving poorly characterized transcription factor copy numbers and gene copy numbers. These differences relative to our experimental conditions and fitting techniques led us to believe that it was important to perform our own analysis of these parameters. After inferring these three MWC parameters (see Chapter 4 for details regarding the inference of <span class="math inline">\(\Delta \varepsilon_{AI}\)</span>, which was fitted separately from <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span>), we were able to predict the input/output response of the system under a broad range of experimental conditions. For example, this framework can predict the response of the system at different repressor copy numbers <span class="math inline">\(R\)</span>, repressor-operator affinities <span class="math inline">\(\Delta\varepsilon_{RA}\)</span>, inducer concentrations <span class="math inline">\(c\)</span>, and gene copy numbers (see Chapter 4).</p>
<h3 id="experimental-design">Experimental Design</h3>
<p>We test our model by predicting the induction profiles for an array of strains that could be made using previously characterized repressor copy numbers and DNA binding energies. Our approach contrasts with previous studies that have parameterized induction curves of simple repression motifs, as these have relied on expression systems where proteins are expressed from plasmids, resulting in highly variable and unconstrained copy numbers <span class="citation" data-cites="Murphy2007 Daber2009 Murphy2010 Daber2011a Sochor2014"> [<a href="#ref-Murphy2010" role="doc-biblioref">4</a>,<a href="#ref-Murphy2007" role="doc-biblioref">5</a>,<a href="#ref-Sochor2014" role="doc-biblioref">20</a>,<a href="#ref-Daber2009" role="doc-biblioref">45</a>,<a href="#ref-Daber2011a" role="doc-biblioref">55</a>]</span>. Instead, our approach relies on a foundation of previous work as depicted in Fig. 1(C). This includes work from our laboratory that used <em>E. coli</em> constructs based on components of the <em>lac</em> system to demonstrate how the Lac repressor (LacI) copy number <span class="math inline">\(R\)</span> and operator binding energy <span class="math inline">\(\Delta\varepsilon_{RA}\)</span> affect gene expression in the absence of inducer <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>]</span>. <span class="citation" data-cites="Rydenfelt2014B"> [<a href="#ref-Rydenfelt2014B" role="doc-biblioref">34</a>]</span> extended the theory used in that work to the case of multiple promoters competing for a given transcription factor, which was validated experimentally by <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">70</a>]</span>, who modified this system to consider expression from multiple-copy plasmids as well as the presence of competing repressor binding sites.</p>
<p>The present study extends this body of work by introducing three additional biophysical parameters–<span class="math inline">\(\Delta\varepsilon_{AI}\)</span>, <span class="math inline">\(K_A\)</span>, and <span class="math inline">\(K_I\)</span>–which capture the allosteric nature of the transcription factor and complement the results shown by <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>]</span> and <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">70</a>]</span>. Although the current work focuses on systems with a single site of repression, in <a href="#ch2_methods">Materials &amp; Methods</a>, we utilize data from <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">70</a>]</span>, in which multiple sites of repression are explored, to characterize the allosteric free energy difference <span class="math inline">\(\Delta\varepsilon_{AI}\)</span> between the repressor’s active and inactive states. As explained in that Section, this additional data set is critical because multiple degenerate sets of parameters can characterize an induction curve equally well, with the <span class="math inline">\(\Delta\varepsilon_{AI}\)</span> parameter compensated by the inducer dissociation constants <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span> (see Chapter 4). After fixing <span class="math inline">\(\Delta\varepsilon_{AI}\)</span> as described in the <a href="#ch2_methods">Materials &amp; Methods</a>, we can use data from single-site simple repression systems to determine the values of <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span>.</p>
<p>We determine the values of <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span> by fitting to a single induction profile using Bayesian inferential methods <span class="citation" data-cites="Sivia2006"> [<a href="#ref-Sivia2006" role="doc-biblioref">22</a>]</span>. We then use Eq. <span class="math inline">\(\ref{eq:fold_change_full}\)</span> to predict gene expression for any concentration of inducer, repressor copy number, and DNA binding energy and compare these predictions against experimental measurements. To obtain induction profiles for a set of strains with varying repressor copy numbers, we used modified <em>lacI</em> ribosomal binding sites from <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>]</span> to generate strains with mean repressor copy number per cell of <span class="math inline">\(R = 22 \pm 4\)</span>, <span class="math inline">\(60 \pm 20\)</span>, <span class="math inline">\(124 \pm 30\)</span>, <span class="math inline">\(260 \pm 40\)</span>, <span class="math inline">\(1220 \pm 160\)</span>, and <span class="math inline">\(1740 \pm 340\)</span>, where the error denotes standard deviation of at least three replicates as measured by <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>]</span>. We note that <span class="math inline">\(R\)</span> refers to the number of repressor dimers in the cell, which is twice the number of repressor tetramers reported by <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>]</span>; since both heads of the repressor are assumed to always be either specifically or non-specifically bound to the genome, the two repressor dimers in each LacI tetramer can be considered independently. Gene expression was measured using a Yellow Fluorescent Protein (YFP) gene, driven by a <em>lacUV5</em> promoter. Each of the six repressor copy number variants were paired with the native O1, O2, or O3 <em>lac</em> operator <span class="citation" data-cites="Oehler1994"> [<a href="#ref-Oehler1994" role="doc-biblioref">54</a>]</span> placed at the YFP transcription start site, thereby generating eighteen unique strains. The repressor-operator binding energies (O1 <span class="math inline">\(\Delta\varepsilon_{RA} = -15.3 \pm 0.2~k_BT\)</span>, O2 <span class="math inline">\(\Delta\varepsilon_{RA} = -13.9~k_BT \pm 0.2\)</span>, and O3 <span class="math inline">\(\Delta\varepsilon_{RA} = -9.7 \pm 0.1~k_BT\)</span>) were previously inferred by measuring the fold-change of the <em>lac</em> system at different repressor copy numbers, where the error arises from model fitting <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>]</span>. Additionally, we were able to obtain the value <span class="math inline">\(\Delta \varepsilon_{AI} = 4.5\; k_BT\)</span> by fitting to previous data as discussed in Materials &amp; Methods. We measure fold-change over a range of known IPTG concentrations <span class="math inline">\(c\)</span>, using <span class="math inline">\(n=2\)</span> inducer binding sites per LacI dimer and approximating the number of non-specific binding sites as the length in base-pairs of the <em>E. coli</em> genome, <span class="math inline">\(N_{NS} = 4.6 \times 10^6\)</span>.</p>
<p>Our experimental pipeline for determining fold-change using flow cytometry is shown in Fig. 3. Briefly, cells were grown to exponential phase, in which gene expression reaches steady state <span class="citation" data-cites="Scott2010"> [<a href="#ref-Scott2010" role="doc-biblioref">17</a>]</span>, under concentrations of the inducer IPTG ranging between 0 and <span class="math inline">\(5\,\text{mM}\)</span>. We measure YFP fluorescence using flow cytometry and automatically gate the data to include only single-cell measurements (see Materials &amp; Methods). To validate the use of flow cytometry, we also measured the fold-change of a subset of strains using the established method of single-cell microscopy (see Chapter 4). We found that the fold-change measurements obtained from microscopy were indistinguishable from that of flow-cytometry and yielded values for the inducer binding constants <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span> that were within error.</p>
<figure>
<img src="ch2_fig03.png" id="fig:ch2_fig03" data-short-caption="An experimental pipeline for high-throughput fold-change measurements" alt="Figure 3: An experimental pipeline for high-throughput fold-change measurements. Cells are grown to exponential steady state and their fluorescence is measured using flow cytometry. Automatic gating methods using forward- and side-scattering are used to ensure that all measurements come from single cells (see Materials &amp; Methods). Mean expression is then quantified at different IPTG concentrations (top, blue histograms) and for a strain without repressor (bottom, green histograms), which shows no response to IPTG as expected. Fold-change is computed by dividing the mean fluorescence in the presence of repressor by the mean fluorescence in the absence of repressor." /><figcaption aria-hidden="true">Figure 3: <strong>An experimental pipeline for high-throughput fold-change measurements.</strong> Cells are grown to exponential steady state and their fluorescence is measured using flow cytometry. Automatic gating methods using forward- and side-scattering are used to ensure that all measurements come from single cells (see <a href="#ch2_methods">Materials &amp; Methods</a>). Mean expression is then quantified at different IPTG concentrations (top, blue histograms) and for a strain without repressor (bottom, green histograms), which shows no response to IPTG as expected. Fold-change is computed by dividing the mean fluorescence in the presence of repressor by the mean fluorescence in the absence of repressor.</figcaption>
</figure>
<h3 id="determination-of-the-in-vivo-mwc-parameters">Determination of the <em>in vivo</em> MWC Parameters</h3>
<p>The three parameters that we tune experimentally are shown in Fig. 4(A), leaving the three allosteric parameters (<span class="math inline">\(\Delta \varepsilon_{AI}\)</span>, <span class="math inline">\(K_A\)</span>, and <span class="math inline">\(K_I\)</span>) to be determined by fitting. We used previous LacI fold-change data <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">70</a>]</span> to infer that <span class="math inline">\(\Delta\varepsilon_{AI} = 4.5~k_BT\)</span> (see Chapter 4). Rather than fitting <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span> to our entire data set of eighteen unique constructs, we performed Bayesian parameter estimation on data from a single strain with <span class="math inline">\(R=260\)</span> and an O2 operator (<span class="math inline">\(\Delta\varepsilon_{RA}=-13.9~k_BT\)</span> <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>]</span>) shown in Fig. 4(D) (white circles). Using Markov Chain Monte Carlo, we determine the most likely parameter values to be <span class="math inline">\(K_A=139^{+29}_{-22} \times 10^{-6} \, \text{M}\)</span> and <span class="math inline">\(K_I=0.53^{+0.04}_{-0.04} \times 10^{-6}\, \text{M}\)</span>, which are the modes of their respective distributions, where the superscripts and subscripts represent the upper and lower bounds of the <span class="math inline">\(95^\text{th}\)</span> percentile of the parameter value distributions (see Fig. 4(B)). Unfortunately, we are not able to make a meaningful value-for-value comparison of our parameters to those of earlier studies <span class="citation" data-cites="Daber2009 Daber2011a"> [<a href="#ref-Daber2009" role="doc-biblioref">45</a>,<a href="#ref-Daber2011a" role="doc-biblioref">55</a>]</span> because of uncertainties in both gene copy number and transcription factor copy numbers in these studies (see Chapter 4). We then predicted the fold-change for the remaining seventeen strains with no further fitting (see Fig. 4(C)-(E)) together with the specific phenotypic properties described in and discussed in detail below (see Fig. 4(F)-(J)). The shaded regions in - denote the 95% credible regions. Factors determining the width of the credible regions are explored in Chapter 4.</p>
<p>We stress that the entire suite of predictions in is based upon the induction profile of a single strain. Our ability to make such a broad range of predictions stems from the fact that our parameters of interest –such as the repressor copy number and DNA binding energy–appear as distinct physical parameters within our model. While the single data set in Fig. 4(D) could also be fit using a Hill function, such an analysis would be unable to predict any of the other curves in the figure (see Chapter 4). Phenomenological expressions such as the Hill function can describe data, but lack predictive power and are thus unable to build our intuition, help us design <em>de novo</em> input-output functions, or guide future experiments <span class="citation" data-cites="Kuhlman2007 Murphy2007"> [<a href="#ref-Murphy2007" role="doc-biblioref">5</a>,<a href="#ref-Kuhlman2007" role="doc-biblioref">64</a>]</span>.</p>
<figure>
<img src="ch2_fig04.png" id="fig:ch2_fig04" data-short-caption="Predicting induction profiles for different biological control parameters" alt="Figure 4: Predicting induction profiles for different biological control parameters. (A) We can quantitatively tune R via ribosomal binding site (RBS) modifications, \Delta\varepsilon_{RA} by mutating the operator sequence, and c by adding different amounts of IPTG to the growth medium. (B) Previous experiments have characterized the R, N_{NS}, \Delta\varepsilon_{RA}, and \Delta\varepsilon_{AI} parameters (see Fig. 1(C)), leaving only the unknown dissociation constants K_A and K_I between the inducer and the repressor in the active and inactive states, respectively. These two parameters can be inferred using Bayesian parameter estimation from a single induction curve. (C-E) Predicted IPTG titration curves for different repressor copy numbers and operator strengths. Titration data for the O2 strain (white circles in Panel ) with R=260, \Delta\varepsilon_{RA} = -13.9~k_BT, n=2, and \Delta\varepsilon_{AI}=4.5~k_BT can be used to determine the thermodynamic parameters K_A=139^{+29}_{-22} \times 10^{-6} \, \text{M} and K_I=0.53^{+0.04}_{-0.04} \times 10^{-6}\, \text{M} (orange line). The remaining solid lines predict the fold-change Eq. \ref{eq:fold_change_full} for all other combinations of repressor copy numbers (shown in the legend) and repressor-DNA binding energies corresponding to the O1 operator (-15.3~k_B T), O2 operator (-13.9~k_B T), and O3 operator (-9.7~k_B T). Error bars of experimental data show the standard error of the mean (eight or more replicates) when this error is not smaller than the diameter of the data point. The shaded regions denote the 95% credible region, although the credible region is obscured when it is thinner than the curve itself. To display the measured fold-change in the absence of inducer, we alter the scaling of the x-axis between 0 and 10^{-7} M to linear rather than logarithmic, as indicated by a dashed line. Additionally, our model allows us to investigate key phenotypic properties of the induction profiles (see Fig. 1(B)). (F-J) Specifically, we show predictions for the (F) leakiness, (G) saturation, (H) dynamic range, (I) [EC_{50}], and (J) effective Hill coefficient of the induction profiles. The Python code (ch2_fig04.py) used to generate this figure can be found on the original paper GitHub repository." /><figcaption aria-hidden="true">Figure 4: <strong>Predicting induction profiles for different biological control parameters.</strong> (A) We can quantitatively tune <span class="math inline">\(R\)</span> via ribosomal binding site (RBS) modifications, <span class="math inline">\(\Delta\varepsilon_{RA}\)</span> by mutating the operator sequence, and <span class="math inline">\(c\)</span> by adding different amounts of IPTG to the growth medium. (B) Previous experiments have characterized the <span class="math inline">\(R\)</span>, <span class="math inline">\(N_{NS}\)</span>, <span class="math inline">\(\Delta\varepsilon_{RA}\)</span>, and <span class="math inline">\(\Delta\varepsilon_{AI}\)</span> parameters (see Fig. 1(C)), leaving only the unknown dissociation constants <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span> between the inducer and the repressor in the active and inactive states, respectively. These two parameters can be inferred using Bayesian parameter estimation from a single induction curve. (C-E) Predicted IPTG titration curves for different repressor copy numbers and operator strengths. Titration data for the O2 strain (white circles in Panel ) with <span class="math inline">\(R=260\)</span>, <span class="math inline">\(\Delta\varepsilon_{RA} = -13.9~k_BT\)</span>, <span class="math inline">\(n=2\)</span>, and <span class="math inline">\(\Delta\varepsilon_{AI}=4.5~k_BT\)</span> can be used to determine the thermodynamic parameters <span class="math inline">\(K_A=139^{+29}_{-22} \times 10^{-6} \, \text{M}\)</span> and <span class="math inline">\(K_I=0.53^{+0.04}_{-0.04} \times 10^{-6}\, \text{M}\)</span> (orange line). The remaining solid lines predict the fold-change Eq. <span class="math inline">\(\ref{eq:fold_change_full}\)</span> for all other combinations of repressor copy numbers (shown in the legend) and repressor-DNA binding energies corresponding to the O1 operator (<span class="math inline">\(-15.3~k_B T\)</span>), O2 operator (<span class="math inline">\(-13.9~k_B T\)</span>), and O3 operator (<span class="math inline">\(-9.7~k_B T\)</span>). Error bars of experimental data show the standard error of the mean (eight or more replicates) when this error is not smaller than the diameter of the data point. The shaded regions denote the 95% credible region, although the credible region is obscured when it is thinner than the curve itself. To display the measured fold-change in the absence of inducer, we alter the scaling of the <span class="math inline">\(x\)</span>-axis between <span class="math inline">\(0\)</span> and <span class="math inline">\(10^{-7}\)</span> M to linear rather than logarithmic, as indicated by a dashed line. Additionally, our model allows us to investigate key phenotypic properties of the induction profiles (see Fig. 1(B)). (F-J) Specifically, we show predictions for the (F) leakiness, (G) saturation, (H) dynamic range, (I) <span class="math inline">\([EC_{50}]\)</span>, and (J) effective Hill coefficient of the induction profiles. The Python code <a href="https://github.com/RPGroup-PBoC/mwc_induction/blob/master/code/figures/fig4.py">(<code>ch2_fig04.py</code>)</a> used to generate this figure can be found on the original paper <a href="https://github.com/RPGroup-PBoC/mwc_induction">GitHub repository.</a></figcaption>
</figure>
<h3 id="comparison-of-experimental-measurements-with-theoretical-predictions">Comparison of Experimental Measurements with Theoretical Predictions</h3>
<p>We tested the predictions shown in Fig. 4 by measuring fold-change induction profiles in strains with a broad range of repressor copy numbers and repressor binding energies as characterized in <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>]</span>. With a few notable exceptions, the results shown in Fig. 5 demonstrate agreement between theory and experiment. We note that there was an apparently systematic shift in the O3 <span class="math inline">\(\Delta\varepsilon_{RA} = -9.7\ k_BT\)</span> strains (Fig. 5(C)) and all of the <span class="math inline">\(R=1220\)</span> and <span class="math inline">\(R =1740\)</span> strains. This may be partially due to imprecise previous determinations of their <span class="math inline">\(\Delta\varepsilon_{RA}\)</span> and <span class="math inline">\(R\)</span> values. By performing a global fit where we infer all parameters including the repressor copy number <span class="math inline">\(R\)</span> and the binding energy <span class="math inline">\(\Delta\varepsilon_{RA}\)</span>, we found better agreement for these strains, although a discrepancy in the steepness of the response for all O3 strains remains (see Chapter 4). We considered a number of hypotheses to explain these discrepancies such as including other states (e.g. non-negligible binding of the inactive repressor), relaxing the weak promoter approximation, and accounting for variations in gene and repressor copy number throughout the cell cycle, but none explained the observed discrepancies. As an additional test of our model, we considered strains using the synthetic Oid operator which exhibits an especially strong binding energy of <span class="math inline">\(\Delta\varepsilon_{RA}=-17~k_B T\)</span> <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>]</span>. The global fit agrees well with the Oid microscopy data, though it asserts a stronger Oid binding energy of <span class="math inline">\(\Delta\varepsilon_{RA}=-17.7~k_B T\)</span> (see Chapter 4).</p>
<p>To ensure that the agreement between our predictions and data is not an accident of the strain we used to perform our fitting, we also inferred <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span> from each of the other strains. As shown in Chapter 4 and Fig. 5(D), the inferred values of <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span> depend minimally upon which strain is chosen, indicating that these parameter values are highly robust. We also performed a global fit using the data from all eighteen strains in which we fitted for the inducer dissociation constants <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span>, the repressor copy number <span class="math inline">\(R\)</span>, and the repressor DNA binding energy <span class="math inline">\(\Delta\varepsilon_{RA}\)</span> (see Chapter 4). The resulting parameter values were nearly identical to those fitted from any single strain. For the remainder of the text we continue using parameters fitted from the strain with <span class="math inline">\(R=260\)</span> repressors and an O2 operator.</p>
<figure>
<img src="ch2_fig05.png" id="fig:ch2_fig05" data-short-caption="Comparison of predictions against measured and inferred data" alt="Figure 5: Comparison of predictions against measured and inferred data. (A-C) Flow cytometry measurements of fold-change over a range of IPTG concentrations for O1, O2, and O3 strains at varying repressor copy numbers, overlaid on the predicted responses. Error bars for the experimental data show the standard error of the mean (eight or more replicates). As discussed in Fig. 4, all of the predicted induction curves were generated prior to measurement by inferring the MWC parameters using a single data set (O2 R=260, shown by white circles in Panel ). The predictions may therefore depend upon which strain is used to infer the parameters. (D) The inferred parameter values of the dissociation constants K_A and K_I using any of the eighteen strains instead of the O2 R=260 strain. Nearly identical parameter values are inferred from each strain, demonstrating that the same set of induction profiles would have been predicted regardless of which strain was chosen. The points show the mode, and the error bars denote the 95\% credible region of the parameter value distribution. Error bars not visible are smaller than the size of the marker. The Python code (ch2_fig05.py) used to generate this figure can be found on the original paper GitHub repository." /><figcaption aria-hidden="true">Figure 5: <strong>Comparison of predictions against measured and inferred data.</strong> (A-C) Flow cytometry measurements of fold-change over a range of IPTG concentrations for O1, O2, and O3 strains at varying repressor copy numbers, overlaid on the predicted responses. Error bars for the experimental data show the standard error of the mean (eight or more replicates). As discussed in Fig. 4, all of the predicted induction curves were generated prior to measurement by inferring the MWC parameters using a single data set (O2 <span class="math inline">\(R=260\)</span>, shown by white circles in Panel ). The predictions may therefore depend upon which strain is used to infer the parameters. (D) The inferred parameter values of the dissociation constants <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span> using any of the eighteen strains instead of the O2 <span class="math inline">\(R=260\)</span> strain. Nearly identical parameter values are inferred from each strain, demonstrating that the same set of induction profiles would have been predicted regardless of which strain was chosen. The points show the mode, and the error bars denote the <span class="math inline">\(95\%\)</span> credible region of the parameter value distribution. Error bars not visible are smaller than the size of the marker. The Python code <a href="https://github.com/RPGroup-PBoC/mwc_induction/blob/master/code/figures/fig5.py">(<code>ch2_fig05.py</code>)</a> used to generate this figure can be found on the original paper <a href="https://github.com/RPGroup-PBoC/mwc_induction">GitHub repository.</a></figcaption>
</figure>
<h3 id="predicting-the-phenotypic-traits-of-the-induction-response">Predicting the Phenotypic Traits of the Induction Response</h3>
<p>A subset of the properties shown in Fig. 1(i.e. the leakiness, saturation, dynamic range, <span class="math inline">\([EC_{50}]\)</span>, and effective Hill coefficient) are of significant interest to synthetic biology. For example, synthetic biology is often focused on generating large responses (i.e. a large dynamic range) or finding a strong binding partner (i.e. a small <span class="math inline">\([EC_{50}]\)</span>) <span class="citation" data-cites="Brophy2014 Shis2014"> [<a href="#ref-Brophy2014" role="doc-biblioref">14</a>,<a href="#ref-Shis2014" role="doc-biblioref">18</a>]</span>. While these properties are all individually informative, when taken together they capture the essential features of the induction response. We reiterate that a Hill function approach cannot predict these features <em>a priori</em> and furthermore requires fitting each curve individually. The MWC model, on the other hand, enables us to quantify how each trait depends upon a single set of physical parameters as shown by Fig. 4(F-J).</p>
<p>We define these five phenotypic traits using expressions derived from the model, Eq. <span class="math inline">\(\ref{eq:fold_change_full}\)</span>. These results build upon extensive work by <span class="citation" data-cites="Martins2011"> [<a href="#ref-Martins2011" role="doc-biblioref">48</a>]</span>, who computed many such properties for ligand-receptor binding within the MWC model. We begin by analyzing the leakiness, which is the minimum fold-change observed in the absence of ligand, given by <span class="math display">\[
\begin{split}
\text{leakiness} &amp;= \text{fold-change}(c=0) \\
&amp;= \left(
    1+\frac{1}{1+e^{-\beta \Delta \varepsilon_{AI} }}
    \frac{R}{N_{NS}}e^{-\beta \Delta\varepsilon_{RA}} \right)^{-1},
\end{split}
\label{eq:leakiness}
\]</span> and the saturation, which is the maximum fold change observed in the presence of saturating ligand, <span class="math display">\[
\begin{split}
\text{saturation} &amp;= \text{fold-change}(c \to \infty) \\
&amp;= \left(
    1+\frac{1}{1+e^{-\beta \Delta \varepsilon_{AI} } 
    \left(\frac{K_A}{K_I}\right)^n }\frac{R}{N_{NS}}
    e^{-\beta \Delta\varepsilon_{RA}} \right)^{-1}.
\end{split}
\label{eq:saturation}
\]</span></p>
<p>Systems that minimize leakiness repress strongly in the absence of effector while systems that maximize saturation have high expression in the presence of effector. Together, these two properties determine the dynamic range of a system’s response, which is given by the difference <span class="math display">\[
    \text{dynamic range} = \text{saturation} - \text{leakiness}.
\label{eq:dynamic_range_def}
\]</span> These three properties are shown in Fig. 4(F-H). We discuss these properties in greater detail in Chapter 4. Fig. 6(A-C) shows that the measurements of these three properties, derived from the fold-change data in the absence of IPTG and the presence of saturating IPTG, closely match the predictions for all three operators.</p>
<figure>
<img src="ch2_fig06.png" id="fig:ch2_fig06" data-short-caption="Predictions and experimental measurements of key properties of induction profiles" alt="Figure 6: Predictions and experimental measurements of key properties of induction profiles. Data for the leakiness, saturation, and dynamic range are obtained from fold-change measurements in Fig. 5 in the absence of IPTG and at saturating concentrations of IPTG. The three repressor-operator binding energies in the legend correspond to the O1 operator (-15.3~k_B T), O2 operator (-13.9~k_B T), and O3 operator (-9.7~k_B T). Both the [EC_{50}] and effective Hill coefficient are inferred by individually fitting each operator-repressor pairing in Fig. 5(A-C) separately to Eq. \ref{eq:fold_change_full} in order to smoothly interpolate between the data points. Error bars for (A-C) represent the standard error of the mean for eight or more replicates; error bars for (D-E) represent the 95% credible region for the parameter found by propagating the credible region of our estimates of K_A and K_I into Eq. \ref{eq:ec50} and Eq. \ref{eq:effective_Hill}. The Python code (ch2_fig06.py) used to generate this figure can be found on the original paper GitHub repository." /><figcaption aria-hidden="true">Figure 6: <strong>Predictions and experimental measurements of key properties of induction profiles.</strong> Data for the leakiness, saturation, and dynamic range are obtained from fold-change measurements in Fig. 5 in the absence of IPTG and at saturating concentrations of IPTG. The three repressor-operator binding energies in the legend correspond to the O1 operator (<span class="math inline">\(-15.3~k_B T\)</span>), O2 operator (<span class="math inline">\(-13.9~k_B T\)</span>), and O3 operator (<span class="math inline">\(-9.7~k_B T\)</span>). Both the <span class="math inline">\([EC_{50}]\)</span> and effective Hill coefficient are inferred by individually fitting each operator-repressor pairing in Fig. 5(A-C) separately to Eq. <span class="math inline">\(\ref{eq:fold_change_full}\)</span> in order to smoothly interpolate between the data points. Error bars for (A-C) represent the standard error of the mean for eight or more replicates; error bars for (D-E) represent the 95% credible region for the parameter found by propagating the credible region of our estimates of <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span> into Eq. <span class="math inline">\(\ref{eq:ec50}\)</span> and Eq. <span class="math inline">\(\ref{eq:effective_Hill}\)</span>. The Python code <a href="https://github.com/RPGroup-PBoC/mwc_induction/blob/master/code/figures/fig6_props.py">(<code>ch2_fig06.py</code>)</a> used to generate this figure can be found on the original paper <a href="https://github.com/RPGroup-PBoC/mwc_induction">GitHub repository.</a></figcaption>
</figure>
<p>Two additional properties of induction profiles are the <span class="math inline">\([EC_{50}]\)</span> and effective Hill coefficient, which determine the range of inducer concentration in which the system’s output goes from its minimum to maximum value. The <span class="math inline">\([EC_{50}]\)</span> denotes the inducer concentration required to generate a system response halfway between its minimum and maximum value, <span class="math display">\[
\text{fold-change}(c = [EC_{50}]) = \frac{\text{leakiness} + \text{saturation}}{2}.
\label{eq:ec50}
\]</span> The effective Hill coefficient <span class="math inline">\(h\)</span>, which quantifies the steepness of the curve at the <span class="math inline">\([EC_{50}]\)</span> <span class="citation" data-cites="Marzen2013"> [<a href="#ref-Marzen2013" role="doc-biblioref">44</a>]</span>, is given by <span class="math display">\[
h = \left( 2 \frac{d}{d \log c} \left[ \log \left( \frac{ \text{fold-change}(c)
- \text{leakiness}}{\text{dynamic range}} \right) \right] \right)_{c =
  [EC_{50}]}. 
\label{eq:effective_Hill}
\]</span> Fig. 4(I),(J) shows how the <span class="math inline">\([EC_{50}]\)</span> and effective Hill coefficient depend on the repressor copy number. In Chapter 4, we discuss the analytic forms of these two properties as well as their dependence on the repressor-DNA binding energy.</p>
<p>Fig. 6(D) and Fig. 6(E) shows the estimated values of the <span class="math inline">\([EC_{50}]\)</span> and the effective Hill coefficient overlaid on the theoretical predictions. Both properties were obtained by fitting Eq. <span class="math inline">\(\ref{eq:fold_change_full}\)</span> to each individual titration curve and computing the <span class="math inline">\([EC_{50}]\)</span> and effective Hill coefficient using Eq. <span class="math inline">\(\ref{eq:ec50}\)</span> and Eq. <span class="math inline">\(\ref{eq:effective_Hill}\)</span>, respectively. We find that the predictions made with the single strain fit closely match those made for each of the strains with O1 and O2 operators, but the predictions for the O3 operator are markedly off. In Chapter 4 we show that the large, asymmetric error bars for the O3 <span class="math inline">\(R=22\)</span> strain arise from its nearly flat response, where the lack of dynamic range makes it impossible to determine the value of the inducer dissociation constants <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span>, as can be seen in the uncertainty of both the <span class="math inline">\([EC_{50}]\)</span> and effective Hill coefficient. Discrepancies between theory and data for O3 are improved, but not fully resolved, by performing a global fit or fitting the MWC model individually to each curve (see Chapter 4). It remains an open question how to account for discrepancies in O3, in particular regarding the significant mismatch between the predicted and fitted effective Hill coefficients.</p>
<h3 id="data-collapse-of-induction-profiles">Data Collapse of Induction Profiles</h3>
<p>Our primary interest heretofore was to determine the system response at a specific inducer concentration, repressor copy number, and repressor-DNA binding energy. However, the cell does not necessarily “care about” the precise number of repressors in the system or the binding energy of an individual operator. The relevant quantity for cellular function is the fold-change enacted by the regulatory system. This raises the question: given a specific value of the fold-change, what combination of parameters will give rise to this desired response? In other words, what trade-offs between the parameters of the system will give rise to the same mean cellular output? These are key questions both for understanding how the system is governed and for engineering specific responses in a synthetic biology context. To address these questions, we follow the data collapse strategy used in a number of previous studies <span class="citation" data-cites="Sourjik2002 Keymer2006 Swem2008"> [<a href="#ref-Sourjik2002" role="doc-biblioref">8</a>,<a href="#ref-Keymer2006" role="doc-biblioref">31</a>,<a href="#ref-Swem2008" role="doc-biblioref">32</a>]</span>, and rewrite Eq. <span class="math inline">\(\ref{eq:fold_change_full}\)</span> as a Fermi function, <span class="math display">\[
\text{fold-change}= \frac{1}{1 + e^{-F(c)}},
\label{eq:free_energy_definition}
\]</span> where <span class="math inline">\(F(c)\)</span> is the free energy of the repressor binding to the operator of interest relative to the unbound operator state in <span class="math inline">\(k_B T\)</span> units <span class="citation" data-cites="Keymer2006 Swem2008 Phillips2015a"> [<a href="#ref-Keymer2006" role="doc-biblioref">31</a>,<a href="#ref-Swem2008" role="doc-biblioref">32</a>,<a href="#ref-Phillips2015a" role="doc-biblioref">67</a>]</span>, which is given by <span class="math display">\[
F(c) = \frac{\Delta\varepsilon_{RA}}{k_BT} - 
\log \frac{\left(1+\frac{c}{K_A}\right)^n}{\left(1+\frac{c}{K_A}\right)^n + 
e^{-\beta \Delta\varepsilon_{AI} }\left(1+\frac{c}{K_I}\right)^n} -
\log \frac{R}{N_{NS}}.
\label{eq:free_energy_MWC_parameters}
\]</span> The first term in <span class="math inline">\(F(c)\)</span> denotes the repressor-operator binding energy, the second the contribution from the inducer concentration, and the last the effect of the repressor copy number. We note that elsewhere, this free energy has been dubbed the Bohr parameter since such families of curves are analogous to the shifts in hemoglobin binding curves at different pHs known as the Bohr effect <span class="citation" data-cites="Mirny2010 Phillips2015a Einav2016"> [<a href="#ref-Mirny2010" role="doc-biblioref">29</a>,<a href="#ref-Einav2016" role="doc-biblioref">40</a>,<a href="#ref-Phillips2015a" role="doc-biblioref">67</a>]</span>.</p>
<p>Instead of analyzing each induction curve individually, the free energy provides a natural means to simultaneously characterize the diversity in our eighteen induction profiles. Fig. 7(A) demonstrates how the various induction curves from Fig. 4(C-E) all collapse onto a single master curve, where points from every induction profile that yield the same fold-change are mapped onto the same free energy. Fig. 7(B) shows this data collapse for the 216 data points in Fig. 5(A-C), demonstrating the close match between the theoretical predictions and experimental measurements across all eighteen strains.</p>
<p>There are many different combinations of parameter values that can result in the same free energy as defined in Eq. <span class="math inline">\(\ref{eq:free_energy_MWC_parameters}\)</span>. For example, suppose a system originally has a fold-change of 0.2 at a specific inducer concentration, and then operator mutations increase the <span class="math inline">\(\Delta\varepsilon_{RA}\)</span> binding energy <span class="citation" data-cites="Garcia2012"> [<a href="#ref-Garcia2012" role="doc-biblioref">3</a>]</span>. While this serves to initially increase both the free energy and the fold-change, a subsequent increase in the repressor copy number could bring the cell back to the original fold-change level. Such trade-offs hint that there need not be a single set of parameters that evoke a specific cellular response, but rather that the cell explores a large but degenerate space of parameters with multiple, equally valid paths.</p>
<figure>
<img src="ch2_fig07.png" id="fig:ch2_fig07" data-short-caption="Fold-change data from a broad collection of different strains collapse onto a single master curve" alt="Figure 7: Fold-change data from a broad collection of different strains collapse onto a single master curve. (A) Any combination of parameters can be mapped to a single physiological response (i.e. fold-change) via the free energy, which encompasses the parametric details of the model. (B) Experimental data from collapse onto a single master curve as a function of the free energy Eq. \ref{eq:free_energy_MWC_parameters}. The free energy for each strain was calculated from Eq. \ref{eq:free_energy_MWC_parameters} using n=2, \Delta\varepsilon_{AI}=4.5~k_BT, K_A=139 \times 10^{-6} \, \text{M}, K_I=0.53 \times 10^{-6}\, \text{M}, and the strain-specific R and \Delta\varepsilon_{RA}. All data points represent the mean, and error bars are the standard error of the mean for eight or more replicates. The Python code (ch2_fig07.py) used to generate this figure can be found on the original paper GitHub repository." /><figcaption aria-hidden="true">Figure 7: <strong>Fold-change data from a broad collection of different strains collapse onto a single master curve.</strong> (A) Any combination of parameters can be mapped to a single physiological response (i.e. fold-change) via the free energy, which encompasses the parametric details of the model. (B) Experimental data from collapse onto a single master curve as a function of the free energy Eq. <span class="math inline">\(\ref{eq:free_energy_MWC_parameters}\)</span>. The free energy for each strain was calculated from Eq. <span class="math inline">\(\ref{eq:free_energy_MWC_parameters}\)</span> using <span class="math inline">\(n=2\)</span>, <span class="math inline">\(\Delta\varepsilon_{AI}=4.5~k_BT\)</span>, <span class="math inline">\(K_A=139 \times 10^{-6} \, \text{M}\)</span>, <span class="math inline">\(K_I=0.53 \times 10^{-6}\, \text{M}\)</span>, and the strain-specific <span class="math inline">\(R\)</span> and <span class="math inline">\(\Delta\varepsilon_{RA}\)</span>. All data points represent the mean, and error bars are the standard error of the mean for eight or more replicates. The Python code <a href="https://github.com/RPGroup-PBoC/mwc_induction/blob/master/code/figures/fig7_collapse.py">(<code>ch2_fig07.py</code>)</a> used to generate this figure can be found on the original paper <a href="https://github.com/RPGroup-PBoC/mwc_induction">GitHub repository.</a></figcaption>
</figure>
<h2 id="discussion">Discussion</h2>
<p>Since the early work by Monod, Wyman, and Changeux <span class="citation" data-cites="Monod1963 MONOD1965"> [<a href="#ref-Monod1963" role="doc-biblioref">7</a>,<a href="#ref-MONOD1965" role="doc-biblioref">57</a>]</span>, an array of biological phenomena has been tied to the existence of macromolecules that switch between inactive and active states. Examples can be found in a wide variety of cellular processes, including ligand-gated ion channels <span class="citation" data-cites="Auerbach2012"> [<a href="#ref-Auerbach2012" role="doc-biblioref">24</a>]</span>, enzymatic reactions <span class="citation" data-cites="Velyvis2007 Einav2016"> [<a href="#ref-Velyvis2007" role="doc-biblioref">39</a>,<a href="#ref-Einav2016" role="doc-biblioref">40</a>]</span>, chemotaxis <span class="citation" data-cites="Keymer2006"> [<a href="#ref-Keymer2006" role="doc-biblioref">31</a>]</span>, quorum sensing <span class="citation" data-cites="Swem2008"> [<a href="#ref-Swem2008" role="doc-biblioref">32</a>]</span>, G-protein coupled receptors <span class="citation" data-cites="Canals2012"> [<a href="#ref-Canals2012" role="doc-biblioref">51</a>]</span>, physiologically important proteins <span class="citation" data-cites="Milo2007 Levantino2012a"> [<a href="#ref-Milo2007" role="doc-biblioref">6</a>,<a href="#ref-Levantino2012a" role="doc-biblioref">38</a>]</span>, and beyond. One of the most ubiquitous examples of allostery is in the context of gene expression, where an array of molecular players bind to transcription factors to influence their ability to regulate gene activity <span class="citation" data-cites="Huang2011 Li2014"> [<a href="#ref-Huang2011" role="doc-biblioref">21</a>,<a href="#ref-Li2014" role="doc-biblioref">58</a>]</span>. A number of studies have focused on developing a quantitative understanding of allosteric regulatory systems. <span class="citation" data-cites="Martins2011 Marzen2013"> [<a href="#ref-Marzen2013" role="doc-biblioref">44</a>,<a href="#ref-Martins2011" role="doc-biblioref">48</a>]</span> analytically derived fundamental properties of the MWC model, including the leakiness and dynamic range described in this work, noting the inherent trade-offs in these properties when tuning the model’s parameters. Work in the Church and Voigt labs, among others, has expanded on the availability of allosteric circuits for synthetic biology <span class="citation" data-cites="Lutz1997 Moon2012 Rogers2015 Rohlhill2017"> [<a href="#ref-Moon2012" role="doc-biblioref">11</a>–<a href="#ref-Rohlhill2017" role="doc-biblioref">13</a>,<a href="#ref-Lutz1997" role="doc-biblioref">43</a>]</span>. Recently, Daber <em>et al.</em> theoretically explored the induction of simple repression within the MWC model <span class="citation" data-cites="Daber2009"> [<a href="#ref-Daber2009" role="doc-biblioref">45</a>]</span> and experimentally measured how mutations alter the induction profiles of transcription factors <span class="citation" data-cites="Daber2011a"> [<a href="#ref-Daber2011a" role="doc-biblioref">55</a>]</span>. Vilar and Saiz analyzed a variety of interactions in inducible <em>lac</em>-based systems including the effects of oligomerization and DNA folding on transcription factor induction <span class="citation" data-cites="Leonor2008 Vilar2013"> [<a href="#ref-Leonor2008" role="doc-biblioref">10</a>,<a href="#ref-Vilar2013" role="doc-biblioref">66</a>]</span>. Other work has attempted to use the <em>lac</em> system to reconcile <em>in vitro</em> and <em>in vivo</em> measurements <span class="citation" data-cites="Tungtur2011 Sochor2014"> [<a href="#ref-Tungtur2011" role="doc-biblioref">19</a>,<a href="#ref-Sochor2014" role="doc-biblioref">20</a>]</span>.</p>
<p>Although this body of work has done much to improve our understanding of allosteric transcription factors, there have been few attempts to explicitly connect quantitative models to experiments. Here, we generate a predictive model of allosteric transcriptional regulation and then test the model against a thorough set of experiments using well-characterized regulatory components. Specifically, we used the MWC model to build upon a well-established thermodynamic model of transcriptional regulation <span class="citation" data-cites="Bintu2005 Garcia2011c"> [<a href="#ref-Bintu2005" role="doc-biblioref">62</a>,<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>]</span>, allowing us to compose the model from a minimal set of biologically meaningful parameters. This model combines both theoretical and experimental insights; for example, rather than considering gene expression directly we analyze the fold-change in expression, where the weak promoter approximation (see ) circumvents uncertainty in the RNAP copy number. The resulting model depended upon experimentally accessible parameters, namely, the repressor copy number, the repressor-DNA binding energy, and the concentration of inducer. We tested these predictions on a range of strains whose repressor copy number spanned two orders of magnitude and whose DNA binding affinity spanned 6 <span class="math inline">\(k_BT\)</span>. We argue that one would not be able to generate such a wide array of predictions by using a Hill function, which abstracts away the biophysical meaning of the parameters into phenomenological parameters <span class="citation" data-cites="Forsen1995"> [<a href="#ref-Forsen1995" role="doc-biblioref">16</a>]</span>.</p>
<p>More precisely, we tested our model in the context of a <em>lac</em>-based simple repression system by first determining the allosteric dissociation constants <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span> from a single induction data set (O2 operator with binding energy <span class="math inline">\(\Delta \varepsilon_{RA} = -13.9~k_BT\)</span> and repressor copy number <span class="math inline">\(R = 260\)</span>) and then using these values to make parameter-free predictions of the induction profiles for seventeen other strains where <span class="math inline">\(\Delta \varepsilon_{RA}\)</span> and <span class="math inline">\(R\)</span> were varied significantly (see ). We next measured the induction profiles of these seventeen strains using flow cytometry and found that our predictions consistently and accurately captured the primary features for each induction data set, as shown in -. Importantly, we find that fitting <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span> to data from any other strain would have resulted in nearly identical predictions (see Chapter 4 for further details). This suggests that a few carefully chosen measurements can lead to a deep quantitative understanding of how simple regulatory systems work without requiring an extensive sampling of strains that span the parameter space. Moreover, the fact that we could consistently achieve reliable predictions after fitting only two free parameters stands in contrast to the common practice of fitting several free parameters simultaneously, which can nearly guarantee an acceptable fit provided that the model roughly resembles the system response, regardless of whether the details of the model are tied to any underlying molecular mechanism.</p>
<p>Beyond observing changes in fold-change as a function of effector concentration, our application of the MWC model allows us to explicitly predict the values of the induction curves’ key parameters, namely, the leakiness, saturation, dynamic range, <span class="math inline">\([EC_{50}]\)</span>, and the effective Hill coefficient (see ). We are consistently able to accurately predict the leakiness, saturation, and dynamic range for each of the strains. For both the O1 and O2 data sets, our model also accurately predicts the effective Hill coefficient and <span class="math inline">\([EC_{50}]\)</span>, though these predictions for O3 are noticeably less accurate. While performing a global fit for all model parameters marginally improves the prediction for O3 (see Chapter 4), we are still unable to accurately predict the effective Hill coefficient or the <span class="math inline">\([EC_{50}]\)</span>. We further tried including additional states (such as allowing the inactive repressor to bind to the operator), relaxing the weak promoter approximation, accounting for changes in gene and repressor copy number throughout the cell cycle <span class="citation" data-cites="Jones2014a"> [<a href="#ref-Jones2014a" role="doc-biblioref">60</a>]</span>, and refitting the original binding energies from <span class="citation" data-cites="Garcia2011b"> [<a href="#ref-Garcia2011b" role="doc-biblioref">27</a>]</span>, but we were still unable to account for the O3 data. It remains an open question as to how the discrepancy between the theory and measurements for O3 can be reconciled.</p>
<p>The dynamic range, which is of considerable interest when designing or characterizing a genetic circuit, is revealed to have an interesting property: although changing the value of <span class="math inline">\(\Delta \varepsilon_{RA}\)</span> causes the dynamic range curves to shift to the right or left, each curve has the same shape and in particular the same maximum value. This means that strains with strong or weak binding energies can attain the same dynamic range when the value of <span class="math inline">\(R\)</span> is tuned to compensate for the binding energy. This feature is not immediately apparent from the IPTG induction curves, which show very low dynamic ranges for several of the O1 and O3 strains. Without the benefit of models that can predict such phenotypic traits, efforts to engineer genetic circuits with allosteric transcription factors must rely on trial and error to achieve specific responses <span class="citation" data-cites="Rogers2015 Rohlhill2017"> [<a href="#ref-Rogers2015" role="doc-biblioref">12</a>,<a href="#ref-Rohlhill2017" role="doc-biblioref">13</a>]</span>.</p>
<p>Despite the diversity observed in the induction profiles of each of our strains, our data are unified by their reliance on fundamental biophysical parameters. In particular, we have shown that our model for fold-change can be rewritten in terms of the free energy Eq. <span class="math inline">\(\ref{eq:free_energy_MWC_parameters}\)</span>, which encompasses all of the physical parameters of the system. This has proven to be an illuminating technique in a number of studies of allosteric proteins <span class="citation" data-cites="Sourjik2002 Keymer2006 Swem2008"> [<a href="#ref-Sourjik2002" role="doc-biblioref">8</a>,<a href="#ref-Keymer2006" role="doc-biblioref">31</a>,<a href="#ref-Swem2008" role="doc-biblioref">32</a>]</span>. Although it is experimentally straightforward to observe system responses to changes in effector concentration <span class="math inline">\(c\)</span>, framing the input-output function in terms of <span class="math inline">\(c\)</span> can give the misleading impression that changes in system parameters lead to fundamentally altered system responses. Alternatively, if one can find the “natural variable” that enables the output to collapse onto a single curve, it becomes clear that the system’s output is not governed by individual system parameters, but rather the contributions of multiple parameters that define the natural variable. When our fold-change data are plotted against the respective free energies for each construct, they collapse cleanly onto a single curve (see). This enables us to analyze how parameters can compensate each other. For example, rather than viewing strong repression as a consequence of low IPTG concentration <span class="math inline">\(c\)</span> or high repressor copy number <span class="math inline">\(R\)</span>, we can now observe that strong repression is achieved when the free energy <span class="math inline">\(F(c) \leq -5 k_BT\)</span>, a condition which can be reached in a number of ways.</p>
<p>While our experiments validated the theoretical predictions in the case of simple repression, we expect the framework presented here to apply much more generally to different biological instances of allosteric regulation. For example, we can use this model to study more complex systems such as when transcription factors interact with multiple operators <span class="citation" data-cites="Bintu2005"> [<a href="#ref-Bintu2005" role="doc-biblioref">62</a>]</span>. We can further explore different regulatory configurations such as corepression, activation, and coactivation, each of which are found in <em>E. coli</em> (see Chapter 4). This work can also serve as a springboard to characterize not just the mean but the full gene expression distribution and thus quantify the impact of noise on the system <span class="citation" data-cites="eldar2010"> [<a href="#ref-eldar2010" role="doc-biblioref">30</a>]</span>. Another extension of this approach would be to theoretically predict and experimentally verify whether the repressor-inducer dissociation constants <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span> or the energy difference <span class="math inline">\(\Delta \varepsilon_{AI}\)</span> between the allosteric states can be tuned by making single amino acid substitutions in the transcription factor <span class="citation" data-cites="Daber2011a Phillips2015a"> [<a href="#ref-Daber2011a" role="doc-biblioref">55</a>,<a href="#ref-Phillips2015a" role="doc-biblioref">67</a>]</span>. Finally, we expect that the kind of rigorous quantitative description of the allosteric phenomenon provided here will make it possible to construct biophysical models of fitness for allosteric proteins similar to those already invoked to explore the fitness effects of transcription factor binding site strengths and protein stability <span class="citation" data-cites="Gerland2002 Berg2004 Zeldovich2008"> [<a href="#ref-Gerland2002" role="doc-biblioref">49</a>,<a href="#ref-Berg2004" role="doc-biblioref">50</a>,<a href="#ref-Zeldovich2008" role="doc-biblioref">52</a>]</span>.</p>
<p>To conclude, we find that our application of the MWC model provides an accurate, predictive framework for understanding simple repression by allosteric transcription factors. To reach this conclusion, we analyzed the model in the context of a well-characterized system, in which each parameter had a clear biophysical meaning. As many of these parameters had been measured or inferred in previous studies, this gave us a minimal model with only two free parameters which we inferred from a single data set. We then accurately predicted the behavior of seventeen other data sets in which repressor copy number and repressor-DNA binding energy were systematically varied. In addition, our model allowed us to understand how key properties such as the leakiness, saturation, dynamic range, <span class="math inline">\([EC_{50}]\)</span>, and effective Hill coefficient depended upon the small set of parameters governing this system. Finally, we show that by framing inducible simple repression in terms of free energy, the data from all of our experimental strains collapse cleanly onto a single curve, illustrating the many ways in which a particular output can be targeted. In total, these results show that a thermodynamic formulation of the MWC model supersedes phenomenological fitting functions for understanding transcriptional regulation by allosteric proteins.</p>
<h2 id="ch2_methods">Materials &amp; Methods</h2>
<h3 id="bacterial-strains-and-dna-constructs">Bacterial Strains and DNA Constructs</h3>
<p>All strains used in these experiments were derived from <em>E. coli</em> K12 MG1655 with the <em>lac</em> operon removed, adapted from those created and described in <span class="citation" data-cites="Garcia2011c Garcia2011b"> [<a href="#ref-Garcia2011b" role="doc-biblioref">27</a>,<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>]</span>. Briefly, the operator variants and YFP reporter gene were cloned into a pZS25 background which contains a <em>lacUV5</em> promoter that drives expression as is shown schematically in . These constructs carried a kanamycin resistance gene and were integrated into the <em>galK</em> locus of the chromosome using <span class="math inline">\(\lambda\)</span> Red recombineering <span class="citation" data-cites="Sharan2009"> [<a href="#ref-Sharan2009" role="doc-biblioref">28</a>]</span>. The <em>lacI</em> gene was constitutively expressed via a P<span class="math inline">\(_\mathrm{LtetO\hbox{-}1}\)</span> promoter <span class="citation" data-cites="Lutz1997"> [<a href="#ref-Lutz1997" role="doc-biblioref">43</a>]</span>, with ribosomal binding site mutations made to vary the LacI copy number as described in <span class="citation" data-cites="Salis2009"> [<a href="#ref-Salis2009" role="doc-biblioref">36</a>]</span> using site-directed mutagenesis (Quickchange II; Stratagene), with further details in <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>]</span>. These <em>lacI</em> constructs carried a chloramphenicol resistance gene and were integrated into the <em>ybcN</em> locus of the chromosome. Final strain construction was achieved by performing repeated P1 transduction <span class="citation" data-cites="Thomason2007"> [<a href="#ref-Thomason2007" role="doc-biblioref">25</a>]</span> of the different operator and <em>lacI</em> constructs to generate each combination used in this work. Integration was confirmed by PCR amplification of the replaced chromosomal region and by sequencing. Primers and final strain genotypes are listed in Appendix <a href="#AppendixStrainlist" data-reference-type="ref" data-reference="AppendixStrainlist">[AppendixStrainlist]</a>.</p>
<p>It is important to note that the rest of the <em>lac</em> operon (<em>lacZYA</em>) was never expressed. The LacY protein is a transmembrane protein which actively transports lactose as well as IPTG into the cell. As LacY was never produced in our strains, we assume that the extracellular and intracellular IPTG concentration was approximately equal due to diffusion across the membrane into the cell as is suggested by previous work <span class="citation" data-cites="FernandezCastane2012"> [<a href="#ref-FernandezCastane2012" role="doc-biblioref">23</a>]</span>.</p>
<p>To make this theory applicable to transcription factors with any number of DNA binding domains, we used a different definition for repressor copy number than has been used previously. We define the LacI copy number as the average number of repressor dimers per cell whereas in <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>]</span>, the copy number is defined as the average number of repressor tetramers in each cell. To motivate this decision, we consider the fact that the LacI repressor molecule exists as a tetramer in <em>E. coli</em> <span class="citation" data-cites="Lewis1996"> [<a href="#ref-Lewis1996" role="doc-biblioref">56</a>]</span> in which a single DNA binding domain is formed from dimerization of LacI proteins, so that wild-type LacI might be described as dimer of dimers. Since each dimer is allosterically independent (i.e. either dimer can be allosterically active or inactive, independent of the configuration of the other dimer) <span class="citation" data-cites="Daber2009"> [<a href="#ref-Daber2009" role="doc-biblioref">45</a>]</span>, a single LacI tetramer can be treated as two functional repressors. Therefore, we have simply multiplied the number of repressors reported in <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">69</a>]</span> by a factor of two. This factor is included as a keyword argument in the numerous Python functions used to perform this analysis, as discussed in the code documentation.</p>
<p>A subset of strains in these experiments were measured using fluorescence microscopy for validation of the flow cytometry data and results. To aid in the high-fidelity segmentation of individual cells, the strains were modified to constitutively express an mCherry fluorophore. This reporter was cloned into a pZS4*1 backbone <span class="citation" data-cites="Lutz1997"> [<a href="#ref-Lutz1997" role="doc-biblioref">43</a>]</span> in which mCherry is driven by the <em>lacUV5</em> promoter. All microscopy and flow cytometry experiments were performed using these strains.</p>
<h3 id="growth-conditions-for-flow-cytometry-measurements">Growth Conditions for Flow Cytometry Measurements</h3>
<p>All measurements were performed with <em>E. coli</em> cells grown to mid-exponential phase in standard M9 minimal media (M9 5X Salts, Sigma-Aldrich M6030; <span class="math inline">\(2\,\text{mM}\)</span> magnesium sulfate, Mallinckrodt Chemicals 6066-04; <span class="math inline">\(100\,\mu\text{M}\)</span> calcium chloride, Fisher Chemicals C79-500) supplemented with 0.5% (w/v) glucose. Briefly, <span class="math inline">\(500\,\mu\text{L}\)</span> cultures of <em>E. coli</em> were inoculated into Lysogeny Broth (LB Miller Powder, BD Medical) from a 50% glycerol frozen stock (-80<span class="math inline">\(^\circ\)</span>C) and were grown overnight in a <span class="math inline">\(2\,\text{mL}\)</span> 96-deep-well plate sealed with a breathable nylon cover (Lab Pak - Nitex Nylon, Sefar America Inc. Cat. No. 241205) with rapid agitation for proper aeration. After approximately <span class="math inline">\(12\)</span> to <span class="math inline">\(15\)</span> hours, the cultures had reached saturation and were diluted 1000-fold into a second <span class="math inline">\(2\,\text{mL}\)</span> 96-deep-well plate where each well contained <span class="math inline">\(500\,\mu\text{L}\)</span> of M9 minimal media supplemented with 0.5% w/v glucose (anhydrous D-Glucose, Macron Chemicals) and the appropriate concentration of IPTG (Isopropyl <span class="math inline">\(\beta\)</span>-D-1 thiogalactopyranoside Dioxane Free, Research Products International). These were sealed with a breathable cover and were allowed to grow for approximately eight hours. Cells were then diluted ten-fold into a round-bottom 96-well plate (Corning Cat. No. 3365) containing <span class="math inline">\(90\,\mu\text{L}\)</span> of M9 minimal media supplemented with 0.5% w/v glucose along with the corresponding IPTG concentrations. For each IPTG concentration, a stock of 100-fold concentrated IPTG in double distilled water was prepared and partitioned into <span class="math inline">\(100\,\mu\text{L}\)</span> aliquots. The same parent stock was used for all experiments described in this work.</p>
<h3 id="sec:flow_cytometry">Flow Cytometry</h3>
<p>Unless explicitly mentioned, all fold-change measurements were collected on a Miltenyi Biotec MACSquant Analyzer 10 Flow Cytometer graciously provided by the Pamela Björkman lab at Caltech. Detailed information regarding the voltage settings of the photo-multiplier detectors can be found in Appendix Table <a href="#table_instrument_param" data-reference-type="ref" data-reference="table_instrument_param">[table_instrument_param]</a>. Prior to each day’s experiments, the analyzer was calibrated using MACSQuant Calibration Beads (Cat. No. 130-093-607) such that day-to-day experiments would be comparable. All YFP fluorescence measurements were collected via <span class="math inline">\(488\,\text{nm}\)</span> laser excitation coupled with a 525/<span class="math inline">\(50\,\text{nm}\)</span> emission filter. Unless otherwise specified, all measurements were taken over the course of two to three hours using automated sampling from a 96-well plate kept at approximately <span class="math inline">\(4^\circ \, \hbox{-} \, 10^\circ\)</span>C on a MACS Chill 96 Rack (Cat. No. 130-094-459). Cells were diluted to a final concentration of approximately <span class="math inline">\(4\times 10^{4}\)</span> cells per <span class="math inline">\(\mu\text{L}\)</span> which corresponded to a flow rate of 2,000-6,000 measurements per second, and acquisition for each well was halted after 100,000 events were detected. Once completed, the data were extracted and immediately processed using the following methods.</p>
<h3 id="unsupervised-gating-of-flow-cytometry-data">Unsupervised Gating of Flow Cytometry Data</h3>
<p>Flow cytometry data will frequently include a number of spurious events or other undesirable data points such as cell doublets and debris. The process of restricting the collected data set to those data determined to be “real” is commonly referred to as gating. These gates are typically drawn manually <span class="citation" data-cites="Maecker2005"> [<a href="#ref-Maecker2005" role="doc-biblioref">41</a>]</span> and restrict the data set to those points which display a high degree of linear correlation between their forward-scatter (FSC) and side-scatter (SSC). The development of unbiased and unsupervised methods of drawing these gates is an active area of research <span class="citation" data-cites="Lo2008 Aghaeepour2013"> [<a href="#ref-Lo2008" role="doc-biblioref">37</a>,<a href="#ref-Aghaeepour2013" role="doc-biblioref">42</a>]</span>. For our purposes, we assume that the fluorescence level of the population should be log-normally distributed about some mean value. With this assumption in place, we developed a method that allows us to restrict the data used to compute the mean fluorescence intensity of the population to the smallest two-dimensional region of the <span class="math inline">\(\log(\mathrm{FSC})\)</span> vs. <span class="math inline">\(\log(\mathrm{SSC})\)</span> space in which 40% of the data is found. This was performed by fitting a bivariate Gaussian distribution and restricting the data used for calculation to those that reside within the 40th percentile. This procedure is described in more detail in the supplementary information as well as in a Jupyter notebook located in this paper’s <a href="https://rpgroup-pboc.github.io/mwc_induction/code/notebooks/unsupervised_gating.html">Github repository</a>.</p>
<h3 id="experimental-determination-of-fold-change">Experimental Determination of Fold-Change</h3>
<p>For each strain and IPTG concentration, the fold-change in gene expression was calculated by taking the ratio of the population mean YFP expression in the presence of LacI repressor to that of the population mean in the absence of LacI repressor. However, the measured fluorescence intensity of each cell also includes the autofluorescence contributed by the weak excitation of the myriad protein and small molecules within the cell. To correct for this background, we computed the fold change as <span class="math display">\[
\text{fold-change} = \frac{\langle I_{R &gt; 0}
\rangle - \langle I_\text{auto}\rangle}{\langle I_{R = 0} \rangle - \langle
I_\text{auto}\rangle},
\label{eq:fold_change_fluor}
\]</span> where <span class="math inline">\(\langle I_{R &gt; 0}\rangle\)</span> is the average cell YFP intensity in the presence of repressor, <span class="math inline">\(\langle I_{R = 0}\rangle\)</span> is the average cell YFP intensity in the absence of repressor, and <span class="math inline">\(\langle I_\text{auto} \rangle\)</span> is the average cell autofluorescence intensity, as measured from cells that lack the <em>lac</em>-YFP construct.</p>
<h3 class="unnumbered" id="bayesian-parameter-estimation">Bayesian Parameter Estimation</h3>
<p>In this work, we determine the the most likely parameter values for the inducer dissociation constants <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span> of the active and inactive state, respectively, using Bayesian methods. We compute the probability distribution of the value of each parameter given the data <span class="math inline">\(D\)</span>, which by Bayes’ theorem is given by <span class="math display">\[
P(K_A, K_I \mid D) = \frac{P(D \mid K_A, K_I)P(K_A,
K_I)}{P(D)},
\label{eq:bayes_theorem}
\]</span> where <span class="math inline">\(D\)</span> is all the data composed of independent variables (repressor copy number <span class="math inline">\(R\)</span>, repressor-DNA binding energy <span class="math inline">\(\Delta\varepsilon_{RA}\)</span>, and inducer concentration <span class="math inline">\(c\)</span>) and one dependent variable (experimental fold-change). <span class="math inline">\(P(D \mid K_A, K_I)\)</span> is the likelihood of having observed the data given the parameter values for the dissociation constants, <span class="math inline">\(P(K_A, K_I)\)</span> contains all the prior information on these parameters, and <span class="math inline">\(P(D)\)</span> serves as a normalization constant, which we can ignore in our parameter estimation. assumes a deterministic relationship between the parameters and the data, so in order to construct a probabilistic relationship as required by , we assume that the experimental fold-change for the <span class="math inline">\(i^\text{th}\)</span> datum given the parameters is of the form <span class="math display">\[
\text{fold-change}
_{\exp}^{(i)} = \left( 1 + \frac{\left(1 + \frac{c^{(i)}}{K_A}\right)^2}{\left(
1 + \frac{c^{(i)}}{K_A}\right)^2 + e^{-\beta \Delta \varepsilon_{AI}} \left(1 +
\frac{c^{(i)}}{K_I} \right)^2} \frac{R^{(i)}}{N_{NS}} e^{-\beta \Delta
\varepsilon_{RA}^{(i)}}\right)^{-1} + \epsilon^{(i)},
\]</span>{eq:fold_change_exp} where <span class="math inline">\(\epsilon^{(i)}\)</span> represents the departure from the deterministic theoretical prediction for the <span class="math inline">\(i^\text{th}\)</span> data point. If we assume that these <span class="math inline">\(\epsilon^{(i)}\)</span> errors are normally distributed with mean zero and standard deviation <span class="math inline">\(\sigma\)</span>, the likelihood of the data given the parameters is of the form <span class="math display">\[
P(D \vert K_A, K_I, \sigma) =
\frac{1}{(2\pi\sigma^2)^{\frac{n}{2}}}\prod\limits_{i=1}^n \exp
\left[-\frac{(\text{fold-change}^{(i)}_{\exp} - \text{fold-change}(K_A, K_I,
R^{(i)}, \Delta\varepsilon_{RA}^{(i)}, c^{(i)}))^2}{2\sigma^2}\right],
\label{eq:likelihood}
\]</span> where <span class="math inline">\(\text{fold-change}^{(i)}_{\text{exp}}\)</span> is the experimental fold-change and <span class="math inline">\(\text{fold-change}(\,\cdots)\)</span> is the theoretical prediction. The product <span class="math inline">\(\prod_{i=1}^n\)</span> captures the assumption that the <span class="math inline">\(n\)</span> data points are independent. Note that the likelihood and prior terms now include the extra unknown parameter <span class="math inline">\(\sigma\)</span>. In applying , a choice of <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span> that provides better agreement between theoretical fold-change predictions and experimental measurements will result in a more probable likelihood.</p>
<p>Both mathematically and numerically, it is convenient to define <span class="math inline">\(\tilde{k}_A = -\log \frac{K_A}{1\,\text{M}}\)</span> and <span class="math inline">\(\tilde{k}_I = -\log \frac{K_I}{1\,\text{M}}\)</span> and fit for these parameters on a log scale. Dissociation constants are scale invariant, so that a change from <span class="math inline">\(10\,\mu\text{M}\)</span> to <span class="math inline">\(1\,\mu\text{M}\)</span> leads to an equivalent increase in affinity as a change from <span class="math inline">\(1\,\mu\text{M}\)</span> to <span class="math inline">\(0.1\,\mu\text{M}\)</span>. With these definitions we assume for the prior <span class="math inline">\(P(\tilde{k}_A, \tilde{k}_I, \sigma)\)</span> that all three parameters are independent. In addition, we assume a uniform distribution for <span class="math inline">\(\tilde{k}_A\)</span> and <span class="math inline">\(\tilde{k}_I\)</span> and a Jeffreys prior <span class="citation" data-cites="Sivia2006"> [<a href="#ref-Sivia2006" role="doc-biblioref">22</a>]</span> for the scale parameter <span class="math inline">\(\sigma\)</span>. This yields the complete prior <span class="math display">\[
P(\tilde{k}_A, \tilde{k}_I, \sigma)
\equiv \frac{1}{(\tilde{k}_A^{\max} - \tilde{k}_A^{\min})}
\frac{1}{(\tilde{k}_I^{\max} - \tilde{k}_I^{\min})}\frac{1}{\sigma}.
\label{eq:ka_ki_prior}
\]</span> These priors are maximally uninformative meaning that they imply no prior knowledge of the parameter values. We defined the <span class="math inline">\(\tilde{k}_A\)</span> and <span class="math inline">\(\tilde{k}_A\)</span> ranges uniform on the range of <span class="math inline">\(-7\)</span> to <span class="math inline">\(7\)</span>, although we note that this particular choice does not affect the outcome provided the chosen range is sufficiently wide.</p>
<p>Putting all these terms together we can now sample from <span class="math inline">\(P(\tilde{k}_A, \tilde{k}_I, \sigma \mid D)\)</span> using Markov chain Monte Carlo (see <a href="https://rpgroup-pboc.github.io/mwc_induction/code/notebooks/bayesian_parameter_estimation">GitHub repository</a>) to compute the most likely parameter as well as the error bars (given by the 95% credible region) for <span class="math inline">\(K_A\)</span> and <span class="math inline">\(K_I\)</span>.</p>
<h3 id="data-curation">Data Curation</h3>
<p>All of the data used in this work as well as all relevant code can be found at this <a href="http://rpgroup-pboc.github.io/mwc_induction">dedicated website</a>. Data were collected, stored, and preserved using the Git version control software in combination with off-site storage and hosting website GitHub. Code used to generate all figures and complete all processing step as and analyses are available on the GitHub repository. Many analysis files are stored as instructive Jupyter Notebooks. The scientific community is invited to fork our repositories and open constructive issues on the <a href="https://www.github.com/rpgroup-pboc/mwc_induction">GitHub repository</a>.</p>
<h2 class="unnumbered" id="references">References</h2>
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-Lanfranco2017" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[1] </div><div class="csl-right-inline">M. F. Lanfranco, F. Gárate, A. J. Engdahl, and R. A. Maillard, <em><span class="nocase">Asymmetric configurations in a reengineered homodimer reveal multiple subunit communication pathways in protein allostery</span></em>, The Journal of Biological Chemistry <strong>292</strong>, 6086 (2017).</div>
</div>
<div id="ref-Harman2001" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[2] </div><div class="csl-right-inline">J. G. Harman, <em><span class="nocase">Allosteric regulation of the cAMP receptor protein</span></em>, Biochimica Et Biophysica Acta (BBA) - Protein Structure and Molecular Enzymology <strong>1547</strong>, 1 (2001).</div>
</div>
<div id="ref-Garcia2012" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[3] </div><div class="csl-right-inline">H. G. Garcia, A. Sanchez, J. Q. Boedicker, M. Osborne, J. Gelles, J. Kondev, and R. Phillips, <em><span class="nocase">Operator sequence alters gene expression independently of transcription factor occupancy in bacteria</span></em>, Cell Reports <strong>2</strong>, 150 (2012).</div>
</div>
<div id="ref-Murphy2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[4] </div><div class="csl-right-inline">K. F. Murphy, R. M. Adams, X. Wang, G. Balázsi, and J. J. Collins, <em><span class="nocase">Tuning and controlling gene expression noise in synthetic gene networks</span></em>, Nucleic Acids Research <strong>38</strong>, 2712 (2010).</div>
</div>
<div id="ref-Murphy2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[5] </div><div class="csl-right-inline">K. F. Murphy, G. Balázsi, and J. J. Collins, <em><span class="nocase">Combinatorial promoter design for engineering noisy gene expression.</span></em>, Proceedings of the National Academy of Sciences <strong>104</strong>, 12726 (2007).</div>
</div>
<div id="ref-Milo2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[6] </div><div class="csl-right-inline">R. Milo, J. H. Hou, M. Springer, M. P. Brenner, and M. W. Kirschner, <em><span class="nocase">The relationship between evolutionary and physiological variation in hemoglobin.</span></em>, Proceedings of the National Academy of Sciences <strong>104</strong>, 16998 (2007).</div>
</div>
<div id="ref-Monod1963" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[7] </div><div class="csl-right-inline">J. Monod, J.-P. Changeux, and F. Jacob, <em><span class="nocase">Allosteric proteins and cellular control systems.</span></em>, Journal of Molecular Biology <strong>6</strong>, 306 (1963).</div>
</div>
<div id="ref-Sourjik2002" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[8] </div><div class="csl-right-inline">V. Sourjik and H. C. Berg, <em><span class="nocase">Receptor sensitivity in bacterial chemotaxis</span></em>, Proceedings of the National Academy of Sciences <strong>99</strong>, 123 (2002).</div>
</div>
<div id="ref-Poelwijk2011" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[9] </div><div class="csl-right-inline">F. J. Poelwijk, M. G. J. deVos, and S. J. Tans, <em>Tradeoffs and Optimality in the Evolution of Gene Regulation</em>, Cell <strong>146</strong>, 462 (2011).</div>
</div>
<div id="ref-Leonor2008" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[10] </div><div class="csl-right-inline">L. Saiz and J. M. G. Vilar, <em>Ab Initio Thermodynamic Modeling of Distal Multisite Transcription Regulation</em>, Nucleic Acids Research <strong>36</strong>, 726 (2008).</div>
</div>
<div id="ref-Moon2012" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[11] </div><div class="csl-right-inline">T. S. Moon, C. Lou, A. Tamsir, B. C. Stanton, and C. A. Voigt, <em>Genetic Programs Constructed from Layered Logic Gates in Single Cells</em>, Nature <strong>491</strong>, 249 (2012).</div>
</div>
<div id="ref-Rogers2015" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[12] </div><div class="csl-right-inline">J. K. Rogers, C. D. Guzman, N. D. Taylor, S. Raman, K. Anderson, and G. M. Church, <em>Synthetic Biosensors for Precise Gene Control and Real-Time Monitoring of Metabolites</em>, Nucleic Acids Research <strong>43</strong>, 7648 (2015).</div>
</div>
<div id="ref-Rohlhill2017" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[13] </div><div class="csl-right-inline">J. Rohlhill, N. R. Sandoval, and E. T. Papoutsakis, <em>Sort-Seq Approach to Engineering a Formaldehyde-Inducible Promoter for Dynamically Regulated <em>Escherichia Coli</em> Growth on Methanol</em>, ACS Synthetic Biology Advance online publication (2017).</div>
</div>
<div id="ref-Brophy2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[14] </div><div class="csl-right-inline">J. A. N. Brophy and C. A. Voigt, <em><span class="nocase">Principles of genetic circuit design</span></em>, Nature Methods <strong>11</strong>, 508 (2014).</div>
</div>
<div id="ref-Boedicker2013b" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[15] </div><div class="csl-right-inline">J. Q. Boedicker, H. G. Garcia, S. Johnson, and R. Phillips, <em>DNA Sequence-Dependent Mechanics and Protein-Assisted Bending in Repressor-Mediated Loop Formation</em>, Physical Biology <strong>10</strong>, 066005 (2013).</div>
</div>
<div id="ref-Forsen1995" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[16] </div><div class="csl-right-inline">S. Forsén and S. Linse, <em>Cooperativity: Over the Hill</em>, Trends in Biochemical Sciences <strong>20</strong>, 495 (1995).</div>
</div>
<div id="ref-Scott2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[17] </div><div class="csl-right-inline">M. Scott, C. W. Gunderson, E. M. Mateescu, Z. Zhang, and T. Hwa, <em><span class="nocase">Interdependence of Cell Growth and Gene Expression: Origins and Consequences.</span></em>, Science <strong>330</strong>, 1099 (2010).</div>
</div>
<div id="ref-Shis2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[18] </div><div class="csl-right-inline">D. L. Shis, F. Hussain, S. Meinhardt, L. Swint-Kruse, and M. R. Bennett, <em><span class="nocase">Modular, Multi-Input Transcriptional Logic Gating with Orthogonal LacI/GalR Family Chimeras</span></em>, ACS Synthetic Biology <strong>3</strong>, 645 (2014).</div>
</div>
<div id="ref-Tungtur2011" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[19] </div><div class="csl-right-inline">S. Tungtur, H. Skinner, H. Zhan, L. Swint-Kruse, and D. Beckett, <em><span class="nocase">In vivo tests of thermodynamic models of transcription repressor function</span></em>, Biophysical Chemistry <strong>159</strong>, 142 (2011).</div>
</div>
<div id="ref-Sochor2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[20] </div><div class="csl-right-inline">M. A. Sochor, <em><span class="nocase"><em>In vitro</em> transcription accurately predicts lac repressor phenotype <em>in vivo</em> in <em>Escherichia coli</em>.</span></em>, PeerJ <strong>2</strong>, e498 (2014).</div>
</div>
<div id="ref-Huang2011" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[21] </div><div class="csl-right-inline">Z. Huang, L. Zhu, Y. Cao, G. Wu, X. Liu, Y. Chen, Q. Wang, T. Shi, Y. Zhao, Y. Wang, W. Li, Y. Li, H. Chen, G. Chen, and J. Zhang, <em>ASD: A Comprehensive Database of Allosteric Proteins and Modulators</em>, Nucleic Acids Research <strong>39</strong>, D663 (2011).</div>
</div>
<div id="ref-Sivia2006" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[22] </div><div class="csl-right-inline">D. Sivia and J. Skilling, <em>Data Analysis: A Bayesian Tutorial</em> (OUP Oxford, 2006).</div>
</div>
<div id="ref-FernandezCastane2012" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[23] </div><div class="csl-right-inline">A. Fernández-Castané, C. E. Vine, G. Caminal, and J. López-Santín, <em><span class="nocase">Evidencing the role of lactose permease in IPTG uptake by <em>Escherichia coli</em> in fed-batch high cell density cultures</span></em>, Journal of Biotechnology <strong>157</strong>, 391 (2012).</div>
</div>
<div id="ref-Auerbach2012" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[24] </div><div class="csl-right-inline">A. Auerbach, <em><span class="nocase">Thinking in cycles: MWC is a good model for acetylcholine receptor-channels.</span></em>, The Journal of Physiology <strong>590</strong>, 93 (2012).</div>
</div>
<div id="ref-Thomason2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[25] </div><div class="csl-right-inline">L. C. Thomason, N. Costantino, and D. L. Court, <em><span class="nocase"><em>E. coli</em> genome manipulation by P1 transduction.</span></em>, Current Protocols in Molecular Biology <strong>Chapter 1</strong>, Unit 1.17 (2007).</div>
</div>
<div id="ref-Setty2003" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[26] </div><div class="csl-right-inline">Y. Setty, A. E. Mayo, M. G. Surette, and U. Alon, <em><span class="nocase">Detailed map of a cis-regulatory input function.</span></em>, Proceedings of the National Academy of Sciences <strong>100</strong>, 7702 (2003).</div>
</div>
<div id="ref-Garcia2011b" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[27] </div><div class="csl-right-inline">H. G. Garcia, H. J. Lee, J. Q. Boedicker, and R. Phillips, <em><span class="nocase">Comparison and Calibration of Different Reporters for Quantitative Analysis of Gene Expression</span></em>, Biophysical Journal <strong>101</strong>, 535 (2011b).</div>
</div>
<div id="ref-Sharan2009" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[28] </div><div class="csl-right-inline">S. K. Sharan, L. C. Thomason, S. G. Kuznetsov, and D. L. Court, <em><span class="nocase">Recombineering: a homologous recombination-based method of genetic engineering</span></em>, Nature Protocols <strong>4</strong>, 206 (2009).</div>
</div>
<div id="ref-Mirny2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[29] </div><div class="csl-right-inline">L. A. Mirny, <em><span class="nocase">Nucleosome-mediated cooperativity between transcription factors.</span></em>, Proceedings of the National Academy of Sciences <strong>107</strong>, 22534 (2010).</div>
</div>
<div id="ref-eldar2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[30] </div><div class="csl-right-inline">A. Eldar and M. Elowitz, <em><span class="nocase">Functional roles for noise in genetic circuits</span></em>, Nature <strong>467</strong>, 167 (2010).</div>
</div>
<div id="ref-Keymer2006" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[31] </div><div class="csl-right-inline">J. E. Keymer, R. G. Endres, M. Skoge, Y. Meir, and N. S. Wingreen, <em><span class="nocase">Chemosensing in <em>Escherichia coli</em>: Two regimes of two-state receptors.</span></em>, Proceedings of the National Academy of Sciences <strong>103</strong>, 1786 (2006).</div>
</div>
<div id="ref-Swem2008" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[32] </div><div class="csl-right-inline">L. R. Swem, D. L. Swem, N. S. Wingreen, and B. L. Bassler, <em><span class="nocase">Deducing Receptor Signaling Parameters from In Vivo Analysis: LuxN/AI-1 Quorum Sensing in Vibrio harveyi</span></em>, Cell <strong>134</strong>, 461 (2008).</div>
</div>
<div id="ref-Vilar2003" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[33] </div><div class="csl-right-inline">J. M. G. Vilar and S. Leibler, <em><span class="nocase">DNA Looping and Physical Constraints on Transcription Regulation</span></em>, Journal of Molecular Biology <strong>331</strong>, 981 (2003).</div>
</div>
<div id="ref-Rydenfelt2014B" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[34] </div><div class="csl-right-inline">M. Rydenfelt, R. S. Cox, H. Garcia, and R. Phillips, <em>Statistical Mechanical Model of Coupled Transcription from Multiple Promoters Due to Transcription Factor Titration</em>, Physical Review E <strong>89</strong>, 012702 (2014).</div>
</div>
<div id="ref-Lindsley2006" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[35] </div><div class="csl-right-inline">J. E. Lindsley and J. Rutter, <em><span class="nocase">Whence cometh the allosterome?</span></em>, Proceedings of the National Academy of Sciences <strong>103</strong>, 10533 (2006).</div>
</div>
<div id="ref-Salis2009" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[36] </div><div class="csl-right-inline">H. M. Salis, E. A. Mirsky, and C. A. Voigt, <em><span class="nocase">Automated design of synthetic ribosome binding sites to control protein expression</span></em>, Nature Biotechnology <strong>27</strong>, 946 (2009).</div>
</div>
<div id="ref-Lo2008" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[37] </div><div class="csl-right-inline">K. Lo, R. R. Brinkman, and R. Gottardo, <em><span class="nocase">Automated gating of flow cytometry data via robust model-based clustering</span></em>, Cytometry Part A <strong>73A</strong>, 321 (2008).</div>
</div>
<div id="ref-Levantino2012a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[38] </div><div class="csl-right-inline">M. Levantino, A. Spilotros, M. Cammarata, G. Schirò, C. Ardiccioni, B. Vallone, M. Brunori, and A. Cupane, <em><span class="nocase">The Monod-Wyman-Changeux allosteric model accounts for the quaternary transition dynamics in wild type and a recombinant mutant human hemoglobin.</span></em>, Proceedings of the National Academy of Sciences <strong>109</strong>, 14894 (2012).</div>
</div>
<div id="ref-Velyvis2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[39] </div><div class="csl-right-inline">A. Velyvis, Y. R. Yang, H. K. Schachman, and L. E. Kay, <em><span class="nocase">A solution NMR study showing that active site ligands and nucleotides directly perturb the allosteric equilibrium in aspartate transcarbamoylase.</span></em>, Proceedings of the National Academy of Sciences <strong>104</strong>, 8815 (2007).</div>
</div>
<div id="ref-Einav2016" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[40] </div><div class="csl-right-inline">T. Einav, L. Mazutis, and R. Phillips, <em><span class="nocase">Statistical Mechanics of Allosteric Enzymes.</span></em>, The Journal of Physical Chemistry B <strong>121</strong>, (2016).</div>
</div>
<div id="ref-Maecker2005" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[41] </div><div class="csl-right-inline">H. T. Maecker, A. Rinfret, P. D’Souza, J. Darden, E. Roig, C. Landry, P. Hayes, J. Birungi, O. Anzala, M. Garcia, A. Harari, I. Frank, R. Baydo, M. Baker, J. Holbrook, J. Ottinger, L. Lamoreaux, C. L. Epling, E. Sinclair, M. A. Suni, K. Punt, S. Calarota, S. El-Bahi, G. Alter, H. Maila, E. Kuta, J. Cox, C. Gray, M. Altfeld, N. Nougarede, J. Boyer, L. Tussey, T. Tobery, B. Bredt, M. Roederer, R. Koup, V. C. Maino, K. Weinhold, G. Pantaleo, J. Gilmour, H. Horton, and R. P. Sekaly, <em><span class="nocase">Standardization of cytokine flow cytometry assays</span></em>, BMC Immunology <strong>6</strong>, 13 (2005).</div>
</div>
<div id="ref-Aghaeepour2013" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[42] </div><div class="csl-right-inline">N. Aghaeepour, G. Finak, The2ptFlowCAP2ptConsortium, The2ptDREAM2ptConsortium, H. Hoos, T. R. Mosmann, R. Brinkman, R. Gottardo, and R. H. Scheuermann, <em><span class="nocase">Critical assessment of automated flow cytometry data analysis techniques</span></em>, Nature Methods <strong>10</strong>, 228 (2013).</div>
</div>
<div id="ref-Lutz1997" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[43] </div><div class="csl-right-inline">R. Lutz and H. Bujard, <em><span class="nocase">Independent and tight regulation of transcriptional units in <em>Escherichia coli</em> via the LacR/O, the TetR/O and AraC/I1-I2 regulatory elements.</span></em>, Nucleic Acids Research <strong>25</strong>, 1203 (1997).</div>
</div>
<div id="ref-Marzen2013" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[44] </div><div class="csl-right-inline">S. Marzen, H. G. Garcia, and R. Phillips, <em><span class="nocase">Statistical mechanics of Monod-Wyman-Changeux (MWC) models</span></em>, Journal of Molecular Biology <strong>425</strong>, 1433 (2013).</div>
</div>
<div id="ref-Daber2009" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[45] </div><div class="csl-right-inline">R. Daber, K. Sharp, and M. Lewis, <em><span>One Is Not Enough</span></em>, Journal of Molecular Biology <strong>392</strong>, 1133 (2009).</div>
</div>
<div id="ref-OGorman1980" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[46] </div><div class="csl-right-inline">R. B. O’Gorman, J. M. Rosenberg, O. B. Kallai, R. E. Dickerson, K. Itakura, A. D. Riggs, and K. S. Matthews, <em><span class="nocase">Equilibrium binding of inducer to lac repressor-operator DNA complex.</span></em>, Journal of Biological Chemistry <strong>255</strong>, 10107 (1980).</div>
</div>
<div id="ref-Weinert2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[47] </div><div class="csl-right-inline">F. M. Weinert, R. C. Brewster, M. Rydenfelt, R. Phillips, and W. K. Kegel, <em><span class="nocase">Scaling of Gene Expression with Transcription-Factor Fugacity</span></em>, Physical Review Letters <strong>113</strong>, 1 (2014).</div>
</div>
<div id="ref-Martins2011" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[48] </div><div class="csl-right-inline">B. M. C. Martins and P. S. Swain, <em><span class="nocase">Trade-Offs and Constraints in Allosteric Sensing</span></em>, PLoS Computational Biology <strong>7</strong>, 1 (2011).</div>
</div>
<div id="ref-Gerland2002" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[49] </div><div class="csl-right-inline">U. Gerland and T. Hwa, <em><span class="nocase">On the Selection and Evolution of Regulatory DNA Motifs</span></em>, Journal of Molecular Evolution <strong>55</strong>, 386 (2002).</div>
</div>
<div id="ref-Berg2004" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[50] </div><div class="csl-right-inline">J. Berg, S. Willmann, and M. Lässig, <em><span class="nocase">Adaptive evolution of transcription factor binding sites</span></em>, BMC Evolutionary Biology <strong>4</strong>, 42 (2004).</div>
</div>
<div id="ref-Canals2012" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[51] </div><div class="csl-right-inline">M. Canals, J. R. Lane, A. Wen, P. J. Scammells, P. M. Sexton, and A. Christopoulos, <em><span class="nocase">A Monod-Wyman-Changeux mechanism can explain G protein-coupled receptor (GPCR) allosteric modulation</span></em>, Journal of Biological Chemistry <strong>287</strong>, 650 (2012).</div>
</div>
<div id="ref-Zeldovich2008" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[52] </div><div class="csl-right-inline">K. B. Zeldovich and E. I. Shakhnovich, <em><span class="nocase">Understanding Protein Evolution: From Protein Physics to Darwinian Selection</span></em>, Annual Review of Physical Chemistry <strong>59</strong>, 105 (2008).</div>
</div>
<div id="ref-Ackers1982" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[53] </div><div class="csl-right-inline">G. K. Ackers, A. D. Johnson, and M. A. Shea, <em><span class="nocase">Quantitative model for gene regulation by lambda phage repressor.</span></em>, Proceedings of the National Academy of Sciences <strong>79</strong>, 1129 (1982).</div>
</div>
<div id="ref-Oehler1994" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[54] </div><div class="csl-right-inline">S. Oehler, M. Amouyal, P. Kolkhof, B. von Wilcken-Bergmann, and B. Müller-Hill, <em><span class="nocase">Quality and position of the three lac operators of <em>E. coli</em> define efficiency of repression.</span></em>, The EMBO Journal <strong>13</strong>, 3348 (1994).</div>
</div>
<div id="ref-Daber2011a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[55] </div><div class="csl-right-inline">R. Daber, M. A. Sochor, and M. Lewis, <em><span class="nocase">Thermodynamic Analysis of Mutant Lac Repressors</span></em>, Journal of Molecular Biology <strong>409</strong>, 76 (2011).</div>
</div>
<div id="ref-Lewis1996" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[56] </div><div class="csl-right-inline">M. Lewis, G. Chang, N. C. Horton, M. A. Kercher, H. C. Pace, M. A. Schumacher, R. G. Brennan, and P. Lu, <em><span class="nocase">Crystal Structure of the Lactose Operon Repressor and its Complexes with DNA and Inducer.</span></em>, Science <strong>271</strong>, 1247 (1996).</div>
</div>
<div id="ref-MONOD1965" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[57] </div><div class="csl-right-inline">J. Monod, J. Wyman, and J.-P. Changeux, <em><span class="nocase">On the Nature of Allosteric Transitions: A Plausible Model</span></em>, Journal of Molecular Biology <strong>12</strong>, 88 (1965).</div>
</div>
<div id="ref-Li2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[58] </div><div class="csl-right-inline">G.-W. Li, D. Burkhardt, C. Gross, and J. S. Weissman, <em><span class="nocase">Quantifying Absolute Protein Synthesis Rates Reveals Principles Underlying Allocation of Cellular Resources</span></em>, Cell <strong>157</strong>, 624 (2014).</div>
</div>
<div id="ref-Buchler2003" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[59] </div><div class="csl-right-inline">N. E. Buchler, U. Gerland, and T. Hwa, <em><span class="nocase">On schemes of combinatorial transcription logic.</span></em>, PNAS <strong>100</strong>, 5136 (2003).</div>
</div>
<div id="ref-Jones2014a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[60] </div><div class="csl-right-inline">D. L. Jones, R. C. Brewster, and R. Phillips, <em><span class="nocase">Promoter architecture dictates cell-to-cell variability in gene expression</span></em>, Science <strong>346</strong>, 1533 (2014).</div>
</div>
<div id="ref-Boedicker2013a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[61] </div><div class="csl-right-inline"><em><span class="nocase">Theoretical and Experimental Dissection of DNA Loop-Mediated Repression.</span></em>, Physical Review Letters <strong>110</strong>, 018101 (2013).</div>
</div>
<div id="ref-Bintu2005" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[62] </div><div class="csl-right-inline">L. Bintu, N. E. Buchler, H. G. Garcia, U. Gerland, T. Hwa, J. Kondev, T. Kuhlman, and R. Phillips, <em><span class="nocase">Transcriptional regulation by the numbers: applications.</span></em>, Curr. Opin. Genet. Dev. <strong>15</strong>, 125 (2005).</div>
</div>
<div id="ref-Bintu2005a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[63] </div><div class="csl-right-inline">L. Bintu, N. E. Buchler, H. G. Garcia, U. Gerland, T. Hwa, J. Kondev, and R. Phillips, <em><span class="nocase">Transcriptional regulation by the numbers: models</span></em>, Current Opinion in Genetics <span>&amp;</span> Development <strong>15</strong>, 116 (2005).</div>
</div>
<div id="ref-Kuhlman2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[64] </div><div class="csl-right-inline">T. Kuhlman, Z. Zhang, M. H. Saier, and T. Hwa, <em><span class="nocase">Combinatorial transcriptional control of the lactose operon of <em>Escherichia coli</em></span>.</em>, Proceedings of the National Academy of Sciences <strong>104</strong>, 6043 (2007).</div>
</div>
<div id="ref-Klumpp2008" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[65] </div><div class="csl-right-inline">S. Klumpp and T. Hwa, <em><span class="nocase">Growth-rate-dependent partitioning of RNA polymerases in bacteria.</span></em>, PNAS <strong>105</strong>, 20245 (2008).</div>
</div>
<div id="ref-Vilar2013" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[66] </div><div class="csl-right-inline">J. M. G. Vilar and L. Saiz, <em><span class="nocase">Reliable Prediction of Complex Phenotypes from a Modular Design in Free Energy Space: An Extensive Exploration of the <em>lac</em> Operon</span></em>, ACS Synthetic Biology <strong>2</strong>, 576 (2013).</div>
</div>
<div id="ref-Phillips2015a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[67] </div><div class="csl-right-inline">R. Phillips, <em><span class="nocase">Napoleon Is in Equilibrium</span></em>, Annual Review of Condensed Matter Physics <strong>6</strong>, 85 (2015).</div>
</div>
<div id="ref-Brewster2012" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[68] </div><div class="csl-right-inline">R. C. Brewster, D. L. Jones, and R. Phillips, <em><span class="nocase">Tuning Promoter Strength through RNA Polymerase Binding Site Design in <em>Escherichia coli</em></span></em>, PLoS Computational Biology <strong>8</strong>, e1002811 (2012).</div>
</div>
<div id="ref-Garcia2011c" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[69] </div><div class="csl-right-inline">H. G. Garcia and R. Phillips, <em><span class="nocase">Quantitative dissection of the simple repression input-output function.</span></em>, Proceedings of the National Academy of Sciences <strong>108</strong>, 12173 (2011a).</div>
</div>
<div id="ref-Brewster2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[70] </div><div class="csl-right-inline">R. C. Brewster, F. M. Weinert, H. G. Garcia, D. Song, M. Rydenfelt, and R. Phillips, <em><span class="nocase">The Transcription Factor Titration Effect Dictates Level of Gene Expression</span></em>, Cell <strong>156</strong>, 1312 (2014).</div>
</div>
</div>
                    </div><!-- /.content -->
                </div><!-- /.col -->
                <div class="col-md-4 col-md-offset-1">
                    <div class="sections-list-wrapper">
                        <div class="sections-list js-sections js-affix js-scrollspy hidden-xs hidden-sm"><span
                                style="font-size: 14pt">Table of Contents</span>
                            <hr>
                        </div><!-- /.sections-list -->
                    </div>
                </div><!-- /.col -->
            </div><!-- /.row -->
        </div><!-- /.container -->
    </div><!-- /.section -->
    
    <div class="section section--grey">
        <div class="container">
            <div class="row">
                <div class="col-md-7">
                    <div id="disqus_thread"></div>
<script>
	/**
	 *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
	 *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables
	 */
	 /*
	var disqus_config = function () {
	this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
		this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
	};
	*/
	(function() {
		var d = document, s = d.createElement('script');

		s.src = 'https://mrazomej-phd.disqus.com/embed.js';

		s.setAttribute('data-timestamp', +new Date());
		(d.head || d.body).appendChild(s);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>

                </div><!-- /.col -->
            </div><!-- /.row -->
        </div><!-- /.container -->
    </div><!-- /.section -->
    
    <div class="js-footer-area">
        
        <nav class="page-nav">
            <div class="container">
                <div class="row">
                    <div class="col-xs-12">
                        
                        <a href="https://mrazomej.github.io/phd/chapter_01"
                            class="page-nav__item page-nav__item--prev">
                            <i class="icon icon--arrow-left"></i>
                            Chapter 1
                        </a><!-- /.page-nav__item -->
                        
                        
                        <a href="https://mrazomej.github.io/phd/chapter_03"
                            class="page-nav__item page-nav__item--next">
                            Chapter 3
                            <i class="icon icon--arrow-right"></i>
                        </a><!-- /.page-nav__item -->
                        
                    </div><!-- /.col -->
                </div><!-- /.row -->
            </div><!-- /.container -->
        </nav><!-- /.page-nav -->
        
        
        
	<footer class="site-footer">
		<div class="container">
			<div class="row">
				<div class="col-sm-6">
					
						<a href="https://mrazomej.github.io/phd//" class="site-footer__logo">Manuel Razo-Mejia | PhD Thesis</a>
					
					
						<hr>
						<p class="site-footer__copyright">Copyright &copy; 2021. - Manuel Razo-Mejia <br>All rights reserved.</p>
					
				</div><!-- /.col -->
				
					<div class="col-sm-6 align-right">
						<ul class="social-list">
							
								<li>
									<a href="https://twitter.com/mrazomej" target="_blank" class="social-list__item social-list__item--twitter">
										<i class="icon icon--twitter"></i>
									</a>
								</li>
							
						</ul><!-- /.social-list -->
					</div><!-- /.col -->
				
			</div><!-- /.row -->
		</div><!-- /.container -->
	</footer><!-- /.site-footer -->


<script src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/jquery.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/bootstrap/affix.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/bootstrap/scrollspy.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/matchHeight.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/scripts.min.js"></script>


<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-109526846-4', 'auto');
	ga('send', 'pageview');
</script>



    </div><!-- /.js-footer-area -->
</body>

</html>