<!DOCTYPE html>
<html>
	<head>
		<meta charset="UTF-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, minimum-scale=1.0">
<title>Chapter II</title>


<script type="text/x-mathjax-config">
	MathJax.Hub.Config({
    "HTML-CSS" : {
        availableFonts : ["STIX"],
        preferredFont : "Gyre Pagella",
        webFont : "STIX-Web",
        imageFont : null
    },
	  TeX: {
		equationNumbers: {
		  autoNumber: "AMS"
		}
	  },
	  tex2jax: {
		inlineMath: [['$','$']], 
		displayMath: [['$$','$$']],
		processEscapes: true,
	  }
	});
  </script>
  <style>
	.eqnos { display: inline-block; position: relative; width: 100%; }
	.eqnos br { display: none; }
	.eqnos-number { position: absolute; right: 0em; top: 50%; line-height: 0; }
  </style>
  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script id="MathJax-script" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>
  

<link rel="icon" href="/favicon.ico" type="image/x-icon">
<link href="https://fonts.googleapis.com/css?family=Nanum+Myeongjo&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Alegreya|Lora&display=swap" rel="stylesheet">

<link rel="stylesheet" href="/doks-theme/assets/css/style.css">

	</head>
	<body class="grey" data-spy="scroll" data-target=".js-scrollspy">
		


	<header class="site-header">
		<div class="container">
			<div class="row">
				<div class="col-xs-12">
					
						<a href="/" class="site-header__logo"><i class="icon icon--chevron-left"></i> Home</a>
					
					
				</div><!-- /.col -->
			</div><!-- /.row -->
		</div><!-- /.container -->
	</header><!-- /.site-header -->


		<div class="hero-subheader" style="background-image: url(http://127.0.0.1:4000/doks-theme/assets/images/migration.jpg); background-position: center; object-fit: cover; width:100%;">

			<div class="container" style="background-color: rgba(0,0,0, 0.75); border:0.75px solid white;">
				<div class="row">
					<div class="col-md-7">
						<div class="align-container" data-mh>
							<div class="align-inner">
								
									<h2 class="hero-subheader__title">Chapter II</h2>
								
								
									<h1 class="hero-subheader__title">A Predictive Theory of Allosteric Induction
</h1>
								
								
								<!-- 
									
										<a href="chapter_01" class="btn btn--dark btn--rounded btn--w-icon btn--w-icon-left">
											<i class="icon icon--arrow-left"></i>	
										</a>
									
									
										<a href="chapter_03" class="btn btn--dark btn--rounded btn--w-icon">
											<i class="icon icon--arrow-right"></i>
										</a>
									
								 -->
							</div><!-- /.align-inner -->
						</div><!-- /.align-container -->
					</div><!-- /.col -->
					
						<div class="col-md-5 col-md-offset-0 hidden-xs hidden-sm">
							<div class="align-container" data-mh>
								<div class="align-inner">
									<div class="hero-subheader__author">
										<p class="hero-subheader__author-title">
												Summary
												<i class="icon icon--chevron-down"></i>
										</p><!-- /.hero-subheader__author-title -->
										<p>TBD.
</p>
									</div><!-- /.hero-subheader__author -->
								</div><!-- /.align-inner -->
							</div><!-- /.align-container -->
						</div><!-- /.col -->
					
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.hero-subheader -->
		
				<nav class="page-nav">
					<div class="container" >
						<div class="row" >
							<div class="col-xs-12">
								
									<a href="chapter_01" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter 1
									</a><!-- /.page-nav__item -->
								
								
									<a href="chapter_03" class="page-nav__item page-nav__item--next">
										Chapter 3
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			

		<div class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-7">
						<div class="content">
							<p>A version of this chapter originally appeared as Razo-Mejia, M.<em>, Barnes, S.L.</em>, Belliveau, N.M.<em>, Chure, G.</em>, Einav, T.*, Lewis, M., and Phillips, R. (2018). Tuning transcriptional regulation through signaling: A predictive theory of allosteric induction. Cell Systems 6, 456-469.e10. DOI:https://doi.org/10.1016/j.cels.2018.02.004. M.R.M, S.L.B, N.M.B, G.C., and T.E. contributed equally to this work from the theoretical underpinnings to the experimental design and execution. M.R.M, S.L.B, N.M.B, G.C, T.E., and R.P. wrote the paper. M.L. provided extensive guidance and advice.</p>
<h1 id="abstract">Abstract</h1>
<p>Allosteric regulation is found across all domains of life, yet we still lack simple, predictive theories that directly link the experimentally tunable parameters of a system to its input-output response. To that end, we present a general theory of allosteric transcriptional regulation using the Monod-Wyman-Changeux model. We rigorously test this model using the ubiquitous simple repression motif in bacteria by first predicting the behavior of strains that span a large range of repressor copy numbers and DNA binding strengths and then constructing and measuring their response. Our model not only accurately captures the induction profiles of these strains but also enables us to derive analytic expressions for key properties such as the dynamic range and <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span>. Finally, we derive an expression for the free energy of allosteric repressors which enables us to collapse our experimental data onto a single master curve that captures the diverse phenomenology of the induction profiles.</p>
<h1 id="introduction">Introduction</h1>
<p>Understanding how organisms sense and respond to changes in their environment has long been a central theme of biological inquiry. At the cellular level, this interaction is mediated by a diverse collection of molecular signaling pathways. A pervasive mechanism of signaling in these pathways is allosteric regulation, in which the binding of a ligand induces a conformational change in some target molecule, triggering a signaling cascade <span class="citation" data-cites="Lindsley2006">[@Lindsley2006]</span>. One of the most important examples of such signaling is offered by transcriptional regulation, where a transcription factor’s propensity to bind to DNA will be altered upon binding to an allosteric effector.</p>
<p>Despite allostery’s ubiquity, we lack a formal, rigorous, and generalizable framework for studying its effects across the broad variety of contexts in which it appears. A key example of this is transcriptional regulation, in which allosteric transcription factors can be induced or corepressed by binding to a ligand. An allosteric transcription factor can adopt multiple conformational states, each of which has its own affinity for the ligand and for its DNA target site. <em>In vitro</em> studies have rigorously quantified the equilibria of different conformational states for allosteric transcription factors and measured the affinities of these states to the ligand <span class="citation" data-cites="Harman2001 Lanfranco2017">[@Harman2001; @Lanfranco2017]</span>. In spite of these experimental observations, the lack of a coherent quantitative model for allosteric transcriptional regulation has made it impossible to predict the behavior of even a simple genetic circuit across a range of regulatory parameters.</p>
<p>The ability to predict circuit behavior robustly— that is, across both broad ranges of parameters and regulatory architectures —is important for multiple reasons. First, in the context of a specific gene, accurate prediction demonstrates that all components relevant to the gene’s behavior have been identified and characterized to sufficient quantitative precision. Second, in the context of genetic circuits in general, robust prediction validates the model that generated the prediction. Possessing a validated model also has implications for future work. For example, when we have sufficient confidence in the model, a single data set can be used to accurately extrapolate a system’s behavior in other conditions. Moreover, there is an essential distinction between a predictive model, which is used to predict a system’s behavior given a set of input variables, and a retroactive model, which is used to describe the behavior of data that has already been obtained. We note that even some of the most careful and rigorous analysis of transcriptional regulation often entails only a retroactive reflection on a single experiment. This raises the fear that each regulatory architecture may require a unique analysis that cannot carry over to other systems, a worry that is exacerbated by the prevalent use of phenomenological functions (e.g. Hill functions or ratios of polynomials) that can analyze a single data set but cannot be used to extrapolate a system’s behavior in other conditions <span class="citation" data-cites="Setty2003 Poelwijk2011 Vilar2013 Rogers2015 Rohlhill2017">[@Setty2003; @Poelwijk2011; @Vilar2013; @Rogers2015; @Rohlhill2017]</span>.</p>
<p>This work explores what happens when theory takes center stage, namely, we first write down the equations governing a system and describe its expected behavior across a wide array of experimental conditions, and only then do we set out to experimentally confirm these results. Building upon previous work <span class="citation" data-cites="Garcia2011 Brewster2014 Weinert2014">[@Garcia2011; @Brewster2014; @Weinert2014]</span> and the work of Monod, Wyman, and Changeux <span class="citation" data-cites="MONOD1965">[@MONOD1965]</span>, we present a statistical mechanical rendering of allostery in the context of induction and corepression (shown schematically in and henceforth referred to as the MWC model) and use it as the basis of parameter-free predictions which we then test experimentally. More specifically, we study the simple repression motif – a widespread bacterial genetic regulatory architecture in which binding of a transcription factor occludes binding of an RNA polymerase, thereby inhibiting transcription initiation. The MWC model stipulates that an allosteric protein fluctuates between two distinct conformations – an active and inactive state – in thermodynamic equilibrium <span class="citation" data-cites="MONOD1965">[@MONOD1965]</span>. During induction, for example, effector binding increases the probability that a repressor will be in the inactive state, weakening its ability to bind to the promoter and resulting in increased expression. To test the predictions of our model across a wide range of operator binding strengths and repressor copy numbers, we design an <em>E. coli</em> genetic construct in which the binding probability of a repressor regulates gene expression of a fluorescent reporter.</p>
<p>In total, the work presented here demonstrates that one extremely compact set of parameters can be applied self-consistently and predictively to different regulatory situations including simple repression on the chromosome, cases in which decoy binding sites for repressor are put on plasmids, cases in which multiple genes compete for the same regulatory machinery, cases involving multiple binding sites for repressor leading to DNA looping, and induction by signaling <span class="citation" data-cites="Garcia2011 Garcia2011B Brewster2012 Boedicker2013a Boedicker2013b Brewster2014">[@Garcia2011; @Garcia2011B; @Brewster2012; @Boedicker2013a; @Boedicker2013b; @Brewster2014]</span>. Thus, rather than viewing the behavior of each circuit as giving rise to its own unique input-output response, the MWC model provides a means to characterize these seemingly diverse behaviors using a single unified framework governed by a small set of parameters.</p>
<figure>
<img src="ch2_fig01.png" id="fig:inducible_types" data-short-caption="Transcriptional regulatory architectures involving an allosteric repressor." alt="Figure 1: Transcription regulation architectures involving an allosteric repressor. We consider a promoter regulated solely by an allosteric repressor. When bound, the repressor prevents RNAP from binding and initiating transcription. Induction is characterized by the addition of an effector which binds to the repressor and stabilizes the inactive state (defined as the state which has a low affinity for DNA), thereby increasing gene expression. In corepression, the effector stabilizes the repressor’s active state and thus further reduces gene expression. We list several characterized examples of induction and corepression that support different physiological roles in E. coli [@Huang2011; @Li2014]. A schematic regulatory response of the two architectures shown in Panel plotting the fold-change in gene expression as a function of effector concentration, where fold-change is defined as the ratio of gene expression in the presence versus the absence of repressor. We consider the following key phenotypic properties that describe each response curve: the minimum response (leakiness), the maximum response (saturation), the difference between the maximum and minimum response (dynamic range), the concentration of ligand which generates a fold-change halfway between the minimal and maximal response ([EC_{50}]), and the log-log slope at the midpoint of the response (effective Hill coefficient). (C) Over time we have refined our understanding of simple repression architectures. A first round of experiments used colorimetric assays and quantitative Western blots to investigate how single-site repression is modified by the repressor copy number and repressor-DNA binding energy [@Garcia2011]. A second round of experiments used video microscopy to probe how the copy number of the promoter and presence of competing repressor binding sites affect gene expression, and we use this data set to determine the free energy difference between the repressor’s inactive and active conformations [@Weinert2014]. Here we used flow cytometry to determine the inducer-repressor dissociation constants and demonstrate that with these parameters we can predict a priori the behavior of the system for any repressor copy number, DNA binding energy, gene copy number, and inducer concentration." /><figcaption aria-hidden="true">Figure 1: <strong>Transcription regulation architectures involving an allosteric repressor.</strong> We consider a promoter regulated solely by an allosteric repressor. When bound, the repressor prevents RNAP from binding and initiating transcription. Induction is characterized by the addition of an effector which binds to the repressor and stabilizes the inactive state (defined as the state which has a low affinity for DNA), thereby increasing gene expression. In corepression, the effector stabilizes the repressor’s active state and thus further reduces gene expression. We list several characterized examples of induction and corepression that support different physiological roles in <em>E. coli</em> <span class="citation" data-cites="Huang2011 Li2014">[@Huang2011; @Li2014]</span>. A schematic regulatory response of the two architectures shown in Panel plotting the fold-change in gene expression as a function of effector concentration, where fold-change is defined as the ratio of gene expression in the presence versus the absence of repressor. We consider the following key phenotypic properties that describe each response curve: the minimum response (leakiness), the maximum response (saturation), the difference between the maximum and minimum response (dynamic range), the concentration of ligand which generates a fold-change halfway between the minimal and maximal response (<span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span>), and the log-log slope at the midpoint of the response (effective Hill coefficient). (C) Over time we have refined our understanding of simple repression architectures. A first round of experiments used colorimetric assays and quantitative Western blots to investigate how single-site repression is modified by the repressor copy number and repressor-DNA binding energy <span class="citation" data-cites="Garcia2011">[@Garcia2011]</span>. A second round of experiments used video microscopy to probe how the copy number of the promoter and presence of competing repressor binding sites affect gene expression, and we use this data set to determine the free energy difference between the repressor’s inactive and active conformations <span class="citation" data-cites="Weinert2014">[@Weinert2014]</span>. Here we used flow cytometry to determine the inducer-repressor dissociation constants and demonstrate that with these parameters we can predict <em>a priori</em> the behavior of the system for any repressor copy number, DNA binding energy, gene copy number, and inducer concentration.</figcaption>
</figure>
<h2 id="references">References</h2>
						</div><!-- /.content -->
					</div><!-- /.col -->
					<div class="col-md-4 col-md-offset-1">
						<div class="sections-list-wrapper">
							<div class="sections-list js-sections js-affix js-scrollspy hidden-xs hidden-sm"><span style="font-size: 14pt">Table of Contents</span><hr></div><!-- /.sections-list -->
						</div>
					</div><!-- /.col -->
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.section -->
		
		<div class="js-footer-area">
			
				<nav class="page-nav">	
					<div class="container">
						<div class="row">
							<div class="col-xs-12">
								
									<a href="chapter_01" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter 1
									</a><!-- /.page-nav__item -->
								
								
									<a href="chapter_03" class="page-nav__item page-nav__item--next">
										Chapter 3
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			
			
			
	<footer class="site-footer">
		<div class="container">
			<div class="row">
				<div class="col-sm-6">
					
						<a href="/" class="site-footer__logo">Manuel Razo-Mejia | PhD Thesis</a>
					
					
						<hr>
						<p class="site-footer__copyright">Copyright &copy; 2021. - Manuel Razo-Mejia <br>All rights reserved.</p>
					
				</div><!-- /.col -->
				
					<div class="col-sm-6 align-right">
						<ul class="social-list">
							
								<li>
									<a href="https://twitter.com/mrazomej" target="_blank" class="social-list__item social-list__item--twitter">
										<i class="icon icon--twitter"></i>
									</a>
								</li>
							
						</ul><!-- /.social-list -->
					</div><!-- /.col -->
				
			</div><!-- /.row -->
		</div><!-- /.container -->
	</footer><!-- /.site-footer -->


<script src="/doks-theme/assets/js/vendor/jquery.min.js"></script>
<script type="text/javascript" src="/doks-theme/assets/js/vendor/bootstrap/affix.min.js"></script>
<script type="text/javascript" src="/doks-theme/assets/js/vendor/bootstrap/scrollspy.min.js"></script>
<script type="text/javascript" src="/doks-theme/assets/js/vendor/matchHeight.min.js"></script>
<script type="text/javascript" src="/doks-theme/assets/js/scripts.min.js"></script>




		</div><!-- /.js-footer-area -->
	</body>
</html>
