<!DOCTYPE html>

<html>

<head>
    <meta charset="UTF-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, minimum-scale=1.0">
<title>Chapter III</title>


<script type="text/x-mathjax-config">
	MathJax.Hub.Config({
    "HTML-CSS" : {
        availableFonts : ["STIX"],
        preferredFont : "Gyre Pagella",
        webFont : "STIX-Web",
        imageFont : null
    },
	  TeX: {
		equationNumbers: {
		  autoNumber: "AMS"
		}
	  },
	  tex2jax: {
		inlineMath: [['$','$']], 
		displayMath: [['$$','$$']],
		processEscapes: true,
	  }
	});
  </script>
<style>
    .eqnos {
        display: inline-block;
        position: relative;
        width: 100%;
    }

    .eqnos br {
        display: none;
    }

    .eqnos-number {
        position: absolute;
        right: 0em;
        top: 50%;
        line-height: 0;
    }
</style>

<script>
    window.MathJax = {
        tex: {
            tags: 'all'
        }
    };
</script>
<script>
    MathJax.Hub.Config({
        tex2jax: {
            inlineMath: [['$', '$'], ['\\(', '\\)']]
        }
    });
</script>
<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>

<link rel="icon" href="https://mrazomej.github.io/phd//favicon.ico"
    type="image/x-icon">
<link href="https://fonts.googleapis.com/css?family=Nanum+Myeongjo&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Alegreya|Lora&display=swap" rel="stylesheet">

<link rel="stylesheet"
    href="https://mrazomej.github.io/phd//doks-theme/assets/css/style.css">
</head>

<body class="grey" data-spy="scroll" data-target=".js-scrollspy">
    


	<header class="site-header">
		<div class="container">
			<div class="row">
				<div class="col-xs-12">
					
						<a href="https://mrazomej.github.io/phd//" class="site-header__logo"><i class="icon icon--chevron-left"></i> Home</a>
					
					
				</div><!-- /.col -->
			</div><!-- /.row -->
		</div><!-- /.container -->
	</header><!-- /.site-header -->


    <div class="hero-subheader" style="background-image: url(https://mrazomej.github.io/phd//doks-theme/assets/images/alaska.jpg); background-position: center; object-fit: cover; width:100%;">

        <div class="container" style="background-color: rgba(0,0,0, 0.75); border:0.75px solid white;">
            <div class="row">
                <div class="col-md-7">
                    <div class="align-container" data-mh>
                        <div class="align-inner">
                            
                            <h2 class="hero-subheader__title">Chapter III</h2>
                            
                            
                            <h1 class="hero-subheader__title">First-principles prediction of the information processing capacity of a simple genetic circuit
</h1>
                            
                            
                            <!-- 
									
										<a href="https://mrazomej.github.io/phd/chapter_02" class="btn btn--dark btn--rounded btn--w-icon btn--w-icon-left">
											<i class="icon icon--arrow-left"></i>	
										</a>
									
									
										<a href="https://mrazomej.github.io/phd/chapter_04" class="btn btn--dark btn--rounded btn--w-icon">
											<i class="icon icon--arrow-right"></i>
										</a>
									
								 -->
                        </div><!-- /.align-inner -->
                    </div><!-- /.align-container -->
                </div><!-- /.col -->
                
                <div class="col-md-5 col-md-offset-0 hidden-xs hidden-sm">
                    <div class="align-container" data-mh>
                        <div class="align-inner">
                            <div class="hero-subheader__author">
                                <p class="hero-subheader__author-title">
                                    Summary
                                    <i class="icon icon--chevron-down"></i>
                                </p><!-- /.hero-subheader__author-title -->
                                <p>Living organisms are constantly sensing intra and extracellular cues and responding accordingly. The quality and precision of such responses can mean the difference between surviving or not certain challenges; therefore, there is a constant selection pressure for cells to gather enough information from any stimulus in order to build an adequate response. In this context, the information that cells can obtain has a precise mathematical definition measured-—just as in computers—-in bits.
In this chapter our goal is to predict how many bits of information can a cell harboring a simple genetic circuit process. To do so, I write down a theoretical model to predict the full distribution of gene expression based on the physics of this molecular process. I calibrate our model with previous information in order to perform parameter-free predictions. To test the model, I compare the predictions with experimental single-cell gene expression data finding great agreement.
</p>
                            </div><!-- /.hero-subheader__author -->
                        </div><!-- /.align-inner -->
                    </div><!-- /.align-container -->
                </div><!-- /.col -->
                
            </div><!-- /.row -->
        </div><!-- /.container -->
    </div><!-- /.hero-subheader -->
    
    <nav class="page-nav">
        <div class="container">
            <div class="row">
                <div class="col-xs-12">
                    
                    <a href="https://mrazomej.github.io/phd/chapter_02"
                        class="page-nav__item page-nav__item--prev">
                        <i class="icon icon--arrow-left"></i>
                        Chapter 2
                    </a><!-- /.page-nav__item -->
                    
                    
                    <a href="https://mrazomej.github.io/phd/chapter_04"
                        class="page-nav__item page-nav__item--next">
                        Chapter 4
                        <i class="icon icon--arrow-right"></i>
                    </a><!-- /.page-nav__item -->
                    
                </div><!-- /.col -->
            </div><!-- /.row -->
        </div><!-- /.container -->
    </nav><!-- /.page-nav -->
    

    <div class="section">
        <div class="container">
            <div class="row">
                <div class="col-md-7">
                    <div class="content">
                        <strong>Published as …</strong>
<hr/>
<p>A version of this chapter originally appeared as Razo-Mejia, M., Marzen, S., Chure, G., Taubman, R., Morrison, M., and Phillips, R. (2020). First-principles prediction of the information processing capacity of a simple genetic circuit. Physical Review E 102, 022404. DOI:https://doi:10.1103/PhysRevE.102.022404.</p>
<h2 id="abstract">Abstract</h2>
<p>Given the stochastic nature of gene expression, genetically identical cells exposed to the same environmental inputs will produce different outputs. This heterogeneity has been hypothesized to have consequences for how cells are able to survive in changing environments. Recent work has explored the use of information theory as a framework to understand the accuracy with which cells can ascertain the state of their surroundings. Yet the predictive power of these approaches is limited and has not been rigorously tested using precision measurements. To that end, we generate a minimal model for a simple genetic circuit in which all parameter values for the model come from independently published data sets. We then predict the information processing capacity of the genetic circuit for a suite of biophysical parameters such as protein copy number and protein-DNA affinity. We compare these parameter-free predictions with an experimental determination of protein expression distributions and the resulting information processing capacity of <em>E. coli</em> cells. We find that our minimal model captures the scaling of the cell-to-cell variability in the data and the inferred information processing capacity of our simple genetic circuit up to a systematic deviation.</p>
<h2 id="introduction">Introduction</h2>
<p>As living organisms thrive in a given environment, they are faced with constant changes in their surroundings. From abiotic conditions such as temperature fluctuations or changes in osmotic pressure, to biological interactions such as cell-to-cell communication in a tissue or in a bacterial biofilm, living organisms of all types sense and respond to external signals. Fig. 1(A) shows a schematic of this process for a bacterial cell sensing a concentration of an extracellular chemical. At the molecular level where signal transduction unfolds mechanistically, there are physical constraints on the accuracy and precision of these responses given by intrinsic stochastic fluctuations <span class="citation" data-cites="Nemenman2010"> [<a href="#ref-Nemenman2010" role="doc-biblioref">1</a>]</span>. This means that two genetically identical cells exposed to the same stimulus will not have identical responses <span class="citation" data-cites="Eldar2010"> [<a href="#ref-Eldar2010" role="doc-biblioref">2</a>]</span>.</p>
<p>One implication of this noise in biological systems is that cells do not have an infinite resolution to distinguish signals and, as a consequence, there is a one-to-many mapping between inputs and outputs. Furthermore, given the limited number of possible outputs, there are overlapping responses between different inputs. This scenario can be mapped to a Bayesian inference problem where cells try to infer the state of the environment from their phenotypic response, as schematized in Fig. 1(B). The question then becomes this: how can one analyze this probabilistic, rather than deterministic, relationship between inputs and outputs? The abstract answer to this question was worked out in 1948 by Claude Shannon who, in his seminal work, founded the field of information theory <span class="citation" data-cites="Shannon1948"> [<a href="#ref-Shannon1948" role="doc-biblioref">3</a>]</span>. Shannon developed a general framework for how to analyze information transmission through noisy communication channels. In his work, Shannon showed that the only quantity that satisfies three reasonable axioms for a measure of uncertainty was of the same functional form as the thermodynamic entropy–thereby christening his metric the information entropy <span class="citation" data-cites="MacKay2003"> [<a href="#ref-MacKay2003" role="doc-biblioref">4</a>]</span>. He also gave a definition, based on this information entropy, for the relationship between inputs and outputs known as the mutual information. The mutual information <span class="math inline">\(I\)</span> between input <span class="math inline">\(c\)</span> and output <span class="math inline">\(p\)</span>, given by <span class="math display">\[
I = \sum_c P(c) \sum_p P(p \mid c) \log_2 \frac{P(p \mid c)}{P(p)},
\label{eq:ch3_eq01}
\]</span> quantifies how much we learn about the state of the input <span class="math inline">\(c\)</span> given that we get to observe the output <span class="math inline">\(p\)</span>. In other words, the mutual information can be thought of as a generalized correlation coefficient that quantifies the degree to which the uncertainty about a random event decreases given the knowledge of the average outcome of another random event <span class="citation" data-cites="Kinney2010"> [<a href="#ref-Kinney2010" role="doc-biblioref">5</a>]</span>.</p>
<p>It is natural to conceive of scenarios in which living organisms that can better resolve signals might have an evolutionary benefit, making it more likely that their offspring will have a fitness advantage <span class="citation" data-cites="Taylor2007"> [<a href="#ref-Taylor2007" role="doc-biblioref">6</a>]</span>. In recent years there has been a growing interest in understanding the theoretical limits on cellular information processing <span class="citation" data-cites="Bialek2005 Gregor2007"> [<a href="#ref-Bialek2005" role="doc-biblioref">7</a>,<a href="#ref-Gregor2007" role="doc-biblioref">8</a>]</span>, and in quantifying how close evolution has pushed cellular signaling pathways to these theoretical limits <span class="citation" data-cites="Tkacik2008 Dubuis2013 Petkova2019"> [<a href="#ref-Tkacik2008" role="doc-biblioref">9</a>–<a href="#ref-Petkova2019" role="doc-biblioref">11</a>]</span>. While these studies have treated the signaling pathway as a “black box,” explicitly ignoring all the molecular interactions taking place in them, other studies have explored the role that molecular players and regulatory architectures have on these information processing tasks <span class="citation" data-cites="Rieckh2014 Ziv2007 Voliotis2014a Tostevin2009 Tkacik2011 Tkacik2008a Tabbaa2014"> [<a href="#ref-Rieckh2014" role="doc-biblioref">12</a>–<a href="#ref-Tabbaa2014" role="doc-biblioref">18</a>]</span>. Despite the great advances in our understanding of the information processing capabilities of molecular mechanisms, the field still lacks a rigorous experimental test of these detailed models with precision measurements on a simple system in which physical parameters can be perturbed. In this work we approach this task with a system that is both theoretically and experimentally tractable in which molecular parameters can be varied in a controlled manner.</p>
<p>Over the last decade the dialogue between theory and experiments in gene regulation has led to predictive power of models not only over the mean level of gene expression, but the noise as a function of relevant parameters such as regulatory protein copy numbers, affinity of these proteins to the DNA promoter, as well as the extracellular concentrations of inducer molecules <span class="citation" data-cites="Golding2005 Garcia2011c Vilar2013 Xu2015"> [<a href="#ref-Golding2005" role="doc-biblioref">19</a>–<a href="#ref-Xu2015" role="doc-biblioref">22</a>]</span>. These models based on equilibrium and non-equilibrium statistical physics have reached a predictive accuracy level such that, for simple cases, it is now possible to design input-output functions <span class="citation" data-cites="Brewster2012 Barnes2019"> [<a href="#ref-Brewster2012" role="doc-biblioref">23</a>,<a href="#ref-Barnes2019" role="doc-biblioref">24</a>]</span>. This opens the opportunity to exploit these predictive models to tackle the question of how much information genetic circuits can process. This question lies at the heart of understanding the precision of the cellular response to environmental signals. Fig. 1(C) schematizes a scenario in which two bacterial strains respond with different levels of precision to three possible environmental states, i.e., inducer concentrations. The overlap between the three different responses is what precisely determines the resolution with which cells can distinguish different inputs. This is analogous to how the point spread function limits the ability to resolve two light emitting point sources.</p>
<p>In this work we follow the same philosophy of theory-experiment dialogue used to determine model parameters to predict from first principles the effect that biophysical parameters such as transcription factor copy number and protein-DNA affinity have on the information processing capacity of a simple genetic circuit. Specifically, to predict the mutual information between an extracellular chemical signal (input <span class="math inline">\(c\)</span>, isopropyl <span class="math inline">\(\beta\)</span>-D-1-thiogalactopyranoside or IPTG in our experimental system) and the corresponding cellular response in the form of protein expression (output <span class="math inline">\(p\)</span>), we must compute the input-output function <span class="math inline">\(P(p \mid c)\)</span>. To do so, we use a master-equation-based model to construct the protein copy number distribution as a function of an extracellular inducer concentration for different combinations of transcription factor copy numbers and binding sites. Having these input-output distributions allows us to compute the mutual information <span class="math inline">\(I\)</span> between inputs and outputs for any arbitrary input distribution <span class="math inline">\(P(c)\)</span>. We opt to compute the channel capacity, i.e., the maximum information that can be processed by this gene regulatory architecture, defined as [Eq:ch3_eq01] maximized over all possible input distributions <span class="math inline">\(P(c)\)</span>. By doing so we examine the physical limits of what cells can do in terms of information processing by harboring these genetic circuits. Nevertheless, given the generality of the input-output function <span class="math inline">\(P(p \mid c)\)</span> we derive, the model presented here can be used to compute the mutual information for any arbitrary input distribution <span class="math inline">\(P(c)\)</span>. All parameters used for our model were inferred from a series of studies that span several experimental techniques <span class="citation" data-cites="Garcia2011c Jones2014a Brewster2014 Razo-Mejia2018"> [<a href="#ref-Garcia2011c" role="doc-biblioref">20</a>,<a href="#ref-Jones2014a" role="doc-biblioref">25</a>–<a href="#ref-Razo-Mejia2018" role="doc-biblioref">27</a>]</span>, allowing us to make parameter-free predictions of this information processing capacity <span class="citation" data-cites="Phillips2019"> [<a href="#ref-Phillips2019" role="doc-biblioref">28</a>]</span>.</p>
<p>These predictions are then contrasted with experimental data, where the channel capacity is inferred from single-cell fluorescence distributions taken at different concentrations of inducer for cells with previously characterized biophysical parameters <span class="citation" data-cites="Garcia2011c Razo-Mejia2018"> [<a href="#ref-Garcia2011c" role="doc-biblioref">20</a>,<a href="#ref-Razo-Mejia2018" role="doc-biblioref">27</a>]</span>. We find that our parameter-free predictions quantitatively track the experimental data up to a systematic deviation. The lack of numerical agreement between our model and the experimental data poses new challenges towards having a foundational, first-principles understanding of the physics of cellular decision-making.</p>
<!-- The reminder of the paper is organized as follows. In we define the minimal
theoretical model and parameter inference for a simple repression genetic
circuit. discusses how all parameters for the minimal model are determined from
published datasets that explore different aspects of the simple repression
motif. computes the moments of the mRNA and protein distributions from this
minimal model. In we explore the consequences of variability in gene copy number
during the cell cycle. In this section we compare experimental and theoretical
quantities related to the moments of the distribution, specifically the
predictions for the fold-change in gene expression (mean expression relative to
an unregulated promoter) and the gene expression noise (standard deviation over
mean). follows with reconstruction of the full mRNA and protein distribution
from the moments using the maximum entropy principle. Finally uses the
distributions from to compute the maximum amount of information that the genetic
circuit can process. Here we again contrast our zero-parameter fit predictions
with experimental inferences of the channel capacity. -->
<figure>
<img src="ch3_fig01.png" id="fig:ch3_fig01" data-short-caption="Cellular signaling systems sense the environment with different degrees of precision" alt="Figure 1: Cellular signaling systems sense the environment with different degrees of precision. (A) Schematic representation of a cell as a noisy communication channel. From an environmental input (inducer molecule concentration) to a phenotypic output (protein expression level), cellular signaling systems can be modeled as noisy communication channels. (B) We treat cellular response to an external stimulus as a Bayesian inference of the state of the environment. As the phenotype (protein level) serves as the internal representation of the environmental state (inducer concentration), the probability of a cell being in a specific environment given this internal representation P(c \mid p) is a function of the probability of the response given that environmental state P(p \mid c). (C) The precision of the inference of the environmental state depends on how well can cells resolve different inputs. For three different levels of input (left panel) the green strain responds more precisely than the purple strain since the output distributions overlap less (middle panel). This allows the green strain to make a more precise inference of the environmental state given a phenotypic response (right panel)." /><figcaption aria-hidden="true">Figure 1: <strong>Cellular signaling systems sense the environment with different degrees of precision</strong>. (A) Schematic representation of a cell as a noisy communication channel. From an environmental input (inducer molecule concentration) to a phenotypic output (protein expression level), cellular signaling systems can be modeled as noisy communication channels. (B) We treat cellular response to an external stimulus as a Bayesian inference of the state of the environment. As the phenotype (protein level) serves as the internal representation of the environmental state (inducer concentration), the probability of a cell being in a specific environment given this internal representation <span class="math inline">\(P(c \mid p)\)</span> is a function of the probability of the response given that environmental state <span class="math inline">\(P(p \mid c)\)</span>. (C) The precision of the inference of the environmental state depends on how well can cells resolve different inputs. For three different levels of input (left panel) the green strain responds more precisely than the purple strain since the output distributions overlap less (middle panel). This allows the green strain to make a more precise inference of the environmental state given a phenotypic response (right panel).</figcaption>
</figure>
<h2 id="results">Results</h2>
<h3 id="sec_model">Minimal model of transcriptional regulation</h3>
<p>As a tractable circuit for which we have control over the parameters both theoretically and experimentally, we chose the so-called simple repression motif, a common regulatory scheme among prokaryotes <span class="citation" data-cites="Rydenfelt2014"> [<a href="#ref-Rydenfelt2014" role="doc-biblioref">29</a>]</span>. This circuit consists of a single promoter with an RNA-polymerase (RNAP) binding site and a single binding site for a transcriptional repressor <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">20</a>]</span>. The regulation due to the repressor occurs via exclusion of the RNAP from its binding site when the repressor is bound, decreasing the likelihood of having a transcription event. As with many important macromolecules, we consider the repressor to be allosteric, meaning that it can exist in two conformations, one in which the repressor is able to bind to the specific binding site (active state) and one in which it cannot bind the specific binding site (inactive state). The environmental signaling occurs via passive import of an extracellular inducer that binds the repressor, shifting the equilibrium between the two conformations of the repressor <span class="citation" data-cites="Razo-Mejia2018"> [<a href="#ref-Razo-Mejia2018" role="doc-biblioref">27</a>]</span>. In previous work we have extensively characterized the mean response of this circuit under different conditions using equilibrium based models <span class="citation" data-cites="Phillips2019"> [<a href="#ref-Phillips2019" role="doc-biblioref">28</a>]</span>. Here we build upon these models to characterize the full distribution of gene expression with parameters such as repressor copy number and its affinity for the DNA being systematically varied.</p>
<p>As the copy number of molecular species is a discrete quantity, chemical master equations have emerged as a useful tool to model their inherent probability distribution <span class="citation" data-cites="Sanchez2013"> [<a href="#ref-Sanchez2013" role="doc-biblioref">30</a>]</span>. In Fig. 2(A) we show the minimal model and the necessary set of parameters needed to compute the full distribution of mRNA and its protein gene product. Specifically, we assume a three-state model where the promoter can be found in a 1) transcriptionally active state (<span class="math inline">\(A\)</span> state), 2) a transcriptionally inactive state without the repressor bound (<span class="math inline">\(I\)</span> state) and 3) a transcriptionally inactive state with the repressor bound (<span class="math inline">\(R\)</span> state). We do not assume that the transition between the active state <span class="math inline">\(A\)</span> and the inactive state <span class="math inline">\(I\)</span> occurs due to RNAP binding to the promoter as the transcription initiation kinetics involve several more steps than simple binding <span class="citation" data-cites="Browning2004"> [<a href="#ref-Browning2004" role="doc-biblioref">31</a>]</span>. We coarse-grain all these steps into effective “<em>on</em>” and “<em>off</em>” states for the promoter, consistent with experiments demonstrating the bursty nature of gene expression in <em>E. coli</em> <span class="citation" data-cites="Golding2005"> [<a href="#ref-Golding2005" role="doc-biblioref">19</a>]</span>. These three states generate a system of coupled differential equations for each of the three state distributions <span class="math inline">\(P_A(m, p; t)\)</span>, <span class="math inline">\(P_I(m, p; t)\)</span> and <span class="math inline">\(P_R(m, p; t)\)</span>, where <span class="math inline">\(m\)</span> and <span class="math inline">\(p\)</span> are the mRNA and protein count per cell, respectively and <span class="math inline">\(t\)</span> is time. Given the rates depicted in Fig. 2(A) we define the system of ODEs for a specific <span class="math inline">\(m\)</span> and <span class="math inline">\(p\)</span>. For the transcriptionally active state, we have <span class="math display">\[
\begin{split}
    \frac{d P_A(m, p)}{dt} &amp;=
    - \overbrace{k^{(p)}_{\text{off}} P_A(m, p)}^{A \rightarrow I} % A -&gt; I
    + \overbrace{k^{(p)}_{\text{on}} P_I(m, p)}^{I \rightarrow A}\\ % I -&gt; A
    &amp;+ \overbrace{r_m P_A(m-1, p)}^{m-1 \rightarrow m} % m-1 -&gt; m
    - \overbrace{r_m P_A(m, p)}^{m \rightarrow m+1}% m -&gt; m+1
    + \overbrace{\gamma _m (m + 1) P_A(m+1 , p)}^{m+1 \rightarrow m} % m+1 -&gt; m
    - \overbrace{\gamma _m m P_A(m , p)}^{m \rightarrow m-1}\\ % m -&gt; m-1
    &amp;+ \overbrace{r_p m P_A(m, p - 1)}^{p-1 \rightarrow p} % p-1 -&gt; p
    - \overbrace{r_p m P_A(m, p)}^{p \rightarrow p+1} % p -&gt; p+1
    + \overbrace{\gamma _p (p + 1) P_A(m, p + 1)}^{p + 1 \rightarrow p} % p+1 -&gt; p
    - \overbrace{\gamma _p p P_A(m, p)}^{p \rightarrow p-1}, % p -&gt; p-1
\end{split}
\label{eq:ch3_eq02}
\]</span> where the state transitions for each term are labeled by overbraces. For the transcriptionally inactive state <span class="math inline">\(I\)</span>, we have <span class="math display">\[
\begin{split}
    \frac{d P_I(m, p)}{dt} &amp;=
    \overbrace{k^{(p)}_{\text{off}} P_A(m, p)}^{A \rightarrow I} % A -&gt; I
    - \overbrace{k^{(p)}_{\text{on}} P_I(m, p)}^{I \rightarrow A} % I -&gt; A
    + \overbrace{k^{(r)}_{\text{off}} P_R(m, p)}^{R \rightarrow I} % R -&gt; I
    - \overbrace{k^{(r)}_{\text{on}} P_I(m, p)}^{I \rightarrow R}\\ % I -&gt; R
    &amp;+ \overbrace{\gamma _m (m + 1) P_I(m+1 , p)}^{m+1 \rightarrow m} % m+1 -&gt; m
    - \overbrace{\gamma _m m P_I(m , p)}^{m \rightarrow m-1}\\ % m -&gt; m-1
    &amp;+ \overbrace{r_p m P_I(m, p - 1)}^{p-1 \rightarrow p} % p-1 -&gt; p
    - \overbrace{r_p m P_I(m, p)}^{p \rightarrow p+1} % p -&gt; p+1
    + \overbrace{\gamma _p (p + 1) P_I(m, p + 1)}^{p + 1 \rightarrow p} % p+1 -&gt; p
    - \overbrace{\gamma _p p P_I(m, p)}^{p \rightarrow p-1}. % p -&gt; p-1
\end{split}
\label{eq:ch3_eq03}
\]</span> And finally, for the repressor bound state <span class="math inline">\(R\)</span>, <span class="math display">\[
\begin{split}
    \frac{d P_R(m, p)}{dt} &amp;=
    - \overbrace{k^{(r)}_{\text{off}} P_R(m, p)}^{R \rightarrow I} % R -&gt; I
    + \overbrace{k^{(r)}_{\text{on}} P_I(m, p)}^{I \rightarrow R}\\ % I -&gt; R
    &amp;+ \overbrace{\gamma _m (m + 1) P_R(m+1 , p)}^{m+1 \rightarrow m} % m+1 -&gt; m
    - \overbrace{\gamma _m m P_R(m , p)}^{m \rightarrow m-1}\\ % m -&gt; m-1
    &amp;+ \overbrace{r_p m P_R(m, p - 1)}^{p-1 \rightarrow p} % p-1 -&gt; p
    - \overbrace{r_p m P_R(m, p)}^{p \rightarrow p+1} % p -&gt; p+1
    + \overbrace{\gamma _p (p + 1) P_R(m, p + 1)}^{p + 1 \rightarrow p} % p+1 -&gt; p
    - \overbrace{\gamma _p p P_R(m, p)}^{p \rightarrow p-1}. % p -&gt; p-1
\end{split}
\label{eq:ch3_eq04}
\]</span> As we will discuss later, the protein degradation term <span class="math inline">\(\gamma _p\)</span> is set to zero since active protein degradation is slow compared to the cell cycle of exponentially growing bacteria, but rather we explicitly implement binomial partitioning of the proteins into daughter cells upon division <span class="citation" data-cites="Maurizi1992"> [<a href="#ref-Maurizi1992" role="doc-biblioref">32</a>]</span>.</p>
<p>It is convenient to rewrite these equations in a compact matrix notation <span class="citation" data-cites="Sanchez2013"> [<a href="#ref-Sanchez2013" role="doc-biblioref">30</a>]</span>. For this we define the vector <span class="math inline">\(\mathbf{P}(m, p)\)</span> as <span class="math display">\[
\mathbf{P}(m, p) = (P_A(m, p), P_I(m, p), P_R(m, p))^T,
\label{eq:ch3_eq05}
\]</span> where <span class="math inline">\(^T\)</span> is the transpose. By defining the matrices <span class="math inline">\(\mathbf{K}\)</span> to contain the promoter state transitions, <span class="math inline">\(\mathbf{R}_m\)</span> and <span class="math inline">\(\mathbf{\Gamma} _m\)</span> to contain the mRNA production and degradation terms, respectively, and <span class="math inline">\(\mathbf{R}_p\)</span> and <span class="math inline">\(\mathbf{\Gamma}_p\)</span> to contain the protein production and degradation terms, respectively, the system of ODEs can then be written as (See for full definition of these matrices) <span class="math display">\[
\begin{split}
    \frac{d \mathbf{P}(m, p)}{dt} &amp;= 
    \left( \mathbf{K} - \mathbf{R}_m -m\mathbf{\Gamma} _m 
    -m \mathbf{R}_p -p\mathbf{\Gamma}_p \right) \mathbf{P}(m, p)\\
    &amp;+ \mathbf{R}_m \mathbf{P}(m-1, p)
    + (m + 1) \mathbf{\Gamma} _m \mathbf{P}(m + 1, p)\\
    &amp;+ m \mathbf{R}_p \mathbf{P}(m, p - 1)
    + (p + 1) \mathbf{\Gamma}_p \mathbf{P}(m, p + 1).
\end{split}
\label{eq:ch3_eq06}
\]</span> Having defined the gene expression dynamics we now proceed to determine all rate parameters in Eq. <span class="math inline">\(\ref{eq:ch3_eq06}\)</span>.</p>
<h3 id="sec_param">Inferring parameters from published data sets</h3>
<p>A decade of research in our group has characterized the simple repression motif with an ever expanding array of predictions and corresponding experiments to uncover the physics of this genetic circuit <span class="citation" data-cites="Phillips2019"> [<a href="#ref-Phillips2019" role="doc-biblioref">28</a>]</span>. In doing so we have come to understand the mean response of a single promoter in the presence of varying levels of repressor copy numbers and repressor-DNA affinities <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">20</a>]</span>, due to the effect that competing binding sites and multiple promoter copies impose <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">26</a>]</span>, and in recent work, assisted by the Monod-Wyman-Changeux (MWC) model, we expanded the scope to the allosteric nature of the repressor <span class="citation" data-cites="Razo-Mejia2018"> [<a href="#ref-Razo-Mejia2018" role="doc-biblioref">27</a>]</span>. All of these studies have exploited the simplicity and predictive power of equilibrium approximations to these non-equilibrium systems <span class="citation" data-cites="Buchler2003"> [<a href="#ref-Buchler2003" role="doc-biblioref">33</a>]</span>. We have also used a similar kinetic model to that depicted in Fig. 2(A) to study the noise in mRNA copy number <span class="citation" data-cites="Jones2014a"> [<a href="#ref-Jones2014a" role="doc-biblioref">25</a>]</span>. Although these studies focus on the same experimental system described by different theoretical frameworks, in earlier work in our laboratory an attempt to unite parametric knowledge across studies based on equilibrium and non-equilibrium models has not been performed previously. As a test case of the depth of our theoretical understanding of this simple transcriptional regulation system we combine all of the studies mentioned above to inform the parameter values of the model presented in Fig. 2(A). Fig. 2(B) schematizes the data sets and experimental techniques used to measure gene expression along with the parameters that can be inferred from them.</p>
<p>Section XXX expands on the details of how the inference was performed for each of the parameters. Briefly, the promoter activation and inactivation rates <span class="math inline">\(k^{(p)}_{\text{on}}\)</span> and <span class="math inline">\(k^{(p)}_{\text{off}}\)</span>, as well as the transcription rate <span class="math inline">\(r_m\)</span> were obtained in units of the mRNA degradation rate <span class="math inline">\(\gamma _m\)</span> by fitting a two-state promoter model (no state <span class="math inline">\(R\)</span> from Fig. 2(A)) <span class="citation" data-cites="Peccoud1995"> [<a href="#ref-Peccoud1995" role="doc-biblioref">34</a>]</span> to mRNA FISH data of an unregulated promoter (no repressor present in the cell) <span class="citation" data-cites="Jones2014a"> [<a href="#ref-Jones2014a" role="doc-biblioref">25</a>]</span>. The repressor on rate is assumed to be of the form <span class="math inline">\(k^{(r)}_{\text{on}} = k_o [R]\)</span> where <span class="math inline">\(k_o\)</span> is a diffusion-limited on rate and <span class="math inline">\([R]\)</span> is the concentration of active repressor in the cell <span class="citation" data-cites="Jones2014a"> [<a href="#ref-Jones2014a" role="doc-biblioref">25</a>]</span>. This concentration of active repressor is at the same time determined by the repressor copy number in the cell, and the fraction of these repressors that are in the active state, i.e. able to bind DNA. Existing estimates of the transition rates between conformations of allosteric molecules set them at the microsecond scale <span class="citation" data-cites="Cui2008"> [<a href="#ref-Cui2008" role="doc-biblioref">35</a>]</span>. By considering this to be representative for our repressor of interest, the separation of time-scales between the rapid conformational changes of the repressor and the slower downstream processes such as the open-complex formation processes allow us to model the probability of the repressor being in the active state as an equilibrium MWC process. The parameters of the MWC model <span class="math inline">\(K_A\)</span>, <span class="math inline">\(K_I\)</span> and <span class="math inline">\(\Delta\varepsilon_{AI}\)</span> were previously characterized from video-microscopy and flow-cytometry data <span class="citation" data-cites="Razo-Mejia2018"> [<a href="#ref-Razo-Mejia2018" role="doc-biblioref">27</a>]</span>. For the repressor off rate, <span class="math inline">\(k^{(r)}_{\text{off}}\)</span>, we take advantage of the fact that the mean mRNA copy number as derived from the model in Fig. 2(A) cast in the language of rates is of the same functional form as the equilibrium model cast in the language of binding energies <span class="citation" data-cites="Phillips2015a"> [<a href="#ref-Phillips2015a" role="doc-biblioref">36</a>]</span>. Therefore the value of the repressor-DNA binding energy <span class="math inline">\(\Delta\varepsilon_r\)</span> constrains the value of the repressor off rate <span class="math inline">\(k^{(r)}_{\text{off}}\)</span>. These constraints on the rates allow us to make self-consistent predictions under both the equilibrium and the kinetic framework. Having all parameters in hand, we can now proceed to solve the gene expression dynamics.</p>
<figure>
<img src="ch3_fig02.png" id="fig:ch3_fig02" data-short-caption="Minimal kinetic model of transcriptional regulation for a simple repression architecture" alt="Figure 2: Minimal kinetic model of transcriptional regulation for a simple repression architecture. (A) Three-state promoter stochastic model of transcriptional regulation by a repressor. The regulation by the repressor occurs via exclusion of the transcription initiation machinery, not allowing the promoter to transition to the transcriptionally active state. All parameters highlighted with colored boxes were determined from published datasets based on the same genetic circuit. Parameters in dashed boxes were taken directly from values reported in the literature or adjusted to satisfy known biological restrictions. (B) Data sets used to infer the parameter values. From left to right Garcia &amp; Phillips  [20] is used to determine k^{(r)}_{\text{off}} and k^{(r)}_{\text{on}}, Brewster et al.  [26] is used to determine \Delta\varepsilon_{AI} and k^{(r)}_{\text{on}}, Razo-Mejia et al.  [27] is used to determine K_A, K_I, and k^{(r)}_{\text{on}} and Jones et al.  [25] is used to determine r_m, k^{(p)}_{\text{on}}, and k^{(p)}_{\text{off}}." /><figcaption aria-hidden="true">Figure 2: <strong>Minimal kinetic model of transcriptional regulation for a simple repression architecture.</strong> (A) Three-state promoter stochastic model of transcriptional regulation by a repressor. The regulation by the repressor occurs via exclusion of the transcription initiation machinery, not allowing the promoter to transition to the transcriptionally active state. All parameters highlighted with colored boxes were determined from published datasets based on the same genetic circuit. Parameters in dashed boxes were taken directly from values reported in the literature or adjusted to satisfy known biological restrictions. (B) Data sets used to infer the parameter values. From left to right Garcia &amp; Phillips <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">20</a>]</span> is used to determine <span class="math inline">\(k^{(r)}_{\text{off}}\)</span> and <span class="math inline">\(k^{(r)}_{\text{on}}\)</span>, Brewster et al. <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">26</a>]</span> is used to determine <span class="math inline">\(\Delta\varepsilon_{AI}\)</span> and <span class="math inline">\(k^{(r)}_{\text{on}}\)</span>, Razo-Mejia et al. <span class="citation" data-cites="Razo-Mejia2018"> [<a href="#ref-Razo-Mejia2018" role="doc-biblioref">27</a>]</span> is used to determine <span class="math inline">\(K_A\)</span>, <span class="math inline">\(K_I\)</span>, and <span class="math inline">\(k^{(r)}_{\text{on}}\)</span> and Jones et al. <span class="citation" data-cites="Jones2014a"> [<a href="#ref-Jones2014a" role="doc-biblioref">25</a>]</span> is used to determine <span class="math inline">\(r_m\)</span>, <span class="math inline">\(k^{(p)}_{\text{on}}\)</span>, and <span class="math inline">\(k^{(p)}_{\text{off}}\)</span>.</figcaption>
</figure>
<h3 id="computing-the-moments-of-the-mrna-and-protein-distributions">Computing the moments of the mRNA and protein distributions</h3>
<p>Finding analytical solutions to chemical master equations is often fraught with difficulty. An alternative approach is to to approximate the distribution. One such scheme of approximation, the maximum entropy principle, makes use of the moments of the distribution to approximate the full distribution. In this section we will demonstrate an iterative algorithm to compute the mRNA and protein distribution moments.</p>
<p>The kinetic model for the simple repression motif depicted in Fig. 2(A) consists of an infinite system of ODEs for each possible pair of mRNA and protein copy number, <span class="math inline">\((m, p)\)</span>. To compute any moment of the distribution, we define a vector <span class="math display">\[
\langle \mathbf{m^x p^y} \rangle \equiv 
(\langle m^x p^y\rangle_A, 
\langle m^x p^y \rangle_I, 
\langle m^x p^y\rangle_R)^T,
\label{eq:ch2_eq07}
\]</span> where <span class="math inline">\(\langle m^x p^y \rangle_S\)</span> is the expected value of <span class="math inline">\(m^x p^y\)</span> in state <span class="math inline">\(S \in \{A, I, R\}\)</span> for <span class="math inline">\(x, y \in \mathbb{N}\)</span>. In other words, just as we defined the vector <span class="math inline">\(\mathbf{P}(m, p)\)</span>, here we define a vector to collect the expected value of each of the promoter states. By definition, any of these moments <span class="math inline">\(\langle m^x p^y \rangle_S\)</span> can be computed as <span class="math display">\[
\langle m^x p^y \rangle_S \equiv 
\sum_{m=0}^\infty \sum_{p=0}^\infty m^x p^y P_S(m, p).
\label{eq:ch3_eq08}
\]</span> Summing over all possible values for <span class="math inline">\(m\)</span> and <span class="math inline">\(p\)</span> in Eq. <span class="math inline">\(\ref{eq:ch3_eq06}\)</span> results in an ODE for any moment of the distribution of the form (See XXX for full derivation) <span class="math display">\[\begin{split}
    \frac{d \mathbf{\langle m^x p^y \rangle }}{dt} &amp;=
    \mathbf{K} \mathbf{\langle m^x p^y \rangle}\\
    &amp;+ \mathbf{R}_m \mathbf{\langle p^y \left[ (m + 1)^x -m^x \right]\rangle }
     + \mathbf{\Gamma}_m \mathbf{\langle m p^y \left[ (m - 1)^x - m^x \right]\rangle}\\
    &amp;+ \mathbf{R}_p \mathbf{\langle m^{(x + 1)} \left[ (p + 1)^y - p^y \right]\rangle}
     + \mathbf{\Gamma}_p \mathbf{\langle m^x p \left[ (p - 1)^y - p^y \right]\rangle }.
\end{split}
\label{eq:ch3_eq09}
\]</span></p>
<p>Given that all transitions in our stochastic model are first order reactions, has no moment-closure problem <span class="citation" data-cites="Voliotis2014a"> [<a href="#ref-Voliotis2014a" role="doc-biblioref">14</a>]</span>. This means that the dynamical equation for a given moment only depends on lower moments (See XXX for full proof). This feature of our model implies, for example, that the second moment of the protein distribution <span class="math inline">\(\langle p^2 \rangle\)</span> depends only on the first two moments of the mRNA distribution <span class="math inline">\(\langle m \rangle\)</span> and <span class="math inline">\(\langle m^2 \rangle\)</span>, the first protein moment <span class="math inline">\(\langle p \rangle\)</span>, and the cross-correlation term <span class="math inline">\(\langle mp \rangle\)</span>. We can therefore define <span class="math inline">\(\boldsymbol{\mu}^{\mathbf{(x, y)}}\)</span> to be a vector containing all moments up to <span class="math inline">\(\mathbf{\langle m^x p^y\rangle}\)</span> for all promoter states, <span class="math display">\[
\boldsymbol{\mu}^{\mathbf{(x, y)}} = \left[ \mathbf{\langle m^0 p^0 \rangle},
\mathbf{\langle m^1 p^0 \rangle},
\ldots, \mathbf{\langle m^x p^y \rangle} \right]^T.
\label{eq:ch3_eq10}
\]</span> Explicitly for the three-state promoter model depicted in Fig. 2(A) this vector takes the form <span class="math display">\[
\boldsymbol{\mu}^{\mathbf{(x, y)}} = 
\left[ 
    \langle m^0 p^0 \rangle_A,
    \langle m^0 p^0 \rangle_I,
    \langle m^0 p^0 \rangle_R,
    \ldots,
    \langle m^x p^y \rangle_A,
    \langle m^x p^y \rangle_I,
    \langle m^x p^y \rangle_R 
\right]^T.
\label{eq:ch3_eq11}
\]</span></p>
<p>Given this definition we can compute the general moment dynamics as <span class="math display">\[
\frac{d \boldsymbol{\mu}^{\mathbf{(x, y)}}}{dt} = \mathbf{A}
\boldsymbol{\mu}^{\mathbf{(x, y)}}, 
\label{eq:ch3_eq12}
\]</span> where <span class="math inline">\(\mathbf{A}\)</span> is a square matrix that contains all the numerical coefficients that relate each of the moments. We can then use Eq. <span class="math inline">\(\ref{eq:ch3_eq09}\)</span> to build matrix <span class="math inline">\(\mathbf{A}\)</span> by iteratively substituting values for the exponents <span class="math inline">\(x\)</span> and <span class="math inline">\(y\)</span> up to a specified value. In the next section, we will use Eq. <span class="math inline">\(\ref{eq:ch3_eq12}\)</span> to numerically integrate the dynamical equations for our moments of interest as cells progress through the cell cycle. We will then use the value of the moments of the distribution to approximate the full gene expression distribution. This method is computationally more efficient than trying to numerically integrate the infinite set of equations describing the full probability distribution <span class="math inline">\(\mathbf{P}(m, p)\)</span>, or using a stochastic algorithm to sample from the distribution.</p>
<h3 id="accounting-for-cell-cycle-dependent-variability-in-gene-dosage">Accounting for cell-cycle dependent variability in gene dosage</h3>
<p>As cells progress through the cell cycle, the genome has to be replicated to guarantee that each daughter cell receives a copy of the genetic material. As replication of the genome can take longer than the total cell cycle, this implies that cells spend part of the cell cycle with multiple copies of each gene depending on the cellular growth rate and the relative position of the gene with respect to the replication origin <span class="citation" data-cites="Bremer1996"> [<a href="#ref-Bremer1996" role="doc-biblioref">37</a>]</span>. Genes closer to the replication origin spend a larger fraction of the cell cycle with multiple copies compared to genes closer to the replication termination site <span class="citation" data-cites="Bremer1996"> [<a href="#ref-Bremer1996" role="doc-biblioref">37</a>]</span>. (A) depicts a schematic of this process where the replication origin (<em>oriC</em>) and the relevant locus for our experimental measurements (<em>galK</em>) are highlighted.</p>
<p>Since this change in gene copy number has been shown to have an effect on cell-to-cell variability in gene expression <span class="citation" data-cites="Jones2014a Peterson2015"> [<a href="#ref-Jones2014a" role="doc-biblioref">25</a>,<a href="#ref-Peterson2015" role="doc-biblioref">38</a>]</span>, we now extend our minimal model to account for these changes in gene copy number during the cell cycle. We reason that the only difference between the single-copy state and the two-copy state of the promoter is a doubling of the mRNA production rate <span class="math inline">\(r_m\)</span>. In particular, the promoter activation and inactivation rates <span class="math inline">\(k^{(p)}_{\text{on}}\)</span> and <span class="math inline">\(k^{(p)}_{\text{off}}\)</span> and the mRNA production rate <span class="math inline">\(r_m\)</span> inferred in assume that cells spend a fraction <span class="math inline">\(f\)</span> of the cell cycle with one copy of the promoter (mRNA production rate <span class="math inline">\(r_m\)</span>) and a fraction <span class="math inline">\((1-f)\)</span> of the cell cycle with two copies of the promoter (mRNA production rate <span class="math inline">\(2 r_m\)</span>). This inference was performed considering that at each cell state the mRNA level immediately reaches the steady state value for the corresponding mRNA production rate. This assumption is justified since the timescale to reach this steady state depends only on the degradation rate <span class="math inline">\(\gamma _m\)</span>, which for the mRNA is much shorter (<span class="math inline">\(\approx 3\)</span> min) than the length of the cell cycle (<span class="math inline">\(\approx\)</span> 60 min for our experimental conditions) <span class="citation" data-cites="Dong1995"> [<a href="#ref-Dong1995" role="doc-biblioref">39</a>]</span>. shows that a model accounting for this gene copy number variability is able to capture data from single molecule mRNA counts of an unregulated (constitutively expressed) promoter.</p>
<p>Given that the protein degradation rate <span class="math inline">\(\gamma _p\)</span> in our model is set by the cell division time, we do not expect that the protein count will reach the corresponding steady state value for each stage in the cell cycle. In other words, cells do not spend long enough with two copies of the promoter for the protein level to reach the steady state value corresponding to a transcription rate of <span class="math inline">\(2 r_m\)</span>. We therefore use the dynamical equations developed in to numerically integrate the time trajectory of the moments of the distribution with the corresponding parameters for each phase of the cell cycle. (B) shows an example corresponding to the mean mRNA level (upper panel) and the mean protein level (lower panel) for the case of the unregulated promoter. Given that we inferred the promoter rate parameters considering that mRNA reaches steady state in each stage, we see that the numerical integration of the equations is consistent with the assumption of having the mRNA reach a stable value in each stage (See (B) upper panel). On the other hand, the mean protein level does not reach a steady state at either of the cellular stages. Nevertheless it is notable that after several cell cycles the trajectory from cycle to cycle follows a repetitive pattern (See (B) lower panel). Previously we have experimentally observed this repetitive pattern by tracking the expression level over time with video microscopy as observed in Fig. 18 of <span class="citation" data-cites="Phillips2019"> [<a href="#ref-Phillips2019" role="doc-biblioref">28</a>]</span>.</p>
<p>To test the effects of including this gene copy number variability in our model we now compare the predictions of the model with experimental data. As detailed in the Methods section, we obtained single-cell fluorescence values of different <em>E. coli</em> strains carrying a YFP gene under the control of the LacI repressor. Each strain was exposed to twelve different input inducer (IPTG) concentrations for <span class="math inline">\(\approx 8\)</span> generations for cells to adapt to the media. The strains imaged spanned three orders of magnitude in repressor copy number and three distinct repressor-DNA affinities. Since growth was asynchronous, we reason that cells were randomly sampled at all stages of the cell cycle. Therefore, when computing statistics from the data such as the mean fluorescence value, in reality we are averaging over the cell cycle. In other words, as depicted in (B), quantities such as the mean protein copy number change over time, i.e. <span class="math inline">\(\langle p \rangle \equiv \langle p(t) \rangle\)</span>. This means that computing the mean of a population of unsynchronized cells is equivalent to averaging this time-dependent mean protein copy number over the span of the cell cycle. Mathematically this is expressed as <span class="math display">\[
\langle p \rangle_c = \int_{t_o}^{t_d} \langle p(t) \rangle P(t) dt,
\label{eq:ch3_eq13}
\]</span> where <span class="math inline">\(\langle p(t) \rangle\)</span> represents the first moment of the protein distribution as computed from , <span class="math inline">\(\langle p\rangle_c\)</span> represents the average protein copy number over a cell cycle, <span class="math inline">\(t_o\)</span> represents the start of the cell cycle, <span class="math inline">\(t_d\)</span> represents the time of cell division, and <span class="math inline">\(P(t)\)</span> represents the probability of any cell being at time <span class="math inline">\(t \in [t_o, t_d]\)</span> of their cell cycle. We do not consider cells uniformly distributed along the cell cycle since it is known that cells age is exponentially distributed, having more younger than older cells at any point in time <span class="citation" data-cites="Powell1956"> [<a href="#ref-Powell1956" role="doc-biblioref">40</a>]</span> (See for further details). All computations hereafter are therefore done by applying an average like that in for the span of a cell cycle. We remind the reader that these time averages are done under a fixed environmental state. It is the trajectory of cells over cell cycles under a constant environment that we need to account for. It is through this averaging over the span of a cell cycle that we turn a periodic process as the one shown in Fig. 3(B) into a stationary process that we can compare with experimental data and, as we will see later, use to reconstruct the steady state gene expression distribution.</p>
<p>Fig. 3(C) compares zero-parameter fit predictions (lines) with experimentally determined quantities (points). The upper row shows the non-dimensional quantity known as the fold-change in gene expression <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">20</a>]</span>. This fold-change is defined as the relative mean gene expression level with respect to an unregulated promoter. For protein this is <span class="math display">\[
\text{fold-change} = 
\frac{\langle p(R &gt; 0) \rangle_c}{\langle p(R = 0) \rangle_c},
\label{eq:ch2_eq14}
\]</span> where <span class="math inline">\(\langle p(R &gt; 0)i \rangle_c\)</span> represents the mean protein count for cells with non-zero repressor copy number count <span class="math inline">\(R\)</span> over the entire cell cycle, and <span class="math inline">\(\langle p(R = 0) \rangle_c\)</span> represents the equivalent for a strain with no repressors present. The experimental points were determined from the YFP fluorescent intensities of cells with varying repressor copy number and a <span class="math inline">\(\Delta lacI\)</span> strain with no repressor gene present (See Methods for further details). The fold-change in gene expression has previously served as a metric to test the validity of equilibrium-based models <span class="citation" data-cites="Phillips2015a"> [<a href="#ref-Phillips2015a" role="doc-biblioref">36</a>]</span>. We note that the curves shown in the upper panel of (C) are consistent with the predictions from equilibrium models <span class="citation" data-cites="Razo-Mejia2018"> [<a href="#ref-Razo-Mejia2018" role="doc-biblioref">27</a>]</span> despite being generated from a clearly non-equilibrium process as shown in (B). The kinetic model from (A) goes beyond the equilibrium picture to generate predictions for moments of the distribution other than the mean mRNA or mean protein count. To test this extended predictive power the lower row of (C) shows the noise in gene expression defined as the standard deviation over the mean protein count, accounting for the changes in gene dosage during the cell cycle. Although our model systematically underestimates the noise in gene expression, the zero-parameter fits capture the scaling of this noise. Possible origins of this systematic discrepancy could be the intrinsic cell-to-cell variability of rate parameters given the variability in the molecular components of the central dogma machinery <span class="citation" data-cites="Jones2014a"> [<a href="#ref-Jones2014a" role="doc-biblioref">25</a>]</span>, or noise generated by irreversible non-equilibrium reactions not explicitly taken into account in our minimal model <span class="citation" data-cites="Grah2020"> [<a href="#ref-Grah2020" role="doc-biblioref">41</a>]</span>. The large errors for the highly repressed strains (lower left panel in (C)) are a result of having a small number in the denominator - mean fluorescence level - when computing the noise. Although the model is still highly informative about the physical nature of how cells regulate their gene expression, the lack of exact numerical agreement between theory and data opens an opportunity to gain new insights into the biophysical origin of cell-to-cell variability. In we explore empirical ways to account for this systematic deviation. We direct the reader to where equivalent predictions are done ignoring the changes in gene dosage due to the replication of the genome.</p>
<figure>
<img src="ch3_fig03.png" id="fig:ch3_fig03" data-short-caption="Accounting for gene copy number variability during the cell cycle." alt="Figure 3: Accounting for gene copy number variability during the cell cycle. (A) Schematic of a replicating bacterial genome. As cells progress through the cell cycle the genome is replicated, duplicating gene copies for a fraction of the cell cycle before the cell divides. oriC indicates the replication origin, and galK indicates the locus at which the YFP reporter construct was integrated. (B) mean (solid line) \pm standard deviation (shaded region) for the mRNA (upper panel) and protein (lower panel) dynamics. Cells spend a fraction of the cell cycle with a single copy of the promoter (light brown) and the rest of the cell cycle with two copies (light yellow). Black arrows indicate time of cell division. (C) Zero parameter-fit predictions (lines) and experimental data (circles) of the gene expression fold-change (upper row) and noise (lower row) for repressor binding sites with different affinities (different columns) and different repressor copy numbers per cell (different lines on each panel). Error bars in data represent the 95% confidence interval on the quantities as computed from 10,000 bootstrap estimates generated from &gt; 500 single-cell fluorescence measurements. In the theory curves, dotted lines indicate plot in linear scale to include zero, while solid lines indicate logarithmic scale. For visual clarity, data points in the noise panel with exceptionally large values coming from highly repressed strains are plotted on a separate panel. The Python code used to generate part (B) (ch3_fig03B.py) and part (C) (ch3_fig03C.py) of this figure can be found on the original paper GitHub repository." /><figcaption aria-hidden="true">Figure 3: <strong>Accounting for gene copy number variability during the cell cycle.</strong> (A) Schematic of a replicating bacterial genome. As cells progress through the cell cycle the genome is replicated, duplicating gene copies for a fraction of the cell cycle before the cell divides. <em>oriC</em> indicates the replication origin, and <em>galK</em> indicates the locus at which the YFP reporter construct was integrated. (B) mean (solid line) <span class="math inline">\(\pm\)</span> standard deviation (shaded region) for the mRNA (upper panel) and protein (lower panel) dynamics. Cells spend a fraction of the cell cycle with a single copy of the promoter (light brown) and the rest of the cell cycle with two copies (light yellow). Black arrows indicate time of cell division. (C) Zero parameter-fit predictions (lines) and experimental data (circles) of the gene expression fold-change (upper row) and noise (lower row) for repressor binding sites with different affinities (different columns) and different repressor copy numbers per cell (different lines on each panel). Error bars in data represent the 95% confidence interval on the quantities as computed from 10,000 bootstrap estimates generated from <span class="math inline">\(&gt; 500\)</span> single-cell fluorescence measurements. In the theory curves, dotted lines indicate plot in linear scale to include zero, while solid lines indicate logarithmic scale. For visual clarity, data points in the noise panel with exceptionally large values coming from highly repressed strains are plotted on a separate panel. The Python code used to generate part (B) <a href="https://github.com/RPGroup-PBoC/chann_cap/blob/master/src/figs/fig03B.py">(<code>ch3_fig03B.py</code>)</a> and part (C) <a href="https://github.com/RPGroup-PBoC/chann_cap/blob/master/src/figs/fig03C.py">(<code>ch3_fig03C.py</code>)</a> of this figure can be found on the original paper <a href="https://github.com/RPGroup-PBoC/chann_cap">GitHub repository.</a></figcaption>
</figure>
<h3 id="maximum-entropy-approximation">Maximum Entropy approximation</h3>
<p>Having numerically computed the moments of the mRNA and protein distributions as cells progress through the cell cycle, we now proceed to make an approximate reconstruction of the full distributions given this limited information. As hinted in the maximum entropy principle, first proposed by E.T. Jaynes in 1957 <span class="citation" data-cites="Jaynes1957"> [<a href="#ref-Jaynes1957" role="doc-biblioref">42</a>]</span>, approximates the entire distribution by maximizing the Shannon entropy subject to constraints given by the values of the moments of the distribution <span class="citation" data-cites="Jaynes1957"> [<a href="#ref-Jaynes1957" role="doc-biblioref">42</a>]</span>. This procedure leads to a probability distribution of the form (See for full derivation) <span class="math display">\[
P(m, p) = \frac{1}{\mathcal{Z}} \exp
\left( - \sum_{(x,y)} \lambda_{(x,y)} m^x p^y \right), 
\label{eq:ch3_eq15}
\]</span> where <span class="math inline">\(\lambda_{(x,y)}\)</span> is the Lagrange multiplier associated with the constraint set by the moment <span class="math inline">\(\langle m^x p^y \rangle\)</span>, and <span class="math inline">\(\mathcal{Z}\)</span> is a normalization constant. The more moments <span class="math inline">\(\langle m^x p^y \rangle\)</span> included as constraints, the more accurate the approximation resulting from becomes.</p>
<p>The computational challenge then becomes an optimization routine in which the values for the Lagrange multipliers <span class="math inline">\(\lambda_{(x,y)}\)</span> that are consistent with the constraints set by the moment values <span class="math inline">\(\langle m^x p^y \rangle\)</span> need to be found. This is computationally more efficient than sampling directly out of the master equation with a stochastic algorithm (see for further comparison between maximum entropy estimates and the Gillespie algorithm). details our implementation of a robust algorithm to find the values of the Lagrange multipliers. (A) shows example predicted protein distributions reconstructed using the first six moments of the protein distribution for a suite of different biophysical parameters and environmental inducer concentrations. As repressor-DNA binding affinity (columns in (A)) and repressor copy number (rows in (A)) are varied, the responses to different signals, i.e. inducer concentrations, overlap to varying degrees. For example, the upper right corner frame with a weak binding site (<span class="math inline">\(\Delta\varepsilon_r = -9.7 \; k_BT\)</span>) and a low repressor copy number (22 repressors per cell) have virtually identical distributions regardless of the input inducer concentration. This means that cells with this set of parameters cannot resolve any difference in the concentration of the signal. As the number of repressors is increased, the degree of overlap between distributions decreases, allowing cells to better resolve the value of the signal input. On the opposite extreme the lower left panel shows a strong binding site (<span class="math inline">\(\Delta\varepsilon_r = -15.3 \; k_BT\)</span>) and a high repressor copy number (1740 repressors per cell). This parameter combination shows overlap between distributions since the high degree of repression centers all distributions towards lower copy numbers, again giving little ability for the cells to resolve the inputs. In (B) and we show the comparison of these predicted cumulative distributions with the experimental single-cell fluorescence distributions. Given the systematic deviation of our predictions for the protein copy number noise highlighted in (C), the theoretical distributions (dashed lines) underestimate the width of the experimental data. We again direct the reader to for an exploration of empirical changes to the moments that improve the agreement of the predictions. In the following section we formalize the notion of how well cells can resolve different inputs from an information theoretic perspective via the channel capacity.</p>
<figure>
<img src="ch3_fig04.png" id="fig:ch3_fig04" data-short-caption="Maximum entropy protein distributions for varying physical parameters." alt="Figure 4: Maximum entropy protein distributions for varying physical parameters. (A) Predicted protein distributions under different inducer (IPTG) concentrations for different combinations of repressor-DNA affinities (columns) and repressor copy numbers (rows). The first six moments of the protein distribution used to constrain the maximum entropy approximation were computed by integrating as cells progressed through the cell cycle as described in . (B) Theory-experiment comparison of predicted fold-change empirical cumulative distribution functions (ECDF). Each panel shows two example concentrations of inducer (colored curves) with their corresponding theoretical predictions (dashed lines). Distributions were normalized to the mean expression value of the unregulated strain in order to compare theoretical predictions in discrete protein counts with experimental fluorescent measurements in arbitrary units. The Python code used to generate part (A) (ch3_fig04A.py) and part (B) (ch3_fig04B.py) of this figure can be found on the original paper GitHub repository." /><figcaption aria-hidden="true">Figure 4: <strong>Maximum entropy protein distributions for varying physical parameters.</strong> (A) Predicted protein distributions under different inducer (IPTG) concentrations for different combinations of repressor-DNA affinities (columns) and repressor copy numbers (rows). The first six moments of the protein distribution used to constrain the maximum entropy approximation were computed by integrating as cells progressed through the cell cycle as described in . (B) Theory-experiment comparison of predicted fold-change empirical cumulative distribution functions (ECDF). Each panel shows two example concentrations of inducer (colored curves) with their corresponding theoretical predictions (dashed lines). Distributions were normalized to the mean expression value of the unregulated strain in order to compare theoretical predictions in discrete protein counts with experimental fluorescent measurements in arbitrary units. The Python code used to generate part (A) <a href="https://github.com/RPGroup-PBoC/chann_cap/blob/master/src/figs/fig04A.py">(<code>ch3_fig04A.py</code>)</a> and part (B) <a href="https://github.com/RPGroup-PBoC/chann_cap/blob/master/src/figs/fig04B.py">(<code>ch3_fig04B.py</code>)</a> of this figure can be found on the original paper <a href="https://github.com/RPGroup-PBoC/chann_cap">GitHub repository.</a></figcaption>
</figure>
<h3 id="theoretical-prediction-of-the-channel-capacity">Theoretical prediction of the channel capacity</h3>
<p>We now turn our focus to the channel capacity, which is a metric by which we can quantify the degree to which cells can measure the environmental state (in this context, the inducer concentration). The channel capacity is defined as the mutual information <span class="math inline">\(I\)</span> between input and output (), maximized over all possible input (IPTG) distributions <span class="math inline">\(P(c)\)</span>. If used as a metric of how reliably a signaling system can infer the state of the external signal, the channel capacity, when measured in bits, is commonly interpreted as the logarithm of the number of states that the signaling system can properly resolve. For example, a signaling system with a channel capacity of <span class="math inline">\(C\)</span> bits is interpreted as being able to resolve <span class="math inline">\(2^C\)</span> states, though channel capacities with fractional values are allowed. We therefore prefer the Bayesian interpretation that the mutual information quantifies the improvement in the inference of the input when considering the output compared to just using the prior distribution of the input by itself for prediction <span class="citation" data-cites="Voliotis2014a Bowsher2014"> [<a href="#ref-Voliotis2014a" role="doc-biblioref">14</a>,<a href="#ref-Bowsher2014" role="doc-biblioref">43</a>]</span>. Under this interpretation a channel capacity of a fractional bit still quantifies an improvement in the ability of the signaling system to infer the value of the extracellular signal compared to having no sensing system at all.</p>
<p>Computing the channel capacity implies optimizing over an infinite space of possible distributions <span class="math inline">\(P(c)\)</span>. For special cases in which the noise is small compared to the dynamic range, approximate analytical equations have been derived <span class="citation" data-cites="Tkacik2008a"> [<a href="#ref-Tkacik2008a" role="doc-biblioref">17</a>]</span>. But given the high cell-to-cell variability that our model predicts, the conditions of the so-called small noise approximation are not satisfied. We therefore appeal to a numerical solution known as the Blahut-Arimoto algorithm <span class="citation" data-cites="Blahut1972"> [<a href="#ref-Blahut1972" role="doc-biblioref">44</a>]</span> (See for further details). (A) shows zero-parameter fit predictions of the channel capacity as a function of the number of repressors for different repressor-DNA affinities (solid lines). These predictions are contrasted with experimental determinations of the channel capacity as inferred from single-cell fluorescence intensity distributions taken over 12 different concentrations of inducer. Briefly, from single-cell fluorescence measurements we can approximate the input-output distribution <span class="math inline">\(P(p \mid c)\)</span>. Once these conditional distributions are fixed, the task of finding the input distribution at channel capacity becomes a computational optimization routine that can be undertaken using conjugate gradient or similar algorithms. For the particular case of the channel capacity on a system with a discrete number of inputs and outputs the Blahut-Arimoto algorithm is built in such a way that it guarantees the convergence towards the optimal input distribution (See for further details). (B) shows example input-output functions for different values of the channel capacity. This illustrates that having access to no information (zero channel capacity) is a consequence of having overlapping input-output functions (lower panel). On the other hand, the more separated the input-output distributions are (upper panel) the higher the channel capacity can be.</p>
<p>All theoretical predictions in (A) are systematically above the experimental data. Although our theoretical predictions in (A) do not numerically match the experimental inference of the channel capacity, the model does capture interesting qualitative features of the data that are worth highlighting. On one extreme, for cells with no transcription factors, there is no information processing potential as this simple genetic circuit would be constitutively expressed regardless of the environmental state. As cells increase the transcription factor copy number, the channel capacity increases until it reaches a maximum before falling back down at high repressor copy number since the promoter would be permanently repressed. The steepness of the increment in channel capacity as well as the height of the maximum expression is highly dependent on the repressor-DNA affinity. For strong binding sites (blue curve in (A)) there is a rapid increment in the channel capacity, but the maximum value reached is smaller compared to a weaker binding site (orange curve in (A)). In we show using the small noise approximation <span class="citation" data-cites="Tkacik2008 Tkacik2008a"> [<a href="#ref-Tkacik2008" role="doc-biblioref">9</a>,<a href="#ref-Tkacik2008a" role="doc-biblioref">17</a>]</span> that if the systematic deviation of our predictions on the cell-to-cell variability was explained with a multiplicative constant, i.e. all noise predictions can be corrected by multiplying them by a single constant, we would expect the channel capacity to be off by a constant additive factor. This factor of <span class="math inline">\(\approx 0.43\)</span> bits can recover the agreement between the model and the experimental data.</p>
<figure>
<img src="ch3_fig05.png" id="fig:ch3_fig05" data-short-caption="Comparison of theoretical and experimental channel capacity." alt="Figure 5: Comparison of theoretical and experimental channel capacity. (A) Channel capacity as inferred using the Blahut-Arimoto algorithm  [44] for varying number of repressors and repressor-DNA affinities. All inferences were performed using 12 IPTG concentrations as detailed in the Methods. Curves represent zero-parameter fit predictions made with the maximum entropy distributions as shown in . Points represent inferences made from single cell fluorescence distributions (See for further details). Theoretical curves were smoothed using a Gaussian kernel to remove numerical precision errors. (B) Example input-output functions in opposite limits of channel capacity. Lower panel illustrates that zero channel capacity indicates that all distributions overlap. Upper panel illustrates that as the channel capacity increases, the separation between distributions increases as well. Arrows point to the corresponding channel capacity computed from the predicted distributions. The Python code used to generate part (A) (ch3_fig05A.py) and part (B) (ch3_fig04B.py) of this figure can be found on the original paper GitHub repository." /><figcaption aria-hidden="true">Figure 5: <strong>Comparison of theoretical and experimental channel capacity.</strong> (A) Channel capacity as inferred using the Blahut-Arimoto algorithm <span class="citation" data-cites="Blahut1972"> [<a href="#ref-Blahut1972" role="doc-biblioref">44</a>]</span> for varying number of repressors and repressor-DNA affinities. All inferences were performed using 12 IPTG concentrations as detailed in the Methods. Curves represent zero-parameter fit predictions made with the maximum entropy distributions as shown in . Points represent inferences made from single cell fluorescence distributions (See for further details). Theoretical curves were smoothed using a Gaussian kernel to remove numerical precision errors. (B) Example input-output functions in opposite limits of channel capacity. Lower panel illustrates that zero channel capacity indicates that all distributions overlap. Upper panel illustrates that as the channel capacity increases, the separation between distributions increases as well. Arrows point to the corresponding channel capacity computed from the predicted distributions. The Python code used to generate part (A) <a href="https://github.com/RPGroup-PBoC/chann_cap/blob/master/src/figs/fig05A.py">(<code>ch3_fig05A.py</code>)</a> and part (B) <a href="https://github.com/RPGroup-PBoC/chann_cap/blob/master/src/figs/fig05B.py">(<code>ch3_fig04B.py</code>)</a> of this figure can be found on the original paper <a href="https://github.com/RPGroup-PBoC/chann_cap">GitHub repository.</a></figcaption>
</figure>
<h2 id="discussion">Discussion</h2>
<p>Building on Shannon’s formulation of information theory, there have been significant efforts using this theoretical framework to understand the information processing capabilities of biological systems, and the evolutionary consequences for organisms harboring signal transduction systems <span class="citation" data-cites="Bergstrom2004 Taylor2007 Tkacik2008 Polani2009 Nemenman2010 Rivoire2011"> [<a href="#ref-Nemenman2010" role="doc-biblioref">1</a>,<a href="#ref-Taylor2007" role="doc-biblioref">6</a>,<a href="#ref-Tkacik2008" role="doc-biblioref">9</a>,<a href="#ref-Bergstrom2004" role="doc-biblioref">45</a>–<a href="#ref-Rivoire2011" role="doc-biblioref">47</a>]</span>. Recently, with the mechanistic dissection of molecular signaling pathways, significant progress has been made on the question of the physical limits of cellular detection and the role that features such as feedback loops play in this task <span class="citation" data-cites="Bialek2005 Libby2007 Tkacik2011 Rhee2012a Voliotis2014a"> [<a href="#ref-Bialek2005" role="doc-biblioref">7</a>,<a href="#ref-Voliotis2014a" role="doc-biblioref">14</a>,<a href="#ref-Tkacik2011" role="doc-biblioref">16</a>,<a href="#ref-Libby2007" role="doc-biblioref">48</a>,<a href="#ref-Rhee2012a" role="doc-biblioref">49</a>]</span>. But the field still lacks a rigorous experimental test of these ideas with precision measurements on a system that is tractable both experimentally and theoretically.</p>
<p>In this paper we take advantage of the recent progress on the quantitative modeling of input-output functions of genetic circuits to build a minimal model of the simple repression motif <span class="citation" data-cites="Phillips2019"> [<a href="#ref-Phillips2019" role="doc-biblioref">28</a>]</span>. By combining a series of studies on this circuit spanning diverse experimental methods for measuring gene expression under a myriad of different conditions, for the first time we possess complete <em>a priori</em> parametric knowledge – allowing us to generate parameter-free predictions for processes related to information processing. Some of the model parameters for our kinetic formulation of the input-output function are informed by inferences made from equilibrium models. We use the fact that if both kinetic and thermodynamic languages describe the same system, the predictions must be self-consistent. In other words, if the equilibrium model can only make statements about the mean mRNA and mean protein copy number because of the way these models are constructed, those predictions must be equivalent to what the kinetic model has to say about these same quantities. This condition therefore constrains the values that the kinetic rates in the model can take. To test whether or not the equilibrium picture can reproduce the predictions made by the kinetic model we compare the experimental and theoretical fold-change in protein copy number for a suite of biophysical parameters and environmental conditions ((C) upper row). The agreement between theory and experiment demonstrates that these two frameworks can indeed make consistent predictions.</p>
<p>The kinetic treatment of the system brings with it increasing predictive power compared to the equilibrium picture. Under the kinetic formulation, the predictions are not limited only to the mean but to any of the moments of the mRNA and protein distributions. Furthermore, our formulation in terms of dynamical equations allows us to account for the time-varying nature of the moments of the mRNA and protein copy numbers. Specifically, since the protein mean lifetime is comparable with the cell cycle length, the protein copy number does not reach a steady state over the duration of the cell cycle. Accounting for this effect increases the expected cell-to-cell variability when measuring non-synchronized cells. We first test these novel predictions by comparing the noise in protein copy number (standard deviation / mean) with experimental data. Our minimal model predicts the noise up to a systematic deviation. The physical or biological origins of this discrepancy remain an open question. In that way the work presented here exposes the status quo of our understanding of gene regulation in bacteria, posing new questions to be answered with future refinements of the model. We then extend our analysis to infer entire protein distributions at different input signal concentrations by using the maximum entropy principle. What this means is that we compute moments of the protein distribution, and then use these moments to build an approximation to the full distribution. These predicted distributions are then compared with experimental single-cell distributions as shown in (B) and . Again, here although our minimal model systematically underestimates the width of the distributions, it informs how changes in parameters such as protein copy number or protein-DNA binding affinity will affect the full probabilistic input-output function of the genetic circuit, up to a multiplicative constant. We then use our model to predict the information processing capacity.</p>
<p>By maximizing the mutual information between input signal concentration and output protein distribution over all possible input distributions, we predict the channel capacity of the system over a suite of biophysical parameters such as varying repressor protein copy number and repressor-DNA binding affinity. Although there is no reason to assume the the simplified synthetic circuit we used as an experimental model operates optimally given the distribution of inputs, the relevance of the channel capacity comes from its interpretation as a metric of the physical limit of how precise an inference cells can make about what the state of the environment is. Our model, despite the systematic deviations, makes non-trivial predictions such as the existence of an optimal repressor copy number for a given repressor-DNA binding energy, predicting the channel capacity up to an additive constant (See ). The origin of this optimal combination of repressor copy number and binding energy differs from previous publications in which an extra term associated with the cost of producing protein was included in the model <span class="citation" data-cites="Tkacik2011"> [<a href="#ref-Tkacik2011" role="doc-biblioref">16</a>]</span>. This optimal parameter combination is a direct consequence of the fact that the LacI repressor cannot be fully deactivated <span class="citation" data-cites="Razo-Mejia2018"> [<a href="#ref-Razo-Mejia2018" role="doc-biblioref">27</a>]</span>. This implies that as the number of repressors increases, a significant number of them are still able to bind to the promoter even at saturating concentrations of inducer. This causes all of the input-output functions to be shift towards low expression levels, regardless of the inducer concentration, decreasing the amount of information that the circuit is able to process. Interestingly, the number of bits predicted and measured in our system is similar to that of the gap genes in the <em>Drosophila</em> embryo <span class="citation" data-cites="Dubuis2013"> [<a href="#ref-Dubuis2013" role="doc-biblioref">10</a>]</span>. Although this is a suggestive numerical correspondence that sets current experimental data on information processing capacity of genetic circuits between 1 and 2 bits, more work is required to fully understand the effect that different regulatory architectures have on the ability to resolve different signals.</p>
<p>We consider it important to highlight the limitations of the work presented here. The previously discussed systematic deviation for the noise and skewness of the predicted distributions (See ), and therefore of the predicted distributions and channel capacity, remains an unresolved question. Our current best hypothesis for the origin of this unaccounted noise pertains to cell-to-cell variability in the central dogma machinery. More specifically, our model does not account for changes in RNAP and sigma factor copy numbers, changes in ribosome numbers, and even the variability in the repressor copy number. This possibility deserves to be addressed in further iterations of our minimal model. Also, as first reported in <span class="citation" data-cites="Razo-Mejia2018"> [<a href="#ref-Razo-Mejia2018" role="doc-biblioref">27</a>]</span>, our model fails to capture the steepness of the fold-change induction curve for the weakest repressor binding site (See (B)). Furthermore the minimal model in (A), despite being widely used, is an oversimplification of the physical picture of how the transcriptional machinery works. The coarse-graining of all the kinetic steps involved in transcription initiation into two effective promoter states – active and inactive – ignores potential kinetic regulatory mechanisms of intermediate states <span class="citation" data-cites="Scholes2017"> [<a href="#ref-Scholes2017" role="doc-biblioref">50</a>]</span>. Moreover it has been argued that despite the fact that the mRNA count distribution does not follow a Poisson distribution, this effect could be caused by unknown factors not at the level of transcriptional regulation <span class="citation" data-cites="Choubey2018"> [<a href="#ref-Choubey2018" role="doc-biblioref">51</a>]</span>.</p>
<p>The findings of this work open the opportunity to accurately test intriguing ideas that connect Shannon’s metric of how accurately a signaling system can infer the state of the environment, with Darwinian fitness <span class="citation" data-cites="Taylor2007"> [<a href="#ref-Taylor2007" role="doc-biblioref">6</a>]</span>. Beautiful work along these lines has been done in the context of the developmental program of the early <em>Drosophila</em> embryo <span class="citation" data-cites="Tkacik2008 Petkova2019"> [<a href="#ref-Tkacik2008" role="doc-biblioref">9</a>,<a href="#ref-Petkova2019" role="doc-biblioref">11</a>]</span>. These studies demonstrated that the input-output function of the pair-rule genes works at channel capacity, suggesting that selection has acted on these signaling pathways, pushing them to operate at the limit of what the physics of these systems allows. Our system differs from the early embryo in the sense that we have a tunable circuit with variable amounts of information processing capabilities. Furthermore, compared with the fly embryo in which the organism tunes both the input and output distributions over evolutionary time, we have experimental control of the distribution of inputs that the cells are exposed to. Consequently this means that instead of seeing the final result of the evolutionary process, we would be able to set different environmental challenges, and track over time the evolution of the population. These experiments could shed light into the suggestive hypothesis of information bits as a trait on which natural selection acts. We see this exciting direction as part of the overall effort in quantitative biology of predicting evolution <span class="citation" data-cites="Lassig2017"> [<a href="#ref-Lassig2017" role="doc-biblioref">52</a>]</span>.</p>
<h2 id="materials-and-methods">Materials and Methods</h2>
<h3 id="e.-coli-strains"><em>E. coli</em> strains</h3>
<p>All strains used in this study were originally made for <span class="citation" data-cites="Razo-Mejia2018"> [<a href="#ref-Razo-Mejia2018" role="doc-biblioref">27</a>]</span>. We chose a subset of three repressor copy numbers that span two orders of magnitude. We refer the reader to <span class="citation" data-cites="Razo-Mejia2018"> [<a href="#ref-Razo-Mejia2018" role="doc-biblioref">27</a>]</span> for details on the construction of these strains. Briefly, the strains have a construct consisting of the <em>lacUV5</em> promoter and one of three possible binding sites for the <em>lac</em> repressor (O1, O2, and O3) controlling the expression of a YFP reporter gene. This construct is integrated into the genome at the <em>galK</em> locus. The number of repressors per cell is varied by changing the ribosomal binding site controlling the translation of the <em>lac</em> repressor gene. The repressor constructs were integrated in the <em>ybcN</em> locus. Finally, all strains used in this work constitutively express an mCherry reporter from a low copy number plasmid. This serves as a volume marker that facilitates the segmentation of cells when processing microscopy images.</p>
<h3 id="growth-conditions">Growth conditions</h3>
<p>For all experiments, cultures were initiated from a 50% glycerol frozen stock at -80<span class="math inline">\(^\circ\)</span>C. Three strains - autofluorescence (<span class="math inline">\(auto\)</span>), <span class="math inline">\(\Delta lacI\)</span> (<span class="math inline">\(\Delta\)</span>), and a strain with a known binding site and repressor copy number (<span class="math inline">\(R\)</span>) - were inoculated into individual tubes with 2 mL of Lysogeny Broth (LB Miller Powder, BD Medical) with 20 <span class="math inline">\(\mu\)</span>g/mL of chloramphenicol and 30 <span class="math inline">\(\mu\)</span>g/mL of kanamycin. These cultures were grown overnight at 37<span class="math inline">\(^\circ\)</span>C with rapid agitation to reach saturation. The saturated cultures were diluted 1:1000 into 500 <span class="math inline">\(\mu\)</span>L of M9 minimal media (M9 5X Salts, Sigma-Aldrich M6030; 2 mM magnesium sulfate, Mallinckrodt Chemicals 6066-04; 100 mM calcium chloride, Fisher Chemicals C79-500) supplemented with 0.5% (w/v) glucose on a 2 mL 96-deep-well plate. The <span class="math inline">\(R\)</span> strain was diluted into 12 different wells with minimal media, each with a different IPTG concentration (0 <span class="math inline">\(\mu\)</span>M, 0.1 <span class="math inline">\(\mu\)</span>M, 5 <span class="math inline">\(\mu\)</span>M, 10 <span class="math inline">\(\mu\)</span>M, 25 <span class="math inline">\(\mu\)</span>M, 50 <span class="math inline">\(\mu\)</span>M, 75 <span class="math inline">\(\mu\)</span>M, 100 <span class="math inline">\(\mu\)</span>M, 250 <span class="math inline">\(\mu\)</span>M, 500 <span class="math inline">\(\mu M\)</span>, 1000 <span class="math inline">\(\mu\)</span>M, 5000 <span class="math inline">\(\mu\)</span>M) while the <span class="math inline">\(auto\)</span> and <span class="math inline">\(\Delta\)</span> strains were diluted into two wells (0 <span class="math inline">\(\mu\)</span>M, 5000 <span class="math inline">\(\mu\)</span>M). Each of the IPTG concentrations came from a single preparation stock kept in 100-fold concentrated aliquots. The 96 well plate was then incubated at 37<span class="math inline">\(^\circ\)</span>C with rapid agitation for 8 hours before imaging.</p>
<h3 id="microscopy-imaging-procedure">Microscopy imaging procedure</h3>
<p>The microscopy pipeline used for this work exactly followed the steps from <span class="citation" data-cites="Razo-Mejia2018"> [<a href="#ref-Razo-Mejia2018" role="doc-biblioref">27</a>]</span>. Briefly, twelve 2% agarose (Life Technologies UltraPure Agarose, Cat.No. 16500100) gels were made out of M9 media (or PBS buffer) with the corresponding IPTG concentration (see growth conditions) and placed between two glass coverslips for them to solidify after microwaving. After the 8 hour incubation in minimal media, 1 <span class="math inline">\(\mu\)</span>L of a 1:10 dilution of the cultures into fresh media or PBS buffer was placed into small squares (roughly 10 mm <span class="math inline">\(\times\)</span> 10 mm) of the different agarose gels. A total of 16 agarose squares - 12 concentrations of IPTG for the <span class="math inline">\(R\)</span> strain, 2 concentrations for the <span class="math inline">\(\Delta\)</span> and 2 for the <span class="math inline">\(auto\)</span> strain - were mounted into a single glass-bottom dish (Ted Pella Wilco Dish, Cat. No. 14027-20) that was sealed with parafilm.</p>
<p>All imaging was done on an inverted fluorescent microscope (Nikon Ti-Eclipse) with custom-built laser illumination system. The YFP fluorescence (quantitative reporter) was imaged with a CrystaLaser 514 nm excitation laser coupled with a laser-optimized (Semrock Cat. No. LF514-C-000) emission filter. All strains, including the <span class="math inline">\(auto\)</span> strain, included a constitutively expressed mCherry protein to aid the segmentation. Therefore, for each image three channels (YFP, On average 30 images with roughly 20 cells per condition were taken. 25 images of a fluorescent slide and 25 images of the camera background noise were taken every imaging session in order to flatten the illumination. The image processing pipeline for this work is exactly the same as in <span class="citation" data-cites="Razo-Mejia2018"> [<a href="#ref-Razo-Mejia2018" role="doc-biblioref">27</a>]</span>.</p>
<h3 id="data-and-code-availability">Data and Code Availability</h3>
<p>All data and custom scripts were collected and stored using Git version control. Code for raw data processing, theoretical analysis, and figure generation is available on the GitHub repository (<a href="https://github.com/RPGroup-PBoC/chann_cap" class="uri">https://github.com/RPGroup-PBoC/chann_cap</a>). The code can also be accessed via the paper website (<a href="https://www.rpgroup.caltech.edu/chann_cap/" class="uri">https://www.rpgroup.caltech.edu/chann_cap/</a>). Raw microscopy data are stored on the CaltechDATA data repository and can be accessed via DOI <a href="https://doi.org/10.22002/d1.1184" class="uri">https://doi.org/10.22002/d1.1184</a>. Bootstrap estimates of experimental channel capacity are also available on the CaltechDATA data repository via <a href="https://doi.org/10.22002/D1.1185" class="uri">https://doi.org/10.22002/D1.1185</a>.</p>
<h2 class="unnumbered" id="references">References</h2>
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-Nemenman2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[1] </div><div class="csl-right-inline">I. Nemenman, <em><span class="nocase">Information theory and adaptation</span></em>, in <em>Quantitative Biology: From Molecular to Cellular Systems</em>, Vol. 30322 (Taylor; Francis, 2010), pp. 1–12.</div>
</div>
<div id="ref-Eldar2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[2] </div><div class="csl-right-inline">A. Eldar and M. B. Elowitz, <em><span class="nocase">Functional roles for noise in genetic circuits</span></em>, Nature <strong>467</strong>, 167 (2010).</div>
</div>
<div id="ref-Shannon1948" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[3] </div><div class="csl-right-inline">C. E. Shannon, <em><span class="nocase">A Mathematical Theory of Communication</span></em>, Bell System Technical Journal <strong>27</strong>, 379 (1948).</div>
</div>
<div id="ref-MacKay2003" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[4] </div><div class="csl-right-inline">D. J. MacKay, <em><span class="nocase">Information theory, inference and learning algorithms</span></em> (Cambridge university press, 2003).</div>
</div>
<div id="ref-Kinney2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[5] </div><div class="csl-right-inline">J. B. Kinney, A. Murugan, C. G. Callan, and E. C. Cox, <em><span class="nocase">Using deep sequencing to characterize the biophysical mechanism of a transcriptional regulatory sequence</span></em>, PNAS <strong>107</strong>, 9158 (2010).</div>
</div>
<div id="ref-Taylor2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[6] </div><div class="csl-right-inline">S. F. Taylor, N. Tishby, and W. Bialek, <em><span class="nocase">Information and fitness</span></em>, ArXiv (2007).</div>
</div>
<div id="ref-Bialek2005" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[7] </div><div class="csl-right-inline">W. Bialek and S. Setayeshgar, <em><span class="nocase">Physical limits to biochemical signaling</span></em>, PNAS <strong>102</strong>, 10040 (2005).</div>
</div>
<div id="ref-Gregor2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[8] </div><div class="csl-right-inline">T. Gregor, D. W. Tank, E. F. Wieschaus, and W. Bialek, <em><span class="nocase">Probing the Limits to Positional Information</span></em>, Cell <strong>130</strong>, 153 (2007).</div>
</div>
<div id="ref-Tkacik2008" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[9] </div><div class="csl-right-inline">G. Tkacik, C. G. Callan, and W. Bialek, <em><span class="nocase">Information flow and optimization in transcriptional regulation</span></em>, PNAS <strong>105</strong>, 12265 (2008).</div>
</div>
<div id="ref-Dubuis2013" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[10] </div><div class="csl-right-inline">J. O. Dubuis, G. Tkacik, E. F. Wieschaus, T. Gregor, and W. Bialek, <em><span class="nocase">Positional information, in bits</span></em>, PNAS <strong>110</strong>, 16301 (2013).</div>
</div>
<div id="ref-Petkova2019" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[11] </div><div class="csl-right-inline">M. D. Petkova, G. Tkačik, W. Bialek, E. F. Wieschaus, and T. Gregor, <em><span class="nocase">Optimal Decoding of Cellular Identities in a Genetic Network</span></em>, Cell <strong>176</strong>, 844 (2019).</div>
</div>
<div id="ref-Rieckh2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[12] </div><div class="csl-right-inline">G. Rieckh and G. Tkačik, <em><span class="nocase">Noise and Information Transmission in Promoters with Multiple Internal States</span></em>, Biophysical Journal <strong>106</strong>, 1194 (2014).</div>
</div>
<div id="ref-Ziv2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[13] </div><div class="csl-right-inline">E. Ziv, I. Nemenman, and C. H. Wiggins, <em><span class="nocase">Optimal Signal Processing in Small Stochastic Biochemical Networks</span></em>, PLoS ONE <strong>2</strong>, e1077 (2007).</div>
</div>
<div id="ref-Voliotis2014a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[14] </div><div class="csl-right-inline">M. Voliotis, R. M. Perrett, C. McWilliams, C. a. McArdle, and C. G. Bowsher, <em><span class="nocase">Information transfer by leaky, heterogeneous, protein kinase signaling systems</span></em>, PNAS <strong>111</strong>, E326 (2014).</div>
</div>
<div id="ref-Tostevin2009" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[15] </div><div class="csl-right-inline">F. Tostevin and P. R. ten Wolde, <em><span class="nocase">Mutual Information between Input and Output Trajectories of Biochemical Networks</span></em>, Physical Review Letters <strong>102</strong>, 218101 (2009).</div>
</div>
<div id="ref-Tkacik2011" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[16] </div><div class="csl-right-inline">G. Tkačik and A. M. Walczak, <em><span class="nocase">Information transmission in genetic regulatory networks: a review</span></em>, Journal of Physics: Condensed Matter <strong>23</strong>, 153102 (2011).</div>
</div>
<div id="ref-Tkacik2008a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[17] </div><div class="csl-right-inline">G. Tkačik, C. G. Callan, and W. Bialek, <em><span class="nocase">Information capacity of genetic regulatory elements</span></em>, Physical Review E <strong>78</strong>, 011910 (2008).</div>
</div>
<div id="ref-Tabbaa2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[18] </div><div class="csl-right-inline">O. P. Tabbaa and C. Jayaprakash, <em><span class="nocase">Mutual information and the fidelity of response of gene regulatory models</span></em>, Physical Biology <strong>11</strong>, 046004 (2014).</div>
</div>
<div id="ref-Golding2005" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[19] </div><div class="csl-right-inline">I. Golding, J. Paulsson, S. M. Zawilski, and E. C. Cox, <em><span class="nocase">Real-Time Kinetics of Gene Activity in Individual Bacteria</span></em>, Cell <strong>123</strong>, 1025 (2005).</div>
</div>
<div id="ref-Garcia2011c" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[20] </div><div class="csl-right-inline">H. G. Garcia and R. Phillips, <em><span class="nocase">Quantitative dissection of the simple repression input-output function.</span></em>, Proceedings of the National Academy of Sciences <strong>108</strong>, 12173 (2011a).</div>
</div>
<div id="ref-Vilar2013" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[21] </div><div class="csl-right-inline">J. M. G. Vilar and L. Saiz, <em><span class="nocase">Reliable Prediction of Complex Phenotypes from a Modular Design in Free Energy Space: An Extensive Exploration of the <em>lac</em> Operon</span></em>, ACS Synthetic Biology <strong>2</strong>, 576 (2013).</div>
</div>
<div id="ref-Xu2015" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[22] </div><div class="csl-right-inline">H. Xu, L. A. Sepúlveda, L. Figard, A. M. Sokac, and I. Golding, <em><span class="nocase">Combining protein and mRNA quantification to decipher transcriptional regulation</span></em>, Nature Methods <strong>12</strong>, 739 (2015).</div>
</div>
<div id="ref-Brewster2012" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[23] </div><div class="csl-right-inline">R. C. Brewster, D. L. Jones, and R. Phillips, <em><span class="nocase">Tuning Promoter Strength through RNA Polymerase Binding Site Design in <em>Escherichia coli</em></span></em>, PLoS Computational Biology <strong>8</strong>, e1002811 (2012).</div>
</div>
<div id="ref-Barnes2019" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[24] </div><div class="csl-right-inline">S. L. Barnes, N. M. Belliveau, W. T. Ireland, J. B. Kinney, and R. Phillips, <em><span class="nocase">Mapping DNA sequence to transcription factor binding energy in vivo</span></em>, PLoS Computational Biology <strong>15</strong>, e1006226 (2019).</div>
</div>
<div id="ref-Jones2014a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[25] </div><div class="csl-right-inline">D. L. Jones, R. C. Brewster, and R. Phillips, <em><span class="nocase">Promoter architecture dictates cell-to-cell variability in gene expression</span></em>, Science <strong>346</strong>, 1533 (2014).</div>
</div>
<div id="ref-Brewster2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[26] </div><div class="csl-right-inline">R. C. Brewster, F. M. Weinert, H. G. Garcia, D. Song, M. Rydenfelt, and R. Phillips, <em><span class="nocase">The Transcription Factor Titration Effect Dictates Level of Gene Expression</span></em>, Cell <strong>156</strong>, 1312 (2014).</div>
</div>
<div id="ref-Razo-Mejia2018" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[27] </div><div class="csl-right-inline">M. Razo-Mejia, S. L. Barnes, N. M. Belliveau, G. Chure, T. Einav, M. Lewis, and R. Phillips, <em><span class="nocase">Tuning Transcriptional Regulation through Signaling: A Predictive Theory of Allosteric Induction</span></em>, Cell Systems <strong>6</strong>, 456 (2018).</div>
</div>
<div id="ref-Phillips2019" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[28] </div><div class="csl-right-inline">R. Phillips, N. M. Belliveau, G. Chure, H. G. Garcia, M. Razo-Mejia, and C. Scholes, <em><span class="nocase">Figure 1 Theory Meets Figure 2 Experiments in the Study of Gene Expression</span></em>, Annual Review of Biophysics <strong>48</strong>, 121 (2019).</div>
</div>
<div id="ref-Rydenfelt2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[29] </div><div class="csl-right-inline">M. Rydenfelt, H. G. Garcia, R. S. Cox, and R. Phillips, <em><span class="nocase">The Influence of Promoter Architectures and Regulatory Motifs on Gene Expression in <em>Escherichia coli</em></span></em>, PLoS ONE <strong>9</strong>, 1 (2014).</div>
</div>
<div id="ref-Sanchez2013" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[30] </div><div class="csl-right-inline">A. Sanchez, S. Choubey, and J. Kondev, <em><span class="nocase">Stochastic models of transcription: From single molecules to single cells</span></em>, Methods <strong>62</strong>, 13 (2013).</div>
</div>
<div id="ref-Browning2004" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[31] </div><div class="csl-right-inline">D. F. Browning and S. J. W. Busby, <em><span class="nocase">The regulation of bacterial transcription initiation</span></em>, Nature Reviews Microbiology <strong>2</strong>, 57 (2004).</div>
</div>
<div id="ref-Maurizi1992" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[32] </div><div class="csl-right-inline">M. R. Maurizi, <em><span class="nocase">Proteases and protein degradation in <em>Escherichia coli</em></span></em>, Experientia <strong>48</strong>, 178 (1992).</div>
</div>
<div id="ref-Buchler2003" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[33] </div><div class="csl-right-inline">N. E. Buchler, U. Gerland, and T. Hwa, <em><span class="nocase">On schemes of combinatorial transcription logic.</span></em>, PNAS <strong>100</strong>, 5136 (2003).</div>
</div>
<div id="ref-Peccoud1995" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[34] </div><div class="csl-right-inline">J. Peccoud and B. Ycart, <em><span class="nocase">Markovian Modeling of Gene-Product Synthesis</span></em>, Theoretical Population Biology <strong>48</strong>, 222 (1995).</div>
</div>
<div id="ref-Cui2008" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[35] </div><div class="csl-right-inline">Q. Cui and M. Karplus, <em><span class="nocase">Allostery and cooperativity revisited</span></em>, Protein Science <strong>17</strong>, 1295 (2008).</div>
</div>
<div id="ref-Phillips2015a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[36] </div><div class="csl-right-inline">R. Phillips, <em><span class="nocase">Napoleon Is in Equilibrium</span></em>, Annual Review of Condensed Matter Physics <strong>6</strong>, 85 (2015).</div>
</div>
<div id="ref-Bremer1996" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[37] </div><div class="csl-right-inline">H. Bremer and P. P. Dennis, <em><span class="nocase">Modulation of Chemical Composition and Other Parameters of the Cell by Growth Rate</span></em>, (n.d.).</div>
</div>
<div id="ref-Peterson2015" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[38] </div><div class="csl-right-inline">J. R. Peterson, J. A. Cole, J. Fei, T. Ha, and Z. A. Luthey-Schulten, <em><span class="nocase">Effects of DNA replication on mRNA noise</span></em>, PNAS <strong>112</strong>, 15886 (2015).</div>
</div>
<div id="ref-Dong1995" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[39] </div><div class="csl-right-inline">H. Dong and C. G. Kurland, <em><span class="nocase">Ribosome Mutants with Altered Accuracy Translate with Reduced Processivity</span></em>, Journal of Molecular Biology <strong>248</strong>, 551 (1995).</div>
</div>
<div id="ref-Powell1956" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[40] </div><div class="csl-right-inline">E. O. Powell, <em><span class="nocase">Growth Rate and Generation Time of Bacteria, with Special Reference to Continuous Culture</span></em>, Journal of General Microbiology <strong>15</strong>, 492 (1956).</div>
</div>
<div id="ref-Grah2020" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[41] </div><div class="csl-right-inline">R. Grah, B. Zoller, and G. Tkacik, <em><span class="nocase">Normative models of enhancer function</span></em>, bioRxiv (2020).</div>
</div>
<div id="ref-Jaynes1957" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[42] </div><div class="csl-right-inline">E. T. Jaynes, <em><span class="nocase">Information Theory and Statistical Mechanics</span></em>, Physical Review <strong>106</strong>, 620 (1957).</div>
</div>
<div id="ref-Bowsher2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[43] </div><div class="csl-right-inline">C. G. Bowsher and P. S. Swain, <em><span class="nocase">Environmental sensing, information transfer, and cellular decision-making</span></em>, Current Opinion in Biotechnology <strong>28</strong>, 149 (2014).</div>
</div>
<div id="ref-Blahut1972" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[44] </div><div class="csl-right-inline">R. Blahut, <em><span class="nocase">Computation of channel capacity and rate-distortion functions</span></em>, IEEE Transactions on Information Theory <strong>18</strong>, 460 (1972).</div>
</div>
<div id="ref-Bergstrom2004" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[45] </div><div class="csl-right-inline">C. T. Bergstrom and M. Lachmann, <em><span class="nocase">Shannon information and biological fitness</span></em>, in <em>Information Theory Workshop. IEEE, 2004</em> (IEEE, 2004), pp. 50–54.</div>
</div>
<div id="ref-Polani2009" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[46] </div><div class="csl-right-inline">D. Polani, <em><span class="nocase">Information: Currency of life?</span></em>, HFSP Journal <strong>3</strong>, 307 (2009).</div>
</div>
<div id="ref-Rivoire2011" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[47] </div><div class="csl-right-inline">O. Rivoire and S. Leibler, <em><span class="nocase">The Value of Information for Populations in Varying Environments</span></em>, Journal of Statistical Physics <strong>142</strong>, 1124 (2011).</div>
</div>
<div id="ref-Libby2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[48] </div><div class="csl-right-inline">E. Libby, T. J. Perkins, and P. S. Swain, <em><span class="nocase">Noisy information processing through transcriptional regulation</span></em>, PNAS <strong>104</strong>, 7151 (2007).</div>
</div>
<div id="ref-Rhee2012a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[49] </div><div class="csl-right-inline">A. Rhee, R. Cheong, and A. Levchenko, <em><span class="nocase">The application of information theory to biochemical signaling systems</span></em>, Physical Biology <strong>9</strong>, 045011 (2012).</div>
</div>
<div id="ref-Scholes2017" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[50] </div><div class="csl-right-inline">C. Scholes, A. H. DePace, and Á. Sánchez, <em><span class="nocase">Combinatorial Gene Regulation through Kinetic Control of the Transcription Cycle</span></em>, Cell Systems <strong>4</strong>, 97 (2017).</div>
</div>
<div id="ref-Choubey2018" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[51] </div><div class="csl-right-inline">S. Choubey, J. Kondev, and A. Sanchez, <em><span class="nocase">Distribution of Initiation Times Reveals Mechanisms of Transcriptional Regulation in Single Cells</span></em>, Biophysical Journal <strong>114</strong>, 2072 (2018).</div>
</div>
<div id="ref-Lassig2017" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[52] </div><div class="csl-right-inline">M. Lässig, V. Mustonen, and A. M. Walczak, <em><span class="nocase">Predicting evolution</span></em>, Nature Ecology <span>&amp;</span> Evolution <strong>1</strong>, 0077 (2017).</div>
</div>
</div>
                    </div><!-- /.content -->
                </div><!-- /.col -->
                <div class="col-md-4 col-md-offset-1">
                    <div class="sections-list-wrapper">
                        <div class="sections-list js-sections js-affix js-scrollspy hidden-xs hidden-sm"><span
                                style="font-size: 14pt">Table of Contents</span>
                            <hr>
                        </div><!-- /.sections-list -->
                    </div>
                </div><!-- /.col -->
            </div><!-- /.row -->
        </div><!-- /.container -->
    </div><!-- /.section -->
    
    <div class="section section--grey">
        <div class="container">
            <div class="row">
                <div class="col-md-7">
                    <div id="disqus_thread"></div>
<script>
	/**
	 *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
	 *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables
	 */
	 /*
	var disqus_config = function () {
	this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
		this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
	};
	*/
	(function() {
		var d = document, s = d.createElement('script');

		s.src = 'https://mrazomej-phd.disqus.com/embed.js';

		s.setAttribute('data-timestamp', +new Date());
		(d.head || d.body).appendChild(s);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>

                </div><!-- /.col -->
            </div><!-- /.row -->
        </div><!-- /.container -->
    </div><!-- /.section -->
    
    <div class="js-footer-area">
        
        <nav class="page-nav">
            <div class="container">
                <div class="row">
                    <div class="col-xs-12">
                        
                        <a href="https://mrazomej.github.io/phd/chapter_02"
                            class="page-nav__item page-nav__item--prev">
                            <i class="icon icon--arrow-left"></i>
                            Chapter 2
                        </a><!-- /.page-nav__item -->
                        
                        
                        <a href="https://mrazomej.github.io/phd/chapter_04"
                            class="page-nav__item page-nav__item--next">
                            Chapter 4
                            <i class="icon icon--arrow-right"></i>
                        </a><!-- /.page-nav__item -->
                        
                    </div><!-- /.col -->
                </div><!-- /.row -->
            </div><!-- /.container -->
        </nav><!-- /.page-nav -->
        
        
        
	<footer class="site-footer">
		<div class="container">
			<div class="row">
				<div class="col-sm-6">
					
						<a href="https://mrazomej.github.io/phd//" class="site-footer__logo">Manuel Razo-Mejia | PhD Thesis</a>
					
					
						<hr>
						<p class="site-footer__copyright">Copyright &copy; 2021. - Manuel Razo-Mejia <br>All rights reserved.</p>
					
				</div><!-- /.col -->
				
					<div class="col-sm-6 align-right">
						<ul class="social-list">
							
								<li>
									<a href="https://twitter.com/mrazomej" target="_blank" class="social-list__item social-list__item--twitter">
										<i class="icon icon--twitter"></i>
									</a>
								</li>
							
						</ul><!-- /.social-list -->
					</div><!-- /.col -->
				
			</div><!-- /.row -->
		</div><!-- /.container -->
	</footer><!-- /.site-footer -->


<script src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/jquery.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/bootstrap/affix.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/bootstrap/scrollspy.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/matchHeight.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/scripts.min.js"></script>


<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-109526846-4', 'auto');
	ga('send', 'pageview');
</script>



    </div><!-- /.js-footer-area -->
</body>

</html>