<!DOCTYPE html>

<html>

<head>
    <meta charset="UTF-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, minimum-scale=1.0">
<title>Chapter I</title>


<script type="text/x-mathjax-config">
	MathJax.Hub.Config({
    "HTML-CSS" : {
        availableFonts : ["STIX"],
        preferredFont : "Gyre Pagella",
        webFont : "STIX-Web",
        imageFont : null
    },
	  TeX: {
		equationNumbers: {
		  autoNumber: "AMS"
		}
	  },
	  tex2jax: {
		inlineMath: [['$','$']], 
		displayMath: [['$$','$$']],
		processEscapes: true,
	  }
	});
  </script>
<style>
    .eqnos {
        display: inline-block;
        position: relative;
        width: 100%;
    }

    .eqnos br {
        display: none;
    }

    .eqnos-number {
        position: absolute;
        right: 0em;
        top: 50%;
        line-height: 0;
    }
</style>

<script>
    window.MathJax = {
        tex: {
            tags: 'all'
        }
    };
</script>
<script>
    MathJax.Hub.Config({
        tex2jax: {
            inlineMath: [['$', '$'], ['\\(', '\\)']]
        }
    });
</script>
<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>

<link rel="icon" href="https://mrazomej.github.io/phd//favicon.ico"
    type="image/x-icon">
<link href="https://fonts.googleapis.com/css?family=Nanum+Myeongjo&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Alegreya|Lora&display=swap" rel="stylesheet">

<link rel="stylesheet"
    href="https://mrazomej.github.io/phd//doks-theme/assets/css/style.css">
</head>

<body class="grey" data-spy="scroll" data-target=".js-scrollspy">
    


	<header class="site-header">
		<div class="container">
			<div class="row">
				<div class="col-xs-12">
					
						<a href="https://mrazomej.github.io/phd//" class="site-header__logo"><i class="icon icon--chevron-left"></i> Home</a>
					
					
				</div><!-- /.col -->
			</div><!-- /.row -->
		</div><!-- /.container -->
	</header><!-- /.site-header -->


    <div class="hero-subheader" style="background-image: url(https://mrazomej.github.io/phd//doks-theme/assets/images/paihia.jpg); background-position: center; object-fit: cover; width:100%;">

        <div class="container" style="background-color: rgba(0,0,0, 0.75); border:0.75px solid white;">
            <div class="row">
                <div class="col-md-7">
                    <div class="align-container" data-mh>
                        <div class="align-inner">
                            
                            <h2 class="hero-subheader__title">Chapter I</h2>
                            
                            
                            <h1 class="hero-subheader__title">Introduction
</h1>
                            
                            
                            <!-- 
									
										<a href="https://mrazomej.github.io/phd/abstract" class="btn btn--dark btn--rounded btn--w-icon btn--w-icon-left">
											<i class="icon icon--arrow-left"></i>	
										</a>
									
									
										<a href="https://mrazomej.github.io/phd/chapter_02" class="btn btn--dark btn--rounded btn--w-icon">
											<i class="icon icon--arrow-right"></i>
										</a>
									
								 -->
                        </div><!-- /.align-inner -->
                    </div><!-- /.align-container -->
                </div><!-- /.col -->
                
                <div class="col-md-5 col-md-offset-0 hidden-xs hidden-sm">
                    <div class="align-container" data-mh>
                        <div class="align-inner">
                            <div class="hero-subheader__author">
                                <p class="hero-subheader__author-title">
                                    Summary
                                    <i class="icon icon--chevron-down"></i>
                                </p><!-- /.hero-subheader__author-title -->
                                <p>TBD
</p>
                            </div><!-- /.hero-subheader__author -->
                        </div><!-- /.align-inner -->
                    </div><!-- /.align-container -->
                </div><!-- /.col -->
                
            </div><!-- /.row -->
        </div><!-- /.container -->
    </div><!-- /.hero-subheader -->
    
    <nav class="page-nav">
        <div class="container">
            <div class="row">
                <div class="col-xs-12">
                    
                    <a href="https://mrazomej.github.io/phd/abstract"
                        class="page-nav__item page-nav__item--prev">
                        <i class="icon icon--arrow-left"></i>
                        Abstract
                    </a><!-- /.page-nav__item -->
                    
                    
                    <a href="https://mrazomej.github.io/phd/chapter_02"
                        class="page-nav__item page-nav__item--next">
                        Chapter 2
                        <i class="icon icon--arrow-right"></i>
                    </a><!-- /.page-nav__item -->
                    
                </div><!-- /.col -->
            </div><!-- /.row -->
        </div><!-- /.container -->
    </nav><!-- /.page-nav -->
    

    <div class="section">
        <div class="container">
            <div class="row">
                <div class="col-md-7">
                    <div class="content">
                        <h1 id="from-bio-to-bit-how-do-cells-sense-the-world-around-them">From Bio to Bit: How do cells sense the world around them?</h1>
<h2 id="introduction">Introduction</h2>
<p>In his classic 1944 book, Schrödinger brought to the attention of the scientific community what he thought were two of the biggest challenges we had ahead of us if we were to understand living systems in the same way we understand the electromagnetic field, or the universal law of gravitation <span class="citation" data-cites="Schrodinger1992"> [<a href="#ref-Schrodinger1992" role="doc-biblioref">1</a>]</span>. The idea that living organisms could be “accounted for” by physics and chemistry brought with it a new agenda on what needed to be done in order to transition from a qualitative and descriptive study of the phenomena of life, to a quantitative and predictive science in the spirit of the physical sciences. Since the publication of the book there has been an enormous amount of progress on our understanding of living systems from a first principles perspective, nevertheless 75 years later and Schrödinger questions are still as relevant and as vibrant as ever before <span class="citation" data-cites="Phillips2021"> [<a href="#ref-Phillips2021" role="doc-biblioref">2</a>]</span>.</p>
<p>One of the defining features of living organisms at all scales is their capacity of gathering information from the environment, encode an internal representation of the state of the environment, and generate a response based on this information processing capacity. Researches in the the field of origins-of-life have gone as far as declaring that life emerged when chemical systems underwent a radical phase transition after which they were able to process and use information and free energy <span class="citation" data-cites="Cronin2016"> [<a href="#ref-Cronin2016" role="doc-biblioref">3</a>]</span>. So, although speculative, it is highly probable that the physical theory that will fulfill Schrödinger’s vision of accounting for the phenomena of life, will be the physics of systems capable of processing information <span class="citation" data-cites="Davies2019"> [<a href="#ref-Davies2019" role="doc-biblioref">4</a>]</span>.</p>
<p>In this context, information does not take the usual generic concept of possessing useful knowledge about something. The kind of information that I am referring to has a precise mathematical definition <span class="citation" data-cites="Adami2016"> [<a href="#ref-Adami2016" role="doc-biblioref">5</a>]</span>. This formal definition of information makes it a metric worth quantifying and predicting in different biological context as theoretical studies suggest that natural selection might act on the ability of an organism to process information <span class="citation" data-cites="Taylor2007"> [<a href="#ref-Taylor2007" role="doc-biblioref">6</a>]</span>. Working out the physical details of how it is that organisms sense the environment, this is, gather information about the state of the environment, encode such information in some shape or form within their physical boundaries, and take an action based on this information is at the core of the state-of-the-art research in biophysics <span class="citation" data-cites="Bialek2012"> [<a href="#ref-Bialek2012" role="doc-biblioref">7</a>]</span>.</p>
<p>The present thesis is an effort towards this vision of understanding biological systems as information processing systems. Our object of study will be gene regulation in bacteria. This particular system has been the subject of study for microbiologists and molecular biologists for decades, and we have come to learn a lot about the microscopic mechanistic details of how bacteria turn on and off their transcriptional machinery <span class="citation" data-cites="Browning2004"> [<a href="#ref-Browning2004" role="doc-biblioref">8</a>]</span>. In particular we will focus on what we think of as the “hydrogen atom” of gene regulation–the so-called simple-repression motif (more on that in the next section). In physics, calling something the hydrogen atom of <span class="math inline">\(X\)</span> means that for the area of study <span class="math inline">\(X\)</span>, this “something” represents the a system simple enough to be amenable to analytical models that can be solve by standard mathematical methods, but rich enough to capture the general features of the phenomena. This simple genetic circuit will allow us to write tractable mathematical models that will guide our experimental efforts with the ultimate goal of testing our understanding of such system when predicting how much information can a bacterium gather from the environment using this genetic module.</p>
<p>Professional biophysicists might wish to skip the rest of this chapter as we will lay the foundations needed for the rest of our enterprise. We will introduce the basics of gene expression modeling and the mathematical concept of information, and work through every single physical and mathematical prerequisite needed for the rest of the thesis. The following chapters are structured as follows: Chapter 2 builds on a decade of understanding this hydrogen atom of gene regulation, and expands the predictive ability of our models by including the effect of environmental effectors. What this means is that we will consider how gene regulation is affected by the presence of an extracellular inducer molecule. Chapter 3 will expand even further our predictive capacities by building a model capable of making predictions about the cell-to-cell variability inherent to all signaling systems working at the molecular scale. Chapter 4 serves as a Supporting Information section for Chapter 2, detailing every calculation and every inference. Likewise, Chapter 5 expands the on Chapter 3, explaining every technical detail.</p>
<h2 id="gene-regulation-as-a-physics-101-problem">Gene regulation as a Physics 101 problem</h2>
<p>As organisms navigate the challenges presented by the environment, they must constantly fight against the will of the second law of thermodynamics to bring them back to an equilibrium state. To face those challenges, cells are equipped with a toolkit of genes written in the language of A, T, C, and G of the genome. We can think of a typical bacteria genome with <span class="math inline">\(\approx 5\times 10^3\)</span> genes as the instruction manual to produce a repertoire of tools that allow cells to thrive under different circumstances that they face throughout their lives. But not all challenges are the same, and therefore not all tools are needed simultaneously. Therefore all organisms are faced with the challenge of orchestrating the expression of the correct subset molecular tools at their disposal when trying to survive on different environments. From cells in the fly embryo expressing different genes that will define their identity on the final body plan of the animal, to a simple bacteria expressing the correct enzymes to process the available nutrients in the environment.</p>
<p>Our understanding of how organisms regulate the expression of their genes is still not as thorough as one might expect given the amount of effort that has gone into this question. Take for example <em>E. coli</em>–arguably the most well characterized model organism–for which we know the regulatory scheme of less than 1/3 of its genes <span class="citation" data-cites="Ireland2020"> [<a href="#ref-Ireland2020" role="doc-biblioref">9</a>]</span>. For more complex organisms such as <em>Drosophila</em>, <em>C. elegans</em>, or even humans we are even more hopeless on getting a holistic view of the regulatory landscape. Nevertheless we would not be doing justice to the great advances in the field if we were to pretend we are completely ignorant about how gene regulation takes place in bacteria. There is a rich mechanistic understanding of how the transcriptional machinery takes the information contained in the DNA and transcribes it into RNA <span class="citation" data-cites="Browning2004"> [<a href="#ref-Browning2004" role="doc-biblioref">8</a>]</span>. The relative simplicity of the process has inspired generations of biophysicists to try to write down minimal models that can describe and predict features of the process of gene regulation <span class="citation" data-cites="Ackers1982 Bintu2005 Kuhlman2007"> [<a href="#ref-Ackers1982" role="doc-biblioref">10</a>–<a href="#ref-Kuhlman2007" role="doc-biblioref">12</a>]</span>.</p>
<p>These modeling efforts come into two main flavors: equilibrium statistical mechanical models, and kinetic models. In the following sections we will introduce the necessary background for both approaches relevant to the rest of the thesis.</p>
<h3 id="minimal-model-of-gene-expression">Minimal model of gene expression</h3>
<p>Let us begin our introduction to gene expression modeling with the simplest example. As shown in Fig. 1(A) we imagine a gene promoter (the region of the gene from which transcriptional regulation takes place) produces mRNA at a constant rate <span class="math inline">\(r_m\)</span>. Each individual mRNA can stochastically decay with a rate <span class="math inline">\(\gamma_m\)</span>. Our interest is to understand how the mRNA count <span class="math inline">\(m\)</span> changes over time given these two competing processes. For that let us write the mRNA count at time <span class="math inline">\(m(t + \Delta t)\)</span>, where <span class="math inline">\(t\)</span> is the time–which we are thinking of as being “right now”–and <span class="math inline">\(\Delta t\)</span> is a tiny time step into the future. The mRNA count can then be predicted by computing <span class="math display">\[
m(t + \Delta t) = m(t) + r_m \Delta t - (\gamma_m \Delta t) m(t),
\]</span> where we can think of <span class="math inline">\(r_m \Delta t\)</span> as the probability of observing a single mRNA being produced in the time interval <span class="math inline">\([t, t + \Delta t]\)</span> (<span class="math inline">\(\Delta t\)</span> is so small that we neglect the possibility of seing multiple mRNAs being produced), and <span class="math inline">\(\gamma_m \Delta t\)</span> the probability of seeing a single mRNA being degraded. But since each mRNA has the same probability of being degraded, the total number of mRNAs that we would see decay in this time window would be the probability per mRNA times the total number of mRNAs. If we send the term <span class="math inline">\(m(t)\)</span> to the left hand side of the equation and divide both sides by <span class="math inline">\(\Delta t\)</span>, we obtain <span class="math display">\[
\frac{m(t + \Delta t) - m(t)}{\Delta t} = r_m - \gamma_m m(t).
\]</span> Upon taking the limit when <span class="math inline">\(\Delta t \rightarrow 0\)</span> we see that the left hand side is the definition of the derivative of the mRNA count with respect to time. We then obtain an ordinary differential equation of the form <span class="math display">\[
\frac{dm}{dt} = r_m - \gamma_m m(t).
\label{eq:dm_dt}
\]</span> Before even attempting to solve <span class="math inline">\(\ref{eq:dm_dt}\)</span> we can perform a qualitative analysis of the dynamics <span class="citation" data-cites="Strogatz2018"> [<a href="#ref-Strogatz2018" role="doc-biblioref">13</a>]</span>. In particular it is useful to plot the contribution to the derivative <span class="math inline">\(dm/dt\)</span> for each of the components (production and degradation) as a function of <span class="math inline">\(m\)</span>. This is shown in Fig. 1(B) where the blue horizontal line <span class="math inline">\(r_m\)</span> shows the production rate–which does not depend on <span class="math inline">\(m\)</span>, and the red line shows the degradation term <span class="math inline">\(m\gamma_m\)</span> which scales linearly with <span class="math inline">\(m\)</span>. Notice that for the degradation term we are not including the negative sign, i.e., we are not plotting <span class="math inline">\(-m \gamma_m\)</span>. The point <span class="math inline">\(m_{ss}\)</span> where both lines intersect represent the point where the production matches the degradation. For all values before <span class="math inline">\(m_{ss}\)</span> the production term is larger than the degradation, which means that for any value <span class="math inline">\(m &lt; m_{ss}\)</span> the derivative is positive (<span class="math inline">\(dm/dt &gt; 0\)</span>), so over time the system will produce more mRNA. The opposite is true for values all values after <span class="math inline">\(m_{ss}\)</span> where the degradation term is larger than the production term, implying that <span class="math inline">\(dm/dt &lt; 0\)</span>. This means that for <span class="math inline">\(m &gt; m_{ss}\)</span> the system will degrade mRNA. This opposite trends point at the idea that <span class="math inline">\(m_{ss}\)</span> must be what is called a stable fix point of the dynamical system. This can schematically be seen at the bottom of Fig. 1(B) where the size of the arrow heads indicates the trend of the system to move either left or right in <span class="math inline">\(m\)</span>. Since all arrows point at the special value <span class="math inline">\(m_{ss}\)</span> we can say that any small perturbation of the system will be dissipated as the system relaxes back to <span class="math inline">\(m_{ss}\)</span>.</p>
<p>This qualitative statement can be confirmed by solving Eq. <span class="math inline">\(\ref{eq:dm_dt}\)</span>. If we define the initial condition <span class="math inline">\(m(t=0) = m_o\)</span> by separation of variables we will obtain a solution of the form <span class="math display">\[
m(t) = m_o e^{-\gamma_m t} + \frac{r_m}{\gamma_m} (1 - e^{-\gamma_m t}).
\]</span> In the limit when <span class="math inline">\(t \rightarrow \infty\)</span> we can see that the steady state solution is given by <span class="math display">\[
m_{ss} = \frac{r_m}{\gamma_m}.
\]</span> Fig. 1(C) shows the time evolution of <span class="math inline">\(m\)</span> for different initial values <span class="math inline">\(m_o\)</span>. We can see that indeed regardless of the initial mRNA count the system relaxes exponentially fast to <span class="math inline">\(m_{ss} = r_m / \gamma_m\)</span>.</p>
<figure>
<img src="ch1_fig01.png" id="fig:ch1_fig01" data-short-caption="Minimal model of gene expression" alt="Figure 1: Minimal model of gene expression. (A) Schematic of the kinetics governing gene expression. mRNA is produced at a constant rate r_m independent of the current mRNA copy number. Degradation of each mRNA occurs at a rate \gamma_m. (B) Example of the qualitative analysis of the mRNA dynamics via a 1D phase-portrait. The differential equation governing the dynamics contains two terms: a constant production rate given by r_m, and a degradation rate \gamma_m m that depends on the current mRNA count. The main plot shows each of the components in the m vs dm/dt plot. Since r_m does not depend on the current number of mRNA it gives a straight production rate as a function of m. The total degradation rate depends linearly with the mRNA copy number, giving a line with slope \gamma_m. When the two components are equal (bot lines crossing), we obtain the steady-state mRNA value m_{ss}. The bottom line shows a qualitative schematic of the flow of the system towards this steady state. The further m is from m_{ss}, the faster it moves towards this point as schematized by the size of the arrows. (C) Example of mRNA dynamics for different initial conditions. Over time all curves converge to the steady-state mRNA value m_{ss}=r_m/\gamma_m. For this plot \gamma_m = 1 and r_m/\gamma_m = 10. The Python code (ch1_fig01C.py) used to generate part (C) of this figure can be found on the thesis GitHub repository." /><figcaption aria-hidden="true">Figure 1: <strong>Minimal model of gene expression.</strong> (A) Schematic of the kinetics governing gene expression. mRNA is produced at a constant rate <span class="math inline">\(r_m\)</span> independent of the current mRNA copy number. Degradation of each mRNA occurs at a rate <span class="math inline">\(\gamma_m\)</span>. (B) Example of the qualitative analysis of the mRNA dynamics via a 1D phase-portrait. The differential equation governing the dynamics contains two terms: a constant production rate given by <span class="math inline">\(r_m\)</span>, and a degradation rate <span class="math inline">\(\gamma_m m\)</span> that depends on the current mRNA count. The main plot shows each of the components in the <span class="math inline">\(m\)</span> vs <span class="math inline">\(dm/dt\)</span> plot. Since <span class="math inline">\(r_m\)</span> does not depend on the current number of mRNA it gives a straight production rate as a function of <span class="math inline">\(m\)</span>. The total degradation rate depends linearly with the mRNA copy number, giving a line with slope <span class="math inline">\(\gamma_m\)</span>. When the two components are equal (bot lines crossing), we obtain the steady-state mRNA value <span class="math inline">\(m_{ss}\)</span>. The bottom line shows a qualitative schematic of the flow of the system towards this steady state. The further <span class="math inline">\(m\)</span> is from <span class="math inline">\(m_{ss}\)</span>, the faster it moves towards this point as schematized by the size of the arrows. (C) Example of mRNA dynamics for different initial conditions. Over time all curves converge to the steady-state mRNA value <span class="math inline">\(m_{ss}=r_m/\gamma_m\)</span>. For this plot <span class="math inline">\(\gamma_m = 1\)</span> and <span class="math inline">\(r_m/\gamma_m = 10\)</span>. The <a href="https://github.com/mrazomej/phd/blob/master/src/chapter_01/code/ch1_fig01C.py">Python code (<code>ch1_fig01C.py</code>)</a> used to generate part (C) of this figure can be found on the thesis <a href="https://github.com/mrazomej/phd">GitHub repository</a>.</figcaption>
</figure>
<p>So far our model assumes a simple constant transcription rate <span class="math inline">\(r_m\)</span>; let us expand this term a little further in order to include regulation by a transcriptional repressor further down the road. We knot that for a transcriptional event to take place the RNA polymerase (RNAP) must bind to the promoter region and undergo a series of irreversible steps such as opening the double helix to initiate the copying of the DNA sequence into mRNA <span class="citation" data-cites="Browning2004"> [<a href="#ref-Browning2004" role="doc-biblioref">8</a>]</span>. If we assume these irreversible steps take place in a much longer time-scale compared to the initial binding and unbinding of the RNAP on the promoter, we can separate these two events in different terms. In particular we can write that the mRNA production happens with a rate <span class="math display">\[
\text{mRNA production} = r_m \cdot p_{\text{bound}},
\label{eq:mRNA_prod}
\]</span> where we split the original production term into two steps: <span class="math inline">\(p_{\text{bound}}\)</span>, the probability of finding an RNAP bound to the promoter, and <span class="math inline">\(r_m\)</span> which captures all of the downstream irreversible steps that take place once the RNAP is engaged in a transcriptional event. A way to think about it–relevant to what I am doing right now as I type my thesis–is to think that the speed at which I type this document has to do with two things: The probability of me being actively working on these notes times the rate at which I type these notes once I engage in the activity. The reason this splitting makes sense is because we can include the effect of the regulation by a transcriptional repressor as a reduction of the time (the probability) that the RNAP can be bound to the promoter by effectively taking away probability mass. Futhermore, since we are assuming that the binding and unbinding of the RNAP happen at a timescale much faster than the downstream events, we can assume this binding reaction is in quasi-equilibrium, for which we can use the powerful theoretical framework of statistical mechanics. Let us now delve into the basics of this physical theory.</p>
<h3 id="the-unreasonable-effectiveness-of-unrealistic-simplifications">The unreasonable effectiveness of unrealistic simplifications</h3>
<p>On the preface of the textbook <em>Molecular Driving Forces</em> Dill and Bromberg introduce the idea of Statistical Mechanics as <em>the unreasonable effectiveness of unrealistic simplifications</em> <span class="citation" data-cites="Dill2010"> [<a href="#ref-Dill2010" role="doc-biblioref">14</a>]</span>. Although one could make the case that all of physics follows this description, it is certainly evident that statistical mechanics is a vivid example of how simple ideas can have profound consequences. Statistical mechanics can be simply defined as the theory that, upon assuming the atomic nature of matter, explains the phenomenology that classical thermodynamics established <span class="citation" data-cites="Dill2010"> [<a href="#ref-Dill2010" role="doc-biblioref">14</a>]</span> from the interactions of the microscopic components of a system. As with any other physical theory, statistical mechanics is built from a set of <em>empirical</em> facts that define “axioms” that we take to be truth. In other words, as Feynman famously described to us: if we want to come up with a new law of nature there is a simple recipe that we must follow:</p>
<ol type="1">
<li><p>We guess the law. Literally. The deepest understanding of physical reality we have up to now come from educated guesses made after a careful observation of nature.</p></li>
<li><p>We compute the consequences of such guess. That is why mathematical theories allow us to sharpen our statements about how we think nature works.</p></li>
<li><p>We compare with experiments/observations. The scientific revolution came about when, after the dark ages, we finally learned it was okay to say “we don’t know.”</p></li>
</ol>
<p>In such simple statement, Feynman tells us, lies the key to science <span class="citation" data-cites="Feynman1965"> [<a href="#ref-Feynman1965" role="doc-biblioref">15</a>]</span>. For our purpose of understanding the basis of statistical mechanics we will argue that the main law upon which the field is founded is given by Boltzmann’s law <span class="math display">\[
\frac{P(E_1)}{P(E_2)} = \frac{e^{-E_1 / k_BT}}{e^{-E_2 / k_BT}}.
\label{eq:boltzmann_law}
\]</span> Let us unpack this equation. The main idea behind statistical mechanics is that macroscopic observables (temperature, pressure, specific heat in classic examples) are emergent properties dictated by the dynamics of the microscopic components of the system. What Boltzmann’s law tells us is that the relative probability of a system in thermal equilibrium to be found in a particular microstate with energy <span class="math inline">\(E_1\)</span> compared to being in a microstate with energy <span class="math inline">\(E_2\)</span> is given by an exponential function of minus the energy of such microstate divided by <span class="math inline">\(k_BT\)</span>, the thermal energy. To give concrete examples of what a microstate can look like, Fig. 2(A) shows three molecular systems relevant for biology. On the first example we have the classic ligand-receptor binding problem; here we imagine that a solution can be discretized into a series of small boxes. In each of these boxes one and only one ligand molecule can fit in. In principle we can list all possible spatial arrangements of ligands. We could then calculate the relative likelihood of finding the system in any of the configurations as long as we can assign an energy value to each of them. The second example focuses on ligand-gated ion channels. In this particular system we care about the state of the ion channel itself–either open or close–and the binding configuration of the ligands. If the channel responds to the concentration of the ligand by changing its probability of gating, that is something we can calculate using equilibrium statistical mechanics. Finally, the third example shows different configurations of a small patch of cell membrane. All deformations of a membrane have energetic costs associated with them. So listing all possible membrane configurations we can calculate what is the most likely shape of a membrane given the forces and stresses acting on it.</p>
<p>The macroscopic states that we get to observe can then be thought of as a coarse-graining of many microstates into a single macrostate. For example, in the case of the ligand-receptor binding, we rarely would care about the specific position of all the ligand molecules in the solution. What we would be interested in is whether or not the ligand is bound to the receptor. We can therefore define as our “macrostate” the particular configuration of the receptor as schematically shown in Fig. 2(B).</p>
<figure>
<img src="ch1_fig02.png" id="fig:ch1_fig02" data-short-caption="Boltzmann&#39;s law and the definition of a micro and macrostate" alt="Figure 2: Boltzmann’s law and the definition of a micro and macrostate. (A) Top panel: ligand-receptor binding microstates. Middle panel: ligand-gated ion channel microstates. Bottom panel: membrane patch deformations. (B) Schematic of the definition of a “macrostate.” In the ligand-receptor binding problem we ignore the spatial configuration of all ligand molecules, and focus on the binding state of the receptor." /><figcaption aria-hidden="true">Figure 2: <strong>Boltzmann’s law and the definition of a micro and macrostate.</strong> (A) Top panel: ligand-receptor binding microstates. Middle panel: ligand-gated ion channel microstates. Bottom panel: membrane patch deformations. (B) Schematic of the definition of a “macrostate.” In the ligand-receptor binding problem we ignore the spatial configuration of all ligand molecules, and focus on the binding state of the receptor.</figcaption>
</figure>
<p>If we want to know the likelihood of finding a particular system in any specific configuration Boltzmann’s law (Eq. <span class="math inline">\(\ref{eq:boltzmann_law}\)</span>) is then telling us a protocol we must follow:</p>
<ol type="1">
<li><p>Enumerate all possible microstates in which the system can be found.</p></li>
<li><p>Compute the energy of each of these microstates.</p></li>
<li><p>Define the “macrostate” we care about by grouping all microstaes that belong to the same macrostate.</p></li>
<li><p>Compute the Boltzmann factor by exponentiating minus the energy divided by the thermal energy.</p></li>
</ol>
<p>To see this protocol in action let us apply it to the calculation of <span class="math inline">\(p_{\text{bound}}\)</span>, the probability of finding an RNAP bound to the promoter. We will go through each of the steps on the protocol and build up the “unrealistic simplifications” that will allow us to make this calculation.</p>
<p><strong>1. Enumerate possible microstates.</strong> We begin by making a drastic coarse-graining of the bacterial genome. For us a genome is simply made out of boxes where the RNAP can bind. We imagine that there is a single site where RNAP can bind specifically–the promoter of interest. There are also <span class="math inline">\(N_{NS} \approx 5\times 10^6\)</span> non-specific binding sites, one per basepair (bp) in the genome. We ignore the fact that the RNAP footprint when it binds to the genome is roughly 30 bp. This assumption is valid if the number of available RNAP molecules is much smaller than the number of non-specific binding sites since it is extremely unlikely that by pure chance two RNAPs would fall next to each other. We also ignore the possibility of RNAP not being bound to the genome. Experimental evidence with mini-cells in which a mutant <em>E. coli</em> that sheds vesicles without segregating a chromosome support this assumption <span class="citation" data-cites="Bintu2005"> [<a href="#ref-Bintu2005" role="doc-biblioref">11</a>]</span>. The exercise then consists of choosing at random one box for each of the <span class="math inline">\(P\)</span> polymerase available to bind. Fig. 3 shows in the first column two possible configurations of our coarse-grained genome.</p>
<p><strong>2. Compute the energy for each microstate.</strong> Let us analyze first the case where all <span class="math inline">\(P\)</span> RNAP molecules are bound non-specifically to the genome. For simplicity we assume that RNAP binds to all <span class="math inline">\(N_{NS}\)</span> non-specific binding sites with the same affinity. We assign this energy to be <span class="math inline">\(\varepsilon_P^{(NS)}\)</span>. This assumption could be relaxed as we explored in <span class="citation" data-cites="Phillips2019"> [<a href="#ref-Phillips2019" role="doc-biblioref">16</a>]</span>, but for now we don’t have to worry about this complication. For a statistical mechanician the assignment of binding energies does not come from some quantum first-principled calculation or anything similar. We simply label the interaction of the RNAP and the rest of the genome with a single value, <span class="math inline">\(\varepsilon_P^{(NS)}\)</span>, that coarse-grains all of the hydrogen bonds and other effects that go into this physical process. Since we have <span class="math inline">\(P\)</span> such polymerases bound non specifically, the energy of any state with a similar configuration is then <span class="math inline">\(P \varepsilon_P^{(NS)}\)</span> as shown in Fig. 3 second column, top row.</p>
<p><strong>3. Define the “macrostate” we care about.</strong> In a sense when we speak about macrostate, it does not necessarily mean something that we can macroscopically observe. What it means is that we group together–a form of coarse-graining–a bunch of states that we take to be functionally equivalent as shown in Fig. 2(B). In our case we only care about whether or not the RNAP is bound to our promoter of interest. The configuration of the rest of the background sites is irrelevant to our question. What this means in practice is that we must compute the degeneracy or multiplicity of our state. In other words, for the <em>specific</em> state shown in the first column/top row of Fig. 3 we know its Boltzmann weight. Eq. <span class="math inline">\(\ref{eq:boltzmann_law}\)</span> tells us that the probability of this particular configuration takes the form <span class="math display">\[
P_{\text{state}} \propto e^{-\beta P \varepsilon_P^{(NS)}},
\]</span> since the <span class="math inline">\(P\)</span> RNAP molecules are bound non specifically. But every single arrangement in which all RNAPs are bound non-specifically has the exact same Boltzmann weight. The question then becomes: how many of such microstates can the system exist in? This is a combinatorics question of the form: in how many different ways can I arrange <span class="math inline">\(P\)</span> molecules into <span class="math inline">\(N_{NS}\)</span> boxes? Which of course the answer is <span class="math display">\[
\text{\# states with all RNAPs bound non-specifically} = 
\frac{N_{NS}!}{P!(N_{NS} - P)!},
\]</span> as shown in the third column of Fig. 3. This multiplicity can be simplified if we consider that <span class="math inline">\(N_{NS} \gg P\)</span>. To more easily visualize how to simplify this let us for a second assume <span class="math inline">\(N_{NS} = 100\)</span> and <span class="math inline">\(P = 3\)</span>. Given the definition of factorials this means that <span class="math display">\[
\frac{N_{NS}!}{(N_{NS} - P)!} = 
\frac{100\cdot 99\cdot 98\cdots97\cdots 2\cdot 1}{97\cdots2\cdot 1} = 
100\cdot 99\cdot 98.
\]</span> Given this result I can simply state that <span class="math inline">\(100\cdot 99\cdot 98 \approx 100^3\)</span> without making such a big mistake. Imagine <span class="math inline">\(N_{NS}\)</span> is in the order of <span class="math inline">\(10^6\)</span>, then the error would become even smaller. That is why, as shown in Fig. 3 third column, we can approximate <span class="math display">\[
\frac{N_{NS}!}{P!(N_{NS} - P)!} \approx \frac{N_{NS}^P}{P!}, \;
\text{for }N_{NS} \gg P.
\]</span> For our other “macrostate” we have the case where only one out of the <span class="math inline">\(P\)</span> RNAPs is bound specifically for the promoter. The way to realize this state is then given by <span class="math display">\[
\small
\text{\# states with one RNAP bound specifically} = 
\frac{N_{NS}!}{(P - 1)!(N_{NS} - (P - 1))!} \approx
\frac{N_{NS}^{P-1}}{(P-1)!}.
\]</span></p>
<p><strong>4. Compute the Boltzmann Factor.</strong> The last step in the protocol is simply to follow the recipe indicated by Eq. <span class="math inline">\(\ref{eq:boltzmann_law}\)</span> where we exponentiate the energy, with the caveat that this time we multiply by the multiplicity that we just mentioned since we are lumping together all microstates into a single functional macrostate. So the boltzmann weight for the unbound <span class="math inline">\(\rho_{\text{unbound}}\)</span> macrostate is given by <span class="math display">\[
\rho_{\text{unbound}} = \frac{N_{NS}^P}{P!}
e^{-\beta P \varepsilon_P^{(NS)}}.
\]</span> For the bound state we have <span class="math display">\[
\rho_{\text{bound}} = \frac{N_{NS}^{P-1}}{(P-1)!}
e^{-\beta \left(\varepsilon_P^{(S)} +  (P - 1) \varepsilon_P^{(NS)}\right)}.
\]</span> For reasons that will become clear later in this chapter once we work with the entropy and derive the Boltzmann distribution, we know that to compute the probability of a specific microstate (or a macrostate) we simply take the Boltzmann weight of the microstate and divide by the <em>sum</em> of all of the other Boltzmann weights of the states available to the system. Therefore, to calculate <span class="math inline">\(p_{\text{bound}}\)</span> we compute <span class="math display">\[
p_{\text{bound}} = 
\frac{\rho_{\text{bound}}}{\rho_{\text{unbound}} + \rho_{\text{bound}}}.
\]</span> Substituting the Boltzmann weights we derived we find <span class="math display">\[
p_{\text{bound}} = 
\frac{
\frac{N_{NS}^{P-1}}{(P-1)!}
e^{-\beta \left(\varepsilon_P^{(S)} +  (P - 1) \varepsilon_P^{(NS)}\right)}
}{
\frac{N_{NS}^{P-1}}{(P-1)!}
e^{-\beta \left(\varepsilon_P^{(S)} +  (P - 1) \varepsilon_P^{(NS)}\right)}
+ 
\frac{N_{NS}^P}{P!}
e^{-\beta P \varepsilon_P^{(NS)}},
}
\]</span> an algebraic nightmare. We can simplify this expression enormously multiplying numerator and denominator by <span class="math inline">\(\rho_{\text{unbound}}^{-1}\)</span>. Upon simplification we find the neat expression <span class="math display">\[
p_{\text{bound}} = 
\frac{
    \frac{P}{N_{NS}} e^{-\beta \Delta \varepsilon_P}
}{
    1 + \frac{P}{N_{NS}} e^{-\beta \Delta \varepsilon_P}
},
\label{eq:pbound_unreg}
\]</span> where <span class="math inline">\(\Delta\varepsilon_P \equiv \varepsilon_P^{(S)} - \varepsilon_P^{(NS)}\)</span>. This simple expression, known as the Langmuir isothermal binding curve tells us that the more RNAPs available (larger <span class="math inline">\(P\)</span>), or the stronger the promoter is (more negative <span class="math inline">\(\Delta\varepsilon_P\)</span>), the more likely it is to find the promoter bound by an RNAP, and according to Eq. <span class="math inline">\(\ref{eq:mRNA_prod}\)</span>, the higher the mRNA production. In the next section we connect this model to experimental measurements.</p>
<figure>
<img src="ch1_fig03.png" id="fig:ch1_fig03" data-short-caption="Statistical Mechanics protocol for RNAP binding" alt="Figure 3: Statistical Mechanics protocol for RNAP binding. On a discretized genome we follow the statistical mechanics protocol to compute the Boltzmann weight of each of the relevant microstates. The P available RNAPs are assumed to have two binding configurations: One specific binding to the promoter of interest (with energy \varepsilon_P^{(S)}) and non-specific to any of the N_{NS} non-specific binding sites (with energy \varepsilon_P^{(NS)})." /><figcaption aria-hidden="true">Figure 3: <strong>Statistical Mechanics protocol for RNAP binding.</strong> On a discretized genome we follow the statistical mechanics protocol to compute the Boltzmann weight of each of the relevant microstates. The <span class="math inline">\(P\)</span> available RNAPs are assumed to have two binding configurations: One specific binding to the promoter of interest (with energy <span class="math inline">\(\varepsilon_P^{(S)}\)</span>) and non-specific to any of the <span class="math inline">\(N_{NS}\)</span> non-specific binding sites (with energy <span class="math inline">\(\varepsilon_P^{(NS)}\)</span>).</figcaption>
</figure>
<h3 id="figure-1-theory-in-gene-regulation">Figure 1 theory in gene regulation</h3>
<p>We began this section with a simple model for the dynamics of mRNA production and degradation. We then expanded our model to deconvolve the production term into the rate at which mRNA is produced by RNAP, and the probability of finding such RNAP bound to the promoter. To calculate this probability we used the statistical mechanics protocol, which culminated in Eq. <span class="math inline">\(\ref{eq:pbound_unreg}\)</span>. So far we are missing two important steps in our logical construction that will lead us to specific quantitative predictions that we can test experimentally:</p>
<ol type="1">
<li><p>The inclusion of a regulatory scheme via a transcriptional repressor.</p></li>
<li><p>The connection of the model with experimentally accessible quantities.</p></li>
</ol>
<p>As hinted at earlier, for a transcriptional repressor we imagine that the effect that the repressor has on the regulation of the gene acts only through changes in <span class="math inline">\(p_{\text{bound}}\)</span>. In order to include the regulation then we add a series of microstates in which rather than having only <span class="math inline">\(P\)</span> RNAP molecules to bind the genome, we also have <span class="math inline">\(R\)</span> repressors that can bind specifically and non-specifically. Through the same statistical mechanics protocol as for the previous case we can arrive to the Boltzmann weights shown for the three “macrostates” in Fig. 4(A). For the regulated case we have that the probability of the promoter being bound by an RNAP takes the form <span class="math display">\[
p_{\text{bound}}(R &gt; 0) = 
\frac{
    \frac{P}{N_{NS}} e^{-\beta \Delta \varepsilon_P}
}{
    1 
    + \frac{P}{N_{NS}} e^{-\beta \Delta \varepsilon_P}
    + \frac{R}{N_{NS}} e^{-\beta \Delta \varepsilon_R}
},
\label{eq:pbound_reg}
\]</span> where <span class="math inline">\(\Delta\varepsilon_R\)</span> is the binding energy difference between the repressor binding to a specific binding site and a non-specific one. Although interesting and insightful, the quantities we have derived so far do not have an immediate <strong>quantitative</strong> prediction we can connect with experimental measurements. For example, for the regulated case, the steady state mRNA count takes the form <span class="math display">\[
m_{ss}(R &gt; 0) = 
\frac{r_m}{\gamma_m} p_{\text{bound}}(R &gt; 0) = 
\frac{r_m}{\gamma_m}
\frac{
    \frac{P}{N_{NS}} e^{-\beta \Delta \varepsilon_P}
}{
    1 
    + \frac{P}{N_{NS}} e^{-\beta \Delta \varepsilon_P}
    + \frac{R}{N_{NS}} e^{-\beta \Delta \varepsilon_R}
}.
\]</span> Determining <span class="math inline">\(r_m\)</span> or <span class="math inline">\(\gamma_m\)</span> directly from experiments, although possible, it represents an enormous technical challenge. A convenient metric we can use instead is what we call the fold-change in gene expression. Fig. 4(B) shows a schematic representation of what we mean by the fold-change. This ratiometric quantity normalizes the expression level of a gene with regulation given by a transcriptional repressor by the expression level of the same gene in the absence of the regulation–via a knock-out of the repressor gene for example. Mathematically this is defined as <span class="math display">\[
\text{fold-change} \equiv \frac{m_{ss}(R &gt; 0)}{m_{ss}(R = 0)}.
\]</span> This expression is convenient because upon taking the ratio of these steady-state mRNA counts the ratio <span class="math inline">\(r_m / \gamma_m\)</span> drops out of the equation. All we are left is then the ratio of the <span class="math inline">\(p_{\text{bound}}\)</span>s <span class="math display">\[
\text{fold-change} = \frac{p_{\text{bound}}(R &gt; 0)}{p_{\text{bound}}(R = 0)}.
\]</span> Substitutin Eqs. <span class="math inline">\(\ref{eq:pbound_unreg}\)</span> and <span class="math inline">\(\ref{eq:pbound_reg}\)</span> results in <span class="math display">\[
\text{fold-change} = 
\frac{
    1 
    + \frac{P}{N_{NS}} e^{-\beta \Delta \varepsilon_P}
}{
    1 
    + \frac{P}{N_{NS}} e^{-\beta \Delta \varepsilon_P}
    + \frac{R}{N_{NS}} e^{-\beta \Delta \varepsilon_R}
}.
\]</span> To simplify this equation further we appeal to some experimental understanding of the bacterial proteome composition <span class="citation" data-cites="Bremer1996 Schmidt2016 Grigorova2006"> [<a href="#ref-Bremer1996" role="doc-biblioref">17</a>–<a href="#ref-Grigorova2006" role="doc-biblioref">19</a>]</span>. RNAP copy number in <em>E. coli</em> is of the order <span class="math inline">\(P \sim 10^3-10^4\)</span> <span class="citation" data-cites="Grigorova2006"> [<a href="#ref-Grigorova2006" role="doc-biblioref">19</a>]</span>. The binding affinity of these promoters is of the order <span class="math inline">\(\Delta\varepsilon_P \sim -2\pm 1\;k_BT\)</span> <span class="citation" data-cites="Bintu2005"> [<a href="#ref-Bintu2005" role="doc-biblioref">11</a>]</span>. Along with the value of <span class="math inline">\(N_{NS}\sim 10^6\)</span> This results in <span class="math display">\[
\frac{P}{N_{NS}} e^{-\beta \Delta \varepsilon_P} \approx
\frac{10^3}{10^6}e^{2.3} \approx
\frac{10^3 \cdot 10}{10^6} \approx 10^{-2} \ll 1,
\]</span> the so-called weak-promoter approximation For the repressor we have that most repressors in <em>E. coli</em> are in the order of <span class="math inline">\(R \sim 10\)</span> <span class="citation" data-cites="Schmidt2016"> [<a href="#ref-Schmidt2016" role="doc-biblioref">18</a>]</span>. Their binding affinities take values between <span class="math inline">\(\Delta\varepsilon_R \sim -15 \pm 5\; k_BT\)</span> <span class="citation" data-cites="Bintu2005"> [<a href="#ref-Bintu2005" role="doc-biblioref">11</a>]</span>. These numerical values then give <span class="math display">\[
\frac{R}{N_{NS}} e^{-\beta \Delta \varepsilon_R} \approx
\frac{10}{10^6}e^{15} \approx
\frac{10 \cdot 10^6}{10^6} \approx 10.
\]</span> If we implement these approximations we can justify simplifying the fold-change equation to take the form <span class="math display">\[
\text{fold-change} \approx
\left(
    1 + \frac{R}{N_{NS}} e^{-\beta\Delta\varepsilon_R}
\right)^{-1}.
\label{eq:fc}
\]</span> As shown in Fig. 4(C) this expression points directly at two experimental knobs that we can tune using molecular biology. We can modify the number of repressors by changing the ribosomal binding site sequence (RBS) of the repressor gene <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">20</a>]</span>. What that means is that with a sequence-dependent manner the ribosome translates mRNAs according to a specific region of the gene known as the RBS <span class="citation" data-cites="Chen2013"> [<a href="#ref-Chen2013" role="doc-biblioref">21</a>]</span>. Furthermore, we can change the affinity of the repressor for its binding site by mutating the binding site itself <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">20</a>]</span>. Fig. 4(D) shows predictions of Eq. <span class="math inline">\(\ref{eq:fc}\)</span> for different binding energies.</p>
<figure>
<img src="ch1_fig04.png" id="fig:ch1_fig04" data-short-caption="Figure 1 theory in transcriptional regulation" alt="Figure 4: Figure 1 theory in transcriptional regulation. (A) States and (normalized) weights for the simple repression motif. The promoter can be found in three states: 1) empty, 2) bound by an RNAP, 3) bound by a repressor. The same statistical mechanics protocol as in Fig. 3 can be used to derive the weights. (B) Schematic of the experimental determination of the fold-change in gene expression. The expression level of a regulated strain is normalized by the expression level of a strain with a knock-out of the repressor. (C) Experimentally accessible knobs predicted from the theoretical model. The number of transcription factors can be tuned by changing the amount of protein produced per mRNA. The binding energy of the repressor can be tuned by mutating the basepairs in the binding site. (D) Fold-change as a function of the repressor copy number for different binding energies." /><figcaption aria-hidden="true">Figure 4: <strong>Figure 1 theory in transcriptional regulation.</strong> (A) States and (normalized) weights for the simple repression motif. The promoter can be found in three states: 1) empty, 2) bound by an RNAP, 3) bound by a repressor. The same statistical mechanics protocol as in Fig. 3 can be used to derive the weights. (B) Schematic of the experimental determination of the fold-change in gene expression. The expression level of a regulated strain is normalized by the expression level of a strain with a knock-out of the repressor. (C) Experimentally accessible knobs predicted from the theoretical model. The number of transcription factors can be tuned by changing the amount of protein produced per mRNA. The binding energy of the repressor can be tuned by mutating the basepairs in the binding site. (D) Fold-change as a function of the repressor copy number for different binding energies.</figcaption>
</figure>
<h2 id="entropy-information-and-the-math-behind-the-bit">Entropy, information, and the math behind the bit</h2>
<h2 class="unnumbered" id="references">References</h2>
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-Schrodinger1992" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[1] </div><div class="csl-right-inline">E. Schrödinger, <em>What Is Life?: With Mind and Matter and Autobiographical Sketches</em> (Cambridge University Press, 1992).</div>
</div>
<div id="ref-Phillips2021" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[2] </div><div class="csl-right-inline">R. Phillips, <em><span class="nocase">Schr<span class="nocase">ö</span>dinger’ <span>“What is Life?”</span> at 75</span></em>, ArXiv 1 (2021).</div>
</div>
<div id="ref-Cronin2016" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[3] </div><div class="csl-right-inline">L. Cronin and S. I. Walker, <em><span class="nocase">Beyond prebiotic chemistry</span></em>, Science (80-. ). <strong>352</strong>, 1174 (2016).</div>
</div>
<div id="ref-Davies2019" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[4] </div><div class="csl-right-inline">P. Davies, <em>The Demon in the Machine: How Hidden Webs of Information Are Solving the Mystery of Life</em> (University of Chicago Press, 2019).</div>
</div>
<div id="ref-Adami2016" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[5] </div><div class="csl-right-inline">C. Adami, <em><span class="nocase">What is information?</span></em>, Philos. Trans. R. Soc. A Math. Phys. Eng. Sci. <strong>374</strong>, 20150230 (2016).</div>
</div>
<div id="ref-Taylor2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[6] </div><div class="csl-right-inline">S. F. Taylor, N. Tishby, and W. Bialek, <em><span class="nocase">Information and fitness</span></em>, ArXiv (2007).</div>
</div>
<div id="ref-Bialek2012" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[7] </div><div class="csl-right-inline">W. Bialek, <em>Biophysics: Searching for Principles</em> (Princeton University Press, 2012).</div>
</div>
<div id="ref-Browning2004" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[8] </div><div class="csl-right-inline">D. F. Browning and S. J. W. Busby, <em><span class="nocase">The regulation of bacterial transcription initiation</span></em>, Nature Reviews Microbiology <strong>2</strong>, 57 (2004).</div>
</div>
<div id="ref-Ireland2020" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[9] </div><div class="csl-right-inline">W. T. Ireland, S. M. Beeler, E. Flores-Bautista, N. S. McCarty, T. Röschinger, N. M. Belliveau, M. J. Sweredoski, A. Moradian, J. B. Kinney, and R. Phillips, <em>Deciphering the Regulatory Genome of Escherichia Coli, One Hundred Promoters at a Time</em>, Elife <strong>9</strong>, e55308 (2020).</div>
</div>
<div id="ref-Ackers1982" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[10] </div><div class="csl-right-inline">G. K. Ackers, A. D. Johnson, and M. A. Shea, <em><span class="nocase">Quantitative model for gene regulation by lambda phage repressor.</span></em>, Proceedings of the National Academy of Sciences <strong>79</strong>, 1129 (1982).</div>
</div>
<div id="ref-Bintu2005" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[11] </div><div class="csl-right-inline">L. Bintu, N. E. Buchler, H. G. Garcia, U. Gerland, T. Hwa, J. Kondev, T. Kuhlman, and R. Phillips, <em><span class="nocase">Transcriptional regulation by the numbers: applications.</span></em>, Curr. Opin. Genet. Dev. <strong>15</strong>, 125 (2005).</div>
</div>
<div id="ref-Kuhlman2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[12] </div><div class="csl-right-inline">T. Kuhlman, Z. Zhang, M. H. Saier, and T. Hwa, <em><span class="nocase">Combinatorial transcriptional control of the lactose operon of <em>Escherichia coli</em></span>.</em>, Proceedings of the National Academy of Sciences <strong>104</strong>, 6043 (2007).</div>
</div>
<div id="ref-Strogatz2018" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[13] </div><div class="csl-right-inline">S. H. Strogatz, <em>Nonlinear Dynamics and Chaos with Student Solutions Manual: With Applications to Physics, Biology, Chemistry, and Engineering</em> (CRC press, 2018).</div>
</div>
<div id="ref-Dill2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[14] </div><div class="csl-right-inline">K. Dill and S. Bromberg, <em>Molecular Driving Forces: Statistical Thermodynamics in Biology, Chemistry, Physics, and Nanoscience</em> (Garland Science, 2010).</div>
</div>
<div id="ref-Feynman1965" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[15] </div><div class="csl-right-inline">R. P. Feynman, R. B. Leighton, and M. Sands, <em>The Feynman Lectures on Physics; Vol. I</em>, American Journal of Physics <strong>33</strong>, 750 (1965).</div>
</div>
<div id="ref-Phillips2019" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[16] </div><div class="csl-right-inline">R. Phillips, N. M. Belliveau, G. Chure, H. G. Garcia, M. Razo-Mejia, and C. Scholes, <em><span class="nocase">Figure 1 Theory Meets Figure 2 Experiments in the Study of Gene Expression</span></em>, Annual Review of Biophysics <strong>48</strong>, 121 (2019).</div>
</div>
<div id="ref-Bremer1996" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[17] </div><div class="csl-right-inline">H. Bremer and P. P. Dennis, <em><span class="nocase">Modulation of Chemical Composition and Other Parameters of the Cell by Growth Rate</span></em>, (n.d.).</div>
</div>
<div id="ref-Schmidt2016" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[18] </div><div class="csl-right-inline">A. Schmidt, K. Kochanowski, S. Vedelaar, E. Ahrne, B. Volkmer, L. Callipo, K. Knoops, M. Bauer, R. Aebersold, and M. Heinemann, <em><span class="nocase">The quantitative and condition-dependent Escherichia coli proteome</span></em>, Nat Biotechnol <strong>34</strong>, 104 (2016).</div>
</div>
<div id="ref-Grigorova2006" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[19] </div><div class="csl-right-inline">I. L. Grigorova, N. J. Phleger, V. K. Mutalik, and C. A. Gross, <em><span class="nocase">Insights into transcriptional regulation and sigma competition from an equilibrium model of RNA polymerase binding to DNA.</span></em>, Proceedings of the National Academy of Sciences <strong>103</strong>, 5332 (2006).</div>
</div>
<div id="ref-Garcia2011c" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[20] </div><div class="csl-right-inline">H. G. Garcia and R. Phillips, <em><span class="nocase">Quantitative dissection of the simple repression input-output function.</span></em>, Proceedings of the National Academy of Sciences <strong>108</strong>, 12173 (2011a).</div>
</div>
<div id="ref-Chen2013" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[21] </div><div class="csl-right-inline">Y.-J. Chen, P. Liu, A. A. K. Nielsen, J. A. N. Brophy, K. Clancy, T. Peterson, and C. A. Voigt, <em><span class="nocase">Characterization of 582 natural and synthetic terminators and quantification of their design constraints</span></em>, Nat. Methods <strong>10</strong>, 659 (2013).</div>
</div>
</div>
                    </div><!-- /.content -->
                </div><!-- /.col -->
                <div class="col-md-4 col-md-offset-1">
                    <div class="sections-list-wrapper">
                        <div class="sections-list js-sections js-affix js-scrollspy hidden-xs hidden-sm"><span
                                style="font-size: 14pt">Table of Contents</span>
                            <hr>
                        </div><!-- /.sections-list -->
                    </div>
                </div><!-- /.col -->
            </div><!-- /.row -->
        </div><!-- /.container -->
    </div><!-- /.section -->
    
    <div class="section section--grey">
        <div class="container">
            <div class="row">
                <div class="col-md-7">
                    <div id="disqus_thread"></div>
<script>
	/**
	 *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
	 *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables
	 */
	 /*
	var disqus_config = function () {
	this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
		this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
	};
	*/
	(function() {
		var d = document, s = d.createElement('script');

		s.src = 'https://mrazomej-phd.disqus.com/embed.js';

		s.setAttribute('data-timestamp', +new Date());
		(d.head || d.body).appendChild(s);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>

                </div><!-- /.col -->
            </div><!-- /.row -->
        </div><!-- /.container -->
    </div><!-- /.section -->
    
    <div class="js-footer-area">
        
        <nav class="page-nav">
            <div class="container">
                <div class="row">
                    <div class="col-xs-12">
                        
                        <a href="https://mrazomej.github.io/phd/abstract"
                            class="page-nav__item page-nav__item--prev">
                            <i class="icon icon--arrow-left"></i>
                            Abstract
                        </a><!-- /.page-nav__item -->
                        
                        
                        <a href="https://mrazomej.github.io/phd/chapter_02"
                            class="page-nav__item page-nav__item--next">
                            Chapter 2
                            <i class="icon icon--arrow-right"></i>
                        </a><!-- /.page-nav__item -->
                        
                    </div><!-- /.col -->
                </div><!-- /.row -->
            </div><!-- /.container -->
        </nav><!-- /.page-nav -->
        
        
        
	<footer class="site-footer">
		<div class="container">
			<div class="row">
				<div class="col-sm-6">
					
						<a href="https://mrazomej.github.io/phd//" class="site-footer__logo">Manuel Razo-Mejia | PhD Thesis</a>
					
					
						<hr>
						<p class="site-footer__copyright">Copyright &copy; 2021. - Manuel Razo-Mejia <br>All rights reserved.</p>
					
				</div><!-- /.col -->
				
					<div class="col-sm-6 align-right">
						<ul class="social-list">
							
								<li>
									<a href="https://twitter.com/mrazomej" target="_blank" class="social-list__item social-list__item--twitter">
										<i class="icon icon--twitter"></i>
									</a>
								</li>
							
						</ul><!-- /.social-list -->
					</div><!-- /.col -->
				
			</div><!-- /.row -->
		</div><!-- /.container -->
	</footer><!-- /.site-footer -->


<script src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/jquery.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/bootstrap/affix.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/bootstrap/scrollspy.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/matchHeight.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/scripts.min.js"></script>


<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-109526846-4', 'auto');
	ga('send', 'pageview');
</script>



    </div><!-- /.js-footer-area -->
</body>

</html>