<!DOCTYPE html>
<html>
	<head>
		<meta charset="UTF-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, minimum-scale=1.0">
<title>Chapter IV</title>


<script type="text/x-mathjax-config">
	MathJax.Hub.Config({
    "HTML-CSS" : {
        availableFonts : ["STIX"],
        preferredFont : "Gyre Pagella",
        webFont : "STIX-Web",
        imageFont : null
    },
	  TeX: {
		equationNumbers: {
		  autoNumber: "AMS"
		}
	  },
	  tex2jax: {
		inlineMath: [['$','$']], 
		displayMath: [['$$','$$']],
		processEscapes: true,
	  }
	});
  </script>
  <style>
	.eqnos { display: inline-block; position: relative; width: 100%; }
	.eqnos br { display: none; }
	.eqnos-number { position: absolute; right: 0em; top: 50%; line-height: 0; }
  </style>
  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script id="MathJax-script" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>
  

<link rel="icon" href="https://mrazomej.github.io/phd//favicon.ico" type="image/x-icon">
<link href="https://fonts.googleapis.com/css?family=Nanum+Myeongjo&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Alegreya|Lora&display=swap" rel="stylesheet">

<link rel="stylesheet" href="https://mrazomej.github.io/phd//doks-theme/assets/css/style.css">

	</head>
	<body class="grey" data-spy="scroll" data-target=".js-scrollspy">
		


	<header class="site-header">
		<div class="container">
			<div class="row">
				<div class="col-xs-12">
					
						<a href="https://mrazomej.github.io/phd//" class="site-header__logo"><i class="icon icon--chevron-left"></i> Home</a>
					
					
				</div><!-- /.col -->
			</div><!-- /.row -->
		</div><!-- /.container -->
	</header><!-- /.site-header -->


		<div class="hero-subheader" style="background-image: url(https://mrazomej.github.io/phd//doks-theme/assets/images/alaska.jpg); background-position: center; object-fit: cover; width:100%;">

			<div class="container" style="background-color: rgba(0,0,0, 0.75); border:0.75px solid white;">
				<div class="row">
					<div class="col-md-7">
						<div class="align-container" data-mh>
							<div class="align-inner">
								
									<h2 class="hero-subheader__title">Chapter IV</h2>
								
								
									<h1 class="hero-subheader__title">Supporting Information for Tuning Transcriptional Regulation through Signaling: A Predictive Theory of Allosteric Induction
</h1>
								
								
								<!-- 
									
										<a href="https://mrazomej.github.io/phd/chapter_03" class="btn btn--dark btn--rounded btn--w-icon btn--w-icon-left">
											<i class="icon icon--arrow-left"></i>	
										</a>
									
									
										<a href="https://mrazomej.github.io/phd/chapter_05" class="btn btn--dark btn--rounded btn--w-icon">
											<i class="icon icon--arrow-right"></i>
										</a>
									
								 -->
							</div><!-- /.align-inner -->
						</div><!-- /.align-container -->
					</div><!-- /.col -->
					
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.hero-subheader -->
		
				<nav class="page-nav">
					<div class="container" >
						<div class="row" >
							<div class="col-xs-12">
								
									<a href="https://mrazomej.github.io/phd/chapter_03" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter 3
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://mrazomej.github.io/phd/chapter_05" class="page-nav__item page-nav__item--next">
										Chapter 5
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			

		<div class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-7">
						<div class="content">
							<hr/>
<p>A version of this chapter originally appeared as Razo-Mejia, M.†, Barnes, S.L.†, Belliveau, N.M.†, Chure, G.†, Einav, T.†, Lewis, M., and Phillips, R. (2018). Tuning transcriptional regulation through signaling: A predictive theory of allosteric induction. Cell Systems 6, 456-469.e10. DOI:https://doi.org/10.1016/j.cels.2018.02.004. † M.R.M, S.L.B, N.M.B, G.C., and T.E. contributed equally to this work from the theoretical underpinnings to the experimental design and execution. M.R.M, S.L.B, N.M.B, G.C, T.E., and R.P. wrote the paper. M.L. provided guidance and advice.</p>
<h2 id="abstract">Abstract</h2>
<p>Allosteric regulation is found across all domains of life, yet we still lack simple, predictive theories that directly link the experimentally tunable parameters of a system to its input-output response. To that end, we present a general theory of allosteric transcriptional regulation using the Monod-Wyman-Changeux model. We rigorously test this model using the ubiquitous simple repression motif in bacteria by first predicting the behavior of strains that span a large range of repressor copy numbers and DNA binding strengths and then constructing and measuring their response. Our model not only accurately captures the induction profiles of these strains but also enables us to derive analytic expressions for key properties such as the dynamic range and <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span>. Finally, we derive an expression for the free energy of allosteric repressors which enables us to collapse our experimental data onto a single master curve that captures the diverse phenomenology of the induction profiles.</p>
<h2 id="inferring-allosteric-parameters-from-previous-data">Inferring Allosteric Parameters from Previous Data</h2>
<p>The fold-change profile described by features three unknown parameters <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. In this section, we explore different conceptual approaches to determining these parameters. We first discuss how the induction titration profile of the simple repression constructs used in this paper are not sufficient to determine all three MWC parameters simultaneously, since multiple degenerate sets of parameters can produce the same fold-change response. We then utilize an additional data set from <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">1</a>]</span> to determine the parameter <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> = 4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span>, after which the remaining parameters <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> can be extracted from any induction profile with no further degeneracy.</p>
<h3 id="degenerate-parameter-values">Degenerate Parameter Values</h3>
<p>In this section, we discuss how multiple sets of parameters may yield identical fold-change profiles. More precisely, we shall show that if we try to fit the data in to the fold-change and extract the three unknown parameters (<span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>), then multiple degenerate parameter sets would yield equally good fits. In other words, this data set alone is insufficient to uniquely determine the actual physical parameter values of the system. This problem persists even when fitting multiple data sets simultaneously as in Section "".</p>
<p>In , we fit the <span class="math inline"><em>R</em> = 260</span> data by fixing <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> to the value shown on the <span class="math inline"><em>x</em></span>-axis and determine the parameters <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> given this constraint. We use the fold-change function but with <span class="math inline"><em>β</em><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> modified to the form <span class="math inline"><em>β</em><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub></span> in to account for the underlying assumptions used when fitting previous data (see Section "" for a full explanation of why this modification is needed).</p>
<p>The best-fit curves for several different values of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> are shown in . Note that these fold-change curves are nearly overlapping, demonstrating that different sets of parameters can yield nearly equivalent responses. Without more data, the relationships between the parameter values shown in represent the maximum information about the parameter values that can be extracted from the data. Additional experiments which independently measure any of these unknown parameters could resolve this degeneracy. For example, NMR measurements could be used to directly measure the fraction <span class="math inline">(1 + <em>e</em><sup> − <em>β</em><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></sup>)<sup> − 1</sup></span> of active repressors in the absence of IPTG <span class="citation" data-cites="Gardino2003 Boulton2016"> [<a href="#ref-Gardino2003" role="doc-biblioref">2</a>,<a href="#ref-Boulton2016" role="doc-biblioref">3</a>]</span>.</p>
<figure>
<img src="ch4_fig01.png" id="fig:ch4_fig01" data-short-caption="Multiple sets of parameters yield identical fold-change responses" alt="Figure 1: Multiple sets of parameters yield identical fold-change responses. (A) The data for the O2 strain (\Delta\varepsilon_{RA} = -13.9~k_BT) with R=260 in XXX was fit using XXX with n=2. \Delta\varepsilon_{AI} is forced to take on the value shown on the x-axis, while the K_A and K_I parameters are fit freely. (B) The resulting best-fit functions for several value of \Delta\varepsilon_{AI} all yield nearly identical fold-change responses." /><figcaption aria-hidden="true">Figure 1: <strong>Multiple sets of parameters yield identical fold-change responses</strong>. (A) The data for the O2 strain (<span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span>) with <span class="math inline"><em>R</em> = 260</span> in XXX was fit using XXX with <span class="math inline"><em>n</em> = 2</span>. <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is forced to take on the value shown on the <span class="math inline"><em>x</em></span>-axis, while the <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> parameters are fit freely. (B) The resulting best-fit functions for several value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> all yield nearly identical fold-change responses.</figcaption>
</figure>
<h3 id="computing-boldsymboldeltavarepsilon_ai">Computing <span class="math inline"><strong>Δ</strong><strong>ε</strong><sub><strong>A</strong><strong>I</strong></sub></span></h3>
<p>As shown in the previous section, the fold-change response of a single strain is not sufficient to determine the three MWC parameters (<span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>), since degenerate sets of parameters yield nearly identical fold-change responses. To circumvent this degeneracy, we now turn to some previous data from the <em>lac</em> system in order to determine the value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> in for the induction of the Lac repressor. Specifically, we consider two previous sets of work from: (1) <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span> and (2) <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">1</a>]</span>, both of which measured fold-change with the same simple repression system in the absence of inducer (<span class="math inline"><em>c</em> = 0</span>) but at various repressor copy numbers <span class="math inline"><em>R</em></span>. The original analysis for both data sets assumed that in the absence of inducer all of the Lac repressors were in the active state. As a result, the effective binding energies they extracted were a convolution of the DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and the allosteric energy difference <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> between the Lac repressor’s active and inactive states. We refer to this convoluted energy value as <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub></span>. We first disentangle the relationship between these parameters in Garcia and Phillips and then use this relationship to extract the value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> from the Brewster et al. dataset.</p>
<p>Garcia and Phillips determined the total repressor copy numbers <span class="math inline"><em>R</em></span> of different strains using quantitative Western blots. Then they measured the fold-change at these repressor copy numbers for simple repression constructs carrying the O1, O2, O3, and Oid <em>lac</em> operators integrated into the chromosome. These data were then fit to the following thermodynamic model to determine the repressor-DNA binding energies <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub></span> for each operator, <span id="eq:ch4_eq01"><span class="math display">$$
\text{fold-change}(c=0) = \left(
1+\frac{R}{N_{NS}}e^{-\beta \Delta\tilde{\varepsilon}_{RA}} \right)^{-1}.
\qquad(1)$$</span></span> Note that this functional form does not exactly match our fold-change in the limit <span class="math inline"><em>c</em> = 0</span>, <span id="eq:ch4_eq02"><span class="math display">$$
\text{fold-change}(c=0) = \left(
1+\frac{1}{1+e^{-\beta \Delta \varepsilon_{AI}}}
\frac{R}{N_{NS}}e^{-\beta \Delta\varepsilon_{RA}} \right)^{-1},
\qquad(2)$$</span></span> since it is missing the factor <span class="math inline">$\frac{1}{1+e^{-\beta \Delta\varepsilon_{AI}}}$</span> which specifies what fraction of repressors are in the active state in the absence of inducer, <span id="eq:ch4_eq03"><span class="math display">$$
\frac{1}{1+e^{-\beta \Delta\varepsilon_{AI}}} = p_A(0).
\qquad(3)$$</span></span> In other words, Garcia and Phillips assumed that in the absence of inducer, all repressors were active. In terms of our notation, the convoluted energy values <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub></span> extracted by Garcia and Phillips (namely, <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub> =  − 15.3 <em>k</em><sub><em>B</em></sub><em>T</em></span> for O1 and <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub> =  − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span> for Oid) represent <span id="eq:ch4_eq04"><span class="math display">$$
\beta \Delta\tilde{\varepsilon}_{RA} = 
\beta \Delta\varepsilon_{RA} - 
\log \left( \frac{1}{1 + e^{-\beta \Delta \varepsilon_{AI}}} \right).
\qquad(4)$$</span></span> Note that if <span class="math inline"><em>e</em><sup> − <em>β</em><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></sup> ≪ 1</span>, then nearly all of the repressors are active in the absence of inducer so that <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub> ≈ <em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>. In simple repression systems where we definitively know the value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>R</em></span>, we can use to determine the value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> by comparing with experimentally determined fold-change values. However, the binding energy values that we use from <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span> are effective parameters <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub></span>. In this case, we are faced with an undetermined system in which we have more variables than equations, and we are thus unable to determine the value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. In order to obtain this parameter, we must turn to a more complex regulatory scenario which provides additional constraints that allow us to fit for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>.</p>
<p>A variation on simple repression in which multiple copies of the promoter are available for repressor binding (for instance, when the simple repression construct is on plasmid) can be used to circumvent the problems that arise when using <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub></span>. This is because the behavior of the system is distinctly different when the number of active repressors <span class="math inline"><em>p</em><sub><em>A</em></sub>(0)<em>R</em></span> is less than or greater than the number of available promoters <span class="math inline"><em>N</em></span>. Repression data for plasmids with known copy number <span class="math inline"><em>N</em></span> allows us to perform a fit for the value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>.</p>
<p>To obtain an expression for a system with multiple promoters <span class="math inline"><em>N</em></span>, we follow <span class="citation" data-cites="Weinert2014"> [<a href="#ref-Weinert2014" role="doc-biblioref">5</a>]</span>, writing the fold-change in terms of the the grand canonical ensemble as <span id="eq:ch4_eq05"><span class="math display">$$
\text{fold-change} = \frac{1}{1 + \lambda_r e^{-\beta \Delta \varepsilon_{RA}}},
\qquad(5)$$</span></span> where <span class="math inline"><em>λ</em><sub><em>r</em></sub> = <em>e</em><sup><em>β</em><em>μ</em></sup></span> is the fugacity and <span class="math inline"><em>μ</em></span> is the chemical potential of the repressor. The fugacity will enable us to easily enumerate the possible states available to the repressor.</p>
<p>To determine the value of <span class="math inline"><em>λ</em><sub><em>r</em></sub></span>, we first consider that the total number of repressors in the system, <span class="math inline"><em>R</em><sub>tot</sub></span>, is fixed and given by <span id="eq:ch4_eq06"><span class="math display"><em>R</em><sub>tot</sub> = <em>R</em><sub><em>S</em></sub> + <em>R</em><sub><em>N</em><em>S</em></sub>,   (6)</span></span> where <span class="math inline"><em>R</em><sub><em>S</em></sub></span> represents the number of repressors specifically bound to the promoter and <span class="math inline"><em>R</em><sub><em>N</em><em>S</em></sub></span> represents the number of repressors nonspecifically bound throughout the genome. The value of <span class="math inline"><em>R</em><sub><em>S</em></sub></span> is given by <span id="eq:ch4_eq07"><span class="math display">$$
R_S = N \frac{\lambda_r 
e^{-\beta \Delta \varepsilon_{RA}}}
{1 + \lambda_r e^{-\beta \Delta \varepsilon_{RA}}},
\qquad(7)$$</span></span> where <span class="math inline"><em>N</em></span> is the number of available promoters in the cell. Note that in counting <span class="math inline"><em>N</em></span>, we do not distinguish between promoters that are on plasmid or chromosomally integrated provided that they both have the same repressor-operator binding energy <span class="citation" data-cites="Weinert2014"> [<a href="#ref-Weinert2014" role="doc-biblioref">5</a>]</span>. The value of <span class="math inline"><em>R</em><sub><em>N</em><em>S</em></sub></span> is similarly give by <span id="eq:ch4_eq08"><span class="math display">$$
R_{NS} = N_{NS} \frac{\lambda_r}{1 + \lambda_r},
\qquad(8)$$</span></span> where <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> is the number of non-specific sites in the cell (recall that we use <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub> = 4.6 × 10<sup>6</sup></span> for <em>E. coli</em>).</p>
<p>Substituting in into the modified yields the form <span id="eq:ch4_eq09"><span class="math display">$$
p_A(0) R_{\text{tot}} = 
\frac{1}{1 + e^{-\beta \Delta \varepsilon_{AI}}}
\left( N \frac{\lambda_r e^{-\beta \Delta \varepsilon_{RA}}}
{1 + \lambda_r e^{-\beta \Delta \varepsilon_{RA}}} + N_{NS} 
\frac{\lambda_r}{1 + \lambda_r} \right),
\qquad(9)$$</span></span> where we recall from that <span class="math inline">$\beta \Delta \varepsilon_{RA} = \beta \Delta \tilde\varepsilon_{RA} + \log{\left(\frac{1}{1 + e^{-\beta \Delta \varepsilon_{AI}}}\right)}.$</span> Numerically solving for <span class="math inline"><em>λ</em><sub><em>r</em></sub></span> and plugging the value back into yields a fold-change function in which the only unknown parameter is <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>.</p>
<p>With these calculations in hand, we can now determine the value of the <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> parameter. shows how different values of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> lead to significantly different fold-change response curves. Thus, analyzing the specific fold-change response of any strain with a known plasmid copy number <span class="math inline"><em>N</em></span> will fix <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. Interestingly, the inflection point of occurs near <span class="math inline"><em>p</em><sub><em>A</em></sub>(0)<em>R</em><sub>tot</sub> = <em>N</em></span> (as shown by the triangles in ), so that merely knowing where the fold-change response transitions from concave down to concave up is sufficient to obtain a rough value for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. We note, however, that for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> ≥ 5 <em>k</em><sub><em>B</em></sub><em>T</em></span>, increasing <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> further does not affect the fold-change because essentially every repressors will be in the active state in this regime. Thus, if the <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is in this regime, we can only bound it from below.</p>
<figure>
<img src="ch4_fig02.png" id="fig:ch4_fig02" data-short-caption="Fold-change of multiple identical genes." alt="Figure 2: Fold-change of multiple identical genes.. (A) In the presence of N=10 identical promoters, the fold-change XXX depends strongly on the allosteric energy difference \Delta\varepsilon_{AI} between the Lac repressor’s active and inactive states. The vertical dotted lines represent the number of repressors at which R_A = N for each value of \Delta \varepsilon_{AI}. (B) Using fold-change measurements from  [1] for the operators and gene copy numbers shown, we can determine the most likely value \Delta\varepsilon_{AI} = 4.5~k_BT for LacI." /><figcaption aria-hidden="true">Figure 2: <strong>Fold-change of multiple identical genes.</strong>. (A) In the presence of <span class="math inline"><em>N</em> = 10</span> identical promoters, the fold-change XXX depends strongly on the allosteric energy difference <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> between the Lac repressor’s active and inactive states. The vertical dotted lines represent the number of repressors at which <span class="math inline"><em>R</em><sub><em>A</em></sub> = <em>N</em></span> for each value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. (B) Using fold-change measurements from <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">1</a>]</span> for the operators and gene copy numbers shown, we can determine the most likely value <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> = 4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span> for LacI.</figcaption>
</figure>
<p>We now analyze experimental induction data for different strains with known plasmid copy numbers to determine <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. shows experimental measurements of fold-change for two O1 promoters with <span class="math inline"><em>N</em> = 64</span> and <span class="math inline"><em>N</em> = 52</span> copy numbers and one Oid promoter with <span class="math inline"><em>N</em> = 10</span> from <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">1</a>]</span>. By fitting these data to , we extracted the parameter value <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> = 4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span>. Substituting this value into shows that 99% of the repressors are in the active state in the absence of inducer and <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub> ≈ <em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>, so that all of the previous energies and calculations made by <span class="citation" data-cites="Garcia2011c Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">1</a>,<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span> were accurate.</p>
<h2 id="induction-of-simple-repression-with-multiple-promoters-or-competitor-sites">Induction of Simple Repression with Multiple Promoters or Competitor Sites</h2>
<p>We made the choice to perform all of our experiments using strains in which a single copy of our simple repression construct had been integrated into the chromosome. This stands in contrast to the methods used by a number of other studies <span class="citation" data-cites="Oehler1994 Setty2003 Oehler2006a Daber2009 Daber2011a Vilar2013 Shis2014 Sochor2014"> [<a href="#ref-Oehler1994" role="doc-biblioref">6</a>–<a href="#ref-Sochor2014" role="doc-biblioref">13</a>]</span>, in which reporter constructs are placed on plasmid, meaning that the number of constructs in the cell is not precisely known. It is also common to express repressor on plasmid to boost its copy number, which results in an uncertain value for repressor copy number. Here we show that our treatment of the MWC model has broad predictive power beyond the single-promoter scenario we explore experimentally, and indeed can account for systems in which multiple promoters compete for the repressor of interest. Additionally, we demonstrate the importance of having precise control over these parameters, as they can have a significant effect on the induction profile.</p>
<h3 id="chemical-potential-formulation-to-calculate-fold-change">Chemical Potential Formulation to Calculate Fold-Change</h3>
<p>In this section, we discuss a simple repression construct which we generalize in two ways from the scenario discussed in the text. First, we will allow the repressor to bind to <span class="math inline"><em>N</em><sub><em>S</em></sub></span> identical specific promoters whose fold-change we are interested in measuring, with each promoter containing a single repressor binding site (<span class="math inline"><em>N</em><sub><em>S</em></sub> = 1</span> in the main text). Second, we consider <span class="math inline"><em>N</em><sub><em>C</em></sub></span> identical competitor sites which do not regulate the promoter of interest, but whose binding energies are substantially stronger than non-specific binding (<span class="math inline"><em>N</em><sub><em>C</em></sub> = 0</span> in the main text). As in the main text, we assume that the rest of the genome contains <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> non-specific binding sites for the repressor. As in Appendix XXX, we can write the fold-change in the grand canonical ensemble as <span id="eq:ch4_eq10"><span class="math display">$$
\text{fold-change} = \frac{1}{1 + \lambda_r e^{-\beta \Delta \varepsilon_{RA}}},
\qquad(10)$$</span></span> where <span class="math inline"><em>λ</em><sub><em>r</em></sub></span> is the fugacity of the repressor and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> represents the energy difference between the repressor’s binding affinity to the specific operator of interest relative to the repressor’s non-specific binding affinity to the rest of the genome.</p>
<p>We now expand our definition of the total number of repressors in the system, <span class="math inline"><em>R</em><sub>tot</sub></span>, so that it is given by <span id="eq:ch4_eq11"><span class="math display"><em>R</em><sub>tot</sub> = <em>R</em><sub><em>S</em></sub> + <em>R</em><sub><em>N</em><em>S</em></sub> + <em>R</em><sub><em>C</em></sub>,   (11)</span></span> where <span class="math inline"><em>R</em><sub><em>S</em></sub></span>, <span class="math inline"><em>R</em><sub><em>N</em><em>S</em></sub></span>, and <span class="math inline"><em>R</em><sub><em>C</em></sub></span> represent the number of repressors bound to the specific promoter, a non-specific binding site, or to a competitor binding site, respectively. The value of <span class="math inline"><em>R</em><sub><em>S</em></sub></span> is given by <span id="eq:ch4_eq12"><span class="math display">$$
R_S = N_S \frac{\lambda_r e^{-\beta \Delta \varepsilon_{RA}}}
{1 + \lambda_r e^{-\beta \Delta \varepsilon_{RA}}},
\qquad(12)$$</span></span> where <span class="math inline"><em>N</em><sub><em>S</em></sub></span> is the number of specific binding sites in the cell. The value of <span class="math inline"><em>R</em><sub><em>N</em><em>S</em></sub></span> is similarly give by <span id="eq:ch4_eq13"><span class="math display">$$
R_{NS} = N_{NS} \frac{\lambda_r}{1 + \lambda_r},
\qquad(13)$$</span></span> where <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> is the number of non-specific sites in the cell (recall that we use <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub> = 4.6 × 10<sup>6</sup></span> for <em>E. coli</em>), and <span class="math inline"><em>R</em><sub><em>C</em></sub></span> is given by <span id="eq:ch4_eq14"><span class="math display">$$
R_C = N_C \frac{\lambda_r e^{-\beta \Delta \varepsilon_C}}
{1 + \lambda_r e^{-\beta \Delta \varepsilon_C}},
\qquad(14)$$</span></span> where <span class="math inline"><em>N</em><sub><em>C</em></sub></span> is the number of competitor sites in the cell and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>C</em></sub></span> is the binding energy of the repressor to the competitor site relative to its non-specific binding energy to the rest of the genome.</p>
<p>To account for the induction of the repressor, we replace the total number of repressors <span class="math inline"><em>R</em><sub>tot</sub></span> in by the number of active repressors in the cell, <span class="math inline"><em>p</em><sub><em>A</em></sub>(<em>c</em>)<em>R</em><sub>tot</sub></span>. Here, <span class="math inline"><em>p</em><sub><em>A</em></sub></span> denotes the probability that the repressor is in the active state (), <span id="eq:ch4_eq15"><span class="math display">$$
p_A(c) = 
\frac{\left(1+\frac{c}{K_A}\right)^n}
{\left(1+\frac{c}{K_A}\right)^n+e^{-\beta \Delta \varepsilon_{AI} }
\left(1+\frac{c}{K_I}\right)^n}.
\qquad(15)$$</span></span> Substituting in into the modified yields the form <span id="eq:ch4_eq16"><span class="math display">$$
p_A(c) R_{\text{tot}} = 
N_S \frac{\lambda_r e^{-\beta \Delta \varepsilon_{RA}}}
{1 + \lambda_r e^{-\beta \Delta \varepsilon_{RA}}} + 
N_{NS} \frac{\lambda_r}{1 + \lambda_r} + 
N_C \frac{\lambda_r e^{-\beta \Delta \varepsilon_C}}
{1 + \lambda_r e^{-\beta \Delta \varepsilon_C}}.
\qquad(16)$$</span></span> For systems where the number of binding sites <span class="math inline"><em>N</em><sub><em>S</em></sub></span>, <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span>, and <span class="math inline"><em>N</em><sub><em>C</em></sub></span> are known, together with the binding affinities <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>C</em></sub></span>, we can solve numerically for <span class="math inline"><em>λ</em><sub><em>r</em></sub></span> and then substitute it into to obtain a fold-change at any concentration of inducer <span class="math inline"><em>c</em></span>. In the following sections, we will theoretically explore the induction curves given by for a number of different combinations of simple repression binding sites, thereby predicting how the system would behave if additional specific or competitor binding sites were introduced.</p>
<h3 id="variable-repressor-copy-number-boldsymbolr-with-multiple-specific-binding-sites-boldsymboln_s-1">Variable Repressor Copy Number (<span class="math inline"><strong>R</strong></span>) with Multiple Specific Binding Sites (<span class="math inline"><strong>N</strong><sub><strong>S</strong></sub> <strong>&gt;</strong> <strong>1</strong></span>)</h3>
<p>In the the main text, we consider the induction profiles of strains with varying <span class="math inline"><em>R</em></span> but a single, specific binding site <span class="math inline"><em>N</em><sub><em>S</em></sub> = 1</span> (see ). Here we predict the induction profiles for similar strains in which <span class="math inline"><em>R</em></span> is varied, but <span class="math inline"><em>N</em><sub><em>S</em></sub> &gt; 1</span>, as shown in . The top row shows induction profiles in which <span class="math inline"><em>N</em><sub><em>S</em></sub> = 10</span> and the bottom row shows profiles in which <span class="math inline"><em>N</em><sub><em>S</em></sub> = 100</span>, assuming three different choices for the specific operator binding sites given by the O1, O2, and O3 operators. These values of <span class="math inline"><em>N</em><sub><em>S</em></sub></span> were chosen to mimic the common scenario in which a promoter construct is placed on either a low or high copy number plasmid. A few features stand out in these profiles. First, as the magnitude of <span class="math inline"><em>N</em><sub><em>S</em></sub></span> surpasses the number of repressors <span class="math inline"><em>R</em></span>, the leakiness begins to increase significantly, since there are no longer enough repressors to regulate all copies of the promoter of interest. Second, in the cases where <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 15.3 <em>k</em><sub><em>B</em></sub><em>T</em></span> for the O1 operator or <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span> for the O2 operator, the profiles where <span class="math inline"><em>N</em><sub><em>S</em></sub> = 100</span> are notably sharper than the profiles where <span class="math inline"><em>N</em><sub><em>S</em></sub> = 10</span>, and it is possible to achieve dynamic ranges approaching 1. Finally, it is interesting to note that the profiles for the O3 operator where <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span> are nearly indifferent to the value of <span class="math inline"><em>N</em><sub><em>S</em></sub></span>.</p>
<figure>
<img src="ch4_fig03.png" id="fig:ch4_fig03" data-short-caption="Induction with variable $\boldsymbol{R}$ and multiple specific binding sites" alt="Figure 3: Induction with variable \boldsymbol{R} and multiple specific binding sites. Induction profiles are shown for strains with variable R and \Delta \varepsilon_{RA} = -15.3, -13.9, or -9.7~k_B T. The number of specific sites, N_S, is held constant at 10 as R and \Delta \varepsilon_{RA} are varied. N_S is held constant at 100 as R and \Delta \varepsilon_{RA} are varied. These situations mimic the common scenario in which a promoter construct is placed on either a low or high copy number plasmid." /><figcaption aria-hidden="true">Figure 3: <strong>Induction with variable <span class="math inline"><strong>R</strong></span> and multiple specific binding sites.</strong> Induction profiles are shown for strains with variable R and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 15.3</span>, <span class="math inline"> − 13.9</span>, or <span class="math inline"> − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span>. The number of specific sites, <span class="math inline"><em>N</em><sub><em>S</em></sub></span>, is held constant at 10 as <span class="math inline"><em>R</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> are varied. <span class="math inline"><em>N</em><sub><em>S</em></sub></span> is held constant at 100 as <span class="math inline"><em>R</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> are varied. These situations mimic the common scenario in which a promoter construct is placed on either a low or high copy number plasmid.</figcaption>
</figure>
<h3 id="variable-number-of-specific-binding-sites-boldsymboln_s-with-fixed-repressor-copy-number-boldsymbolr">Variable Number of Specific Binding Sites <span class="math inline"><strong>N</strong><sub><strong>S</strong></sub></span> with Fixed Repressor Copy Number (<span class="math inline"><strong>R</strong></span>)</h3>
<p>The second set of scenarios we consider is the case in which the repressor copy number <span class="math inline"><em>R</em> = 260</span> is held constant while the number of specific promoters <span class="math inline"><em>N</em><sub><em>S</em></sub></span> is varied (see ). Again we see that leakiness is increased significantly when <span class="math inline"><em>N</em><sub><em>S</em></sub> &gt; <em>R</em></span>, though all profiles for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span> exhibit high leakiness, making the effect less dramatic for this operator. Additionally, we find again that adjusting the number of specific sites can produce induction profiles with maximal dynamic ranges. In particular, the O1 and O2 profiles with <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 15.3</span> and <span class="math inline"> − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span>, respectively, have dynamic ranges approaching 1 for <span class="math inline"><em>N</em><sub><em>S</em></sub> = 50</span> and 100.</p>
<figure>
<img src="ch4_fig04.png" id="fig:ch4_fig04" data-short-caption="Induction with variable specific sites and fixed $\boldsymbol{R}$" alt="Figure 4: Induction with variable specific sites and fixed \boldsymbol{R}. Induction profiles are shown for strains with R=260 and \Delta \varepsilon_{RA} = -15.3~k_BT, \Delta \varepsilon_{RA} = -13.9~k_BT, or \Delta \varepsilon_{RA} = -9.7~k_BT. The number of specific sites N_S is varied from 1 to 500." /><figcaption aria-hidden="true">Figure 4: <strong>Induction with variable specific sites and fixed <span class="math inline"><strong>R</strong></span>.</strong> Induction profiles are shown for strains with <span class="math inline"><em>R</em> = 260</span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 15.3 <em>k</em><sub><em>B</em></sub><em>T</em></span>, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span>, or <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span>. The number of specific sites <span class="math inline"><em>N</em><sub><em>S</em></sub></span> is varied from 1 to 500.</figcaption>
</figure>
<h3 id="competitor-binding-sites">Competitor Binding Sites</h3>
<p>An intriguing scenario is presented by the possibility of competitor sites elsewhere in the genome. This serves as a model for situations in which a promoter of interest is regulated by a transcription factor that has multiple targets. This is highly relevant, as the majority of transcription factors in <em>E. coli</em> have at least two known binding sites, with approximately 50 transcription factors having more than ten known binding sites <span class="citation" data-cites="Rydenfelt2014 Schmidt2015"> [<a href="#ref-Rydenfelt2014" role="doc-biblioref">14</a>,<a href="#ref-Schmidt2015" role="doc-biblioref">15</a>]</span>. If the number of competitor sites and their average binding energy is known, however, they can be accounted for in the model. Here, we predict the induction profiles for strains in which <span class="math inline"><em>R</em> = 260</span> and <span class="math inline"><em>N</em><sub><em>S</em></sub> = 1</span>, but there is a variable number of competitor sites <span class="math inline"><em>N</em><sub><em>C</em></sub></span> with a strong binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>C</em></sub> =  − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span>. In the presence of such a strong competitor, when <span class="math inline"><em>N</em><sub><em>C</em></sub> &gt; <em>R</em></span> the leakiness is greatly increased, as many repressors are siphoned into the pool of competitor sites. This is most dramatic for the case where <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span>, in which it appears that no repression occurs at all when <span class="math inline"><em>N</em><sub><em>C</em></sub> = 500</span>. Interestingly, when <span class="math inline"><em>N</em><sub><em>C</em></sub> &lt; <em>R</em></span> the effects of the competitor are not especially notable.</p>
<figure>
<img src="ch4_fig05.png" id="fig:ch4_fig05" data-short-caption="Induction with variable competitor sites, a single specific site, and fixed $\boldsymbol{R}$" alt="Figure 5: Induction with variable competitor sites, a single specific site, and fixed \boldsymbol{R}. Induction profiles are shown for strains with R=260, N_s=1, and \Delta \varepsilon_{RA} = -15.3~k_B T for the O1 operator, \Delta \varepsilon_{RA} = -13.9~k_B T for the O2 operator, or \Delta \varepsilon_{RA} = -9.7~k_B T for the O3 operator. The number of specific sites, N_C, is varied from 1 to 500. This mimics the common scenario in which a transcription factor has multiple binding sites in the genome." /><figcaption aria-hidden="true">Figure 5: <strong>Induction with variable competitor sites, a single specific site, and fixed <span class="math inline"><strong>R</strong></span>.</strong> Induction profiles are shown for strains with <span class="math inline"><em>R</em> = 260</span>, <span class="math inline"><em>N</em><sub><em>s</em></sub> = 1</span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 15.3 <em>k</em><sub><em>B</em></sub><em>T</em></span> for the O1 operator, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span> for the O2 operator, or <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span> for the O3 operator. The number of specific sites, <span class="math inline"><em>N</em><sub><em>C</em></sub></span>, is varied from 1 to 500. This mimics the common scenario in which a transcription factor has multiple binding sites in the genome.</figcaption>
</figure>
<h3 id="properties-of-the-induction-response">Properties of the Induction Response</h3>
<p>As discussed in the main body of the paper, our treatment of the MWC model allows us to predict key properties of induction responses. Here, we consider the leakiness, saturation, and dynamic range (see ) by numerically solving in the absence of inducer, <span class="math inline"><em>c</em> = 0</span>, and in the presence of saturating inducer <span class="math inline"><em>c</em> → ∞</span>. Using , the former case is given by <span id="eq:ch4_eq17"><span class="math display">$$
R_{\text{tot}} \frac{1}{1 + e^{-\beta \Delta \varepsilon_{AI}}} =
N_S \frac{\lambda_r e^{-\beta \Delta \varepsilon_{RA}}}
{1 + \lambda_r e^{-\beta \Delta \varepsilon_{RA}}} + 
N_{NS} \frac{\lambda_r}{1 + \lambda_r} +
N_C \frac{\lambda_r e^{-\beta \Delta \varepsilon_C}}
{1 + \lambda_r e^{-\beta \Delta \varepsilon_C}},
\qquad(17)$$</span></span> whereupon substituting in the value of <span class="math inline"><em>λ</em><sub><em>r</em></sub></span> into will yield the leakiness. Similarly, the limit of saturating inducer is found by determining <span class="math inline"><em>λ</em><sub><em>r</em></sub></span> from the form <span id="eq:ch4_eq18"><span class="math display">$$
R_{\text{tot}} 
\frac{1}{1 + e^{-\beta \Delta \varepsilon_{AI}} \left(\frac{K_A}{K_I} \right)^2}
= N_S \frac{\lambda_r e^{-\beta \Delta \varepsilon_{RA}}}
{1 + \lambda_r e^{-\beta \Delta \varepsilon_{RA}}} + N_{NS}
\frac{\lambda_r}{1 + \lambda_r} +
N_C \frac{\lambda_r e^{-\beta \Delta \varepsilon_C}}
{1 + \lambda_r e^{-\beta \Delta \varepsilon_C}}.
\qquad(18)$$</span></span></p>
<p>In we show how the leakiness, saturation, and dynamic range vary with <span class="math inline"><em>R</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> in systems with <span class="math inline"><em>N</em><sub><em>S</em></sub> = 10</span> or <span class="math inline"><em>N</em><sub><em>S</em></sub> = 100</span>. An inflection point occurs where <span class="math inline"><em>N</em><sub><em>S</em></sub> = <em>R</em></span>, with leakiness and dynamic range behaving differently when <span class="math inline"><em>R</em> &lt; <em>N</em><sub><em>S</em></sub></span> than when <span class="math inline"><em>R</em> &gt; <em>N</em><sub><em>S</em></sub></span>. This transition is more dramatic for <span class="math inline"><em>N</em><sub><em>S</em></sub> = 100</span> than for <span class="math inline"><em>N</em><sub><em>S</em></sub> = 10</span>. Interestingly, the saturation values consistently approach 1, indicating that full induction is easier to achieve when multiple specific sites are present. Moreover, dynamic range values for O1 and O2 strains with <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 15.3</span> and <span class="math inline"> − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span> approach 1 when <span class="math inline"><em>R</em> &gt; <em>N</em><sub><em>S</em></sub></span>, although when <span class="math inline"><em>N</em><sub><em>S</em></sub> = 10</span> there is a slight downward dip owing to saturation values of less than 1 at high repressor copy numbers.</p>
<figure>
<img src="ch4_fig06.png" id="fig:ch4_fig06" data-short-caption="Phenotypic properties of induction with multiple specific binding sites" alt="Figure 6: Phenotypic properties of induction with multiple specific binding sites. The leakiness , saturation , and dynamic range are shown for systems with number of specific binding sites N_S = 10 or N_S = 100 . The dashed vertical line indicates the point at which N_S = R." /><figcaption aria-hidden="true">Figure 6: <strong>Phenotypic properties of induction with multiple specific binding sites.</strong> The leakiness , saturation , and dynamic range are shown for systems with number of specific binding sites <span class="math inline"><em>N</em><sub><em>S</em></sub> = 10</span> or <span class="math inline"><em>N</em><sub><em>S</em></sub> = 100</span> . The dashed vertical line indicates the point at which <span class="math inline"><em>N</em><sub><em>S</em></sub> = <em>R</em></span>.</figcaption>
</figure>
<p>In we similarly show how the leakiness, saturation, and dynamic range vary with <span class="math inline"><em>R</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> in systems with <span class="math inline"><em>N</em><sub><em>S</em></sub> = 1</span> and multiple competitor sites <span class="math inline"><em>N</em><sub><em>C</em></sub> = 10</span> or <span class="math inline"><em>N</em><sub><em>C</em></sub> = 100</span>. Each of the competitor sites has a binding energy of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>C</em></sub> =  − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span>. The phenotypic profiles are very similar to those for multiple specific sites shown in , with sharper transitions at <span class="math inline"><em>R</em> = <em>N</em><sub><em>C</em></sub></span> due to the greater binding strength of the competitor site. This indicates that introducing competitors has much the same effect on the induction phenotypes as introducing additional specific sites, as in either case the influence of the repressors is dampened when there are insufficient repressors to interact with all of the specific binding sites.</p>
<figure>
<img src="ch4_fig07.png" id="fig:ch4_fig07" data-short-caption="Phenotypic properties of induction with a single specific site and multiple competitor sites" alt="Figure 7: Phenotypic properties of induction with a single specific site and multiple competitor sites. The leakiness , saturation , and dynamic range are shown for systems with a single specific binding site N_S = 1 and a number of competitor sites N_C = 10 or N_C = 100 . All competitor sites have a binding energy of \Delta \varepsilon_C = -17.0~k_BT. The dashed vertical line indicates the point at which N_C = R." /><figcaption aria-hidden="true">Figure 7: <strong>Phenotypic properties of induction with a single specific site and multiple competitor sites.</strong> The leakiness , saturation , and dynamic range are shown for systems with a single specific binding site <span class="math inline"><em>N</em><sub><em>S</em></sub> = 1</span> and a number of competitor sites <span class="math inline"><em>N</em><sub><em>C</em></sub> = 10</span> or <span class="math inline"><em>N</em><sub><em>C</em></sub> = 100</span> . All competitor sites have a binding energy of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>C</em></sub> =  − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span>. The dashed vertical line indicates the point at which <span class="math inline"><em>N</em><sub><em>C</em></sub> = <em>R</em></span>.</figcaption>
</figure>
<p>This section of the appendix gives a quantitative analysis of the nuances imposed on induction response in the case of systems involving multiple gene copies as are found in the vast majority of studies on induction. In these cases, the intrinsic parameters of the MWC model get entangled with the parameters describing gene copy number.</p>
<h2 id="flow-cytometry">Flow Cytometry</h2>
<p>In this section, we provide information regarding the equipment used to make experimental measurements of the fold-change in gene expression in the interests of transparency and reproducibility. We also provide a summary of our unsupervised method of gating the flow cytometry measurements for consistency between experimental runs.</p>
<h3 id="equipment">Equipment</h3>
<p>Due to past experience using the Miltenyi Biotec MACSQuant flow cytometer during the Physiology summer course at the Marine Biological Laboratory, we used the same flow cytometer for the formal measurements in this work graciously provided by the Pamela Björkman lab at Caltech. All measurements were made using an excitation wavelength of <span class="math inline">488 nm</span> with an emission filter set of 525/<span class="math inline">50 nm</span>. This excitation wavelength provides approximately 40% of the maximum YFP absorbance <span class="citation" data-cites="SpectraViewer"> [<a href="#ref-SpectraViewer" role="doc-biblioref">16</a>]</span>, and this was found to be sufficient for the purposes of these experiments. A useful feature of modern flow cytometry is the high-sensitivity signal detection through the use of photomultiplier tubes (PMT) whose response can be tuned by adjusting the voltage. Thus, the voltage for the forward-scatter (FSC), side-scatter (SSC), and gene expression measurements were tuned manually to maximize the dynamic range between autofluorescence signal and maximal expression without losing the details of the population distribution. Once these voltages were determined, they were used for all subsequent measurements. Extremely low signal producing particles were discarded before data storage by setting a basal voltage threshold, thus removing the majority of spurious events. The various instrument settings for data collection are given in tbl. 1.</p>
<div id="tbl:ch4_tab01">
<table>
<caption>Table 1: <strong>Instrument settings for data collection using the Miltenyi Biotec MACSQuant flow cytometer.</strong> All experimental measurements were collected using these values.</caption>
<thead>
<tr class="header">
<th><strong>Laser</strong></th>
<th><strong>Channel</strong></th>
<th><strong>Sensor Voltage</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><span class="math inline">488 nm</span></td>
<td>Forward-Scatter (FSC)</td>
<td><span class="math inline">423 V</span></td>
</tr>
<tr class="even">
<td><span class="math inline">488 nm</span></td>
<td>Side-Scatter (SSC)</td>
<td><span class="math inline">537 V</span></td>
</tr>
<tr class="odd">
<td><span class="math inline">488 nm</span></td>
<td>Intensity (B1 Filter, 525/50nm)</td>
<td><span class="math inline">790 V</span></td>
</tr>
<tr class="even">
<td><span class="math inline">488 nm</span></td>
<td>Trigger (debris threshold)</td>
<td><span class="math inline">24.5 V</span></td>
</tr>
</tbody>
</table>
</div>
<h3 id="experimental-measurement">Experimental Measurement</h3>
<p>Prior to each day’s experiments, the analyzer was calibrated using MACSQuant Calibration Beads (Cat. No. 130-093-607) such that day-to-day experiments would be comparable. A single data set consisted of seven bacterial strains, all sharing the same operator, with varying repressor copy numbers (<span class="math inline"><em>R</em> = 0</span>, 22, 60, 124, 260, 1220, and 1740), in addition to an autofluorescent strain, under twelve IPTG concentrations. Data collection took place over two to three hours. During this time, the cultures were held at approximately 4<span class="math inline"><sup>∘</sup></span>C by placing the 96-well plate on a MACSQuant ice block. Because the ice block thawed over the course of the experiment, the samples measured last were approximately at room temperature. This means that samples may have grown slightly by the end of the experiment. To confirm that this continued growth did not alter the measured results, a subset of experiments were run in reverse meaning that the fully induced cultures were measured first and the uninduced samples last. The plate arrangements and corresponding fold-change measurements are shown in and , respectively. The measured fold-change values in the reverse ordered plate appear to be drawn from the same distribution as those measured in the forward order, meaning that any growth that might have taken place during the experiment did not significantly affect the results. Both the forward and reverse data sets were used in our analysis.</p>
<figure>
<img src="ch4_fig08.png" id="fig:ch4_fig08" data-short-caption="Plate arrangements for flow cytometry." alt="Figure 8: Plate arrangements for flow cytometry. (A) Samples were measured primarily in the forward arrangement with a subset of samples measured in reverse. The black arrow indicates the order in which samples were processed by the flow cytometer. (B) The experimentally measured fold-change values for the two sets of plate arrangements show that samples measured in the forward arrangement appear to be indistinguishable from those measured in reverse order." /><figcaption aria-hidden="true">Figure 8: <strong>Plate arrangements for flow cytometry.</strong> (A) Samples were measured primarily in the forward arrangement with a subset of samples measured in reverse. The black arrow indicates the order in which samples were processed by the flow cytometer. (B) The experimentally measured fold-change values for the two sets of plate arrangements show that samples measured in the forward arrangement appear to be indistinguishable from those measured in reverse order.</figcaption>
</figure>
<h3 id="unsupervised-gating">Unsupervised Gating</h3>
<p>Flow cytometry data will frequently include a number of spurious events or other undesirable data points such as cell doublets and debris. The process of restricting the collected data set to those data determined to be “real” is commonly referred to as gating. These gates are typically drawn manually <span class="citation" data-cites="Maecker2005"> [<a href="#ref-Maecker2005" role="doc-biblioref">17</a>]</span> and restrict the data set to those points which display a high degree of linear correlation between their forward-scatter (FSC) and side-scatter (SSC). The development of unbiased and unsupervised methods of drawing these gates is an active area of research <span class="citation" data-cites="Lo2008 Aghaeepour2013"> [<a href="#ref-Lo2008" role="doc-biblioref">18</a>,<a href="#ref-Aghaeepour2013" role="doc-biblioref">19</a>]</span>.</p>
<p>For this study, we used an automatic unsupervised gating procedure to filter the flow cytometry data based on the front and side-scattering values returned by the MACSQuant flow cytometer. We assume that the region with highest density of points in these two channels corresponds to single-cell measurements. Everything extending outside of this region was discarded in order to exclude sources of error such as cell clustering, particulates, or other spurious events.</p>
<p>In order to define the gated region we fit a two-dimensional Gaussian function to the <span class="math inline">log<sub>10</sub></span> forward-scattering (FSC) and the <span class="math inline">log<sub>10</sub></span> side-scattering (SSC) data. We then kept a fraction <span class="math inline"><em>α</em> ∈ [0, 1]</span> of the data by defining an elliptical region given by <span id="eq:ch4_eq19"><span class="math display">(<strong>x</strong>−<strong>μ</strong>)<sup><em>T</em></sup><strong>Σ</strong><sup> − 1</sup>(<strong>x</strong>−<strong>μ</strong>) ≤ <em>χ</em><sub><em>α</em></sub><sup>2</sup>(<em>p</em>),   (19)</span></span> where <span class="math inline"><strong>x</strong></span> is the <span class="math inline">2 × 1</span> vector containing the <span class="math inline">log (FSC)</span> and <span class="math inline">log (SSC)</span>, <span class="math inline"><strong>μ</strong></span> is the <span class="math inline">2 × 1</span> vector representing the mean values of <span class="math inline">log (FSC)</span> and <span class="math inline">log (SSC)</span> as obtained from fitting a two-dimensional Gaussian to the data, and <span class="math inline"><strong>Σ</strong></span> is the <span class="math inline">2 × 2</span> covariance matrix also obtained from the Gaussian fit. <span class="math inline"><em>χ</em><sub><em>α</em></sub><sup>2</sup>(<em>p</em>)</span> is the quantile function for probability <span class="math inline"><em>p</em></span> of the chi-squared distribution with two degrees of freedom. shows an example of different gating contours that would arise from different values of <span class="math inline"><em>α</em></span> in . In this work, we chose <span class="math inline"><em>α</em> = 0.4</span> which we deemed was a sufficient constraint to minimize the noise in the data. As explained in Appendix XXX we compared our high throughput flow cytometry data with single cell microscopy, confirming that the automatic gating did not introduce systematic biases to the analysis pipeline. The specific code where this gating is implemented can be found in <a href="https://github.com/RPGroup-PBoC/mwc_induction/blob/master/code/analysis/unsupervised_gating.ipynb">GitHub repository</a>.</p>
<figure>
<img src="ch4_fig09.png" id="fig:ch4_fig09" data-short-caption="Representative unsupervised gating contours." alt="Figure 9: Representative unsupervised gating contours. Points indicate individual flow cytometry measurements of forward scatter and side scatter. Colored points indicate arbitrary gating contours ranging from 100% (\alpha = 1.0) to 5% (\alpha = 0.05). All measurements for this work were made computing the mean fluorescence from the 40^\text{th} percentile (\alpha = 0.4), shown as orange points." /><figcaption aria-hidden="true">Figure 9: <strong>Representative unsupervised gating contours.</strong> Points indicate individual flow cytometry measurements of forward scatter and side scatter. Colored points indicate arbitrary gating contours ranging from 100% (<span class="math inline"><em>α</em> = 1.0</span>) to 5% (<span class="math inline"><em>α</em> = 0.05</span>). All measurements for this work were made computing the mean fluorescence from the 40<span class="math inline"><sup>th</sup></span> percentile (<span class="math inline"><em>α</em> = 0.4</span>), shown as orange points.</figcaption>
</figure>
<h3 id="comparison-of-flow-cytometry-with-other-methods">Comparison of Flow Cytometry with Other Methods</h3>
<p>Previous work from our lab experimentally determined fold-change for similar simple repression constructs using a variety of different measurement methods <span class="citation" data-cites="Garcia2011b Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">1</a>,<a href="#ref-Garcia2011b" role="doc-biblioref">20</a>]</span>. Garcia and Phillips used the same background strains as the ones used in this work, but gene expression was measured with Miller assays based on colorimetric enzymatic reactions with the LacZ protein <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span>. <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">1</a>]</span> used a LacI dimer with the tetramerization region replaced with an mCherry tag, where the fold-change was measured as the ratio of the gene expression rate rather than a single snapshot of the gene output.</p>
<p>shows the comparison of these methods along with the flow cytometry method used in this work. The consistency of these three readouts validates the quantitative use of flow cytometry and unsupervised gating to determine the fold-change in gene expression. However, one important caveat revealed by this figure is that the sensitivity of flow cytometer measurements is not sufficient to accurately determine the fold-change for the high repressor copy number strains in O1 without induction. Instead, a method with a large dynamic range such as the Miller assay is needed to accurately resolve the fold-change at such low expression levels.</p>
<figure>
<img src="ch4_fig10.png" id="fig:ch4_fig10" data-short-caption="Comparison of experimental methods to determine the fold-change." alt="Figure 10: Comparison of experimental methods to determine the fold-change. The fold-change in gene expression for equivalent simple-repression constructs has been determined using three independent methods: flow cytometry (this work), colorimetric miller assays  [4], and video microscopy  [1]. All three methods give consistent results, although flow cytometry measurements lose accuracy for fold-change less than 10^{-2}. note that the repressor-dna binding energies \delta\varepsilon_{ra} used for the theoretical predictions were determined in  [4]." /><figcaption aria-hidden="true">Figure 10: <strong>Comparison of experimental methods to determine the fold-change.</strong> The fold-change in gene expression for equivalent simple-repression constructs has been determined using three independent methods: flow cytometry (this work), colorimetric miller assays <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span>, and video microscopy <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">1</a>]</span>. All three methods give consistent results, although flow cytometry measurements lose accuracy for fold-change less than <span class="math inline">10<sup> − 2</sup></span>. note that the repressor-dna binding energies <span class="math inline"><em>δ</em><em>ε</em><sub><em>r</em><em>a</em></sub></span> used for the theoretical predictions were determined in <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span>.</figcaption>
</figure>
<h2 id="single-cell-microscopy">Single-Cell Microscopy</h2>
<p>In this section, we detail the procedures and results from single-cell microscopy verification of our flow cytometry measurements. Our previous measurements of fold-change in gene expression have been measured using bulk-scale Miller assays <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span> or through single-cell microscopy <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">1</a>]</span>. In this work, flow cytometry was an attractive method due to the ability to screen through many different strains at different concentrations of inducer in a short amount of time. To verify our results from flow cytometry, we examined two bacterial strains with different repressor-DNA binding energies (<span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>) of <span class="math inline"> − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span> and <span class="math inline"> − 15.3 <em>k</em><sub><em>B</em></sub><em>T</em></span> with <span class="math inline"><em>R</em> = 260</span> repressors per cell using fluorescence microscopy and estimated the values of the parameters <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> for direct comparison between the two methods. For a detailed explanation of the Python code implementation of the processing steps described below, please see this paper’s <a href="https://rpgroup-pboc.github.io/mwc_induction/code/notebooks/unsupervised_gating.html">GitHub repository</a>. An outline of our microscopy workflow can be seen in .</p>
<h3 id="strains-and-growth-conditions">Strains and Growth Conditions</h3>
<p>Cells were grown in an identical manner to those used for measurement via flow cytometry (see Methods). Briefly, cells were grown overnight (between 10 and 13 hours) to saturation in rich media broth (LB) with <span class="math inline">100 <em>μ</em>g ⋅ mL<sup> − 1</sup></span> spectinomycin in a deep-well 96 well plate at <span class="math inline">37<sup>∘</sup>C</span>. These cultures were then diluted 1000-fold into <span class="math inline">500 <em>μ</em>L</span> of M9 minimal medium supplemented with 0.5% glucose and the appropriate concentration of the inducer IPTG. Strains were allowed to grow at <span class="math inline">37<sup>∘</sup>C</span> with vigorous aeration for approximately 8 hours. Prior to mounting for microscopy, the cultures were diluted 10-fold into M9 glucose minimal medium in the absence of IPTG. Each construct was measured using the same range of inducer concentration values as was performed in the flow cytometry measurements (between <span class="math inline">100 nM</span> and <span class="math inline">5 mM</span> IPTG). Each condition was measured in triplicate in microscopy whereas approximately ten measurements were made using flow cytometry.</p>
<h3 id="imaging-procedure">Imaging Procedure</h3>
<figure>
<img src="ch4_fig11.png" id="fig:ch4_fig11" data-short-caption="Experimental workflow for single-cell microscopy" alt="Figure 11: Experimental workflow for single-cell microscopy. For comparison with the flow cytometry results, the cells were grown in an identical manner to those described in the main text. Once cells had reached mid to late exponential growth, the cultures were diluted and placed on agarose substrates and imaged under 100\times magnification. Regions of interest representing cellular mass were segmented and average single-cell intensities were computed. The means of the distributions were used to compute the fold-change in gene expression." /><figcaption aria-hidden="true">Figure 11: <strong>Experimental workflow for single-cell microscopy</strong>. For comparison with the flow cytometry results, the cells were grown in an identical manner to those described in the main text. Once cells had reached mid to late exponential growth, the cultures were diluted and placed on agarose substrates and imaged under 100<span class="math inline">×</span> magnification. Regions of interest representing cellular mass were segmented and average single-cell intensities were computed. The means of the distributions were used to compute the fold-change in gene expression.</figcaption>
</figure>
<p>During the last hour of cell growth, an agarose mounting substrate was prepared containing the appropriate concentration of the IPTG inducer. This mounting substrate was composed of M9 minimal medium supplemented with 0.5% glucose and 2% agarose (Life Technologies UltraPure Agarose, Cat. No. 16500100). This solution was heated in a microwave until molten followed by addition of the IPTG to the appropriate final concentration. This solution was then thoroughly mixed and a <span class="math inline">500 <em>μ</em>L</span> aliquot was sandwiched between two glass coverslips and was allowed to solidify.</p>
<p>Once solid, the agarose substrates were cut into approximately <span class="math inline">10 mm × 10 mm</span> squares. An aliquot of one to two microliters of the diluted cell suspension was then added to each pad. For each concentration of inducer, a sample of the autofluorescence control, the <span class="math inline"><em>Δ</em><em>l</em><em>a</em><em>c</em><em>I</em></span> constitutive expression control, and the experimental strain was prepared yielding a total of thirty-six agarose mounts per experiment. These samples were then mounted onto two glass-bottom dishes (Ted Pella Wilco Dish, Cat. No. 14027-20) and sealed with parafilm.</p>
<p>All imaging was performed on a Nikon Ti-Eclipse inverted fluorescent microscope outfitted with a custom-built laser illumination system and operated by the open-source MicroManager control software <span class="citation" data-cites="Edelstein2014"> [<a href="#ref-Edelstein2014" role="doc-biblioref">21</a>]</span>. The YFP fluorescence was imaged using a CrystaLaser <span class="math inline">514 nm</span> excitation laser coupled with a laser-optimized (Semrock Cat. No. LF514-C-000) emission filter.</p>
<p>For each sample, between fifteen and twenty positions were imaged allowing for measurement of several hundred cells. At each position, a phase contrast image, an mCherry image, and a YFP image were collected in that order with exposures on a time scale of ten to twenty milliseconds. For each channel, the same exposure time was used across all samples in a given experiment. All images were collected and stored in <code>ome.tiff</code> format. All microscopy images are available on the CaltechDATA online repository under DOI: 10.22002/D1.229.</p>
<h3 id="image-processing">Image Processing</h3>
<p><strong>Correcting Uneven Illumination</strong></p>
<figure>
<img src="ch4_fig12.png" id="fig:ch4_fig12" data-short-caption="Correction for uneven illumination." alt="Figure 12: Correction for uneven illumination. A representative image of the illumination profile of the 512\,\text{nm} excitation beam on a homogeneously fluorescent slide is shown in the left panel. This is corrected for using equation and is shown in the right panel." /><figcaption aria-hidden="true">Figure 12: <strong>Correction for uneven illumination.</strong> A representative image of the illumination profile of the <span class="math inline">512 nm</span> excitation beam on a homogeneously fluorescent slide is shown in the left panel. This is corrected for using equation and is shown in the right panel.</figcaption>
</figure>
<p>The excitation laser has a two-dimensional gaussian profile. To minimize non-uniform illumination of a single field of view, the excitation beam was expanded to illuminate an area larger than that of the camera sensor. While this allowed for an entire field of view to be illuminated, there was still approximately a 10% difference in illumination across both dimensions. This nonuniformity was corrected for in post-processing by capturing twenty images of a homogeneously fluorescent plastic slide (Autofluorescent Plastic Slides, Chroma Cat. No. 920001) and averaging to generate a map of illumination intensity at any pixel <span class="math inline"><em>I</em><sub>YFP</sub></span>. To correct for shot noise in the camera (Andor iXon+ 897 EMCCD), twenty images were captured in the absence of illumination using the exposure time used for the experimental data. Averaging over these images produced a map of background noise at any pixel <span class="math inline"><em>I</em><sub>dark</sub></span>. To perform the correction, each fluorescent image in the experimental acquisition was renormalized with respect to these average maps as <span id="eq:ch4_eq20"><span class="math display">$$
I_\text{flat} = 
\frac{I - I_\text{dark}}{I_\text{YFP} - 
I_\text{dark}}\langle I_\text{YFP} - I_\text{dark} \rangle,
\qquad(20)$$</span></span> where <span class="math inline"><em>I</em><sub>flat</sub></span> is the renormalized image and <span class="math inline"><em>I</em></span> is the original fluorescence image. An example of this correction can be seen in .</p>
<p><strong>Cell Segmentation</strong></p>
<p>Each bacterial strain constitutively expressed an mCherry fluorophore from a low copy-number plasmid. This served as a volume marker of cell mass allowing us to segment individual cells through edge detection in fluorescence. We used the Marr-Hildreth edge detector <span class="citation" data-cites="Marr1980"> [<a href="#ref-Marr1980" role="doc-biblioref">22</a>]</span> which identifies edges by taking the second derivative of a lightly Gaussian blurred image. Edges are identified as those regions which cross from highly negative to highly positive values or vice-versa within a specified neighborhood. Bacterial cells were defined as regions within an intact and closed identified edge. All segmented objects were then labeled and passed through a series of filtering steps.</p>
<p>To ensure that primarily single cells were segmented, we imposed area and eccentricity bounds. We assumed that single cells projected into two dimensions are roughly <span class="math inline">2 <em>μ</em>m</span> long and <span class="math inline">1 <em>μ</em>m</span> wide, so that cells are likely to have an area between <span class="math inline">0.5 <em>μ</em>m<sup>2</sup></span> and <span class="math inline">6 <em>μ</em>m</span>. To determine the eccentricity bounds, we assumed that the a single cell can be approximated by an ellipse with semi-major (<span class="math inline"><em>a</em></span>) and semi-minor (<span class="math inline"><em>b</em></span>) axis lengths of <span class="math inline">0.5 <em>μ</em>m</span> and <span class="math inline">0.25 <em>μ</em>m</span>, respectively. The eccentricity of this hypothetical cell can be computed as <span id="eq:ch4_eq21"><span class="math display">$$
\text{eccentricity} = \sqrt{1 - \left(\frac{b}{a}\right)^2},
\qquad(21)$$</span></span> yielding a value of approximately 0.8. Any objects with an eccentricity below this value were not considered to be single cells. After imposing both an area () and eccentricity filter (), the remaining objects were considered cells of interest () and the mean fluorescence intensity of each cell was extracted.</p>
<figure>
<img src="ch4_fig13.png" id="fig:ch4_fig13" data-short-caption="Segmentation of single bacterial cells" alt="Figure 13: Segmentation of single bacterial cells. Objects were selected if they had an eccentricity greater than 0.8 and an area between 0.5\,\mu\text{m}^2 and 6\,\mu\text{m}^2. Highlighted in blue are the regions considered to be representative of single cells. The black lines correspond to the empirical cumulative distribution functions for the parameter of interest. A representative final segmentation mask is shown in which segmented cells are depicted in cyan over the phase contrast image." /><figcaption aria-hidden="true">Figure 13: <strong>Segmentation of single bacterial cells.</strong> Objects were selected if they had an eccentricity greater than 0.8 and an area between <span class="math inline">0.5 <em>μ</em>m<sup>2</sup></span> and <span class="math inline">6 <em>μ</em>m<sup>2</sup></span>. Highlighted in blue are the regions considered to be representative of single cells. The black lines correspond to the empirical cumulative distribution functions for the parameter of interest. A representative final segmentation mask is shown in which segmented cells are depicted in cyan over the phase contrast image.</figcaption>
</figure>
<p><strong>Calculation of Fold-Change</strong></p>
<p>Cells exhibited background fluorescence even in the absence of an expressed fluorophore. We corrected for this autofluorescence contribution to the fold-change calculation by subtracting the mean YFP fluorescence of cells expressing only the mCherry volume marker from each experimental measurement. The fold-change in gene expression was therefore calculated as <span id="eq:ch4_eq22"><span class="math display">$$
\text{fold-change} = 
\frac{\langle I_{R &gt; 0} \rangle - \langle I_\text{auto} \rangle}
{\langle I_{R = 0} \rangle - \langle I_\text{auto} \rangle},
\qquad(22)$$</span></span> where <span class="math inline">⟨<em>I</em><sub><em>R</em> &gt; 0</sub>⟩</span> is the mean fluorescence intensity of cells expressing LacI repressors, <span class="math inline">⟨<em>I</em><sub>auto</sub>⟩</span> is the mean intensity of cells expressing only the mCherry volume marker, and <span class="math inline">⟨<em>I</em><sub><em>R</em> = 0</sub>⟩</span> is the mean fluorescence intensity of cells in the absence of LacI. These fold-change values were very similar to those obtained through flow cytometry and were well described using the thermodynamic parameters used in the main text. With these experimentally measured fold-change values, the best-fit parameter values of the model were inferred and compared to those obtained from flow cytometry.</p>
<h3 id="parameter-estimation-and-comparison">Parameter Estimation and Comparison</h3>
<figure>
<img src="ch4_fig14.png" id="fig:ch4_fig14" data-short-caption="Comparison of measured fold-change between flow cytometry and single-cell microscopy" alt="Figure 14: Comparison of measured fold-change between flow cytometry and single-cell microscopy. Experimentally measured fold-change values obtained through single-cell microscopy and flow cytometry are shown as white filled and solid colored circles, respectively. Solid and dashed lines indicate the predicted behavior using the most likely parameter values of K_A and K_I inferred from flow cytometry data and microscopy data, respectively. The red and blue plotting elements correspond to the different operators O1 and O2 with binding energies \Delta\varepsilon_{RA} of -13.9~k_BT and -15.3~k_BT, respectively  [4]. The marginalized posterior distributions for K_A and K_I are shown in the top and bottom panel, respectively. The posterior distribution determined using the microscopy data is wider than that computed using the flow cytometry data due to a smaller fig collection of data sets (three for microscopy and ten for flow cytometry)." /><figcaption aria-hidden="true">Figure 14: <strong>Comparison of measured fold-change between flow cytometry and single-cell microscopy.</strong> Experimentally measured fold-change values obtained through single-cell microscopy and flow cytometry are shown as white filled and solid colored circles, respectively. Solid and dashed lines indicate the predicted behavior using the most likely parameter values of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> inferred from flow cytometry data and microscopy data, respectively. The red and blue plotting elements correspond to the different operators O1 and O2 with binding energies <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> of <span class="math inline"> − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span> and <span class="math inline"> − 15.3 <em>k</em><sub><em>B</em></sub><em>T</em></span>, respectively <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span>. The marginalized posterior distributions for <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are shown in the top and bottom panel, respectively. The posterior distribution determined using the microscopy data is wider than that computed using the flow cytometry data due to a smaller fig collection of data sets (three for microscopy and ten for flow cytometry).</figcaption>
</figure>
<p>To confirm quantitative consistency between flow cytometry and microscopy, the parameter values of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> were also estimated from three biological replicates of IPTG titration curves obtained by microscopy for strains with <span class="math inline"><em>R</em> = 260</span> and operators O1 and O2. shows the data from these measurements (orange circles) and the ten biological replicates from our flow cytometry measurements (blue circles), along with the fold-change predictions from each inference. In comparison with the values obtained by flow cytometry, each parameter estimate overlapped with the 95% credible region of our flow cytometry estimates, as shown in . Specifically, these values were <span class="math inline"><em>K</em><sub><em>A</em></sub> = 142<sub> − 34</sub><sup> + 40</sup> <em>μ</em>M</span> and <span class="math inline"><em>K</em><sub><em>I</em></sub> = 0.6<sub> − 0.1</sub><sup> + 0.1</sup> <em>μ</em>M</span> from microscopy and <span class="math inline"><em>K</em><sub><em>A</em></sub> = 149<sub> − 12</sub><sup> + 14</sup> <em>μ</em>M</span> and <span class="math inline"><em>K</em><sub><em>I</em></sub> = 0.57<sub> − 0.02</sub><sup> + 0.03</sup> <em>μ</em>M</span> from the flow cytometry data. We note that the credible regions from the microscopy data shown in are much broader than those from flow cytometry due to the fewer number of replicates performed.</p>
<h2 id="fold-change-sensitivity-analysis">Fold-Change Sensitivity Analysis</h2>
<p>In we found that the width of the credible regions varied widely depending on the repressor copy number <span class="math inline"><em>R</em></span> and repressor operator binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>. More precisely, the credible regions were much narrower for low repressor copy numbers <span class="math inline"><em>R</em></span> and weak binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>. In this section, we explain how this behavior comes about. We focus our attention on the maximum fold-change in the presence of saturating inducer given by . While it is straightforward to consider the width of the credible regions at any other inducer concentration, shows that the credible region are widest at saturation.</p>
<p>The width of the credible regions corresponds to how sensitive the fold-change is to the fit values of the dissociation constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>. To be quantitative, we define <span id="eq:ch4_eq23"><span class="math display"><em>Δ</em>fold-change<sub><em>K</em><sub><em>A</em></sub></sub> ≡ fold-change(<em>K</em><sub><em>A</em></sub>, <em>K</em><sub><em>I</em></sub><sup>fit</sup>) − fold-change(<em>K</em><sub><em>A</em></sub><sup>fit</sup>, <em>K</em><sub><em>I</em></sub><sup>fit</sup>),   (23)</span></span> the difference between the fold-change at a particular <span class="math inline"><em>K</em><sub><em>A</em></sub></span> value relative to the best-fit dissociation constant <span class="math inline"><em>K</em><sub><em>A</em></sub><sup>fit</sup> = 139 × 10<sup> − 6</sup> M</span>. For simplicity, we keep the inactive state dissociation constant fixed at its best-fit value <span class="math inline"><em>K</em><sub><em>I</em></sub><sup>fit</sup> = 0.53 × 10<sup> − 6</sup> M</span>. A larger difference <span class="math inline"><em>Δ</em>fold-change<sub><em>K</em><sub><em>A</em></sub></sub></span> implies a wider credible region. Similarly, we define the analogous quantity <span id="eq:ch4_eq24"><span class="math display"><em>Δ</em>fold-change<sub><em>K</em><sub><em>I</em></sub></sub> = fold-change(<em>K</em><sub><em>A</em></sub><sup>fit</sup>, <em>K</em><sub><em>I</em></sub>) − fold-change(<em>K</em><sub><em>A</em></sub><sup>fit</sup>, <em>K</em><sub><em>I</em></sub><sup>fit</sup>)   (24)</span></span> to measure the sensitivity of the fold-change to <span class="math inline"><em>K</em><sub><em>I</em></sub></span> at a fixed <span class="math inline"><em>K</em><sub><em>A</em></sub><sup>fit</sup></span>. shows both of these quantities in the limit <span class="math inline"><em>c</em> → ∞</span> for different repressor-DNA binding energies <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and repressor copy numbers <span class="math inline"><em>R</em></span>. See our <a href="https://github.com/RPGroup-PBoC/mwc_induction/blob/master/code/analysis/sensitivity_analysis.ipynb">GitHub repository</a> for the code that reproduces these plots.</p>
<p>To understand how the width of the credible region scales with <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>R</em></span>, we can Taylor expand the difference in fold-change to first order, <span class="math inline">$\Delta \text{fold-change}_{K_A} \approx \frac{\partial \text{fold-change}}{\partial K_A} \left( K_A - K_A^{\text{fit}} \right)$</span>, where the partial derivative has the form <span id="eq:ch4_eq25"><span class="math display">$$
\frac{\partial \text{fold-change}}{\partial K_A} =
\frac{e^{-\beta \Delta \varepsilon_{AI}} 
\frac{n}{K_I}\left(\frac{K_A}{K_I}\right)^{n-1}}
{\left( 1 + e^{-\beta \Delta \varepsilon_{AI}} 
\left(\frac{K_A}{K_I}\right)^n \right)^2}\frac{R}{N_{NS}}
e^{-\beta \Delta\varepsilon_{RA}} 
\left(
1+\frac{1}{1+e^{-\beta \Delta \varepsilon_{AI} }
\left(\frac{K_A}{K_I}\right)^n }
\frac{R}{N_{NS}}e^{-\beta \Delta\varepsilon_{RA}}
\right)^{-2}.
\qquad(25)$$</span></span> Similarly, the Taylor expansion <span class="math inline">$\Delta \text{fold-change}_{K_I} \approx \frac{\partial  \text{fold-change}}{\partial K_I} \left( K_I - K_I^{\text{fit}} \right)$</span> features the partial derivative <span id="eq:ch4_eq26"><span class="math display">$$
\frac{\partial \text{fold-change}}{\partial K_I} = 
-\frac{e^{-\beta \Delta \varepsilon_{AI}} 
\frac{n}{K_I}\left(\frac{K_A}{K_I}\right)^{n}}
{\left( 1 + e^{-\beta \Delta \varepsilon_{AI}} 
\left(\frac{K_A}{K_I}\right)^n \right)^2}\frac{R}{N_{NS}}
e^{-\beta \Delta\varepsilon_{RA}}
\left(
1+\frac{1}{1+e^{-\beta \Delta \varepsilon_{AI} }
\left(\frac{K_A}{K_I}\right)^n }\frac{R}{N_{NS}}
e^{-\beta \Delta\varepsilon_{RA}} \right)^{-2}.
\qquad(26)$$</span></span> From , we find that both <span class="math inline"><em>Δ</em>fold-change<sub><em>K</em><sub><em>A</em></sub></sub></span> and <span class="math inline"><em>Δ</em>fold-change<sub><em>K</em><sub><em>I</em></sub></sub></span> increase in magnitude with <span class="math inline"><em>R</em></span> and decrease in magnitude with <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>. Accordingly, we expect that the O3 strains (with the least negative <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>) and the strains with the smallest repressor copy number will lead to partial derivatives with smaller magnitude and hence to tighter credible regions. Indeed, this prediction is carried out in XXX.</p>
<p>Lastly, we note that enable us to quantify the scaling relationship between the width of the credible region and the two quantities <span class="math inline"><em>R</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>. For example, for the O3 strains, where the fold-change at saturating inducer concentration is <span class="math inline"> ≈ 1</span>, the right-most term in both equations which equals the fold-change squared is roughly 1. Therefore, we find that both <span class="math inline">$\frac{\partial \text{fold-change}}{\partial K_A}$</span> and <span class="math inline">$\frac{\partial \text{fold-change}}{\partial K_I}$</span> scale linearly with <span class="math inline"><em>R</em></span> and <span class="math inline"><em>e</em><sup> − <em>β</em><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></sup></span>. Thus the width of the <span class="math inline"><em>R</em> = 22</span> strain will be roughly 1/1000 as large as that of the <span class="math inline"><em>R</em> = 1740</span> strain; similarly, the width of the O3 curves will be roughly 1/1000 the width of the O1 curves.</p>
<figure>
<img src="ch4_fig15.png" id="fig:ch4_fig15" data-short-caption="Determining how sensitive the fold-change values are to the fit values of the dissociation constants" alt="Figure 15: Determining how sensitive the fold-change values are to the fit values of the dissociation constants. The difference \Delta \text{fold-change}_{K_A} in fold change when the dissociation constant K_A is slightly offset from its best-fit value K_A=139^{+29}_{-22} \times 10^{-6} \, \text{M}, as given by . Fold-change is computed in the limit of saturating inducer concentration (c \to \infty, see ) where the credible regions in are widest. The O3 strain (\Delta\varepsilon_{RA} = -9.7~k_B T) is about 1/1000 as sensitive as the O1 operator to perturbations in the parameter values, and hence its credible region is roughly 1/1000 as wide. All curves were made using R = 260. As in Panel , but plotting the sensitivity of fold-change to the K_I parameter relative to the best-fit value K_I=0.53^{+0.04}_{-0.04} \times 10^{-6}\, \text{M}. Note that only the magnitude, and not the sign, of this difference describes the sensitivity of each parameter. Hence, the O3 strain is again less sensitive than the O1 and O2 strains. As in Panel , but showing how the fold-change sensitivity for different repressor copy numbers. The strains with lower repressor copy number are less sensitive to changes in the dissociation constants, and hence their corresponding curves in have tighter credible regions. All curves were made using \Delta\varepsilon_{RA} = -13.9~k_B T. As in Panel , the sensitivity of fold-change with respect to K_I is again smallest (in magnitude) for the low repressor copy number strains." /><figcaption aria-hidden="true">Figure 15: <strong>Determining how sensitive the fold-change values are to the fit values of the dissociation constants.</strong> The difference <span class="math inline"><em>Δ</em>fold-change<sub><em>K</em><sub><em>A</em></sub></sub></span> in fold change when the dissociation constant <span class="math inline"><em>K</em><sub><em>A</em></sub></span> is slightly offset from its best-fit value <span class="math inline"><em>K</em><sub><em>A</em></sub> = 139<sub> − 22</sub><sup> + 29</sup> × 10<sup> − 6</sup> M</span>, as given by . Fold-change is computed in the limit of saturating inducer concentration (<span class="math inline"><em>c</em> → ∞</span>, see ) where the credible regions in are widest. The O3 strain (<span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span>) is about 1/1000 as sensitive as the O1 operator to perturbations in the parameter values, and hence its credible region is roughly 1/1000 as wide. All curves were made using <span class="math inline"><em>R</em> = 260</span>. As in Panel , but plotting the sensitivity of fold-change to the <span class="math inline"><em>K</em><sub><em>I</em></sub></span> parameter relative to the best-fit value <span class="math inline"><em>K</em><sub><em>I</em></sub> = 0.53<sub> − 0.04</sub><sup> + 0.04</sup> × 10<sup> − 6</sup> M</span>. Note that only the magnitude, and not the sign, of this difference describes the sensitivity of each parameter. Hence, the O3 strain is again less sensitive than the O1 and O2 strains. As in Panel , but showing how the fold-change sensitivity for different repressor copy numbers. The strains with lower repressor copy number are less sensitive to changes in the dissociation constants, and hence their corresponding curves in have tighter credible regions. All curves were made using <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span>. As in Panel , the sensitivity of fold-change with respect to <span class="math inline"><em>K</em><sub><em>I</em></sub></span> is again smallest (in magnitude) for the low repressor copy number strains.</figcaption>
</figure>
<h2 id="alternate-characterizations-of-induction">Alternate Characterizations of Induction</h2>
<p>In this section we discuss a different way to describe the induction data, namely, through using the conventional Hill approach. We first demonstrate how using a Hill function to characterize a single induction curve enables us to extract features (such as the midpoint and sharpness) of that single response, but precludes any predictions of the other seventeen strains. We then discuss how a thermodynamic model of simple repression coupled with a Hill approach to the induction response can both characterize an induction profile and predict the response of all eighteen strains, although we argue that such a description provides no insight into the allosteric nature of the protein and how mutations to the repressor would affect induction. We conclude the section by discussing the differences between such a model and the statistical mechanical model used in the main text.</p>
<h3 id="fitting-induction-curves-using-a-hill-function-approach">Fitting Induction Curves using a Hill Function Approach</h3>
<p>The Hill equation is a phenomenological function commonly used to describe data with a sigmoidal profile <span class="citation" data-cites="Murphy2007 Murphy2010 Rogers2015"> [<a href="#ref-Murphy2007" role="doc-biblioref">23</a>–<a href="#ref-Rogers2015" role="doc-biblioref">25</a>]</span>. Its simplicity and ability to estimate the cooperativity of a system (through the Hill coefficient) has led to its widespread use in many domains of biology <span class="citation" data-cites="Frank2013"> [<a href="#ref-Frank2013" role="doc-biblioref">26</a>]</span>. Nevertheless, the Hill function is often criticized as a physically unrealistic model and the extracted Hill coefficient is often difficult to contextualize in the physics of a system <span class="citation" data-cites="Weiss1997"> [<a href="#ref-Weiss1997" role="doc-biblioref">27</a>]</span>. In the present work, we note that a Hill function, even if it is only used because of its simplicity, presents no mechanism to understand how a regulatory system’s behavior will change if physical parameters such as repressor copy number or operator binding energy are varied. In addition, the Hill equation provides no foundation to explore how mutating the repressor (e.g., at its inducer-binding interface) would modify its induction profile, although statistical mechanical models have proved capable of characterizing such scenarios <span class="citation" data-cites="Keymer2006 Swem2008 Einav2016"> [<a href="#ref-Keymer2006" role="doc-biblioref">28</a>–<a href="#ref-Einav2016" role="doc-biblioref">30</a>]</span>.</p>
<p>Consider the general Hill equation for a single induction profile given by <span id="eq:ch4_eq27"><span class="math display">$$
\text{fold-change} = (\text{leakiness}) + (\text{dynamic range}) 
\frac{\left( \frac{c}{K} \right)^n}{1 + \left( \frac{c}{K} \right)^n},
\qquad(27)$$</span></span> where, as in the main text, the leakiness represents the minimum fold-change, the dynamic range represents the difference between the maximum and minimum fold-change, <span class="math inline"><em>K</em></span> is the repressor-inducer dissociation constant, and <span class="math inline"><em>n</em></span> denotes the Hill coefficient that characterizes the sharpness of the curve (<span class="math inline"><em>n</em> &gt; 1</span> signifies positive cooperativity, <span class="math inline"><em>n</em> = 1</span> denotes no cooperativity, and <span class="math inline"><em>n</em> &lt; 1</span> represents negative cooperativity). shows how the individual induction profiles can be fit (using the same Bayesian methods as described in Section "") to this Hill response, yielding a similar response to that shown in . However, characterizing the induction response in this manner is unsatisfactory because each curve must be fit independently thus removing our predictive power for other repressor copy numbers and binding sites.</p>
<p>The fitted parameters obtained from this approach are shown in . These are rather unsatisfactory because they do not clearly reflect the properties of the physical system under consideration. For example, the dissociation constant <span class="math inline"><em>K</em></span> between LacI and inducer should not be affected by either the copy number of the repressor or the DNA binding energy, and yet we see upward trends as <span class="math inline"><em>R</em></span> is increased or the binding energy is decreased. Here, the <span class="math inline"><em>K</em></span> parameter ultimately describes the midpoint of the induction curve and therefore cannot strictly be considered a dissociation constant. Similarly, the Hill coefficient <span class="math inline"><em>n</em></span> does not directly represent the cooperativity between the repressor and the inducer as the molecular details of the copy number and DNA binding strength are subsumed in this parameter as well. While the leakiness and dynamic range describe important phenotypic properties of the induction response, this Hill approach leaves us with no means to predict them for other strains. In summary, the Hill equation cannot predict how an induction profile varies with repressor copy number, operator binding energy, or how mutations will alter the induction profile. To that end, we turn to a more sophisticated approach where we use the Hill function to describe the available fraction of repressor as a function of inducer concentration.</p>
<figure>
<img src="ch4_fig16.png" id="fig:ch4_fig16" data-short-caption="Hill function and MWC analysis of each induction profile" alt="Figure 16: Hill function and MWC analysis of each induction profile. Data for each individual strain was fit to the general Hill function in XXX. (A) strains with O1 binding site, (B) strains with O2 binding site, and (C) strains with O3 binding site. Shaded regions indicate the bounds of the 95% credible region." /><figcaption aria-hidden="true">Figure 16: <strong>Hill function and MWC analysis of each induction profile.</strong> Data for each individual strain was fit to the general Hill function in XXX. (A) strains with O1 binding site, (B) strains with O2 binding site, and (C) strains with O3 binding site. Shaded regions indicate the bounds of the 95% credible region.</figcaption>
</figure>
<figure>
<img src="ch4_fig17.png" id="fig:ch4_fig17" data-short-caption="Parameter values for the Hill equation fit to each individual titration" alt="Figure 17: Parameter values for the Hill equation fit to each individual titration. The resulting fit parameters from the Hill function fits of Fig. 16 are summarized. The large parameter intervals for many of the O3 strains are due to the flatter induction profile (as seen by its smaller dynamic range), and the ability for a large range of K and n values to describe the data." /><figcaption aria-hidden="true">Figure 17: <strong>Parameter values for the Hill equation fit to each individual titration.</strong> The resulting fit parameters from the Hill function fits of Fig. 16 are summarized. The large parameter intervals for many of the O3 strains are due to the flatter induction profile (as seen by its smaller dynamic range), and the ability for a large range of <span class="math inline"><em>K</em></span> and <span class="math inline"><em>n</em></span> values to describe the data.</figcaption>
</figure>
<h3 id="fitting-induction-curves-using-a-combination-thermodynamic-model-and-hill-function-approach">Fitting Induction Curves using a Combination Thermodynamic Model and Hill Function Approach</h3>
<p>Motivated by the inability in the previous section to characterize all eighteen strains using the Hill function with a single set of parameters, here we combine the Hill approach with a thermodynamic model of simple repression to garner predictive power. More specifically, we will use the thermodynamic model in but substitute the statistical model in with the phenomenological Hill function XXX.</p>
<p>Following , fold-change is given by <span id="eq:ch4_eq28"><span class="math display">$$
\text{fold-change} = \left( 1 + p_A(c) \frac{R}{N_{NS}}e^{-\beta
\Delta\varepsilon_{RA}} \right)^{-1}
\qquad(28)$$</span></span> where the Hill function <span id="eq:ch4_eq29"><span class="math display">$$
p_A(c) = p_A^{\text{max}} - p_A^{\text{range}}
\frac{\left( \frac{c}{K_D} \right)^n}{1 + \left( \frac{c}{K_D} \right)^n}
\qquad(29)$$</span></span> represents the fraction of repressors in the allosterically active state, with <span class="math inline"><em>p</em><sub><em>A</em></sub><sup>max</sup></span> denoting the fraction of active repressors in the absence of inducer and <span class="math inline"><em>p</em><sub><em>A</em></sub><sup>max</sup> − <em>p</em><sub><em>A</em></sub><sup>range</sup></span> the minimum fraction of active repressors in the presence of saturating inducer. The Hill function characterizes the inducer-repressor binding while the thermodynamic model with the known constants <span class="math inline"><em>R</em></span>, <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> describes how the induction profile changes with repressor copy number and repressor-operator binding energy.</p>
<p>As in the main text, we can fit the four Hill parameters – the vertical shift and stretch parameters <span class="math inline"><em>p</em><sub><em>A</em></sub><sup>max</sup></span> and <span class="math inline"><em>p</em><sub><em>A</em></sub><sup>range</sup></span>, the Hill coefficient <span class="math inline"><em>n</em></span>, and the inducer-repressor dissociation constant <span class="math inline"><em>K</em><sub><em>D</em></sub></span>–for a single induction curve and then use the fully characterized to describe the response of each of the eighteen strains. shows this process carried out by fitting the O2 <span class="math inline"><em>R</em> = 260</span> strain (white circles in Panel XXX) and predicting the behavior of the remaining seventeen strains.</p>
<figure>
<img src="ch4_fig18.png" id="fig:ch4_fig18" data-short-caption="A thermodynamic model coupled with a Hill analysis can characterize induction" alt="Figure 18: A thermodynamic model coupled with a Hill analysis can characterize induction. Combining a thermodynamic model of simple repression with the Hill function to characterize the repressor-inducer binding successfully characterizes the induction profiles of all eighteen strains. As in the main text, data was only fit for the O2 R=260 strain using and the parameters p_A^{\text{max}} =0.90^{+0.03}_{-0.01}, p_A^{\text{range}} = -0.90^{+0.02}_{-0.03}, n = 1.6_{-0.1}^{+0.2}, and K_D = 4^{+2}_{-1} \times 10^{-6}\,\text{M}. Shaded regions indicate bounds of the 95% credible region." /><figcaption aria-hidden="true">Figure 18: <strong>A thermodynamic model coupled with a Hill analysis can characterize induction.</strong> Combining a thermodynamic model of simple repression with the Hill function to characterize the repressor-inducer binding successfully characterizes the induction profiles of all eighteen strains. As in the main text, data was only fit for the O2 <span class="math inline"><em>R</em> = 260</span> strain using and the parameters <span class="math inline"><em>p</em><sub><em>A</em></sub><sup>max</sup> = 0.90<sub> − 0.01</sub><sup> + 0.03</sup></span>, <span class="math inline"><em>p</em><sub><em>A</em></sub><sup>range</sup> =  − 0.90<sub> − 0.03</sub><sup> + 0.02</sup></span>, <span class="math inline"><em>n</em> = 1.6<sub> − 0.1</sub><sup> + 0.2</sup></span>, and <span class="math inline"><em>K</em><sub><em>D</em></sub> = 4<sub> − 1</sub><sup> + 2</sup> × 10<sup> − 6</sup> M</span>. Shaded regions indicate bounds of the 95% credible region.</figcaption>
</figure>
<p>Although the curves in are nearly identical to those in (which were made using the MWC model ), we stress that the Hill function approach is more complex than the MWC model (containing four parameters instead of three) and it obscures the relationships to the physical parameters of the system. For example, it is not clear whether the fit parameter <span class="math inline"><em>K</em><sub><em>D</em></sub> = 4<sub> − 1</sub><sup> + 2</sup> × 10<sup> − 6</sup> M</span> relays the dissociation constant between the inducer and active-state repressor, between the inducer and the inactive-state repressor, or some mix of the two quantities.</p>
<p>In addition, the MWC model naturally suggests further quantitative tests for the fold-change relationship. For example, mutating the repressor’s inducer binding site would likely alter the repressor-inducer dissociation constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and it would be interesting to find out if such mutations also modify the allosteric energy difference <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> between the repressor’s active and inactive conformations. For our purposes, the Hill function falls short of the connection to the physics of the system and provides no intuition about how transcription depends upon such mutations. For these reasons, we present the thermodynamic model coupled with the statistical mechanical MWC model approach in the paper.</p>
<h2 id="global-fit-of-all-parameters">Global Fit of All Parameters</h2>
<p>In the main text, we used the repressor copy numbers <span class="math inline"><em>R</em></span> and repressor-DNA binding energies <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> as reported by <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span>. However, any error in these previous measurements of <span class="math inline"><em>R</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> will necessarily propagate into our own fold-change predictions. In this section we take an alternative approach to fitting the physical parameters of the system to that used in the main text. First, rather than fitting only a single strain, we fit the entire data set in along with microscopy data for the synthetic operator Oid (see Appendix XXX). In addition, we also simultaneously fit the parameters <span class="math inline"><em>R</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> using the prior information given by the previous measurements. By using the entire data set and fitting all of the parameters, we obtain the best possible characterization of the statistical mechanical parameters of the system given our current state of knowledge. As a point of reference, we state all of the parameters of the MWC model derived in the text in Table XXX.</p>
<p>To fit all of the parameters simultaneously, we follow a similar approach to the one detailed in the Methods section. Briefly, we perform a Bayesian parameter estimation of the dissociation constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, the six different repressor copy numbers <span class="math inline"><em>R</em></span> corresponding to the six <em>lacI</em> ribosomal binding sites used in our work, and the four different binding energies <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> characterizing the four distinct operators used to make the experimental strains. As in the main text, we fit the logarithms <span class="math inline">$\tilde{k}_A = -\log \frac{K_A}{1\,\text{M}}$</span> and <span class="math inline">$\tilde{k}_I = -\log \frac{K_I}{1\,\text{M}}$</span> of the dissociation constants which grants better numerical stability.</p>
<p>As in , we assume that deviations of the experimental fold-change from the theoretical predictions are normally distributed with mean zero and standard deviation <span class="math inline"><em>σ</em></span>. We begin by writing Bayes’ theorem, <span id="eq:ch4_eq30"><span class="math display">$$
P(\tilde{k}_A, \tilde{k}_I, \mathbf{R},
\mathbf{\Delta\boldsymbol{\varepsilon}_{\textbf{RA}}}, \sigma \mid D) =
\frac{P(D \mid \tilde{k}_A, \tilde{k}_I, \mathbf{R},
\mathbf{\Delta\boldsymbol{\varepsilon}_{\textbf{RA}}}, \sigma) P(\tilde{k}_A,
\tilde{k}_I, \mathbf{R},
\mathbf{\Delta\boldsymbol{\varepsilon}_{\textbf{RA}}}, \sigma)}{P(D)},
\qquad(30)$$</span></span> where <span class="math inline"><strong>R</strong></span> is an array containing the six different repressor copy numbers to be fit, <span class="math inline"><strong>Δ</strong><strong>ε</strong><sub><strong>RA</strong></sub></span> is an array containing the four binding energies to be fit, and <span class="math inline"><em>D</em></span> is the experimental fold-change data. The term <span class="math inline"><em>P</em>(<em>k̃</em><sub><em>A</em></sub>, <em>k̃</em><sub><em>I</em></sub>, <strong>R</strong>, <strong>Δ</strong><strong>ε</strong><sub><strong>RA</strong></sub>, <em>σ</em> ∣ <em>D</em>)</span> gives the probability distributions of all of the parameters given the data. The term <span class="math inline"><em>P</em>(<em>D</em> ∣ <em>k̃</em><sub><em>A</em></sub>, <em>k̃</em><sub><em>I</em></sub>, <strong>R</strong>, <strong>Δ</strong><strong>ε</strong><sub><strong>RA</strong></sub>, <em>σ</em>)</span> represents the likelihood of having observed our experimental data given some value for each parameter. <span class="math inline"><em>P</em>(<em>k̃</em><sub><em>A</em></sub>, <em>k̃</em><sub><em>I</em></sub>, <strong>R</strong>, <strong>Δ</strong><strong>ε</strong><sub><strong>RA</strong></sub>, <em>σ</em>)</span> contains all the prior information on the values of these parameters. Lastly, <span class="math inline"><em>P</em>(<em>D</em>)</span> serves as a normalization constant and hence can be ignored.</p>
<p>Given <span class="math inline"><em>n</em></span> independent measurements of the fold-change, the first term in can be written as <span id="eq:ch4_eq31"><span class="math display">$$
P(D \mid \tilde{k}_A, \tilde{k}_I, \mathbf{R},
\mathbf{\Delta\boldsymbol{\varepsilon}_{\textbf{RA}}}, \sigma) =
\frac{1}{(2\pi\sigma^2)^{\frac{n}{2}}}\prod\limits_{i=1}^n \exp
\left[-\frac{(\text{fc}^{(i)}_{\exp} - \text{fc}(\tilde{k}_A, \tilde{k}_I,
R^{(i)}, \Delta\varepsilon_{RA}^{(i)}, c^{(i)}))^2}{2\sigma^2}\right],
\qquad(31)$$</span></span> where <span class="math inline">fc<sub>exp</sub><sup>(<em>i</em>)</sup></span> is the <span class="math inline"><em>i</em><sup>th</sup></span> experimental fold-change and <span class="math inline">fc( ⋅  ⋅  ⋅ )</span> is the theoretical prediction. Note that the standard deviation <span class="math inline"><em>σ</em></span> of this distribution is not known and hence needs to be included as a parameter to be fit.</p>
<p>The second term in represents the prior information of the parameter values. We assume that all parameters are independent of each other, so that <span id="eq:ch4_eq32"><span class="math display"><em>P</em>(<em>k̃</em><sub><em>A</em></sub>, <em>k̃</em><sub><em>I</em></sub>, <strong>R</strong>, <strong>Δ</strong><strong>ε</strong><sub><strong>RA</strong></sub>, <em>σ</em>) = <em>P</em>(<em>k̃</em><sub><em>A</em></sub>) ⋅ <em>P</em>(<em>k̃</em><sub><em>I</em></sub>) ⋅ ∏<sub><em>i</em></sub><em>P</em>(<em>R</em><sup>(<em>i</em>)</sup>) ⋅ ∏<sub><em>j</em></sub><em>P</em>(<em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub><sup>(<em>j</em>)</sup>) ⋅ <em>P</em>(<em>σ</em>),   (32)</span></span> where the superscript <span class="math inline">(<em>i</em>)</span> indicates the repressor copy number of index <span class="math inline"><em>i</em></span> and the superscript <span class="math inline">(<em>j</em>)</span> denotes the binding energy of index <span class="math inline"><em>j</em></span>. As above, we note that a prior must also be included for the unknown parameter <span class="math inline"><em>σ</em></span>.</p>
<p>Because we knew nothing about the values of <span class="math inline"><em>k̃</em><sub><em>A</em></sub></span>, <span class="math inline"><em>k̃</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>σ</em></span> before performing the experiment, we assign maximally uninformative priors to each of these parameters. More specifically, we assign uniform priors to <span class="math inline"><em>k̃</em><sub><em>A</em></sub></span> and <span class="math inline"><em>k̃</em><sub><em>I</em></sub></span> and a Jeffreys prior to <span class="math inline"><em>σ</em></span>, indicating that <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>σ</em></span> are scale parameters <span class="citation" data-cites="Sivia2006"> [<a href="#ref-Sivia2006" role="doc-biblioref">31</a>]</span>. We do, however, have prior information for the repressor copy numbers and the repressor-DNA binding energies from <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span>. This prior knowledge is included within our model using an informative prior for these two parameters, which we assume to be Gaussian. Hence each of the <span class="math inline"><em>R</em><sup>(<em>i</em>)</sup></span> repressor copy numbers to be fit satisfies <span id="eq:ch4_eq33"><span class="math display">$$
P(R^{(i)}) = \frac{1}{\sqrt{2\pi\sigma_{R_i}^2}} 
\exp \left(-\frac{(R^{(i)} - \bar{R}^{(i)})^2}{2 \sigma_{R_i}^2} \right),
\qquad(33)$$</span></span> where <span class="math inline"><em>R̄</em><sup>(<em>i</em>)</sup></span> is the mean repressor copy number and <span class="math inline"><em>σ</em><sub><em>R</em><sub><em>i</em></sub></sub></span> is the variability associated with this parameter as reported in <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span>. Note that we use the given value of <span class="math inline"><em>σ</em><sub><em>R</em><sub><em>i</em></sub></sub></span> from previous measurements rather than leaving this as a free parameter.</p>
<p>Similarly, the binding energies <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub><sup>(<em>j</em>)</sup></span> are also assumed to have a Gaussian informative prior of the same form. We write it as <span id="eq:ch4_eq34"><span class="math display">$$
P(\Delta\varepsilon_{RA}^{(j)}) = 
\frac{1}{\sqrt{2\pi\sigma_{\varepsilon_j}^2}}
\exp \left(- \frac{(\Delta\varepsilon_{RA}^{(j)} -
\Delta\bar{\varepsilon}_{RA}^{(j)})^2}{2 \sigma_{\varepsilon_j}^2} \right),
\qquad(34)$$</span></span> where <span class="math inline"><em>Δ</em><em>ε̄</em><sub><em>R</em><em>A</em></sub><sup>(<em>j</em>)</sup></span> is the binding energy and <span class="math inline"><em>σ</em><sub><em>ε</em><sub><em>j</em></sub></sub></span> is the variability associated with that parameter around the mean value as reported in <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span> .</p>
<p>The <span class="math inline"><em>σ</em><sub><em>R</em><sub><em>i</em></sub></sub></span> and <span class="math inline"><em>σ</em><sub><em>ε</em><sub><em>j</em></sub></sub></span> parameters will constrain the range of values for <span class="math inline"><em>R</em><sup>(<em>i</em>)</sup></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub><sup>(<em>j</em>)</sup></span> found from the fitting. For example, if for some <span class="math inline"><em>i</em></span> the standard deviation <span class="math inline"><em>σ</em><sub><em>R</em><sub><em>i</em></sub></sub></span> is very small, it implies a strong confidence in the previously reported value. Mathematically, the exponential in will ensure that the best-fit <span class="math inline"><em>R</em><sup>(<em>i</em>)</sup></span> lies within a few standard deviations of <span class="math inline"><em>R̄</em><sup>(<em>i</em>)</sup></span>. Since we are interested in exploring which values could give the best fit, the errors are taken to be wide enough to allow the parameter estimation to freely explore parameter space in the vicinity of the best estimates. Putting all these terms together, we use Markov chain Monte Carlo to sample the posterior distribution <span class="math inline"><em>P</em>(<em>k̃</em><sub><em>A</em></sub>, <em>k̃</em><sub><em>I</em></sub>, <strong>R</strong>, <strong>Δ</strong><strong>ε</strong><sub><strong>RA</strong></sub>, <em>σ</em> ∣ <em>D</em>)</span>, enabling us to determine both the most likely value for each physical parameter as well as its associated credible region (see the <a href="https://rpgroup-pboc.github.io/mwc_induction/code/notebooks/global_fits.html">GitHub repository</a> for the implementation).</p>
<p>shows the result of this global fit. When compared with we can see that fitting for the binding energies and the repressor copy numbers improves the agreement between the theory and the data. Table XXX summarizes the values of the parameters as obtained with this MCMC parameter inference. We note that even though we allowed the repressor copy numbers and repressor-DNA binding energies to vary, the resulting fit values were very close to the previously reported values. The fit values of the repressor copy numbers were all within one standard deviation of the previous reported values provided in <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span>. And although some of the repressor-DNA binding energies differed by a few standard deviations from the reported values, the differences were always less than <span class="math inline">1 <em>k</em><sub><em>B</em></sub><em>T</em></span>, which represents a small change in the biological scales we are considering. The biggest discrepancy between our fit values and the previous measurements arose for the synthetic Oid operator, which we discuss in more detail in Appendix XXX.</p>
<p>XXX shows the same key properties as in XXX, but uses the parameters obtained from this global fitting approach. We note that even by increasing the number of degrees of freedom in our fit, the result does not change substantially, due to in general, only minor improvements between the theoretical curves and data. For the O3 operator data, again, agreement between the predicted <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span> and the effective Hill coefficient remain poor due the theory being unable to capture the steepness of the response curves.</p>
<div id="tbl:ch4_tbl02">
<table>
<caption>Table 2: <strong>Key model parameters for induction of an allosteric repressor.</strong></caption>
<colgroup>
<col style="width: 30%" />
<col style="width: 69%" />
</colgroup>
<thead>
<tr class="header">
<th><strong>Parameter</strong></th>
<th><strong>Description</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><span class="math inline"><em>c</em></span></td>
<td>Concentration of the inducer</td>
</tr>
<tr class="even">
<td><span class="math inline"><em>K</em><sub><em>A</em></sub>, <em>K</em><sub><em>I</em></sub></span></td>
<td>Dissociation constant between an inducer and the repressor in the active/inactive state</td>
</tr>
<tr class="odd">
<td><span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span></td>
<td>The difference between the free energy of repressor in the inactive and active states</td>
</tr>
<tr class="even">
<td><span class="math inline"><em>Δ</em><em>ε</em><sub><em>P</em></sub></span></td>
<td>Binding energy between the RNAP and its specific binding site</td>
</tr>
<tr class="odd">
<td><span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub>, <em>Δ</em><em>ε</em><sub><em>R</em><em>I</em></sub></span></td>
<td>Binding energy between the operator and the active/inactive repressor</td>
</tr>
<tr class="even">
<td><span class="math inline"><em>n</em></span></td>
<td>Number of inducer binding sites per repressor</td>
</tr>
<tr class="odd">
<td><span class="math inline"><em>P</em></span></td>
<td>Number of RNAP</td>
</tr>
<tr class="even">
<td><span class="math inline"><em>R</em><sub><em>A</em></sub>, <em>R</em><sub><em>I</em></sub>, <em>R</em></span></td>
<td>Number of active/inactive/total repressors</td>
</tr>
<tr class="odd">
<td><span class="math inline">$p_A = \frac{R_A}{R}$</span></td>
<td>Probability that a repressor will be in the active state</td>
</tr>
<tr class="even">
<td><span class="math inline"><em>p</em><sub>bound</sub></span></td>
<td>Probability that an RNAP is bound to the promoter of interest, assumed to be proportional to gene expression</td>
</tr>
<tr class="odd">
<td><span class="math inline">fold-change</span></td>
<td>Ratio of gene expression in the presence of repressor to that in the absence of repressor</td>
</tr>
<tr class="even">
<td><span class="math inline"><em>F</em></span></td>
<td>Free energy of the system</td>
</tr>
<tr class="odd">
<td><span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span></td>
<td>The number of non-specific binding sites for the repressor in the genome</td>
</tr>
<tr class="even">
<td><span class="math inline">$\beta = \frac{1}{k_B T}$</span></td>
<td>The inverse product of the Boltzmann constant <span class="math inline"><em>k</em><sub><em>B</em></sub></span> and the temperature <span class="math inline"><em>T</em></span> of the system</td>
</tr>
</tbody>
</table>
</div>
<figure>
<img src="ch4_fig19.png" id="fig:ch4_fig19" data-short-caption="Global fit of dissociation constants, repressor copy numbers and binding energies" alt="Figure 19: Global fit of dissociation constants, repressor copy numbers and binding energies. Theoretical predictions resulting from simultaneously fitting the dissociation constants K_A and K_I, the six repressor copy numbers R, and the four repressor-DNA binding energies \Delta\varepsilon_{RA} using the entire data set from XXX as well as the microscopy data for the Oid operator. Error bars of experimental data show the standard error of the mean (eight or more replicates) and shaded regions denote the 95% credible region. Where error bars are not visible, they are smaller than the point itself. For the Oid operator, all of the data points are shown since a smaller number of replicates were taken. The shaded regions are significantly smaller than in XXX because this fit was based on all data points, and hence the fit parameters are much more tightly constrained. The dashed lines at 0 IPTG indicates a linear scale, whereas solid lines represent a log scale." /><figcaption aria-hidden="true">Figure 19: <strong>Global fit of dissociation constants, repressor copy numbers and binding energies.</strong> Theoretical predictions resulting from simultaneously fitting the dissociation constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, the six repressor copy numbers <span class="math inline"><em>R</em></span>, and the four repressor-DNA binding energies <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> using the entire data set from XXX as well as the microscopy data for the Oid operator. Error bars of experimental data show the standard error of the mean (eight or more replicates) and shaded regions denote the 95% credible region. Where error bars are not visible, they are smaller than the point itself. For the Oid operator, all of the data points are shown since a smaller number of replicates were taken. The shaded regions are significantly smaller than in XXX because this fit was based on all data points, and hence the fit parameters are much more tightly constrained. The dashed lines at 0 IPTG indicates a linear scale, whereas solid lines represent a log scale.</figcaption>
</figure>
<figure>
<img src="ch4_fig20.png" id="fig:ch4_fig20" data-short-caption="Key properties of induction profiles as predicted with a global fit using all available data" alt="Figure 20: Key properties of induction profiles as predicted with a global fit using all available data. Data for the (A) leakiness, (B) saturation, and (C) dynamic range are obtained from fold-change measurements in XXX in the absence and presence of IPTG. All prediction curves were generated using the parameters listed in . Both the (D) [EC_{50}] and (E) effective Hill coefficient are inferred by individually fitting all parameters–K_A,\, K_I,\, R,\, \Delta\varepsilon_{RA}–to each operator-repressor pairing in XXX(A)-(C) separately to Eq. ¿eq:fold_change_full? in order to smoothly interpolate between the data points. Note that where error bars are not visible, this indicates that the error bars are smaller than the point itself." /><figcaption aria-hidden="true">Figure 20: <strong>Key properties of induction profiles as predicted with a global fit using all available data.</strong> Data for the (A) leakiness, (B) saturation, and (C) dynamic range are obtained from fold-change measurements in XXX in the absence and presence of IPTG. All prediction curves were generated using the parameters listed in . Both the (D) <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span> and (E) effective Hill coefficient are inferred by individually fitting all parameters–<span class="math inline"><em>K</em><sub><em>A</em></sub>, <em>K</em><sub><em>I</em></sub>, <em>R</em>, <em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>–to each operator-repressor pairing in XXX(A)-(C) separately to Eq. <strong>¿eq:fold_change_full?</strong> in order to smoothly interpolate between the data points. Note that where error bars are not visible, this indicates that the error bars are smaller than the point itself.</figcaption>
</figure>
<div id="tbl:ch4_tbl03">
<table>
<caption>Table 3: <strong>Global fit of all parameter values using the entire data set in XXX.</strong> In addition to fitting the repressor inducer dissociation constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> as was done in the text, we also fit the repressor DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> as well as the repressor copy numbers <span class="math inline"><em>R</em></span> for each strain. The middle columns show the previously reported values for all <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>R</em></span> values, with <span class="math inline">±</span> representing the standard deviation of three replicates. The right column shows the global fits from this work, with the subscript and superscript notation denoting the 95% credible region. Note that there is overlap between all of the repressor copy numbers and that the net difference in the repressor-DNA binding energies is less than <span class="math inline">1 <em>k</em><sub><em>B</em></sub><em>T</em></span>. The logarithms <span class="math inline">$\tilde{k}_A = -\log \frac{K_A}{1\,\text{M}}$</span> and <span class="math inline">$\tilde{k}_I = -\log \frac{K_I}{1\,\text{M}}$</span> of the dissociation constants were fit for numerical stability.</caption>
<thead>
<tr class="header">
<th></th>
<th><strong>Reported Values <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span></strong></th>
<th><strong>Global Fit</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><span class="math inline"><em>k̃</em><sub><em>A</em></sub></span></td>
<td><span class="math inline">−</span></td>
<td><span class="math inline"> − 5.33<sub> − 0.05</sub><sup> + 0.06</sup></span></td>
</tr>
<tr class="even">
<td><span class="math inline"><em>k̃</em><sub><em>I</em></sub></span></td>
<td><span class="math inline">−</span></td>
<td><span class="math inline">0.31<sub> − 0.06</sub><sup> + 0.05</sup></span></td>
</tr>
<tr class="odd">
<td><span class="math inline"><em>K</em><sub><em>A</em></sub></span></td>
<td><span class="math inline">−</span></td>
<td><span class="math inline">205<sub> − 12</sub><sup> + 11</sup> <em>μ</em>M</span></td>
</tr>
<tr class="even">
<td><span class="math inline"><em>K</em><sub><em>I</em></sub></span></td>
<td><span class="math inline">−</span></td>
<td><span class="math inline">0.73<sub> − 0.04</sub><sup> + 0.04</sup> <em>μ</em>M</span></td>
</tr>
<tr class="odd">
<td><span class="math inline"><em>R</em><sub>22</sub></span></td>
<td><span class="math inline">22 ± 4</span></td>
<td><span class="math inline">20<sub> − 1</sub><sup> + 1</sup></span></td>
</tr>
<tr class="even">
<td><span class="math inline"><em>R</em><sub>60</sub></span></td>
<td><span class="math inline">60 ± 20</span></td>
<td><span class="math inline">74<sub> − 3</sub><sup> + 4</sup></span></td>
</tr>
<tr class="odd">
<td><span class="math inline"><em>R</em><sub>124</sub></span></td>
<td><span class="math inline">124 ± 30</span></td>
<td><span class="math inline">130<sub> − 6</sub><sup> + 6</sup></span></td>
</tr>
<tr class="even">
<td><span class="math inline"><em>R</em><sub>260</sub></span></td>
<td><span class="math inline">260 ± 40</span></td>
<td><span class="math inline">257<sub> − 11</sub><sup> + 9</sup></span></td>
</tr>
<tr class="odd">
<td><span class="math inline"><em>R</em><sub>1220</sub></span></td>
<td><span class="math inline">1220 ± 160</span></td>
<td><span class="math inline">1191<sub> − 55</sub><sup> + 32</sup></span></td>
</tr>
<tr class="even">
<td><span class="math inline"><em>R</em><sub>1740</sub></span></td>
<td><span class="math inline">1740 ± 340</span></td>
<td><span class="math inline">1599<sub> − 87</sub><sup> + 75</sup></span></td>
</tr>
<tr class="odd">
<td>O1 <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span></td>
<td><span class="math inline"> − 15.3 ± 0.2 <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td><span class="math inline"> − 15.2<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
</tr>
<tr class="even">
<td>O2 <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span></td>
<td><span class="math inline"> − 13.9 ± 0.2 <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td><span class="math inline"> − 13.6<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
</tr>
<tr class="odd">
<td>O3 <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span></td>
<td><span class="math inline"> − 9.7 ± 0.1 <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td><span class="math inline"> − 9.4<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
</tr>
<tr class="even">
<td>Oid <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span></td>
<td><span class="math inline"> − 17.0 ± 0.2 <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td><span class="math inline"> − 17.7<sub> − 0.1</sub><sup> + 0.2</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
</tr>
</tbody>
</table>
</div>
<h2 id="applicability-of-theory-to-the-oid-operator-sequence">Applicability of Theory to the Oid Operator Sequence</h2>
<p>In addition to the native operator sequences (O1, O2, and O3) considered in the main text, we were also interested in testing our model predictions against the synthetic Oid operator. In contrast to the other operators, Oid is one base pair shorter in length (<span class="math inline">20 bp</span>), is fully symmetric, and is known to provide stronger repression than the native operator sequences considered so far. While the theory should be similarly applicable, measuring the lower fold-changes associated with this YFP construct was expected to be near the sensitivity limit for our flow cytometer, due to the especially strong binding energy of Oid (<span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span>) <span class="citation" data-cites="Garcia2011b"> [<a href="#ref-Garcia2011b" role="doc-biblioref">20</a>]</span>. Accordingly, fluorescence data for Oid were obtained using microscopy, which is more sensitive than flow cytometry. Appendix <a href="#AppendixMicroscopy" data-reference-type="ref" data-reference="AppendixMicroscopy">[AppendixMicroscopy]</a> gives a detailed explanation of how microscopy measurements were used to obtain induction curves.</p>
<p>We follow the approach of the main text and make fold-change predictions based on the parameter estimates from our strain with <span class="math inline"><em>R</em> = 260</span> and an O2 operator. These predictions are shown in , where we also plot data taken in triplicate for strains containing <span class="math inline"><em>R</em> = 22</span>, 60, and 124, obtained by single-cell microscopy. We find that the data are systematically below the theoretical predictions. We also considered our global fitting approach (see Appendix <a href="#appendix_global_fit" data-reference-type="ref" data-reference="appendix_global_fit">[appendix_global_fit]</a>) to see whether we might find better agreement with the observed data. Interestingly, we findthat the majority of the parameters remain largely unchanged, but our estimate for the Oid binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> is shifted to <span class="math inline"> − 17.7 <em>k</em><sub><em>B</em></sub><em>T</em></span> instead of the value <span class="math inline"> − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span> found by <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span>. In we again plot the Oid fold-change data but with theoretical predictions using the new estimate for the Oid binding energy from our global fit and find substantially better agreement.</p>
<figure>
<img src="ch4_fig21.png" id="fig:ch4_fig21" data-short-caption="Predictions of fold-change for strains with an Oid binding sequence versus experimental measurements with different repressor copy numbers" alt="Figure 21: Predictions of fold-change for strains with an Oid binding sequence versus experimental measurements with different repressor copy numbers. Experimental data is plotted against the parameter-free predictions that are based on our fit to the O2 strain with R=260. Here we use the previously measured binding energy \Delta\varepsilon_{RA}=-17.0~k_BT  [4]. The same experimental data is plotted against the best-fit parameters using the complete O1, O2, O3, and Oid data sets to infer K_A, K_I, repressor copy numbers, and the binding energies of all operators (see Appendix [appendix_global_fit]). Here the major difference in the inferred parameters is a shift in the binding energy for Oid from \Delta\varepsilon_{RA}=-17.0~k_BT to \Delta\varepsilon_{RA}=-17.7~k_BT, which now shows agreement between the theoretical predictions and experimental data. Shaded regions from the theoretical curves denote the 95% credible region. These are narrower in Panel because the inference of parameters was performed with much more data, and hence the best-fit values are more tightly constrained. Individual data points are shown due to the small number of replicates. The dashed lines at 0 IPTG indicate a linear scale, whereas solid lines represent a log scale." /><figcaption aria-hidden="true">Figure 21: <strong>Predictions of fold-change for strains with an Oid binding sequence versus experimental measurements with different repressor copy numbers.</strong> Experimental data is plotted against the parameter-free predictions that are based on our fit to the O2 strain with <span class="math inline"><em>R</em> = 260</span>. Here we use the previously measured binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span>. The same experimental data is plotted against the best-fit parameters using the complete O1, O2, O3, and Oid data sets to infer <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, repressor copy numbers, and the binding energies of all operators (see Appendix <a href="#appendix_global_fit" data-reference-type="ref" data-reference="appendix_global_fit">[appendix_global_fit]</a>). Here the major difference in the inferred parameters is a shift in the binding energy for Oid from <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span> to <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 17.7 <em>k</em><sub><em>B</em></sub><em>T</em></span>, which now shows agreement between the theoretical predictions and experimental data. Shaded regions from the theoretical curves denote the 95% credible region. These are narrower in Panel because the inference of parameters was performed with much more data, and hence the best-fit values are more tightly constrained. Individual data points are shown due to the small number of replicates. The dashed lines at 0 IPTG indicate a linear scale, whereas solid lines represent a log scale.</figcaption>
</figure>
<p>Fig. 22 shows the cumulative data from <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span> and <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">1</a>]</span>, as well as our data with <span class="math inline"><em>c</em> = 0 <em>μ</em>M</span>, which all measured fold-change for the same simple repression architecture utilizing different reporters and measurement techniques. We find that the binding energies from the global fit, including <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 17.7 <em>k</em><sub><em>B</em></sub><em>T</em></span>, compare reasonably well with all previous measurements.</p>
<figure>
<img src="ch4_fig22.png" id="fig:ch4_fig22" data-short-caption="Comparison of fold-change predictions based on binding energies from Garcia and Phillips and those inferred from this work" alt="Figure 22: Comparison of fold-change predictions based on binding energies from Garcia and Phillips and those inferred from this work. Fold-change curves for the different repressor-DNA binding energies \Delta\varepsilon_{RA} are plotted as a function of repressor copy number when IPTG concentration c=0. Solid curves use the binding energies determined from  [4], while the dashed curves use the inferred binding energies we obtained when performing a global fit of K_A, K_I, repressor copy numbers, and the binding energies using all available data from our work. Fold-change measurements from our experiments (outlined circles)  [4] (solid circles), and  [1] (diamonds) show that the small shifts in binding energy that we infer are still in agreement with prior data. Note that only a single flow cytometry data point is shown for Oid from this study, since the R=60 and R=124 curves from had extremely low fold-change in the absence of inducer (c=0) so as to be indistinguishable from autofluorescence, and in fact their fold-change values in this limit were negative and hence do not appear on this plot." /><figcaption aria-hidden="true">Figure 22: <strong>Comparison of fold-change predictions based on binding energies from Garcia and Phillips and those inferred from this work.</strong> Fold-change curves for the different repressor-DNA binding energies <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> are plotted as a function of repressor copy number when IPTG concentration <span class="math inline"><em>c</em> = 0</span>. Solid curves use the binding energies determined from <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span>, while the dashed curves use the inferred binding energies we obtained when performing a global fit of <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, repressor copy numbers, and the binding energies using all available data from our work. Fold-change measurements from our experiments (outlined circles) <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span> (solid circles), and <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">1</a>]</span> (diamonds) show that the small shifts in binding energy that we infer are still in agreement with prior data. Note that only a single flow cytometry data point is shown for Oid from this study, since the <span class="math inline"><em>R</em> = 60</span> and <span class="math inline"><em>R</em> = 124</span> curves from had extremely low fold-change in the absence of inducer (<span class="math inline"><em>c</em> = 0</span>) so as to be indistinguishable from autofluorescence, and in fact their fold-change values in this limit were negative and hence do not appear on this plot.</figcaption>
</figure>
<h2 id="comparison-of-parameter-estimation-and-fold-change-predictions-across-strains">Comparison of Parameter Estimation and Fold-Change Predictions across Strains</h2>
<p>The inferred parameter values for <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> in the main text were determined by fitting to induction fold-change measurements from a single strain (<span class="math inline"><em>R</em> = 260</span>, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span>, <span class="math inline"><em>n</em> = 2</span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> = 4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span>). After determining these parameters, we were able to predict the fold-change of the remaining strains without any additional fitting. However, the theory should be independent of the specific strain used to estimate <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>; using any alternative strain to fit <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> should yield similar predictions. For the sake of completeness, here we discuss the values for <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> that are obtained by fitting to each of the induction data sets individually. These fit parameters are shown in of the main text, where we find close agreement between strains, but with some deviation and poorer inferences observed with the O3 operator strains. Overall, we find that regardless of which strain is chosen to determine the unknown parameters, the predictions laid out by the theory closely match the experimental measurements. Here we present a comparison of the strain specific predictions and measured fold-change data for each of the three operators considered.</p>
<p>We follow the approach taken in the main text and use to infer values for <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> by fitting to each combination of binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and repressor copy number <span class="math inline"><em>R</em></span>. We then use these fitted parameters to predict the induction curves of all other strains. In we plot these fold-change predictions along with experimental data for each of our strains that contains an O1 operator. To make sense of this plot consider the first row as an example. In the first row, <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> were estimated using data from the strain containing <span class="math inline"><em>R</em> = 22</span> and an O1 operator (top leftmost plot, shaded in gray). The remaining plots in this row show the predicted fold-change using these values for <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>. In each row, we then infer <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> using data from a strain containing a different repressor copy number (<span class="math inline"><em>R</em> = 60</span> in the second row, <span class="math inline"><em>R</em> = 124</span> in the third row, and so on). In and , we similarly apply this inference to our strains with O2 and O3 operators, respectively. We note that the overwhelming majority of predictions closely match the experimental data.The notable exception is that using the <span class="math inline"><em>R</em> = 22</span> strain provides poor predictions for the strains with large copy numbers (especially <span class="math inline"><em>R</em> = 1220</span> and <span class="math inline"><em>R</em> = 1740</span>), though it should be noted that predictions made from the <span class="math inline"><em>R</em> = 22</span> strain have considerably broader credible regions. This loss in predictive power is due to the poorer estimates of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> for the <span class="math inline"><em>R</em> = 22</span> strain as shown in .</p>
<figure>
<img src="ch4_fig23.png" id="fig:ch4_fig23" data-short-caption="O1 strain fold-change predictions based on strain-specific parameter estimation of $\boldsymbol{K_A}$ and $\boldsymbol{K_I}$" alt="Figure 23: O1 strain fold-change predictions based on strain-specific parameter estimation of \boldsymbol{K_A} and \boldsymbol{K_I}. Fold-change in expression is plotted as a function of IPTG concentration for all strains containing an O1 operator. The solid points correspond to the mean experimental value. The solid lines correspond to using the parameter estimates of K_A and K_I. Each row uses a single set of parameter values based on the strain noted on the left axis. The shaded plots along the diagonal are those where the parameter estimates are plotted along with the data used to infer them. Values for repressor copy number and operator binding energy are from  [4]. The shaded region on the curve represents the uncertainty from our parameter estimates and reflect the 95% highest probability density region of the parameter predictions." /><figcaption aria-hidden="true">Figure 23: <strong>O1 strain fold-change predictions based on strain-specific parameter estimation of <span class="math inline"><strong>K</strong><sub><strong>A</strong></sub></span> and <span class="math inline"><strong>K</strong><sub><strong>I</strong></sub></span>.</strong> Fold-change in expression is plotted as a function of IPTG concentration for all strains containing an O1 operator. The solid points correspond to the mean experimental value. The solid lines correspond to using the parameter estimates of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>. Each row uses a single set of parameter values based on the strain noted on the left axis. The shaded plots along the diagonal are those where the parameter estimates are plotted along with the data used to infer them. Values for repressor copy number and operator binding energy are from <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span>. The shaded region on the curve represents the uncertainty from our parameter estimates and reflect the 95% highest probability density region of the parameter predictions.</figcaption>
</figure>
<figure>
<img src="ch4_fig24.png" id="fig:ch4_fig24" data-short-caption="O2 strain fold-change predictions based on strain-specific parameter estimation of $\boldsymbol{K_A}$ and $\boldsymbol{K_I}$" alt="Figure 24: O2 strain fold-change predictions based on strain-specific parameter estimation of \boldsymbol{K_A} and \boldsymbol{K_I}. Fold-change in expression is plotted as a function of IPTG concentration for all strains containing an O2 operator. The plots and data shown are analogous to , but for the O2 operator." /><figcaption aria-hidden="true">Figure 24: <strong>O2 strain fold-change predictions based on strain-specific parameter estimation of <span class="math inline"><strong>K</strong><sub><strong>A</strong></sub></span> and <span class="math inline"><strong>K</strong><sub><strong>I</strong></sub></span>.</strong> Fold-change in expression is plotted as a function of IPTG concentration for all strains containing an O2 operator. The plots and data shown are analogous to , but for the O2 operator.</figcaption>
</figure>
<figure>
<img src="ch4_fig25.png" id="fig:ch4_fig25" data-short-caption="O3 strain fold-change predictions based on strain-specific parameter estimation of $\boldsymbol{K_A}$ and $\boldsymbol{K_I}$" alt="Figure 25: O3 strain fold-change predictions based on strain-specific parameter estimation of \boldsymbol{K_A} and \boldsymbol{K_I}. Fold-change in expression is plotted as a function of IPTG concentration for all strains containing an O3 operator. The plots and data shown are analogous to , but for the O3 operator. We note that when using the R=22 O3 strain to predict K_A and K_I, the large uncertainty in the estimates of these parameters (see ) leads to correspondingly wider credible regions." /><figcaption aria-hidden="true">Figure 25: <strong>O3 strain fold-change predictions based on strain-specific parameter estimation of <span class="math inline"><strong>K</strong><sub><strong>A</strong></sub></span> and <span class="math inline"><strong>K</strong><sub><strong>I</strong></sub></span>.</strong> Fold-change in expression is plotted as a function of IPTG concentration for all strains containing an O3 operator. The plots and data shown are analogous to , but for the O3 operator. We note that when using the <span class="math inline"><em>R</em> = 22</span> O3 strain to predict <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, the large uncertainty in the estimates of these parameters (see ) leads to correspondingly wider credible regions.</figcaption>
</figure>
<h2 id="properties-of-induction-titration-curves">Properties of Induction Titration Curves</h2>
<p>In this section, we expand on the phenotypic properties of the induction response that were explored in the main text (see ). We begin by expanding on our discussion of dynamic range and then show the analytic form of the <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span> for simple repression.</p>
<p>As stated in the main text, the dynamic range is defined as the difference between the maximum and minimum system response, or equivalently, as the difference between the saturation and leakiness of the system. Using , the dynamic range is given by <span id="eq:ch4_eq35"><span class="math display">$$
\text{dynamic range} = 
\left(
1+\frac{1}{1+e^{-\beta \Delta \varepsilon_{AI} }
 \left(\frac{K_A}{K_I}\right)^n }\frac{R}{N_{NS}}
 e^{-\beta \Delta\varepsilon_{RA}} \right)^{-1} - 
 \left(
1+\frac{1}{1+e^{-\beta \Delta \varepsilon_{AI} }}
\frac{R}{N_{NS}}e^{-\beta \Delta\varepsilon_{RA}} \right)^{-1}.
\qquad(35)$$</span></span> The dynamic range, along with saturation and leakiness were plotted with our experimental data in - as a function of repressor copy number. shows how these properties are expected to vary as a function of the repressor-operator binding energy. Note that the resulting curves for all three properties have the same shape as in -, since the dependence of the fold-change upon the repressor copy number and repressor-operator binding energy are both contained in a single multiplicative term, <span class="math inline"><em>R</em><em>e</em><sup> − <em>β</em><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></sup></span>. Hence, increasing <span class="math inline"><em>R</em></span> on a logarithmic scale (as in -) is equivalent to decreasing <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> on a linear scale (as in ).</p>
<p>An interesting aspect of the dynamic range is that it exhibits a peak as a function of either the repressor copy number (or equivalently of the repressor-operator binding energy). Differentiating the dynamic range and setting it equal to zero, we find that this peak occurs at <span id="eq:ch4_eq36"><span class="math display">$$
\frac{R^*}{N_{NS}} = e^{-\beta (\Delta\varepsilon_{AI} -
\Delta\varepsilon_{RA})} \sqrt{e^{\Delta\varepsilon_{AI}} + 1} 
\sqrt{e^{\Delta\varepsilon_{AI}} + \left( \frac{K_A}{K_I} \right)^n}.
\qquad(36)$$</span></span> The magnitude of the peak is given by <span id="eq:ch4_eq37"><span class="math display">$$
\text{max dynamic range} = 
\frac{\left( \sqrt{e^{\Delta\varepsilon_{AI}} + 1} - 
\sqrt{e^{\Delta\varepsilon_{AI}} + \left( \frac{K_A}{K_I} \right)^n} 
\right)^2}{\left( \frac{K_A}{K_I} \right)^n - 1},
\qquad(37)$$</span></span> which is independent of the repressor-operator binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> or <span class="math inline"><em>R</em></span>, and will only cause a shift in the location of the peak but not its magnitude.</p>
<figure>
<img src="ch4_fig26.png" id="fig:ch4_fig26" data-short-caption="Dependence of leakiness, saturation, and dynamic range on the operator binding energy and repressor copy number." alt="Figure 26: Dependence of leakiness, saturation, and dynamic range on the operator binding energy and repressor copy number. Increasing repressor copy number or decreasing the repressor-operator binding energy suppresses gene expression and decreases both the leakiness and saturation. The dynamic range retains its shape but shifts right as the repressor copy number increases. The peak in the dynamic range can be understood by considering the two extremes for \Delta \varepsilon_{RA}: for small repressor-operator binding energies, the leakiness is small but the saturation increases with \Delta \varepsilon_{RA}; for lare repressor-operator binding energies the saturation is near unity and the leakiness increases with \Delta \varepsilon_{RA}, thereby decreasing the dynamic range. Repressor copy number does not affect the maximum dynamic range (see ). Circles, diamonds, and squares represent \Delta \varepsilon_{RA} values for the O1, O2, and O3 operators, respectively, demonstrating the expected values of the properties using those strains." /><figcaption aria-hidden="true">Figure 26: <strong>Dependence of leakiness, saturation, and dynamic range on the operator binding energy and repressor copy number.</strong> Increasing repressor copy number or decreasing the repressor-operator binding energy suppresses gene expression and decreases both the leakiness and saturation. The dynamic range retains its shape but shifts right as the repressor copy number increases. The peak in the dynamic range can be understood by considering the two extremes for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>: for small repressor-operator binding energies, the leakiness is small but the saturation increases with <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>; for lare repressor-operator binding energies the saturation is near unity and the leakiness increases with <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>, thereby decreasing the dynamic range. Repressor copy number does not affect the maximum dynamic range (see ). Circles, diamonds, and squares represent <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> values for the O1, O2, and O3 operators, respectively, demonstrating the expected values of the properties using those strains.</figcaption>
</figure>
<p>We now consider the two remaining properties, the <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span> and effective Hill coefficient, which determine the horizontal properties of a system - that is, they determine the range of inducer concentration in which the system’s response goes from its minimum to maximum values. The <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span> denotes the inducer concentration required to generate fold-change halfway between its minimum and maximum value and was defined implicitly in . For the simple repression system, the <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span> is given by <span id="eq:ch4_eq38"><span class="math display">$$
\frac{[EC_{50}]}{K_A} = 
\frac{\frac{K_A}{K_I} - 1}
{\frac{K_A}{K_I} - \left( \frac{\left( 1 + \frac{R}{N_{NS} }
e^{-\beta \Delta\varepsilon_{RA}} \right) + 
\left( \frac{K_A}{K_I} \right)^n 
\left( 2 e^{-\beta \Delta \varepsilon_{AI}} + 
\left( 1 + \frac{R}{N_{NS}} e^{-\beta \Delta\varepsilon_{RA}} \right) 
\right) }
{ 2 \left( 1 + \frac{R}{N_{NS}} e^{-\beta \Delta\varepsilon_{RA}} \right) +
 e^{-\beta \Delta \varepsilon_{AI}} + 
 \left( \frac{K_A}{K_I} \right)^n e^{-\beta \Delta \varepsilon_{AI}} } 
 \right)^{\frac{1}{n}}} - 1.
\qquad(38)$$</span></span> Using this expression, we can then find the effective Hill coefficient <span class="math inline"><em>h</em></span>, which equals twice the log-log slope of the normalized fold-change evaluated at <span class="math inline"><em>c</em> = [<em>E</em><em>C</em><sub>50</sub>]</span> (see ). In - we show how these two properties vary with repressor copy number, and in we demonstrate how they depend on the repressor-operator binding energy. Both the <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span> and <span class="math inline"><em>h</em></span> vary significantly with repressor copy number for sufficiently strong operator binding energies. Interestingly, for weak operator binding energies on the order of the O3 operator, it is predicted that the effective Hill coefficient should not vary with repressor copy number. In addition, the maximum possible Hill coefficient is roughly 1.75, which stresses the point that the effective Hill coefficient should not be interpreted as the number of inducer binding sites, which is exactly 2.</p>
<figure>
<img src="ch4_fig27.png" id="fig:ch4_fig27" data-short-caption="**$\boldsymbol{[EC_{50}]}$ and effective Hill coefficient depend strongly on repressor copy number and operator binding energy." alt="Figure 27: \boldsymbol{[EC_{50}]} and effective Hill coefficient depend strongly on repressor copy number and operator binding energy. [EC_{50}] values range from very small and tightly clustered at weak operator binding energies (e.g. O3) to relatively large and spread out for stronger operator binding energies (O1 and O2). The effective Hill coefficient generally decreases with increasing repressor copy number, indicating a flatter normalized response. The maximum possible Hill coefficient is roughly 1.75 for all repressor-operator binding energies. Circles, diamonds, and squares represent \Delta \varepsilon_{RA} values for the O1, O2, and O3 operators, respectively." /><figcaption aria-hidden="true">Figure 27: <strong><span class="math inline"><strong>[</strong><strong>E</strong><strong>C</strong><sub><strong>50</strong></sub><strong>]</strong></span> and effective Hill coefficient depend strongly on repressor copy number and operator binding energy.</strong> <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span> values range from very small and tightly clustered at weak operator binding energies (e.g. O3) to relatively large and spread out for stronger operator binding energies (O1 and O2). The effective Hill coefficient generally decreases with increasing repressor copy number, indicating a flatter normalized response. The maximum possible Hill coefficient is roughly 1.75 for all repressor-operator binding energies. Circles, diamonds, and squares represent <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> values for the O1, O2, and O3 operators, respectively.</figcaption>
</figure>
<h2 id="applications-to-other-regulatory-architectures">Applications to Other Regulatory Architectures</h2>
<p>In this section, we discuss how the theoretical framework presented in this work is sufficiently general to include a variety of regulatory architectures outside of simple repression by LacI. We begin by noting that the exact same formula for fold-change given in can also describe corepression. We then demonstrate how our model can be generalized to include other architectures, such as a coactivator binding to an activator to promote gene expression. In each case, we briefly describe the system and describe its corresponding theoretical description. For further details, we invite the interested reader to read <span class="citation" data-cites="Bintu2005 Marzen2013"> [<a href="#ref-Bintu2005" role="doc-biblioref">32</a>,<a href="#ref-Marzen2013" role="doc-biblioref">33</a>]</span>.</p>
<h3 id="corepression">Corepression</h3>
<p>Consider a regulatory architecture where binding of a transcriptional repressor occludes the binding of RNAP to the DNA. A corepressor molecule binds to the repressor and shifts its allosteric equilibrium towards the active state in which it binds more tightly to the DNA, thereby decreasing gene expression (in contrast, an inducer shifts the allosteric equilibrium towards the inactive state where the repressor binds more weakly to the DNA). As in the main text, we can enumerate the states and statistical weights of the promoter and the allosteric states of the repressor. We note that these states and weights exactly match and yield the same fold-change equation as XXX, <span id="eq:ch4_eq39"><span class="math display">$$
\text{fold-change} \approx 
\left(1 + \frac{\left(1 + \frac{c}{K_A}\right)^n}{
\left(1 + \frac{c}{K_A}\right)^n + e^{\beta\Delta\varepsilon_{AI}}
\left(1 + \frac{c}{K_I}\right)^n} \frac{R}{N_{NS}}
e^{-\beta\Delta\varepsilon_{RA}}\right)^{-1},
\qquad(39)$$</span></span> where <span class="math inline"><em>c</em></span> now represents the concentration of the corepressor molecule. Mathematically, the difference between these two architectures can be seen in the relative sizes of the dissociation constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> between the inducer and repressor in the active and inactive states, respectively. The corepressor is defined by <span class="math inline"><em>K</em><sub><em>A</em></sub> &lt; <em>K</em><sub><em>I</em></sub></span>, since the corepressor favors binding to the repressor’s active state; an inducer must satisfy <span class="math inline"><em>K</em><sub><em>I</em></sub> &lt; <em>K</em><sub><em>A</em></sub></span>, as was found in the main text from the induction data (see ). Much as was performed in the main text, we can make some predictions about the how the response of a corepressor. In , we show how varying the repressor copy number <span class="math inline"><em>R</em></span> and the repressor-DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> influence the response. We draw the reader’s attention to the decrease in fold-change as the concentration of effector is increased.</p>
<h3 id="activation">Activation</h3>
<p>We now turn to the case of activation. While this architecture was not studied in this work, we wish to demonstrate how the framework presented here can be extended to include transcription factors other than repressors. To that end, we consider a transcriptional activator which binds to DNA and aids in the binding of RNAP through energetic interaction term <span class="math inline"><em>ε</em><sub><em>A</em><em>P</em></sub></span>. Note that in this architecture, binding of the activator does not occlude binding of the polymerase. Binding of a coactivator molecule shifts its allosteric equilibrium towards the active state (<span class="math inline"><em>K</em><sub><em>A</em></sub> &lt; <em>K</em><sub><em>I</em></sub></span>), where the activator is more likely to be bound to the DNA and promote expression. Enumerating all of the states and statistical weights of this architecture and making the approximation that the promoter is weak generates a fold-change equation of the form <span id="eq:ch4_eq40"><span class="math display">$$
\text{fold-change} = 
\frac{1 + \frac{\left(1 + \frac{c}{K_A}\right)^n}{
\left( 1 + \frac{c}{K_A}\right)^n + e^{\beta\Delta\varepsilon_{AI}}
\left(1 + \frac{c}{K_I}\right)^n}\frac{A}{N_{NS}}
e^{-\beta\Delta\varepsilon_{AA}}e^{-\beta\varepsilon_{AP}}}{
1 + \frac{\left(1 + \frac{c}{K_A}\right)^n}{
\left(1 + \frac{c}{K_A}\right)^n + e^{\beta\Delta\varepsilon_{AI}}
\left(1 + \frac{c}{K_I}\right)^n}\frac{A}{N_{NS}}
e^{-\beta\Delta\varepsilon_{AA}}},
\qquad(40)$$</span></span> where <span class="math inline"><em>A</em></span> is the total number of activators per cell, <span class="math inline"><em>c</em></span> is the concentration of a coactivator molecule, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>A</em></sub></span> is the binding energy of the activator to the DNA in the active allosteric state, and <span class="math inline"><em>ε</em><sub><em>A</em><em>P</em></sub></span> is the interaction energy between the activator and the RNAP. Unlike in the cases of induction and corepression, the fold-change formula for activation includes terms from when the RNAP is bound by itself on the DNA as well as when both RNAP and the activator are simultaneously bound to the DNA. explores predictions of the fold-change in gene expression by manipulating the activator copy number, DNA binding energy, and the polymerase-activator interaction energy. Note that with this activation scheme, the fold-change must necessarily be greater than one. An interesting feature of these predictions is the observation that even small changes in the interaction energy (<span class="math inline"> &lt; 0.5 <em>k</em><sub><em>B</em></sub><em>T</em></span>) can result in dramatic increase in fold-change.</p>
<p>As in the case of induction, the is straightforward to generalize. For example, the relative values of <span class="math inline"><em>K</em><sub><em>I</em></sub></span> and <span class="math inline"><em>K</em><sub><em>A</em></sub></span> can be switched such that <span class="math inline"><em>K</em><sub><em>I</em></sub> &lt; <em>K</em><sub><em>A</em></sub></span> in which the secondary molecule drives the activator to assume the inactive state represents induction of an activator. While these cases might be viewed as separate biological phenomena, mathematically they can all be described by the same underlying formalism.</p>
<figure>
<img src="ch4_fig28.png" id="fig:ch4_fig28" data-short-caption="Representative fold-change predictions for allosteric corepression and activation" alt="Figure 28: Representative fold-change predictions for allosteric corepression and activation. (A) Contrary to the case of induction described in the main text, addition of a corepressor decreases fold-change in gene expression. The left and right panels demonstrate how varying the values of the repressor copy number R and repressor-DNA binding energy \Delta\varepsilon_{RA}, respectively, change the predicted response profiles. (B) In the case of inducible activation, binding of an effector molecule to an activator transcription factor increases the fold-change in gene expression. Note that for activation, the fold-change is greater than 1. The left and center panels show how changing the activator copy number A and activator-DNA binding energy \Delta\varepsilon_{AA} alter response, respectively. The right panel shows how varying the polymerase-activator interaction energy \varepsilon_{AP} alters the fold-change. Relatively small perturbations to this energetic parameter drastically changes the level of activation and plays a major role in dictating the dynamic range of the system." /><figcaption aria-hidden="true">Figure 28: <strong>Representative fold-change predictions for allosteric corepression and activation.</strong> (A) Contrary to the case of induction described in the main text, addition of a corepressor decreases fold-change in gene expression. The left and right panels demonstrate how varying the values of the repressor copy number <span class="math inline"><em>R</em></span> and repressor-DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>, respectively, change the predicted response profiles. (B) In the case of inducible activation, binding of an effector molecule to an activator transcription factor increases the fold-change in gene expression. Note that for activation, the fold-change is greater than 1. The left and center panels show how changing the activator copy number <span class="math inline"><em>A</em></span> and activator-DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>A</em></sub></span> alter response, respectively. The right panel shows how varying the polymerase-activator interaction energy <span class="math inline"><em>ε</em><sub><em>A</em><em>P</em></sub></span> alters the fold-change. Relatively small perturbations to this energetic parameter drastically changes the level of activation and plays a major role in dictating the dynamic range of the system.</figcaption>
</figure>
<p>Note that a complete list of references can be found in the main text associated with this work.</p>
<h2 id="definition-of-the-non-specific-background-n_ns">Definition of the non-specific background <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span></h2>
<p>In this section we will explore the definition of the non-specific background <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span>. As raised by an anonymous reviewer the nature of this parameter seems to raise some controversy on what the right value should be, or whether or not the arbitrary definition of its value should also be applied to the <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> parameter.</p>
<p>Specifically during the first round a reviewer did not like the idea that the value of <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub> = 4.6 × 10<sup>6</sup></span> assumed that the entirety of the genome was available for non-specific binding of the repressor. We will consider how reasonable this is at the end of the section. However, As we will show first, the specific value of <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> is analogous to the zero potential energy or the reference concentration state. It is only the free energy differences that matter at the end of the day. For the second round of reviews the same reviewer was willing to agree on our point if and only if we were to acknowledge that other parameters such as <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>, the free energy difference between the active and inactive state of the repressor, also had an arbitrary definition that could be set to any value. In this section we will show as well that such a statement is an erroneous interpretation of the parameters. This free energy difference value cannot be re-defined to take any value if one is to be consistent with the experimental data.</p>
<p>Let us start by showing why the specific value of <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> is not the key variable. Under the weak promoter approximation the fold-change equation is equivalent to a two-state Fermi function of having the promoter being occupied by a repressor or having an empty promoter. This is <span id="eq:ch4_eq41"><span class="math display">$$
\text{fold-change} \rightarrow p^r_{bound} = \frac{1}{ 1 +
\frac{R}{N_{NS}} e^{-\beta\Delta\varepsilon_{RA}}}.
\qquad(41)$$</span></span> This expression can be rewritten as <span id="eq:ch4_eq42"><span class="math display">$$
p^r_{bound} = \frac{1}{ 1 + e^{-\beta\Delta E}},
\qquad(42)$$</span></span> where <span class="math inline"><em>Δ</em><em>E</em></span> is the free energy difference between the empty and occupied promoter. This definition implies that <span id="eq:ch4_eq43"><span class="math display">$$
\Delta E\equiv \overbrace{\Delta\varepsilon_{RA}}^{\text{enthalpic term}}
  - \overbrace{k_BT \ln \left( \frac{R}{N_{NS}} \right)}^{\text{entropic term}}.
\qquad(43)$$</span></span></p>
<p>Given that the parameter <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> is inferred rather than directly measured, this puts us in the position of being able to redefine <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> at will as long as <span class="math inline"><em>Δ</em><em>E</em></span> is in accordance to the experimental data. In other words the parameter that matters is the free energy difference, rather than its individual components. For example, if for a given operator and a given repressor copy number we choose a different value of <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span>, it still should hold true that <span id="eq:ch4_eq44"><span class="math display">$$
\Delta E= \Delta\varepsilon_{RA}' - k_BT \ln \left( \frac{R}{N_{NS}'} \right),
\qquad(44)$$</span></span> where <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub>′</span> is the changed value of the non-specific background and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub>′</span> is a different value for the repressor binding energy that compensates for the difference in the non-specific background.</p>
<p>Let <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub>′ ≡ <em>α</em><em>N</em><sub><em>N</em><em>S</em></sub></span>, since the value of <span class="math inline"><em>Δ</em><em>E</em></span> has to be preserved it should be true that <span id="eq:ch4_eq45"><span class="math display">$$
\Delta E= \Delta\varepsilon_{RA}' - 
k_BT \ln \left( \frac{R}{\alpha N_{NS}} \right)
= \Delta\varepsilon_{RA} - k_BT \ln \left( \frac{R}{N_{NS}} \right).
\qquad(45)$$</span></span> Solving for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub>′</span> gives <span id="eq:ch4_eq46"><span class="math display">$$
\begin{split}
\Delta\varepsilon_{RA}' &amp;= \Delta\varepsilon_{RA} + 
k_BT \ln \left( \frac{N_{NS}}{\alpha N_{NS}} \right)\\
&amp;= \Delta\varepsilon_{RA} - k_BT \ln \alpha.
\end{split}
\qquad(46)$$</span></span></p>
<p>Eq. <strong>¿eq:ch4_eRA_redef?</strong> implies that we can redefine <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> to be any value as long as <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> compensates to maintain the value of <span class="math inline"><em>Δ</em><em>E</em></span>. This statement holds true whether we are considering a single promoter or multiple promoters. The same cannot be said about the <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> parameter. The parameter <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> by itself sets the fraction of inactive repressors in the absence of inducer via <span id="eq:ch4_eq47"><span class="math display">$$
p_{act} = \frac{1}{1 + e^{-\beta\Delta\varepsilon_{AI}}},
\qquad(47)$$</span></span> where we have again a Fermi function for a two-state system in which the repressor can be in an active or inactive state.</p>
<p>As shown before, the reason why we could define <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> to be any value is because the parameter that matters is itself <span class="math inline"><em>Δ</em><em>E</em></span> the free energy difference. Therefore the repressor binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> could compensate for changes in the value of <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span>. For the case of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> tells us that <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> has no entropic term that can be compensated with an enthalpic term, or vice versa.</p>
<p>One could argue that for the case of a single promoter the fold-change equation does allow this parameter to be redefined in an arbitrary way since the full equation in the absence of inducer can be written as <span id="eq:ch4_eq48"><span class="math display">$$
\text{fold-change} = \frac{1}{
1 + \left( \frac{1}{1 + e^{-\beta\Delta\varepsilon_{AI}}} \right)
\frac{R}{N_{NS}} e^{-\beta\Delta\varepsilon_{RA}}}.
\qquad(48)$$</span></span> So when we define the free energy <span class="math inline"><em>Δ</em><em>E</em></span> we would include an extra term of the form <span id="eq:ch4_eq49"><span class="math display">$$
\Delta E= \Delta\varepsilon_{RA} - 
k_BT \left[ \ln \left( \frac{R}{N_{NS}} \right) +
\ln \left( \frac{1}{1 + e^{-\beta\Delta\varepsilon_{AI}}} \right) \right].
\qquad(49)$$</span></span> If we were only to use the statement brought up by the anonymous reviewer would be true since changes in <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> could be compensated by changes in <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> or <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span>. But as specified in appendix <a href="#AppendixModel" data-reference-type="ref" data-reference="AppendixModel">[AppendixModel]</a> this is not the case for cells with multiple promoters.</p>
<p>The case of multiple promoters can be handled using the Canonical ensemble as in or using the Grand Canonical ensemble as detailed in <span class="citation" data-cites="Weinert2014"> [<a href="#ref-Weinert2014" role="doc-biblioref">5</a>]</span>. Our point is more clearly seen in the case of the Canonical ensemble. Under this formalism the fold-change equation is given by <span class="citation" data-cites="Brewster2014"> [<a href="#ref-Brewster2014" role="doc-biblioref">1</a>]</span> <span id="eq:ch4_eq50"><span class="math display">$$
\text{fold-change} = \frac{
\sum_{m=0}^{\min (N,R)} \frac{R!}{(N_{NS})^m (R - m)!}
{N \choose m} e^{-\beta m \Delta\varepsilon_{RA}}(N - m)
}{
N \sum_{m=0}^{\min (N,R)} \frac{R!}{(N_{NS})^m (R - m)!}
{N \choose m} e^{-\beta m \Delta\varepsilon_{RA}}
},
\qquad(50)$$</span></span> where <span class="math inline"><em>N</em></span> is the number of promoters. Notice that we can group the terms including <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> as <span id="eq:ch4_eq51"><span class="math display">$$
\text{fold-change} = \frac{
\sum_{m=0}^{\min (N,R)} \frac{R!}{(R - m)!}
{N \choose m} \left(
\frac{e^{-\beta \Delta\varepsilon_{RA}}}{N_{NS}}\right)^m (N - m)
}{
N \sum_{m=0}^{\min (N,R)} \frac{R!}{(R - m)!}
{N \choose m} \left(\frac{e^{-\beta \Delta\varepsilon_{RA}}}{N_{NS}}\right)^m},
\qquad(51)$$</span></span> to highlight that it is a combination of these two parameters that matters, rather than their individual values. For the case of the <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> parameter this is not the case. Every term containing <span class="math inline"><em>R</em></span> on is effectively multiplied by . Since these terms are included inside the factorials it is not true that a simple compensation by the other parameters allow us to define <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> to be any value. Therefore as defined in appendix <a href="#AppendixModel" data-reference-type="ref" data-reference="AppendixModel">[AppendixModel]</a> the parameter <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> can be independently inferred using multiple promoter measurements of fold change.</p>
<p>As a final note, we can also check whether <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub> = 4.6 × 10<sup>6</sup></span> is at all a reasonable value to use. One potential point of concern is whether the chromosomal DNA is occupied by other transcription factors that may reduce the availability of the DNA for repressor or RNAP to bind. Here we consider data from a recent census of protein abundance across the <em>E. coli</em> genome. In that work, Schmidt <em>et al.</em> <span class="citation" data-cites="Schmidt2015"> [<a href="#ref-Schmidt2015" role="doc-biblioref">15</a>]</span> measured the protein copy number across more than half the coding genes (greater than 95% by total protein mass). During exponential growth in M9 minimal media with 0.5 % glucose, they find that about 6 % of the protein mass, or 311,000 monomer copies per cell, are proteins such as transcription factors that will be bound to the DNA (about two-thirds of these are nucleoid-associated proteins such as HNS and HU).</p>
<p>To make a simple estimate of DNA occupancy, let us assume that all transcription factors bind DNA as dimers and occupy a DNA length of 15 bp (this appears to vary from 7 bp to 38 bp in <em>E. coli</em> on RegulonDB <span class="citation" data-cites="Gama-Castro2016"> [<a href="#ref-Gama-Castro2016" role="doc-biblioref">34</a>]</span>), we find that about 2.3 kbp or about half of the genome will be occupied. In the most extreme case we could assume that this fraction is totally inaccessible, which would reduce <span class="math inline"><em>N</em><sub><em>N</em></sub><em>S</em></span> by a factor of about <span class="math inline">2</span>. Applying this to equation <a href="#eq_eRA_redef" data-reference-type="ref" data-reference="eq_eRA_redef">[eq_eRA_redef]</a>, we see that this has a negligible effect on the actual binding energy that we would infer, and only corresponds to a change in energy <span class="math inline"><em>ε</em><sub><em>R</em><em>A</em></sub></span> by about 0.7 <span class="math inline"><em>k</em><sub><em>B</em></sub><em>T</em></span>.</p>
<h2 id="AppendixSteadyState">Measurement of Steady State</h2>
<p>All measurements have been performed with cells in exponential growth phase, where we expect an average expression to be maintained across the cell population. Here we wanted to use one of our strains (O2 <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span>, <span class="math inline"><em>R</em> = 260</span>) to show that gene expression is under steady state for our experimental conditions. As a reminder, under our standard protocol we begin by growing an overnight culture in Lysogeny Broth for each of the required strains. After approximately <span class="math inline">12</span> hours, the saturated cultures are diluted 1000-fold into a <span class="math inline">2 mL</span> 96-deep-well plate. Each well contains <span class="math inline">500 <em>μ</em>L</span> of M9 minimal media supplemented with 0.5% w/v glucose and the appropriate IPTG inducer concentration.</p>
<p>Here we follow the protocol as noted above, but take measurements in one hour increments after the 1000-fold dilution. We performed this in triplicate with our O2 <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span>, <span class="math inline"><em>R</em> = 260</span> strain (IPTG inducer concentration <span class="math inline"><em>c</em> = 50 <em>μ</em>M</span>), and also include an autofluorescence strain and O2 <span class="math inline"><em>Δ</em><em>l</em><em>a</em><em>c</em><em>I</em></span> strain. In we plot the optical density (OD<span class="math inline"><sub>600<em>n</em><em>m</em></sub></span>) as a function of time and see that growth is reasonably consistent between strains and their replicates. The shaded gray bar indicates an OD 0.3, which is the density at which we typically make our measurements. In , we shown the associated fold-change measurements (using flow cytometry). While it does look like there is a steady increase in fold-change from 0 to 4 hours, it seems to level off past this time point. However, there was also a large degree of variation in our measurements making it difficult to say that fold-change is not changing over time.</p>
<p>In we also plot the raw fluorescence values against the measured OD<span class="math inline"><sub>600<em>n</em><em>m</em></sub></span> values. The variation is rather large, but it does appear that the overall expression is relatively constant across two decades of OD<span class="math inline"><sub>600<em>n</em><em>m</em></sub></span>.</p>
<figure>
<img src="ch4_fig31.png" id="fig:ch4_fig31" data-short-caption="Time course measurement of single cell fluorescence by flow cytometry - data set 1" alt="Figure 29: Time course measurement of single cell fluorescence by flow cytometry - data set 1. Flow cytometry measurements were performed at different time points following a 1000-fold dilution of an overnight culture. Cell strains were grown in M9 minimal media supplemented with 0.5% w/v glucose and IPTG c=50\,\mu\text{M}. OD_{600nm} measurements are shown for the three strains. The fold change is calculated for each measurement shown in Panel . Note that each measurements represents a different culture grown in a 96 deep well plate." /><figcaption aria-hidden="true">Figure 29: <strong>Time course measurement of single cell fluorescence by flow cytometry - data set 1</strong>. Flow cytometry measurements were performed at different time points following a 1000-fold dilution of an overnight culture. Cell strains were grown in M9 minimal media supplemented with 0.5% w/v glucose and IPTG <span class="math inline"><em>c</em> = 50 <em>μ</em>M</span>. OD<span class="math inline"><sub>600<em>n</em><em>m</em></sub></span> measurements are shown for the three strains. The fold change is calculated for each measurement shown in Panel . Note that each measurements represents a different culture grown in a 96 deep well plate.</figcaption>
</figure>
<figure>
<img src="ch4_fig32.png" id="fig:ch4_fig32" data-short-caption="Time course measurement of single cell fluorescence versus OD$_{600nm}$ - data set 1" alt="Figure 30: Time course measurement of single cell fluorescence versus OD_{600nm} - data set 1. Fluorescence measurements used to calculate fold-change from are plotted against their OD_{600nm}. Error bars represent standard deviation from the triplicate culture measurements from growth in a 96 deep well plate." /><figcaption aria-hidden="true">Figure 30: <strong>Time course measurement of single cell fluorescence versus OD<span class="math inline"><sub>600<em>n</em><em>m</em></sub></span> - data set 1</strong>. Fluorescence measurements used to calculate fold-change from are plotted against their OD<span class="math inline"><sub>600<em>n</em><em>m</em></sub></span>. Error bars represent standard deviation from the triplicate culture measurements from growth in a 96 deep well plate.</figcaption>
</figure>
<p>In a separate set of replicates, we observed more consistent fold-change measurements over these later time points. In we show the average single cell fluorescence from these measurements. While there does appear to be a downward trend in both the <span class="math inline"><em>R</em> = 260</span> strain and the <span class="math inline"><em>Δ</em></span><em>lacI</em> strain, this is perhaps due to cultures leaving exponential growth (mistakenly, OD<span class="math inline"><sub>600<em>n</em><em>m</em></sub></span> was not measured in this attempt). In contrast to the data in , we found that fold-change did not appreciably change across these measurements (). Given the differences across these two sets of experiments, it will be important to perform more experiments before drawing any definite opinions about the above results.</p>
<figure>
<img src="ch4_fig33.png" id="fig:ch4_fig33" data-short-caption="Time course measurement of single cell fluorescence by flow cytometry - data set 2" alt="Figure 31: Time course measurement of single cell fluorescence by flow cytometry - data set 2. Flow cytometry measurements were performed at different time points following a 1000-fold dilution of an overnight culture. Cell strains were grown in M9 minimal media supplemented with 0.5% w/v glucose and IPTG c=50\,\mu\text{M}. Mean fluorescence values are shown for strain O2 \Delta\varepsilon_{RA} = -13.9~k_BT with R=260, O2 \Delta lacI, and an autofluorescence strain. Data points represent measurements from separate 500\,\mu\text{L} cell cultures. The fold change is calculated for each measurement shown in Panel ." /><figcaption aria-hidden="true">Figure 31: <strong>Time course measurement of single cell fluorescence by flow cytometry - data set 2</strong>. Flow cytometry measurements were performed at different time points following a 1000-fold dilution of an overnight culture. Cell strains were grown in M9 minimal media supplemented with 0.5% w/v glucose and IPTG <span class="math inline"><em>c</em> = 50 <em>μ</em>M</span>. Mean fluorescence values are shown for strain O2 <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span> with <span class="math inline"><em>R</em> = 260</span>, O2 <span class="math inline"><em>Δ</em><em>l</em><em>a</em><em>c</em><em>I</em></span>, and an autofluorescence strain. Data points represent measurements from separate <span class="math inline">500 <em>μ</em>L</span> cell cultures. The fold change is calculated for each measurement shown in Panel .</figcaption>
</figure>
<h3 id="e.-coli-primer-and-strain-list"><em>E. coli</em> Primer and Strain List</h3>
<p>Here we provide additional details about the genotypes of the strains used, as well as the primer sequences used to generate them. <em>E. coli</em> strains were derived from K12 MG1655. For those containing <span class="math inline"><em>R</em> = 22</span>, we used strain HG104 which additionally has the <em>lacYZA</em> operon deleted (positions 360,483 to 365,579) but still contains the native <em>lacI</em> locus. All other strains used strain HG105, where both the <em>lacYZA</em> and <em>lacI</em> operons have both been deleted (positions 360,483 to 366,637).</p>
<p>All 25x+11-yfp expression constructs were integrated at the <em>galK</em> locus (between positions 1,504,078 and 1,505,112) while the 3*1x-lacI constructs were integrated at the <em>ybcN</em> locus (between positions 1,287,628 and 1,288,047). Integration was performed with <span class="math inline"><em>λ</em></span> Red recombineering <span class="citation" data-cites="Sharan2009"> [<a href="#ref-Sharan2009" role="doc-biblioref">35</a>]</span> as described in <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span> using the primers listed in Table <a href="#table_primers" data-reference-type="ref" data-reference="table_primers">[table_primers]</a>. We follow the notation of Lutz and Bujard <span class="citation" data-cites="Lutz1997"> [<a href="#ref-Lutz1997" role="doc-biblioref">36</a>]</span> for the nomenclature of the different constructs used. Specifically, the first number refers to the antibiotic resistance cassette that is present for selection (2 = kanamycin, 3 = chloramphenicol, and 4 = spectinomycin) and the second number refers to the promoter used to drive expression of either YFP or LacI (1 = <span class="math inline"><em>P</em><sub><em>L</em><em>t</em><em>e</em><em>t</em><em>O</em> − 1</sub></span>, and 5 = <em>lacUV5</em>). Note that in 25x+11-yfp, x refers to the LacI operator used, which is centered at +11 (or alternatively, begins at the transcription start site). For the different LacI constructs, 3*1x-lacI, x refers to the different ribosomal binding site modifications that provide different repressor copy numbers and follows from <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span>. The asterisk refers to the presence of FLP recombinase sites flanking the chloramphenicol resistance gene that can be used to lose this resistance. However, we maintained the resistance gene in our constructs. A summary of the final genotypes of each strain is listed in Table <a href="#table_strains" data-reference-type="ref" data-reference="table_strains">[table_strains]</a>. In addition each strain also contained the plasmid pZS4*1-mCherry and provided constitutive expression of the mCherry fluorescent protein. This pZS plasmid is a low copy (SC101 origin of replication) where like with 3*1x-lacI, mCherry is driven by a <span class="math inline"><em>P</em><sub><em>L</em><em>t</em><em>e</em><em>t</em><em>O</em> − 1</sub></span> promoter.</p>
<div id="tbl:ch4_tbl04">
<table>
<caption>Table 4:  Each strain contains a unique operator-yfp construct for measurement of fluorescence and <span class="math inline"><em>R</em></span> refers to the dimer copy number as measured by <span class="citation" data-cites="Garcia2011c"> [<a href="#ref-Garcia2011c" role="doc-biblioref">4</a>]</span>.</caption>
<thead>
<tr class="header">
<th><strong>Strain</strong></th>
<th><strong>Genotype</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>O1, <span class="math inline"><em>R</em> = 0</span></td>
<td>HG105::galK &lt;&gt;25O1+11-yfp</td>
</tr>
<tr class="even">
<td>O1, <span class="math inline"><em>R</em> = 22</span></td>
<td>HG104::galK &lt;&gt;25O1+11-yfp</td>
</tr>
<tr class="odd">
<td>O1, <span class="math inline"><em>R</em> = 60</span></td>
<td>HG105::galK &lt;&gt;25O1+11-yfp, ybcN &lt;&gt;3*1RBS1147-lacI</td>
</tr>
<tr class="even">
<td>O1, <span class="math inline"><em>R</em> = 124</span></td>
<td>HG105::galK &lt;&gt;25O1+11-yfp, ybcN &lt;&gt;3*1RBS1027-lacI</td>
</tr>
<tr class="odd">
<td>O1, <span class="math inline"><em>R</em> = 260</span></td>
<td>HG105::galK &lt;&gt;25O1+11-yfp, ybcN &lt;&gt;3*1RBS446-lacI</td>
</tr>
<tr class="even">
<td>O1, <span class="math inline"><em>R</em> = 1220</span></td>
<td>HG105::galK &lt;&gt;25O1+11-yfp, ybcN &lt;&gt;3*1RBS1-lacI</td>
</tr>
<tr class="odd">
<td>O1, <span class="math inline"><em>R</em> = 1740</span></td>
<td>HG105::galK &lt;&gt;25O1+11-yfp, ybcN &lt;&gt;3*1-lacI (RBS1L)</td>
</tr>
<tr class="even">
<td>O2, <span class="math inline"><em>R</em> = 0</span></td>
<td>HG105::galK &lt;&gt;25O2+11-yfp</td>
</tr>
<tr class="odd">
<td>O2, <span class="math inline"><em>R</em> = 22</span></td>
<td>HG104::galK &lt;&gt;25O2+11-yfp</td>
</tr>
<tr class="even">
<td>O2, <span class="math inline"><em>R</em> = 60</span></td>
<td>HG105::galK &lt;&gt;25O2+11-yfp, ybcN &lt;&gt;3*1RBS1147-lacI</td>
</tr>
<tr class="odd">
<td>O2, <span class="math inline"><em>R</em> = 124</span></td>
<td>HG105::galK &lt;&gt;25O2+11-yfp, ybcN &lt;&gt;3*1RBS1027-lacI</td>
</tr>
<tr class="even">
<td>O2, <span class="math inline"><em>R</em> = 260</span></td>
<td>HG105::galK &lt;&gt;25O2+11-yfp, ybcN &lt;&gt;3*1RBS446-lacI</td>
</tr>
<tr class="odd">
<td>O2, <span class="math inline"><em>R</em> = 1220</span></td>
<td>HG105::galK &lt;&gt;25O2+11-yfp, ybcN &lt;&gt;3*1RBS1-lacI</td>
</tr>
<tr class="even">
<td>O2, <span class="math inline"><em>R</em> = 1740</span></td>
<td>HG105::galK &lt;&gt;25O2+11-yfp, ybcN &lt;&gt;3*1-lacI (RBS1L)</td>
</tr>
<tr class="odd">
<td>O3, <span class="math inline"><em>R</em> = 0</span></td>
<td>HG105::galK &lt;&gt;25O3+11-yfp</td>
</tr>
<tr class="even">
<td>O3, <span class="math inline"><em>R</em> = 22</span></td>
<td>HG104::galK &lt;&gt;25O3+11-yfp</td>
</tr>
<tr class="odd">
<td>O3, <span class="math inline"><em>R</em> = 60</span></td>
<td>HG105::galK &lt;&gt;25O3+11-yfp, ybcN &lt;&gt;3*1RBS1147-lacI</td>
</tr>
<tr class="even">
<td>O3, <span class="math inline"><em>R</em> = 124</span></td>
<td>HG105::galK &lt;&gt;25O3+11-yfp, ybcN &lt;&gt;3*1RBS1027-lacI</td>
</tr>
<tr class="odd">
<td>O3, <span class="math inline"><em>R</em> = 260</span></td>
<td>HG105::galK &lt;&gt;25O3+11-yfp, ybcN &lt;&gt;3*1RBS446-lacI</td>
</tr>
<tr class="even">
<td>O3, <span class="math inline"><em>R</em> = 1220</span></td>
<td>HG105::galK &lt;&gt;25O3+11-yfp, ybcN &lt;&gt;3*1RBS1-lacI</td>
</tr>
<tr class="odd">
<td>O3, <span class="math inline"><em>R</em> = 1740</span></td>
<td>HG105::galK &lt;&gt;25O3+11-yfp, ybcN &lt;&gt;3*1-lacI (RBS1L)</td>
</tr>
<tr class="even">
<td>Oid, <span class="math inline"><em>R</em> = 0</span></td>
<td>HG105::galK &lt;&gt;25Oid+11-yfp</td>
</tr>
<tr class="odd">
<td>Oid, <span class="math inline"><em>R</em> = 22</span></td>
<td>HG104::galK &lt;&gt;25Oid+11-yfp</td>
</tr>
<tr class="even">
<td>Oid, <span class="math inline"><em>R</em> = 60</span></td>
<td>HG105::galK &lt;&gt;25Oid+11-yfp, ybcN &lt;&gt;3*1RBS1147-lacI</td>
</tr>
<tr class="odd">
<td>Oid, <span class="math inline"><em>R</em> = 124</span></td>
<td>HG105::galK &lt;&gt;25Oid+11-yfp, ybcN &lt;&gt;3*1RBS1027-lacI</td>
</tr>
<tr class="even">
<td>Oid, <span class="math inline"><em>R</em> = 260</span></td>
<td>HG105::galK &lt;&gt;25Oid+11-yfp, ybcN &lt;&gt;3*1RBS446-lacI</td>
</tr>
<tr class="odd">
<td>Oid, <span class="math inline"><em>R</em> = 1220</span></td>
<td>HG105::galK &lt;&gt;25Oid+11-yfp, ybcN &lt;&gt;3*1RBS1-lacI</td>
</tr>
<tr class="even">
<td>Oid, <span class="math inline"><em>R</em> = 1740</span></td>
<td>HG105::galK &lt;&gt;25Oid+11-yfp, ybcN &lt;&gt;3*1-lacI (RBS1L)</td>
</tr>
</tbody>
</table>
</div>
<h2 class="unnumbered" id="references">References</h2>
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-Brewster2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[1] </div><div class="csl-right-inline">R. C. Brewster, F. M. Weinert, H. G. Garcia, D. Song, M. Rydenfelt, and R. Phillips, <em><span class="nocase">The Transcription Factor Titration Effect Dictates Level of Gene Expression</span></em>, Cell <strong>156</strong>, 1312 (2014).</div>
</div>
<div id="ref-Gardino2003" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[2] </div><div class="csl-right-inline">A. K. Gardino, B. F. Volkman, H. S. Cho, S.-Y. Lee, D. E. Wemmer, and D. Kern, <em><span class="nocase">The NMR Solution Structure of BeF3-Activated Spo0F Reveals the Conformational Switch in a Phosphorelay System</span></em>, Journal of Molecular Biology <strong>331</strong>, 245 (2003).</div>
</div>
<div id="ref-Boulton2016" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[3] </div><div class="csl-right-inline">S. Boulton and G. Melacini, <em><span class="nocase">Advances in NMR Methods To Map Allosteric Sites: From Models to Translation</span></em>, Chemical Reviews <strong>116</strong>, 6267 (2016).</div>
</div>
<div id="ref-Garcia2011c" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[4] </div><div class="csl-right-inline">H. G. Garcia and R. Phillips, <em><span class="nocase">Quantitative dissection of the simple repression input-output function.</span></em>, Proceedings of the National Academy of Sciences <strong>108</strong>, 12173 (2011a).</div>
</div>
<div id="ref-Weinert2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[5] </div><div class="csl-right-inline">F. M. Weinert, R. C. Brewster, M. Rydenfelt, R. Phillips, and W. K. Kegel, <em><span class="nocase">Scaling of Gene Expression with Transcription-Factor Fugacity</span></em>, Physical Review Letters <strong>113</strong>, 1 (2014).</div>
</div>
<div id="ref-Oehler1994" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[6] </div><div class="csl-right-inline">S. Oehler, M. Amouyal, P. Kolkhof, B. von Wilcken-Bergmann, and B. Müller-Hill, <em><span class="nocase">Quality and position of the three lac operators of <em>E. coli</em> define efficiency of repression.</span></em>, The EMBO Journal <strong>13</strong>, 3348 (1994).</div>
</div>
<div id="ref-Setty2003" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[7] </div><div class="csl-right-inline">Y. Setty, A. E. Mayo, M. G. Surette, and U. Alon, <em><span class="nocase">Detailed map of a cis-regulatory input function.</span></em>, Proceedings of the National Academy of Sciences <strong>100</strong>, 7702 (2003).</div>
</div>
<div id="ref-Oehler2006a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[8] </div><div class="csl-right-inline">S. Oehler, S. Alberti, and B. Müller-Hill, <em><span class="nocase">Induction of the <em>lac</em> promoter in the absence of DNA loops and the stoichiometry of induction</span></em>, Nucleic Acids Research <strong>34</strong>, 606 (2006).</div>
</div>
<div id="ref-Daber2009" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[9] </div><div class="csl-right-inline">R. Daber, K. Sharp, and M. Lewis, <em><span>One Is Not Enough</span></em>, Journal of Molecular Biology <strong>392</strong>, 1133 (2009).</div>
</div>
<div id="ref-Daber2011a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[10] </div><div class="csl-right-inline">R. Daber, M. A. Sochor, and M. Lewis, <em><span class="nocase">Thermodynamic Analysis of Mutant Lac Repressors</span></em>, Journal of Molecular Biology <strong>409</strong>, 76 (2011).</div>
</div>
<div id="ref-Vilar2013" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[11] </div><div class="csl-right-inline">J. M. G. Vilar and L. Saiz, <em><span class="nocase">Reliable Prediction of Complex Phenotypes from a Modular Design in Free Energy Space: An Extensive Exploration of the <em>lac</em> Operon</span></em>, ACS Synthetic Biology <strong>2</strong>, 576 (2013).</div>
</div>
<div id="ref-Shis2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[12] </div><div class="csl-right-inline">D. L. Shis, F. Hussain, S. Meinhardt, L. Swint-Kruse, and M. R. Bennett, <em><span class="nocase">Modular, Multi-Input Transcriptional Logic Gating with Orthogonal LacI/GalR Family Chimeras</span></em>, ACS Synthetic Biology <strong>3</strong>, 645 (2014).</div>
</div>
<div id="ref-Sochor2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[13] </div><div class="csl-right-inline">M. A. Sochor, <em><span class="nocase"><em>In vitro</em> transcription accurately predicts lac repressor phenotype <em>in vivo</em> in <em>Escherichia coli</em>.</span></em>, PeerJ <strong>2</strong>, e498 (2014).</div>
</div>
<div id="ref-Rydenfelt2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[14] </div><div class="csl-right-inline">M. Rydenfelt, H. G. Garcia, R. S. Cox, and R. Phillips, <em><span class="nocase">The Influence of Promoter Architectures and Regulatory Motifs on Gene Expression in <em>Escherichia coli</em></span></em>, PLoS ONE <strong>9</strong>, 1 (2014).</div>
</div>
<div id="ref-Schmidt2015" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[15] </div><div class="csl-right-inline">A. Schmidt, K. Kochanowski, S. Vedelaar, E. Ahrné, B. Volkmer, L. Callipo, K. Knoops, M. Bauer, R. Aebersold, and M. Heinemann, <em><span class="nocase">The quantitative and condition-dependent <em>Escherichia coli</em> proteome</span></em>, Nature Biotechnology <strong>34</strong>, 104 (2015).</div>
</div>
<div id="ref-SpectraViewer" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[16] </div><div class="csl-right-inline">Chroma2ptTechnology2ptCorporation, <em>Chroma Spectra Viewer</em>, (2016).</div>
</div>
<div id="ref-Maecker2005" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[17] </div><div class="csl-right-inline">H. T. Maecker, A. Rinfret, P. D’Souza, J. Darden, E. Roig, C. Landry, P. Hayes, J. Birungi, O. Anzala, M. Garcia, A. Harari, I. Frank, R. Baydo, M. Baker, J. Holbrook, J. Ottinger, L. Lamoreaux, C. L. Epling, E. Sinclair, M. A. Suni, K. Punt, S. Calarota, S. El-Bahi, G. Alter, H. Maila, E. Kuta, J. Cox, C. Gray, M. Altfeld, N. Nougarede, J. Boyer, L. Tussey, T. Tobery, B. Bredt, M. Roederer, R. Koup, V. C. Maino, K. Weinhold, G. Pantaleo, J. Gilmour, H. Horton, and R. P. Sekaly, <em><span class="nocase">Standardization of cytokine flow cytometry assays</span></em>, BMC Immunology <strong>6</strong>, 13 (2005).</div>
</div>
<div id="ref-Lo2008" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[18] </div><div class="csl-right-inline">K. Lo, R. R. Brinkman, and R. Gottardo, <em><span class="nocase">Automated gating of flow cytometry data via robust model-based clustering</span></em>, Cytometry Part A <strong>73A</strong>, 321 (2008).</div>
</div>
<div id="ref-Aghaeepour2013" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[19] </div><div class="csl-right-inline">N. Aghaeepour, G. Finak, The2ptFlowCAP2ptConsortium, The2ptDREAM2ptConsortium, H. Hoos, T. R. Mosmann, R. Brinkman, R. Gottardo, and R. H. Scheuermann, <em><span class="nocase">Critical assessment of automated flow cytometry data analysis techniques</span></em>, Nature Methods <strong>10</strong>, 228 (2013).</div>
</div>
<div id="ref-Garcia2011b" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[20] </div><div class="csl-right-inline">H. G. Garcia, H. J. Lee, J. Q. Boedicker, and R. Phillips, <em><span class="nocase">Comparison and Calibration of Different Reporters for Quantitative Analysis of Gene Expression</span></em>, Biophysical Journal <strong>101</strong>, 535 (2011b).</div>
</div>
<div id="ref-Edelstein2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[21] </div><div class="csl-right-inline">A. D. Edelstein, M. A. Tsuchida, N. Amodaj, H. Pinkard, R. D. Vale, and N. Stuurman, <em><span class="nocase">Advanced methods of microscope control using <span class="math inline"><em>μ</em></span>Manager software</span></em>, Journal of Biological Methods <strong>1</strong>, 10 (2014).</div>
</div>
<div id="ref-Marr1980" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[22] </div><div class="csl-right-inline">D. Marr and E. Hildreth, <em><span class="nocase">Theory of edge detection</span></em>, Proceedings of the Royal Society B: Biological Sciences <strong>207</strong>, 187 (1980).</div>
</div>
<div id="ref-Murphy2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[23] </div><div class="csl-right-inline">K. F. Murphy, G. Balázsi, and J. J. Collins, <em><span class="nocase">Combinatorial promoter design for engineering noisy gene expression.</span></em>, Proceedings of the National Academy of Sciences <strong>104</strong>, 12726 (2007).</div>
</div>
<div id="ref-Murphy2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[24] </div><div class="csl-right-inline">K. F. Murphy, R. M. Adams, X. Wang, G. Balázsi, and J. J. Collins, <em><span class="nocase">Tuning and controlling gene expression noise in synthetic gene networks</span></em>, Nucleic Acids Research <strong>38</strong>, 2712 (2010).</div>
</div>
<div id="ref-Rogers2015" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[25] </div><div class="csl-right-inline">J. K. Rogers, C. D. Guzman, N. D. Taylor, S. Raman, K. Anderson, and G. M. Church, <em>Synthetic Biosensors for Precise Gene Control and Real-Time Monitoring of Metabolites</em>, Nucleic Acids Research <strong>43</strong>, 7648 (2015).</div>
</div>
<div id="ref-Frank2013" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[26] </div><div class="csl-right-inline">S. Frank, <em><span class="nocase">Input-output relations in biological systems: measurement, information and the Hill equation.</span></em>, Biology Direct <strong>8</strong>, 31 (2013).</div>
</div>
<div id="ref-Weiss1997" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[27] </div><div class="csl-right-inline">J. N. Weiss, <em><span class="nocase">The Hill equation revisited: uses and misuses.</span></em>, The FASEB Journal <strong>11</strong>, 835 (1997).</div>
</div>
<div id="ref-Keymer2006" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[28] </div><div class="csl-right-inline">J. E. Keymer, R. G. Endres, M. Skoge, Y. Meir, and N. S. Wingreen, <em><span class="nocase">Chemosensing in <em>Escherichia coli</em>: Two regimes of two-state receptors.</span></em>, Proceedings of the National Academy of Sciences <strong>103</strong>, 1786 (2006).</div>
</div>
<div id="ref-Swem2008" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[29] </div><div class="csl-right-inline">L. R. Swem, D. L. Swem, N. S. Wingreen, and B. L. Bassler, <em><span class="nocase">Deducing Receptor Signaling Parameters from In Vivo Analysis: LuxN/AI-1 Quorum Sensing in Vibrio harveyi</span></em>, Cell <strong>134</strong>, 461 (2008).</div>
</div>
<div id="ref-Einav2016" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[30] </div><div class="csl-right-inline">T. Einav, L. Mazutis, and R. Phillips, <em><span class="nocase">Statistical Mechanics of Allosteric Enzymes.</span></em>, The Journal of Physical Chemistry B <strong>121</strong>, (2016).</div>
</div>
<div id="ref-Sivia2006" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[31] </div><div class="csl-right-inline">D. Sivia and J. Skilling, <em>Data Analysis: A Bayesian Tutorial</em> (OUP Oxford, 2006).</div>
</div>
<div id="ref-Bintu2005" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[32] </div><div class="csl-right-inline">L. Bintu, N. E. Buchler, H. G. Garcia, U. Gerland, T. Hwa, J. Kondev, T. Kuhlman, and R. Phillips, <em><span class="nocase">Transcriptional regulation by the numbers: applications.</span></em>, Curr. Opin. Genet. Dev. <strong>15</strong>, 125 (2005).</div>
</div>
<div id="ref-Marzen2013" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[33] </div><div class="csl-right-inline">S. Marzen, H. G. Garcia, and R. Phillips, <em><span class="nocase">Statistical mechanics of Monod-Wyman-Changeux (MWC) models</span></em>, Journal of Molecular Biology <strong>425</strong>, 1433 (2013).</div>
</div>
<div id="ref-Gama-Castro2016" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[34] </div><div class="csl-right-inline">S. Gama-Castro, H. Salgado, A. Santos-Zavaleta, D. Ledezma-Tejeida, L. Muñiz-Rascado, J. S. García-Sotelo, K. Alquicira-Hernández, I. Martínez-Flores, L. Pannier, J. A. Castro-Mondragón, A. Medina-Rivera, H. Solano-Lira, C. Bonavides-Martínez, E. Pérez-Rueda, S. Alquicira-Hernández, L. Porrón-Sotelo, A. López-Fuentes, A. Hernández-Koutoucheva, V. D. Moral-Chávez, F. Rinaldi, and J. Collado-Vides, <em><span class="nocase">RegulonDB version 9.0: high-level integration of gene regulation, coexpression, motif clustering and beyond</span></em>, Nucleic Acids Research <strong>44</strong>, D133 (2016).</div>
</div>
<div id="ref-Sharan2009" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[35] </div><div class="csl-right-inline">S. K. Sharan, L. C. Thomason, S. G. Kuznetsov, and D. L. Court, <em><span class="nocase">Recombineering: a homologous recombination-based method of genetic engineering</span></em>, Nature Protocols <strong>4</strong>, 206 (2009).</div>
</div>
<div id="ref-Lutz1997" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[36] </div><div class="csl-right-inline">R. Lutz and H. Bujard, <em><span class="nocase">Independent and tight regulation of transcriptional units in <em>Escherichia coli</em> via the LacR/O, the TetR/O and AraC/I1-I2 regulatory elements.</span></em>, Nucleic Acids Research <strong>25</strong>, 1203 (1997).</div>
</div>
</div>
						</div><!-- /.content -->
					</div><!-- /.col -->
					<div class="col-md-4 col-md-offset-1">
						<div class="sections-list-wrapper">
							<div class="sections-list js-sections js-affix js-scrollspy hidden-xs hidden-sm"><span style="font-size: 14pt">Table of Contents</span><hr></div><!-- /.sections-list -->
						</div>
					</div><!-- /.col -->
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.section -->
		
			<div class="section section--grey">
				<div class="container">
					<div class="row">
						<div class="col-md-7">
							<div id="disqus_thread"></div>
<script>
	/**
	 *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
	 *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables
	 */
	 /*
	var disqus_config = function () {
	this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
		this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
	};
	*/
	(function() {
		var d = document, s = d.createElement('script');

		s.src = 'https://mrazomej-phd.disqus.com/embed.js';

		s.setAttribute('data-timestamp', +new Date());
		(d.head || d.body).appendChild(s);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>

						</div><!-- /.col -->
					</div><!-- /.row -->
				</div><!-- /.container -->
			</div><!-- /.section -->
		
		<div class="js-footer-area">
			
				<nav class="page-nav">	
					<div class="container">
						<div class="row">
							<div class="col-xs-12">
								
									<a href="https://mrazomej.github.io/phd/chapter_03" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter 3
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://mrazomej.github.io/phd/chapter_05" class="page-nav__item page-nav__item--next">
										Chapter 5
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			
			
			
	<footer class="site-footer">
		<div class="container">
			<div class="row">
				<div class="col-sm-6">
					
						<a href="https://mrazomej.github.io/phd//" class="site-footer__logo">Manuel Razo-Mejia | PhD Thesis</a>
					
					
						<hr>
						<p class="site-footer__copyright">Copyright &copy; 2021. - Manuel Razo-Mejia <br>All rights reserved.</p>
					
				</div><!-- /.col -->
				
					<div class="col-sm-6 align-right">
						<ul class="social-list">
							
								<li>
									<a href="https://twitter.com/mrazomej" target="_blank" class="social-list__item social-list__item--twitter">
										<i class="icon icon--twitter"></i>
									</a>
								</li>
							
						</ul><!-- /.social-list -->
					</div><!-- /.col -->
				
			</div><!-- /.row -->
		</div><!-- /.container -->
	</footer><!-- /.site-footer -->


<script src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/jquery.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/bootstrap/affix.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/bootstrap/scrollspy.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/vendor/matchHeight.min.js"></script>
<script type="text/javascript" src="https://mrazomej.github.io/phd//doks-theme/assets/js/scripts.min.js"></script>


<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-109526846-4', 'auto');
	ga('send', 'pageview');
</script>



		</div><!-- /.js-footer-area -->
	</body>
</html>
